{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "185caf7a-68ac-4dbf-acc4-24057692f9b2",
   "metadata": {},
   "outputs": [],
   "source": [
    "from rdflib import ConjunctiveGraph\n",
    "from SPARQLWrapper import SPARQLWrapper, JSON\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib_venn import venn2\n",
    "import numpy as np\n",
    "from IPython.display import display, HTML\n",
    "\n",
    "\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "def remoteQuery(query, endpoint):\n",
    "    endpoint.setQuery(query)\n",
    "    try:\n",
    "        result = endpoint.queryAndConvert()\n",
    "        pd.set_option(\"display.max_rows\",None,\"display.max_colwidth\",6000,\"display.width\",6000,)\n",
    "        df = pd.DataFrame(result['results']['bindings'])\n",
    "        df = df.applymap(lambda x: x['value'])\n",
    "        return df\n",
    "        #return (result['results']['bindings'])\n",
    "    except Exception as e:\n",
    "        print(e)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "11179c2e-b528-4899-916b-6a861ca82cb1",
   "metadata": {},
   "outputs": [],
   "source": [
    "ep_biotools = SPARQLWrapper(\"http://localhost:7200/repositories/Project25\")\n",
    "ep_biotools.setReturnFormat(JSON)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "919832a8-92b4-469a-851a-142f2e7e34e6",
   "metadata": {
    "tags": []
   },
   "source": [
    "# Profondeur TOPIC"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8359bec3-caa7-4d77-ba03-e6469abd3e82",
   "metadata": {},
   "source": [
    "### Sequencing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "b9c4da44-b5f8-4c6a-816c-212e357fcc3d",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "#?topic \n",
    "#?label \n",
    "#(COUNT(?topic) as ?count) \n",
    "?x\n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "   }.\n",
    " #  FILTER (?topic = <http://edamontology.org/topic_3168>)\n",
    "}\n",
    "#GROUP BY ?topic ?label\n",
    "#ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3448c547-3a41-4cc4-9f47-dbeddaefc402",
   "metadata": {},
   "outputs": [],
   "source": [
    "print(f\"topic 'sequencing': {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "76a3121d-dd93-439d-acd0-fdc37ae6cfda",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "\n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3168>)\n",
    "} \n",
    "GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "ce11242e-a659-4a0e-8f0d-ec5259bea36d",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "#?topic \n",
    "#?label \n",
    "#(COUNT(?topic) as ?count) \n",
    "?x\n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "    }\n",
    "    FILTER(?topic != <http://edamontology.org/topic_3168>)\n",
    "    #FILTER (?topic = <http://edamontology.org/topic_3168>)\n",
    "} \n",
    "#GROUP BY ?topic ?label\n",
    "#ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "d838d34e-4c9c-442b-8e6c-5ab9e24e4524",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'sequencing'seul: 2054\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>x</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>https://bio.tools/16s-itgdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>https://bio.tools/2-kupl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>https://bio.tools/3cnet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>https://bio.tools/3gold</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>https://bio.tools/959_nematode_genomes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>https://bio.tools/ACACIA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>https://bio.tools/ACES</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>https://bio.tools/ADAPTABLE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>https://bio.tools/ADS-HCSpark</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>https://bio.tools/AJIA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>https://bio.tools/APSiC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>https://bio.tools/ARGA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>https://bio.tools/ASRD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>https://bio.tools/AcetoBase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>https://bio.tools/Afann</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>https://bio.tools/AluScanCNV2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>https://bio.tools/AmpliSolve</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>https://bio.tools/AmpliconTagger</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>https://bio.tools/Anchor_TFBS_prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>https://bio.tools/AnnotSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>https://bio.tools/AnthOligo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>https://bio.tools/AsyRW</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>https://bio.tools/Atropos</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>https://bio.tools/AutoPVS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>https://bio.tools/BAM-matcher</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>https://bio.tools/BEAN-counter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>https://bio.tools/BEable-GPS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>https://bio.tools/BGVD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>https://bio.tools/BLMRM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>https://bio.tools/BSA4Yeast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>https://bio.tools/Bcool</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>https://bio.tools/BioMedR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>https://bio.tools/C3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>https://bio.tools/C3D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>https://bio.tools/CLINE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>https://bio.tools/CPSR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>https://bio.tools/CSI_NGS_Portal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>https://bio.tools/Cas-analyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>https://bio.tools/Cascabel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>https://bio.tools/Circle-Map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>https://bio.tools/ClinVAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>https://bio.tools/ClonalTREE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>https://bio.tools/CodAn</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>https://bio.tools/Conbase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>https://bio.tools/ConnectedReads</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>https://bio.tools/CottonGen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>https://bio.tools/CytoConverter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>https://bio.tools/DAMEfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>https://bio.tools/DAMIAN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>https://bio.tools/DISSEQT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>https://bio.tools/DNABarcodeCompatibility</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>https://bio.tools/DROPA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>https://bio.tools/DecontaMiner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>https://bio.tools/DeepHF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>https://bio.tools/DeepSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>https://bio.tools/DeepSpCas9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>https://bio.tools/DesignSignatures</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>https://bio.tools/EPIP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>https://bio.tools/EPIVAN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>https://bio.tools/ERINS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>https://bio.tools/ETI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>https://bio.tools/EphaGen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>https://bio.tools/EvoFreq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>https://bio.tools/FACEPAI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>https://bio.tools/FLEA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>https://bio.tools/FastProNGS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>https://bio.tools/ForestQC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>https://bio.tools/FrogCap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>https://bio.tools/GAEseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>https://bio.tools/GASAL2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>https://bio.tools/GEAN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>https://bio.tools/GEAR-base</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>https://bio.tools/GLANET</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>https://bio.tools/GRANVIL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>https://bio.tools/Genesis-indel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>https://bio.tools/GenoGeographer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>https://bio.tools/GenomeQC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>https://bio.tools/GkmExplain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>https://bio.tools/Graphtyper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>https://bio.tools/HAF-pipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>https://bio.tools/HIV-DRLink</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>https://bio.tools/HaTSPiL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>https://bio.tools/HeveaDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>https://bio.tools/HiCBricks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>https://bio.tools/ICGRM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>https://bio.tools/IMDA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>https://bio.tools/Immcantation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>https://bio.tools/ImtRDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>https://bio.tools/InferAMP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>https://bio.tools/Intelli-NGS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>https://bio.tools/JAMP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>https://bio.tools/LEMON</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>https://bio.tools/LFastqC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>https://bio.tools/LRScaf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>https://bio.tools/LRTK-SIM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>https://bio.tools/LULU</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>https://bio.tools/Loxodonta_Localizer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>https://bio.tools/MADloy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>https://bio.tools/MAtCHap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>https://bio.tools/MCSeEd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>https://bio.tools/MG-RAST_API_explorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>https://bio.tools/MIAmS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>https://bio.tools/MapCaller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>https://bio.tools/MaveDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>https://bio.tools/MiDRMpol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>https://bio.tools/MitoIMP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>https://bio.tools/MitoImpute</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>https://bio.tools/MoMI-G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>https://bio.tools/MorphoCatcher</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>https://bio.tools/MsPAC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>https://bio.tools/MtBNN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>https://bio.tools/MultiGuideScan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>https://bio.tools/MyelinJ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>https://bio.tools/NG-TAS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>https://bio.tools/NGSEP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>https://bio.tools/NGSReadsTreatment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>https://bio.tools/NIPTeR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>https://bio.tools/NUMT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>https://bio.tools/NUQA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>https://bio.tools/NanoCaller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>https://bio.tools/NanoDJ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>https://bio.tools/NexGenEx-Tom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>https://bio.tools/OC_manager</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>https://bio.tools/OutbreakFinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>https://bio.tools/P3BSseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>https://bio.tools/PASTMUS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>https://bio.tools/PERs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>https://bio.tools/PHAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>https://bio.tools/PHG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>https://bio.tools/PIRATE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>https://bio.tools/PSI-genome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>https://bio.tools/PVCTools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>https://bio.tools/PacBio_Sequencing_Simulator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>https://bio.tools/ParLECH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>https://bio.tools/PeCanPIE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>https://bio.tools/PhageXpress</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>https://bio.tools/PhytoTypeDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>https://bio.tools/PipeMEM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>https://bio.tools/PopLDdecay</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>https://bio.tools/PrimerMiner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>https://bio.tools/PyBSASeq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>https://bio.tools/PypeAmplicon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>https://bio.tools/RACE-SEQ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>https://bio.tools/RAM-ESVM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>https://bio.tools/REPIC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>https://bio.tools/RGAAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>https://bio.tools/RNAdt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>https://bio.tools/RaGOO</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>https://bio.tools/Rbbt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>https://bio.tools/RelocaTE2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>https://bio.tools/S-conLSH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>https://bio.tools/SCSsim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>https://bio.tools/SECAPR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>https://bio.tools/SEQdata-BEACON</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>https://bio.tools/SGL-SVM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>https://bio.tools/SINGLe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>https://bio.tools/SLAM-DUNK</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>https://bio.tools/SLAST</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>https://bio.tools/SLIM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>https://bio.tools/SLR-superscaffolder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>https://bio.tools/SOLQC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>https://bio.tools/SPDI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>https://bio.tools/SPINE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>https://bio.tools/SSRSC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>https://bio.tools/STing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>https://bio.tools/SVJedi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>https://bio.tools/SWAPCounter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>https://bio.tools/Sentieon_DNASeq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>https://bio.tools/SeqDivA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>https://bio.tools/SeroCall</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>https://bio.tools/SimRVSequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>https://bio.tools/SimkaMin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>https://bio.tools/SnpHub</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>https://bio.tools/SparkGA2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>https://bio.tools/SyRI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>https://bio.tools/T-lex3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>https://bio.tools/TDNAscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>https://bio.tools/TEF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>https://bio.tools/TGS-GapCloser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>https://bio.tools/TPES</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>https://bio.tools/Telescope</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>https://bio.tools/Telescope-expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>https://bio.tools/Tersect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>https://bio.tools/Tn-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>https://bio.tools/TransPrise</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>https://bio.tools/TreeFix-TP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>https://bio.tools/UMI-VarCal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>https://bio.tools/Uniqprimer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>https://bio.tools/VAPPER</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>https://bio.tools/VCF-Server</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>https://bio.tools/VISION</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>https://bio.tools/VPOT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>https://bio.tools/VariCarta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>https://bio.tools/Varstation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>https://bio.tools/VikNGS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>https://bio.tools/VirBin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>https://bio.tools/WHdenovo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>https://bio.tools/WIT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>https://bio.tools/Whisper_2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>https://bio.tools/XIBD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>https://bio.tools/Xylella</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>https://bio.tools/YALFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>https://bio.tools/YAPP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>https://bio.tools/Yersinia</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>https://bio.tools/ab12phylo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>https://bio.tools/abba</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>https://bio.tools/abcrpred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>https://bio.tools/abdiver</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>https://bio.tools/abra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>https://bio.tools/accuenhancer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>https://bio.tools/ace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>https://bio.tools/ace_insect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>https://bio.tools/aces-conservation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>https://bio.tools/adaptest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>https://bio.tools/addtag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>https://bio.tools/adfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>https://bio.tools/adtex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>https://bio.tools/afid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>https://bio.tools/agenda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>https://bio.tools/agile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>https://bio.tools/airlift</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>https://bio.tools/aitac</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>https://bio.tools/akdeseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>https://bio.tools/algaepath</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>https://bio.tools/aliclu</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>https://bio.tools/align-m</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>https://bio.tools/alignment-annotator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>https://bio.tools/alignmentviewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>https://bio.tools/alignparse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>https://bio.tools/allelicimbalance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>https://bio.tools/almostsignificant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>https://bio.tools/alntools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>https://bio.tools/ambar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>https://bio.tools/ambp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>https://bio.tools/amlvaran</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>https://bio.tools/ampdeep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>https://bio.tools/amphoranet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>https://bio.tools/amplicondesign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>https://bio.tools/amplify</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>https://bio.tools/amplikyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>https://bio.tools/amplimap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>https://bio.tools/ampliqueso</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>https://bio.tools/anaquin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>https://bio.tools/ancestralclust</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>https://bio.tools/ancestree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>https://bio.tools/andes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>https://bio.tools/aneufinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>https://bio.tools/angsd-wrapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>https://bio.tools/annonex2embl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>https://bio.tools/annotategenes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>https://bio.tools/annovar_annotate_variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>https://bio.tools/annovar_summarize</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>https://bio.tools/anota2seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>https://bio.tools/answer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>https://bio.tools/anti-crisprdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>https://bio.tools/anubis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>https://bio.tools/ape_plasmid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>https://bio.tools/apeglm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>https://bio.tools/aperture</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>https://bio.tools/apollo-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>https://bio.tools/arachne</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>https://bio.tools/aramis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>https://bio.tools/arg-shine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>https://bio.tools/argyle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>https://bio.tools/ariadne</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>https://bio.tools/ariadne-assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>https://bio.tools/arks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>https://bio.tools/arrayexpresshts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>https://bio.tools/arrmnormalization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>https://bio.tools/artificial-landscape-single-objective-problems</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>https://bio.tools/arts-db</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>https://bio.tools/asaim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>https://bio.tools/asap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>https://bio.tools/asap-assemble</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>https://bio.tools/asap_pipeline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>https://bio.tools/asc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>https://bio.tools/asepcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>https://bio.tools/asf-fast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>https://bio.tools/aspicov</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>https://bio.tools/assemblx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>https://bio.tools/asseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>https://bio.tools/asterias</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>https://bio.tools/atacseqqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>https://bio.tools/atlas2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>https://bio.tools/atlas_suite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>https://bio.tools/atram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>https://bio.tools/atria</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>https://bio.tools/auspp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>https://bio.tools/authentict</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>https://bio.tools/autocloner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>https://bio.tools/autoseqman</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>https://bio.tools/baalchip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>https://bio.tools/baca</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>https://bio.tools/bacarscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>https://bio.tools/baget</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>https://bio.tools/bait</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>https://bio.tools/bam2fastq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>https://bio.tools/bamsi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>https://bio.tools/bamstats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>https://bio.tools/bamtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>https://bio.tools/bankit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>https://bio.tools/bar-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>https://bio.tools/barcode_generator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>https://bio.tools/barcoder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>https://bio.tools/barcodetrackr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>https://bio.tools/barcosel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>https://bio.tools/barcrawl_bartab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>https://bio.tools/basdas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>https://bio.tools/base</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>https://bio.tools/base-antibody</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>https://bio.tools/base-genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>https://bio.tools/basic4cseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>https://bio.tools/basics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>https://bio.tools/basicstarrseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>https://bio.tools/batchqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>https://bio.tools/batl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>https://bio.tools/batmeth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>https://bio.tools/bayexer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>https://bio.tools/bayseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>https://bio.tools/bbmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>https://bio.tools/bbrowser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>https://bio.tools/bc-store</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>https://bio.tools/bcf-tools-index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>https://bio.tools/bcf_view_mpileup</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>https://bio.tools/bco-app</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>https://bio.tools/bcreval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>https://bio.tools/bcseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>https://bio.tools/bddb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>https://bio.tools/beadarraysnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>https://bio.tools/bedtools_genomecoveragebed_histogram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>https://bio.tools/bedtools_mergebedgraph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>https://bio.tools/bfc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>https://bio.tools/bfcounter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>https://bio.tools/bidas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>https://bio.tools/biforce_toolbox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>https://bio.tools/bigbwa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>https://bio.tools/bigld</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>https://bio.tools/bigmelon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>https://bio.tools/bigpre</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>https://bio.tools/bigtop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>https://bio.tools/bima</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>https://bio.tools/binosnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>https://bio.tools/binquasi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>https://bio.tools/bio-docklets</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>https://bio.tools/bio-tradis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>https://bio.tools/bio3d-web</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>https://bio.tools/biobakery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>https://bio.tools/biocgenerics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>https://bio.tools/biojs-io-sam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>https://bio.tools/bionanoanalyst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>https://bio.tools/biosequenceassembler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>https://bio.tools/bioseqzip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>https://bio.tools/biosvd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>https://bio.tools/biotmle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>https://bio.tools/biovizbase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>https://bio.tools/biq_analyzer_himod</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>https://bio.tools/biq_analyzer_ht</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>https://bio.tools/bis-snp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>https://bio.tools/biseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>https://bio.tools/bison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>https://bio.tools/bisulfighter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>https://bio.tools/bitmapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>https://bio.tools/blasr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>https://bio.tools/blast2srs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>https://bio.tools/bless</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>https://bio.tools/blight</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>https://bio.tools/blue</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>https://bio.tools/bm-bc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>https://bio.tools/bmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>https://bio.tools/bmt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>https://bio.tools/bnd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>https://bio.tools/boardion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>https://bio.tools/bolotie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>https://bio.tools/bowtie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>https://bio.tools/bphunter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>https://bio.tools/bpla_kernel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>https://bio.tools/breakalign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>https://bio.tools/breakpointR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>https://bio.tools/breakpointer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>https://bio.tools/brec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>https://bio.tools/breseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>https://bio.tools/breseq_bam2aln</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>https://bio.tools/browservizdemo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>https://bio.tools/brumir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>https://bio.tools/bs-seeker2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>https://bio.tools/bsaseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>https://bio.tools/bseqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>https://bio.tools/bsseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>https://bio.tools/bstools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>https://bio.tools/buddysuite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>https://bio.tools/bugsplit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>https://bio.tools/bumhmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>https://bio.tools/bwa-meme</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>https://bio.tools/bwt-sw</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>https://bio.tools/bwtool</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>https://bio.tools/c-rnncrispr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>https://bio.tools/cabgen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>https://bio.tools/cafe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>https://bio.tools/cafe_sequence_analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>https://bio.tools/cagefightr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>https://bio.tools/cager</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>https://bio.tools/camur</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>https://bio.tools/cancer_predisposition_sequencing_reporter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>https://bio.tools/caneapp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>https://bio.tools/canpredict</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>https://bio.tools/cant-hyd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>https://bio.tools/cap3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>https://bio.tools/capsim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>https://bio.tools/capturecompendium</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>https://bio.tools/car</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>https://bio.tools/care</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>https://bio.tools/carmo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>https://bio.tools/carpools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>https://bio.tools/cas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>https://bio.tools/cas-seed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>https://bio.tools/cas_image_analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>https://bio.tools/cascollect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>https://bio.tools/casper_merging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>https://bio.tools/cato</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>https://bio.tools/cato_Connectivity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>https://bio.tools/ccd2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>https://bio.tools/cdna-detector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>https://bio.tools/cellcoal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>https://bio.tools/cellhashr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>https://bio.tools/cfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>https://bio.tools/cgb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>https://bio.tools/cgmaptools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>https://bio.tools/cgv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>https://bio.tools/cgview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>https://bio.tools/chaincleaner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>https://bio.tools/changegfffeatures</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>https://bio.tools/chaos_dialign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>https://bio.tools/charplant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>https://bio.tools/chia-pet2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>https://bio.tools/chimerdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>https://bio.tools/chip-rna-seqpro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>https://bio.tools/chipanalyser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>https://bio.tools/chipexoqual</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>https://bio.tools/chipmunk_chiphorde</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>https://bio.tools/chipseqspike</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>https://bio.tools/chiron</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>https://bio.tools/chr7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>https://bio.tools/chromebat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>https://bio.tools/cindex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>https://bio.tools/classifycnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>https://bio.tools/clc_assembler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>https://bio.tools/clc_find_variations</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>https://bio.tools/clc_mapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>https://bio.tools/clc_mapper_legacy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>https://bio.tools/clc_mapping_info</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>https://bio.tools/clean_ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>https://bio.tools/clean_reads</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>https://bio.tools/clearcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>https://bio.tools/clineplotr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>https://bio.tools/clinical_ngs_db</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>https://bio.tools/clinqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>https://bio.tools/clip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>https://bio.tools/clipsv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>https://bio.tools/clomial</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>https://bio.tools/clonaltree_b_cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>https://bio.tools/clonocalc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>https://bio.tools/clonomatch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>https://bio.tools/cloudSPAdes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>https://bio.tools/clover</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>https://bio.tools/clover_cluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>https://bio.tools/clue</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>https://bio.tools/clue-kinetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>https://bio.tools/clusterflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>https://bio.tools/clusterizebyslidingwindows</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>https://bio.tools/clusterseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>https://bio.tools/clustperturb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>https://bio.tools/cmatch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>https://bio.tools/cmc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>https://bio.tools/cmic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>https://bio.tools/cmr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>https://bio.tools/cmvdrg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>https://bio.tools/cn3d</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>https://bio.tools/cnanorm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>https://bio.tools/cnaseg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>https://bio.tools/cnn-svr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>https://bio.tools/cnver</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>https://bio.tools/cnvfilter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>https://bio.tools/cnvtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>https://bio.tools/cocoa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>https://bio.tools/codacore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>https://bio.tools/codehop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>https://bio.tools/codex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>https://bio.tools/codex_ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>https://bio.tools/cohesindb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>https://bio.tools/collapsereads</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>https://bio.tools/colocr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>https://bio.tools/coloweb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>https://bio.tools/compairr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>https://bio.tools/compass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>https://bio.tools/compass_sim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>https://bio.tools/complet_plus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>https://bio.tools/condetri</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>https://bio.tools/conodictor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>https://bio.tools/contamld</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>https://bio.tools/contigextender</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>https://bio.tools/contignet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>https://bio.tools/contra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>https://bio.tools/convert2annovar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>https://bio.tools/convida</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>https://bio.tools/copla</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>https://bio.tools/corgen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>https://bio.tools/correct_bacode</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>https://bio.tools/coseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>https://bio.tools/cov2html</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>https://bio.tools/covid-19-mutation-browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>https://bio.tools/covid-deeppredictor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>https://bio.tools/covid-profiler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>https://bio.tools/coxbase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>https://bio.tools/cplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>https://bio.tools/cpss</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>https://bio.tools/createcontrolsubset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>https://bio.tools/crisp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>https://bio.tools/crisp_metabolomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>https://bio.tools/crispr-cbei</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>https://bio.tools/crisprbase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>https://bio.tools/crisprnano</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>https://bio.tools/crisprvariants</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>https://bio.tools/crisprverse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>https://bio.tools/crisprvi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>https://bio.tools/crispy-pop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>https://bio.tools/crop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>https://bio.tools/crosslink</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>https://bio.tools/cryptogenotyper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>https://bio.tools/csar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>https://bio.tools/csar_scaffolding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>https://bio.tools/csaw</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>https://bio.tools/csq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>https://bio.tools/ctdnatools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>https://bio.tools/ctk</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>https://bio.tools/cuda-ec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>https://bio.tools/cuffdiff</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>https://bio.tools/curc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>https://bio.tools/cushaw3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>https://bio.tools/customprodb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>https://bio.tools/cusvardb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>https://bio.tools/cutadapt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>https://bio.tools/cuteSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>https://bio.tools/cview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>https://bio.tools/cwSDTWnano</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>https://bio.tools/cwdprnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>https://bio.tools/cyber-t</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>https://bio.tools/cytodx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>https://bio.tools/cytomapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>https://bio.tools/cytospill</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>https://bio.tools/cytotree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>https://bio.tools/czeum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>https://bio.tools/d-pbwt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>https://bio.tools/dada2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>https://bio.tools/damageprofiler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>https://bio.tools/damirseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>https://bio.tools/dario</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>https://bio.tools/dbMMR-Chinese</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>https://bio.tools/dbamp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>https://bio.tools/dbscan-swa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>https://bio.tools/ddbj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>https://bio.tools/deBGA-VARA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>https://bio.tools/deGSM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>https://bio.tools/deSAMBA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>https://bio.tools/debar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>https://bio.tools/debreak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>https://bio.tools/decompTumor2Sig</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>https://bio.tools/decomplexdisease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>https://bio.tools/decontam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>https://bio.tools/deconvtest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>https://bio.tools/deepac-live</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>https://bio.tools/deepark</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>https://bio.tools/deepcage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>https://bio.tools/deepgengrep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>https://bio.tools/deepgmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>https://bio.tools/deepgrn</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>https://bio.tools/deephage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>https://bio.tools/deephic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>https://bio.tools/deephost</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>https://bio.tools/deepimagetranslator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>https://bio.tools/deepmr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>https://bio.tools/deepn</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>https://bio.tools/deeprepeat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>https://bio.tools/deepvf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>https://bio.tools/deepvisp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>https://bio.tools/defcom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>https://bio.tools/defume</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>https://bio.tools/delly2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>https://bio.tools/denovoid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>https://bio.tools/deploid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>https://bio.tools/deqor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>https://bio.tools/descan2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>https://bio.tools/desingle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>https://bio.tools/desp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>https://bio.tools/detect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>https://bio.tools/dgraph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>https://bio.tools/diapasef</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>https://bio.tools/dictybase_update_2011</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>https://bio.tools/diffcoexp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>https://bio.tools/diffhic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>https://bio.tools/dinamo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>https://bio.tools/dintd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>https://bio.tools/disambiguate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>https://bio.tools/discosnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>https://bio.tools/discount-kmer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>https://bio.tools/distangsd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>https://bio.tools/distmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>https://bio.tools/divis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>https://bio.tools/diyabc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>https://bio.tools/dmchmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>https://bio.tools/dmrcate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>https://bio.tools/dmrscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>https://bio.tools/dmrseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>https://bio.tools/dna-features-viewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>https://bio.tools/dna-mp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>https://bio.tools/dna-reed-solomon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>https://bio.tools/dna_for_windows</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>https://bio.tools/dnaasm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>https://bio.tools/dnamodannot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>618</th>\n",
       "      <td>https://bio.tools/dnannotator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>619</th>\n",
       "      <td>https://bio.tools/dnashape</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>620</th>\n",
       "      <td>https://bio.tools/dnasmart</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>621</th>\n",
       "      <td>https://bio.tools/dnmso</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>622</th>\n",
       "      <td>https://bio.tools/dolphinnext</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>623</th>\n",
       "      <td>https://bio.tools/dominoeffect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>624</th>\n",
       "      <td>https://bio.tools/dotlet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>625</th>\n",
       "      <td>https://bio.tools/dpcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>626</th>\n",
       "      <td>https://bio.tools/drgap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>627</th>\n",
       "      <td>https://bio.tools/drimpute</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>628</th>\n",
       "      <td>https://bio.tools/driverrwh</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>629</th>\n",
       "      <td>https://bio.tools/dropletutils</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>https://bio.tools/dscheck</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>631</th>\n",
       "      <td>https://bio.tools/dsm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>632</th>\n",
       "      <td>https://bio.tools/dss</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>633</th>\n",
       "      <td>https://bio.tools/dupchecker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>634</th>\n",
       "      <td>https://bio.tools/dupre</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>635</th>\n",
       "      <td>https://bio.tools/e2g</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>636</th>\n",
       "      <td>https://bio.tools/eVIDENCE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>637</th>\n",
       "      <td>https://bio.tools/ea-utils</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>https://bio.tools/eagle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>639</th>\n",
       "      <td>https://bio.tools/eagle_R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>640</th>\n",
       "      <td>https://bio.tools/eagleimp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>641</th>\n",
       "      <td>https://bio.tools/eagleview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>https://bio.tools/earrings</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>643</th>\n",
       "      <td>https://bio.tools/easymap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>644</th>\n",
       "      <td>https://bio.tools/easymicroplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>645</th>\n",
       "      <td>https://bio.tools/easyparallel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>646</th>\n",
       "      <td>https://bio.tools/ecr_browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>https://bio.tools/ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>648</th>\n",
       "      <td>https://bio.tools/edge_covid-19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>649</th>\n",
       "      <td>https://bio.tools/ednaflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>650</th>\n",
       "      <td>https://bio.tools/eigendel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>https://bio.tools/elprep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>https://bio.tools/emdomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>https://bio.tools/enano</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>https://bio.tools/enhanceratlas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>https://bio.tools/enhort</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>656</th>\n",
       "      <td>https://bio.tools/enngene</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>https://bio.tools/enologos</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>658</th>\n",
       "      <td>https://bio.tools/enrichmir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>659</th>\n",
       "      <td>https://bio.tools/enterix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>https://bio.tools/epigenetic-target-profiler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>https://bio.tools/epimolas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>662</th>\n",
       "      <td>https://bio.tools/eqtlseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>https://bio.tools/erne</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>https://bio.tools/errorx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>https://bio.tools/esprit-forest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>https://bio.tools/esshi-c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>https://bio.tools/est2assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>668</th>\n",
       "      <td>https://bio.tools/euadb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>https://bio.tools/eventpointer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>670</th>\n",
       "      <td>https://bio.tools/evol2circos</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>671</th>\n",
       "      <td>https://bio.tools/evolstm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>672</th>\n",
       "      <td>https://bio.tools/exodus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>673</th>\n",
       "      <td>https://bio.tools/exomedepth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>674</th>\n",
       "      <td>https://bio.tools/exomepicks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>675</th>\n",
       "      <td>https://bio.tools/expressionatlas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>676</th>\n",
       "      <td>https://bio.tools/ezcount</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>677</th>\n",
       "      <td>https://bio.tools/ezmito</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>678</th>\n",
       "      <td>https://bio.tools/f-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>https://bio.tools/fada</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>https://bio.tools/famdenovo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>https://bio.tools/fasparser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>https://bio.tools/fast-hbr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>https://bio.tools/fasta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>684</th>\n",
       "      <td>https://bio.tools/fastafrombed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>https://bio.tools/fastafs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>https://bio.tools/fastml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>https://bio.tools/fastpcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>https://bio.tools/fastqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>https://bio.tools/fastqc_toolshed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>690</th>\n",
       "      <td>https://bio.tools/fastqcls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>691</th>\n",
       "      <td>https://bio.tools/fastqins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>692</th>\n",
       "      <td>https://bio.tools/fastquick</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>693</th>\n",
       "      <td>https://bio.tools/fastqz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>694</th>\n",
       "      <td>https://bio.tools/fastseg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>https://bio.tools/fastv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>https://bio.tools/fccac</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>https://bio.tools/feature-viewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>https://bio.tools/filtercontrol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>https://bio.tools/filterffpe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>700</th>\n",
       "      <td>https://bio.tools/fings</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>701</th>\n",
       "      <td>https://bio.tools/fish</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>702</th>\n",
       "      <td>https://bio.tools/flare</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>703</th>\n",
       "      <td>https://bio.tools/flare_inference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>704</th>\n",
       "      <td>https://bio.tools/flash</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>705</th>\n",
       "      <td>https://bio.tools/flt3-itd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>706</th>\n",
       "      <td>https://bio.tools/flype</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>https://bio.tools/fogsaa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>https://bio.tools/forge2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>709</th>\n",
       "      <td>https://bio.tools/fork-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>710</th>\n",
       "      <td>https://bio.tools/fqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>711</th>\n",
       "      <td>https://bio.tools/fqcleaner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>712</th>\n",
       "      <td>https://bio.tools/fqduplicate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>713</th>\n",
       "      <td>https://bio.tools/fqquality</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>714</th>\n",
       "      <td>https://bio.tools/framed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>715</th>\n",
       "      <td>https://bio.tools/freecyto</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>716</th>\n",
       "      <td>https://bio.tools/frogancestrycalc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>717</th>\n",
       "      <td>https://bio.tools/frogs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>718</th>\n",
       "      <td>https://bio.tools/frogs_demultiplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>719</th>\n",
       "      <td>https://bio.tools/frogs_normalisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>720</th>\n",
       "      <td>https://bio.tools/frogs_preprocess</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>https://bio.tools/fsbc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>722</th>\n",
       "      <td>https://bio.tools/fsuite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>723</th>\n",
       "      <td>https://bio.tools/funtoonorm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>724</th>\n",
       "      <td>https://bio.tools/fusionanalyser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>725</th>\n",
       "      <td>https://bio.tools/gadem</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>726</th>\n",
       "      <td>https://bio.tools/galru</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>727</th>\n",
       "      <td>https://bio.tools/gam-ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>728</th>\n",
       "      <td>https://bio.tools/gambit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>729</th>\n",
       "      <td>https://bio.tools/gaml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>730</th>\n",
       "      <td>https://bio.tools/gamola2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>https://bio.tools/gamp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>732</th>\n",
       "      <td>https://bio.tools/gatb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>733</th>\n",
       "      <td>https://bio.tools/gatekeeper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>734</th>\n",
       "      <td>https://bio.tools/gatk</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>https://bio.tools/gatk_haplotype_caller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>https://bio.tools/gatk_unified_genotyper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>https://bio.tools/gatk_validate_variants</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>https://bio.tools/gatk_variant_apply_recalibration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>https://bio.tools/gatk_variant_eval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>740</th>\n",
       "      <td>https://bio.tools/gatk_variant_filtration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>741</th>\n",
       "      <td>https://bio.tools/gatk_variant_recalibrator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>742</th>\n",
       "      <td>https://bio.tools/gavisunk</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>743</th>\n",
       "      <td>https://bio.tools/gc-profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>744</th>\n",
       "      <td>https://bio.tools/gcapc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>745</th>\n",
       "      <td>https://bio.tools/gcup</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>746</th>\n",
       "      <td>https://bio.tools/gdc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>https://bio.tools/gear</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>748</th>\n",
       "      <td>https://bio.tools/gecont_2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>749</th>\n",
       "      <td>https://bio.tools/gedi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>750</th>\n",
       "      <td>https://bio.tools/gedpm-cg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>https://bio.tools/gelquest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>https://bio.tools/gem_library</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>753</th>\n",
       "      <td>https://bio.tools/gemprospector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>754</th>\n",
       "      <td>https://bio.tools/gemse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>https://bio.tools/genBaRcode</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>756</th>\n",
       "      <td>https://bio.tools/genbank_to_gene</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>757</th>\n",
       "      <td>https://bio.tools/gencore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>758</th>\n",
       "      <td>https://bio.tools/gendecoder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>759</th>\n",
       "      <td>https://bio.tools/gene-counter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>760</th>\n",
       "      <td>https://bio.tools/gene2oligo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>761</th>\n",
       "      <td>https://bio.tools/gene4hl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>762</th>\n",
       "      <td>https://bio.tools/genebasis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>763</th>\n",
       "      <td>https://bio.tools/genedata_expressionist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>https://bio.tools/genefas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>https://bio.tools/genefisher</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>https://bio.tools/genegeneinter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>https://bio.tools/genegenie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>https://bio.tools/geneseqtofamily</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>769</th>\n",
       "      <td>https://bio.tools/genomebrowse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>770</th>\n",
       "      <td>https://bio.tools/genomicinteractions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>771</th>\n",
       "      <td>https://bio.tools/genomicscores</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>https://bio.tools/genomix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>773</th>\n",
       "      <td>https://bio.tools/genoscapist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>774</th>\n",
       "      <td>https://bio.tools/genovault</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>775</th>\n",
       "      <td>https://bio.tools/genoviewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>776</th>\n",
       "      <td>https://bio.tools/gess_v2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>777</th>\n",
       "      <td>https://bio.tools/gexplora</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>778</th>\n",
       "      <td>https://bio.tools/gfapy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>779</th>\n",
       "      <td>https://bio.tools/ggb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>https://bio.tools/ggd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>https://bio.tools/ggmotif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>782</th>\n",
       "      <td>https://bio.tools/ggmsa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>783</th>\n",
       "      <td>https://bio.tools/ggseqlogo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>784</th>\n",
       "      <td>https://bio.tools/ghostipy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>https://bio.tools/gips</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>786</th>\n",
       "      <td>https://bio.tools/glamst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>787</th>\n",
       "      <td>https://bio.tools/glapd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>788</th>\n",
       "      <td>https://bio.tools/glassesviewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>789</th>\n",
       "      <td>https://bio.tools/glycopost</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>790</th>\n",
       "      <td>https://bio.tools/gm-pep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>791</th>\n",
       "      <td>https://bio.tools/gmap_iit_store</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>https://bio.tools/gmcloser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>793</th>\n",
       "      <td>https://bio.tools/gmql</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>https://bio.tools/gnocis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>795</th>\n",
       "      <td>https://bio.tools/gnome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>796</th>\n",
       "      <td>https://bio.tools/gnosis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>797</th>\n",
       "      <td>https://bio.tools/go-crispr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>798</th>\n",
       "      <td>https://bio.tools/goblet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>799</th>\n",
       "      <td>https://bio.tools/golden_helix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>800</th>\n",
       "      <td>https://bio.tools/gowinda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>https://bio.tools/gpart</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>802</th>\n",
       "      <td>https://bio.tools/gpu-darwin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>803</th>\n",
       "      <td>https://bio.tools/grabseqs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>804</th>\n",
       "      <td>https://bio.tools/gracy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>805</th>\n",
       "      <td>https://bio.tools/grade</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>https://bio.tools/graph4med</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>https://bio.tools/graphbin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>https://bio.tools/graphery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>https://bio.tools/graphomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>https://bio.tools/grasp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>https://bio.tools/grasp_web</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>812</th>\n",
       "      <td>https://bio.tools/grimoirelab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>813</th>\n",
       "      <td>https://bio.tools/grmt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>814</th>\n",
       "      <td>https://bio.tools/groc-svs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>815</th>\n",
       "      <td>https://bio.tools/grohmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>https://bio.tools/gs-preprocess</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>817</th>\n",
       "      <td>https://bio.tools/gsalign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>818</th>\n",
       "      <td>https://bio.tools/gto</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>819</th>\n",
       "      <td>https://bio.tools/gtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>820</th>\n",
       "      <td>https://bio.tools/gtrellis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>821</th>\n",
       "      <td>https://bio.tools/gtseq-pipeline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>https://bio.tools/guidance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>823</th>\n",
       "      <td>https://bio.tools/guidance2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>824</th>\n",
       "      <td>https://bio.tools/gunc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>825</th>\n",
       "      <td>https://bio.tools/guppy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>826</th>\n",
       "      <td>https://bio.tools/gwas3d</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>https://bio.tools/h-blast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>828</th>\n",
       "      <td>https://bio.tools/h3m2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>829</th>\n",
       "      <td>https://bio.tools/hadoop-cnv-rf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>830</th>\n",
       "      <td>https://bio.tools/hadoopcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>831</th>\n",
       "      <td>https://bio.tools/hapedit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>832</th>\n",
       "      <td>https://bio.tools/haphpipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>833</th>\n",
       "      <td>https://bio.tools/haploclique</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>834</th>\n",
       "      <td>https://bio.tools/haplodmf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>835</th>\n",
       "      <td>https://bio.tools/haploscribe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>836</th>\n",
       "      <td>https://bio.tools/haplotracker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>837</th>\n",
       "      <td>https://bio.tools/haplotypetools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>https://bio.tools/harmonies</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>839</th>\n",
       "      <td>https://bio.tools/harsh</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>https://bio.tools/haslr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>841</th>\n",
       "      <td>https://bio.tools/hast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>842</th>\n",
       "      <td>https://bio.tools/hdam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>843</th>\n",
       "      <td>https://bio.tools/heap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>844</th>\n",
       "      <td>https://bio.tools/heat_seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>845</th>\n",
       "      <td>https://bio.tools/helixvis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>https://bio.tools/hemomips</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>847</th>\n",
       "      <td>https://bio.tools/heuraa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>848</th>\n",
       "      <td>https://bio.tools/hexse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>849</th>\n",
       "      <td>https://bio.tools/hgt-id</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>850</th>\n",
       "      <td>https://bio.tools/hhrep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>851</th>\n",
       "      <td>https://bio.tools/hic-te</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>852</th>\n",
       "      <td>https://bio.tools/hifi-se</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>853</th>\n",
       "      <td>https://bio.tools/hipstr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>854</th>\n",
       "      <td>https://bio.tools/hireadsprocessor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>855</th>\n",
       "      <td>https://bio.tools/hitec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>856</th>\n",
       "      <td>https://bio.tools/hitrace-web</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>857</th>\n",
       "      <td>https://bio.tools/hitselect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>858</th>\n",
       "      <td>https://bio.tools/hive</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>859</th>\n",
       "      <td>https://bio.tools/hive_panel_explorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>860</th>\n",
       "      <td>https://bio.tools/hivintact</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>861</th>\n",
       "      <td>https://bio.tools/hla-net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>https://bio.tools/hlareporter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>863</th>\n",
       "      <td>https://bio.tools/hmd-arg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>864</th>\n",
       "      <td>https://bio.tools/hmm-frame</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>865</th>\n",
       "      <td>https://bio.tools/hmvv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>866</th>\n",
       "      <td>https://bio.tools/hnatyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>867</th>\n",
       "      <td>https://bio.tools/hophage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>868</th>\n",
       "      <td>https://bio.tools/hops_malt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>869</th>\n",
       "      <td>https://bio.tools/hpg_aligner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>870</th>\n",
       "      <td>https://bio.tools/hpg_pore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>871</th>\n",
       "      <td>https://bio.tools/hsst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>872</th>\n",
       "      <td>https://bio.tools/htaadvar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>873</th>\n",
       "      <td>https://bio.tools/htscluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>874</th>\n",
       "      <td>https://bio.tools/htseq-clip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>875</th>\n",
       "      <td>https://bio.tools/htseqcount</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>876</th>\n",
       "      <td>https://bio.tools/htseqqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>877</th>\n",
       "      <td>https://bio.tools/htseqtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>878</th>\n",
       "      <td>https://bio.tools/htsfilter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>879</th>\n",
       "      <td>https://bio.tools/htsint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>880</th>\n",
       "      <td>https://bio.tools/htsqualc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>881</th>\n",
       "      <td>https://bio.tools/hubdesign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>882</th>\n",
       "      <td>https://bio.tools/hyb4mc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>883</th>\n",
       "      <td>https://bio.tools/hybrid-denovo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>884</th>\n",
       "      <td>https://bio.tools/hybrid-shrec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>885</th>\n",
       "      <td>https://bio.tools/hydrogen_bonding_does</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>886</th>\n",
       "      <td>https://bio.tools/hyindel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>887</th>\n",
       "      <td>https://bio.tools/hyper-testing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>888</th>\n",
       "      <td>https://bio.tools/hyperex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>889</th>\n",
       "      <td>https://bio.tools/iDASH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>890</th>\n",
       "      <td>https://bio.tools/iDHS-DMCAC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>891</th>\n",
       "      <td>https://bio.tools/iRODS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>892</th>\n",
       "      <td>https://bio.tools/iVariantGuide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>893</th>\n",
       "      <td>https://bio.tools/iadmix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>894</th>\n",
       "      <td>https://bio.tools/ibex_imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>https://bio.tools/icat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>896</th>\n",
       "      <td>https://bio.tools/icc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>897</th>\n",
       "      <td>https://bio.tools/icheck</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>898</th>\n",
       "      <td>https://bio.tools/icrisee</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>899</th>\n",
       "      <td>https://bio.tools/id-taxer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>900</th>\n",
       "      <td>https://bio.tools/ideal_expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>901</th>\n",
       "      <td>https://bio.tools/ideal_rna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>902</th>\n",
       "      <td>https://bio.tools/idhs-dasts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>903</th>\n",
       "      <td>https://bio.tools/idhs-deep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>904</th>\n",
       "      <td>https://bio.tools/idrmutpred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>905</th>\n",
       "      <td>https://bio.tools/iexcerno</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>https://bio.tools/ifcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>907</th>\n",
       "      <td>https://bio.tools/igblast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>https://bio.tools/igidiva</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>909</th>\n",
       "      <td>https://bio.tools/igmat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>910</th>\n",
       "      <td>https://bio.tools/igrepertoireconstructor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>911</th>\n",
       "      <td>https://bio.tools/ikaraj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>912</th>\n",
       "      <td>https://bio.tools/ikernint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>913</th>\n",
       "      <td>https://bio.tools/illumina</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>914</th>\n",
       "      <td>https://bio.tools/illuminaio</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>915</th>\n",
       "      <td>https://bio.tools/imas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>916</th>\n",
       "      <td>https://bio.tools/imetagene</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>917</th>\n",
       "      <td>https://bio.tools/img_vr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>918</th>\n",
       "      <td>https://bio.tools/imgt_v-quest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>919</th>\n",
       "      <td>https://bio.tools/imman</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>https://bio.tools/improver</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>921</th>\n",
       "      <td>https://bio.tools/impulsede</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>922</th>\n",
       "      <td>https://bio.tools/impulsede2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>923</th>\n",
       "      <td>https://bio.tools/indelgt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>924</th>\n",
       "      <td>https://bio.tools/ingap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>925</th>\n",
       "      <td>https://bio.tools/ingap-family</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>926</th>\n",
       "      <td>https://bio.tools/inoiset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>927</th>\n",
       "      <td>https://bio.tools/insector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>928</th>\n",
       "      <td>https://bio.tools/insertionmapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>929</th>\n",
       "      <td>https://bio.tools/insilicosimmolbioexpt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>930</th>\n",
       "      <td>https://bio.tools/intact-app</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>931</th>\n",
       "      <td>https://bio.tools/intad</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>932</th>\n",
       "      <td>https://bio.tools/intansv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>933</th>\n",
       "      <td>https://bio.tools/integrate-vis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>934</th>\n",
       "      <td>https://bio.tools/integron_gene_fetch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>935</th>\n",
       "      <td>https://bio.tools/intemap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>936</th>\n",
       "      <td>https://bio.tools/interactivecomplexheatmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>937</th>\n",
       "      <td>https://bio.tools/interactomeseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>938</th>\n",
       "      <td>https://bio.tools/interest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>939</th>\n",
       "      <td>https://bio.tools/intsim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>940</th>\n",
       "      <td>https://bio.tools/invbfm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>941</th>\n",
       "      <td>https://bio.tools/inversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>942</th>\n",
       "      <td>https://bio.tools/ionbreeders</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>943</th>\n",
       "      <td>https://bio.tools/ioncram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>944</th>\n",
       "      <td>https://bio.tools/ipd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>945</th>\n",
       "      <td>https://bio.tools/ipd-pathogen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>946</th>\n",
       "      <td>https://bio.tools/ipd-pipeline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>947</th>\n",
       "      <td>https://bio.tools/ipromoter-seqvec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>948</th>\n",
       "      <td>https://bio.tools/iroar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>949</th>\n",
       "      <td>https://bio.tools/iscdnase-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>950</th>\n",
       "      <td>https://bio.tools/iseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>951</th>\n",
       "      <td>https://bio.tools/isfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>952</th>\n",
       "      <td>https://bio.tools/isith</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>953</th>\n",
       "      <td>https://bio.tools/isofinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>954</th>\n",
       "      <td>https://bio.tools/isoglyp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>955</th>\n",
       "      <td>https://bio.tools/isolde</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>956</th>\n",
       "      <td>https://bio.tools/isomir-sea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>957</th>\n",
       "      <td>https://bio.tools/itagplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>958</th>\n",
       "      <td>https://bio.tools/itercluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>959</th>\n",
       "      <td>https://bio.tools/iteremoval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>960</th>\n",
       "      <td>https://bio.tools/itsonewb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>961</th>\n",
       "      <td>https://bio.tools/iupred3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>962</th>\n",
       "      <td>https://bio.tools/ivar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>963</th>\n",
       "      <td>https://bio.tools/ivygapse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>964</th>\n",
       "      <td>https://bio.tools/j-space</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>965</th>\n",
       "      <td>https://bio.tools/jamm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>https://bio.tools/janggu</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>https://bio.tools/jmorp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>https://bio.tools/juicebox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>https://bio.tools/juicer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>https://bio.tools/k-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>971</th>\n",
       "      <td>https://bio.tools/kage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>972</th>\n",
       "      <td>https://bio.tools/kantoo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>973</th>\n",
       "      <td>https://bio.tools/karga</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>974</th>\n",
       "      <td>https://bio.tools/kataegis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>975</th>\n",
       "      <td>https://bio.tools/keras_dna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>976</th>\n",
       "      <td>https://bio.tools/kgem</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>977</th>\n",
       "      <td>https://bio.tools/kic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>978</th>\n",
       "      <td>https://bio.tools/kmer-ssr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>979</th>\n",
       "      <td>https://bio.tools/kmer2snp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>980</th>\n",
       "      <td>https://bio.tools/kmerPyramid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>981</th>\n",
       "      <td>https://bio.tools/kmeranalyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>982</th>\n",
       "      <td>https://bio.tools/kmergo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>983</th>\n",
       "      <td>https://bio.tools/kmerstream</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>984</th>\n",
       "      <td>https://bio.tools/knime4ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>985</th>\n",
       "      <td>https://bio.tools/kontaminant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>https://bio.tools/kpath</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>987</th>\n",
       "      <td>https://bio.tools/ktrim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>988</th>\n",
       "      <td>https://bio.tools/lanceotron</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>989</th>\n",
       "      <td>https://bio.tools/lancet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>990</th>\n",
       "      <td>https://bio.tools/layercake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>991</th>\n",
       "      <td>https://bio.tools/lcqs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>992</th>\n",
       "      <td>https://bio.tools/ldak-gbat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>993</th>\n",
       "      <td>https://bio.tools/ledpred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>994</th>\n",
       "      <td>https://bio.tools/lfc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>https://bio.tools/libcsam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>https://bio.tools/lightassembler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>https://bio.tools/lighter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>https://bio.tools/lime</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>https://bio.tools/linda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1000</th>\n",
       "      <td>https://bio.tools/lineagepulse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>https://bio.tools/lineagespot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1002</th>\n",
       "      <td>https://bio.tools/linflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>https://bio.tools/links</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1004</th>\n",
       "      <td>https://bio.tools/linksnps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1005</th>\n",
       "      <td>https://bio.tools/linnorm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1006</th>\n",
       "      <td>https://bio.tools/llamanade</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1007</th>\n",
       "      <td>https://bio.tools/lmcgst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1008</th>\n",
       "      <td>https://bio.tools/localngsrelate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1009</th>\n",
       "      <td>https://bio.tools/lochap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1010</th>\n",
       "      <td>https://bio.tools/loci2path</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1011</th>\n",
       "      <td>https://bio.tools/locsmoc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1012</th>\n",
       "      <td>https://bio.tools/locuszoom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1013</th>\n",
       "      <td>https://bio.tools/lookseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1014</th>\n",
       "      <td>https://bio.tools/loopviz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1015</th>\n",
       "      <td>https://bio.tools/loqum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1016</th>\n",
       "      <td>https://bio.tools/lordFAST</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1017</th>\n",
       "      <td>https://bio.tools/lordec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1018</th>\n",
       "      <td>https://bio.tools/loretta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1019</th>\n",
       "      <td>https://bio.tools/lowcomplexity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1020</th>\n",
       "      <td>https://bio.tools/lox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1021</th>\n",
       "      <td>https://bio.tools/lrod</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1022</th>\n",
       "      <td>https://bio.tools/ltr_finder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1023</th>\n",
       "      <td>https://bio.tools/lumi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1024</th>\n",
       "      <td>https://bio.tools/lumi-pcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1025</th>\n",
       "      <td>https://bio.tools/luxrep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>https://bio.tools/lymanalyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1027</th>\n",
       "      <td>https://bio.tools/m3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1028</th>\n",
       "      <td>https://bio.tools/m3c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1029</th>\n",
       "      <td>https://bio.tools/macrosight</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1030</th>\n",
       "      <td>https://bio.tools/mad</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1031</th>\n",
       "      <td>https://bio.tools/madap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1032</th>\n",
       "      <td>https://bio.tools/mafsnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1033</th>\n",
       "      <td>https://bio.tools/maisen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1034</th>\n",
       "      <td>https://bio.tools/maker-p</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1035</th>\n",
       "      <td>https://bio.tools/mako</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1036</th>\n",
       "      <td>https://bio.tools/malacoda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1037</th>\n",
       "      <td>https://bio.tools/mam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1038</th>\n",
       "      <td>https://bio.tools/mammalian_gene_collection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1039</th>\n",
       "      <td>https://bio.tools/mapdamage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1040</th>\n",
       "      <td>https://bio.tools/mapdatage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1041</th>\n",
       "      <td>https://bio.tools/mapgl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1042</th>\n",
       "      <td>https://bio.tools/marbl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1043</th>\n",
       "      <td>https://bio.tools/marvin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1044</th>\n",
       "      <td>https://bio.tools/masc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>https://bio.tools/mashmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1046</th>\n",
       "      <td>https://bio.tools/mastro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1047</th>\n",
       "      <td>https://bio.tools/matchseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1048</th>\n",
       "      <td>https://bio.tools/mavid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1049</th>\n",
       "      <td>https://bio.tools/maxalign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1050</th>\n",
       "      <td>https://bio.tools/maxatac</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>https://bio.tools/mbcb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1052</th>\n",
       "      <td>https://bio.tools/mbg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053</th>\n",
       "      <td>https://bio.tools/mbv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1054</th>\n",
       "      <td>https://bio.tools/mccortex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1055</th>\n",
       "      <td>https://bio.tools/mcna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1056</th>\n",
       "      <td>https://bio.tools/mdbenchmark</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1057</th>\n",
       "      <td>https://bio.tools/mdts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1058</th>\n",
       "      <td>https://bio.tools/me-scan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1059</th>\n",
       "      <td>https://bio.tools/meat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1060</th>\n",
       "      <td>https://bio.tools/mecaf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1061</th>\n",
       "      <td>https://bio.tools/mecat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1062</th>\n",
       "      <td>https://bio.tools/meconcord</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1063</th>\n",
       "      <td>https://bio.tools/medea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1064</th>\n",
       "      <td>https://bio.tools/medips</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1065</th>\n",
       "      <td>https://bio.tools/meerkat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1066</th>\n",
       "      <td>https://bio.tools/mefit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1067</th>\n",
       "      <td>https://bio.tools/megahit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1068</th>\n",
       "      <td>https://bio.tools/megamapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1069</th>\n",
       "      <td>https://bio.tools/megapath</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1070</th>\n",
       "      <td>https://bio.tools/mendel-gpu</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1071</th>\n",
       "      <td>https://bio.tools/mercator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1072</th>\n",
       "      <td>https://bio.tools/mercator_plant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1073</th>\n",
       "      <td>https://bio.tools/mergingdataclusterize</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1074</th>\n",
       "      <td>https://bio.tools/merit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1075</th>\n",
       "      <td>https://bio.tools/mesa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1076</th>\n",
       "      <td>https://bio.tools/met</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1077</th>\n",
       "      <td>https://bio.tools/meta-align</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1078</th>\n",
       "      <td>https://bio.tools/metabcc-lr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1079</th>\n",
       "      <td>https://bio.tools/metabin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1080</th>\n",
       "      <td>https://bio.tools/metacoag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1081</th>\n",
       "      <td>https://bio.tools/metacram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1082</th>\n",
       "      <td>https://bio.tools/metacv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1083</th>\n",
       "      <td>https://bio.tools/metafunc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1084</th>\n",
       "      <td>https://bio.tools/metafunprimer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1085</th>\n",
       "      <td>https://bio.tools/metagenesense</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1086</th>\n",
       "      <td>https://bio.tools/metagenomeseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1087</th>\n",
       "      <td>https://bio.tools/metamos</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1088</th>\n",
       "      <td>https://bio.tools/metapepseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1089</th>\n",
       "      <td>https://bio.tools/metaseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1090</th>\n",
       "      <td>https://bio.tools/metasv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1091</th>\n",
       "      <td>https://bio.tools/metatrass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>https://bio.tools/methcoresprofiler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>https://bio.tools/methgo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>https://bio.tools/methimpute</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>https://bio.tools/methinheritsim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>https://bio.tools/methpanel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1097</th>\n",
       "      <td>https://bio.tools/methpipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1098</th>\n",
       "      <td>https://bio.tools/methtargetedngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1099</th>\n",
       "      <td>https://bio.tools/methylcoder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1100</th>\n",
       "      <td>https://bio.tools/methylextract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1101</th>\n",
       "      <td>https://bio.tools/methylflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1102</th>\n",
       "      <td>https://bio.tools/methylinheritance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1103</th>\n",
       "      <td>https://bio.tools/methylmnm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1104</th>\n",
       "      <td>https://bio.tools/methylsig</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1105</th>\n",
       "      <td>https://bio.tools/methylstar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1106</th>\n",
       "      <td>https://bio.tools/methylumi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1107</th>\n",
       "      <td>https://bio.tools/methylviewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1108</th>\n",
       "      <td>https://bio.tools/mfcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1109</th>\n",
       "      <td>https://bio.tools/mgfr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1110</th>\n",
       "      <td>https://bio.tools/mgip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1111</th>\n",
       "      <td>https://bio.tools/mgtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1112</th>\n",
       "      <td>https://bio.tools/mica-aligner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1113</th>\n",
       "      <td>https://bio.tools/micc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1114</th>\n",
       "      <td>https://bio.tools/micro-primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1115</th>\n",
       "      <td>https://bio.tools/microbegps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1116</th>\n",
       "      <td>https://bio.tools/microclass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1117</th>\n",
       "      <td>https://bio.tools/microhapdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1118</th>\n",
       "      <td>https://bio.tools/microrazers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1119</th>\n",
       "      <td>https://bio.tools/microtss</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1120</th>\n",
       "      <td>https://bio.tools/mimsi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1121</th>\n",
       "      <td>https://bio.tools/mindel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1122</th>\n",
       "      <td>https://bio.tools/miner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1123</th>\n",
       "      <td>https://bio.tools/miniception</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1124</th>\n",
       "      <td>https://bio.tools/minimac4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1125</th>\n",
       "      <td>https://bio.tools/minion_synthmetagenome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1126</th>\n",
       "      <td>https://bio.tools/minotour</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1127</th>\n",
       "      <td>https://bio.tools/mip_scaffolder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1128</th>\n",
       "      <td>https://bio.tools/mipgen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1129</th>\n",
       "      <td>https://bio.tools/mircat2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1130</th>\n",
       "      <td>https://bio.tools/mirdeep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1131</th>\n",
       "      <td>https://bio.tools/mirgalaxy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1132</th>\n",
       "      <td>https://bio.tools/mirmod</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1133</th>\n",
       "      <td>https://bio.tools/mirna_digger</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1134</th>\n",
       "      <td>https://bio.tools/mirnatap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1135</th>\n",
       "      <td>https://bio.tools/mirplant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1136</th>\n",
       "      <td>https://bio.tools/mirplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1137</th>\n",
       "      <td>https://bio.tools/mirprof</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1138</th>\n",
       "      <td>https://bio.tools/mirspring</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1139</th>\n",
       "      <td>https://bio.tools/missmethyl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1140</th>\n",
       "      <td>https://bio.tools/mitodataclean</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1141</th>\n",
       "      <td>https://bio.tools/mitoflex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1142</th>\n",
       "      <td>https://bio.tools/mitolink</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1143</th>\n",
       "      <td>https://bio.tools/mitoscape</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1144</th>\n",
       "      <td>https://bio.tools/mixindependr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1145</th>\n",
       "      <td>https://bio.tools/mlgt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1146</th>\n",
       "      <td>https://bio.tools/mlrho</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1147</th>\n",
       "      <td>https://bio.tools/mmr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1148</th>\n",
       "      <td>https://bio.tools/mnhn-tree-tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1149</th>\n",
       "      <td>https://bio.tools/mobilomefinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1150</th>\n",
       "      <td>https://bio.tools/mobster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1151</th>\n",
       "      <td>https://bio.tools/modil</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1152</th>\n",
       "      <td>https://bio.tools/monas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1153</th>\n",
       "      <td>https://bio.tools/moscato</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1154</th>\n",
       "      <td>https://bio.tools/motif_scraper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1155</th>\n",
       "      <td>https://bio.tools/motifstack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1156</th>\n",
       "      <td>https://bio.tools/motimul</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1157</th>\n",
       "      <td>https://bio.tools/motto</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1158</th>\n",
       "      <td>https://bio.tools/motulizer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1159</th>\n",
       "      <td>https://bio.tools/movis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1160</th>\n",
       "      <td>https://bio.tools/mpd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1161</th>\n",
       "      <td>https://bio.tools/mpd-primer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1162</th>\n",
       "      <td>https://bio.tools/mpileup</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1163</th>\n",
       "      <td>https://bio.tools/mpileup_vcf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1164</th>\n",
       "      <td>https://bio.tools/mpra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1165</th>\n",
       "      <td>https://bio.tools/mpradecoder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1166</th>\n",
       "      <td>https://bio.tools/mreps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1167</th>\n",
       "      <td>https://bio.tools/msa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>https://bio.tools/msaprobs-mpi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1169</th>\n",
       "      <td>https://bio.tools/mscfs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1170</th>\n",
       "      <td>https://bio.tools/msifinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1171</th>\n",
       "      <td>https://bio.tools/msisensor-ct</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1172</th>\n",
       "      <td>https://bio.tools/msrcall</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1173</th>\n",
       "      <td>https://bio.tools/mssr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1174</th>\n",
       "      <td>https://bio.tools/mtaxi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1175</th>\n",
       "      <td>https://bio.tools/mtb-report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1176</th>\n",
       "      <td>https://bio.tools/multi-car</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1177</th>\n",
       "      <td>https://bio.tools/multi-csar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1178</th>\n",
       "      <td>https://bio.tools/multi-harmony</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1179</th>\n",
       "      <td>https://bio.tools/multibreak-sv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1180</th>\n",
       "      <td>https://bio.tools/multicrispr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1181</th>\n",
       "      <td>https://bio.tools/multigems</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1182</th>\n",
       "      <td>https://bio.tools/multigwas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1183</th>\n",
       "      <td>https://bio.tools/multiqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1184</th>\n",
       "      <td>https://bio.tools/multisnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1185</th>\n",
       "      <td>https://bio.tools/mummer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1186</th>\n",
       "      <td>https://bio.tools/muplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1187</th>\n",
       "      <td>https://bio.tools/mural</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>https://bio.tools/mustguseal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1189</th>\n",
       "      <td>https://bio.tools/mutaid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1190</th>\n",
       "      <td>https://bio.tools/mutationTCN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1191</th>\n",
       "      <td>https://bio.tools/mutation_maker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1192</th>\n",
       "      <td>https://bio.tools/mutationdistiller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1193</th>\n",
       "      <td>https://bio.tools/mutualfold</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1194</th>\n",
       "      <td>https://bio.tools/mvppt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1195</th>\n",
       "      <td>https://bio.tools/myvcf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1196</th>\n",
       "      <td>https://bio.tools/nadfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>https://bio.tools/nano2ngs-muta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>https://bio.tools/nanobase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>https://bio.tools/nanobodies</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>https://bio.tools/nanoclust</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>https://bio.tools/nanocov19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1202</th>\n",
       "      <td>https://bio.tools/nanoforms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1203</th>\n",
       "      <td>https://bio.tools/nanomark</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1204</th>\n",
       "      <td>https://bio.tools/nanomlst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1205</th>\n",
       "      <td>https://bio.tools/nanonet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1206</th>\n",
       "      <td>https://bio.tools/nanopack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1207</th>\n",
       "      <td>https://bio.tools/nanopanel2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1208</th>\n",
       "      <td>https://bio.tools/nanopolish</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1209</th>\n",
       "      <td>https://bio.tools/nanopore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1210</th>\n",
       "      <td>https://bio.tools/nanosim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1211</th>\n",
       "      <td>https://bio.tools/nanosnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1212</th>\n",
       "      <td>https://bio.tools/nanostr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1213</th>\n",
       "      <td>https://bio.tools/nanotube</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1214</th>\n",
       "      <td>https://bio.tools/nast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1215</th>\n",
       "      <td>https://bio.tools/nast-ier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1216</th>\n",
       "      <td>https://bio.tools/navise</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1217</th>\n",
       "      <td>https://bio.tools/ncbi_assembly_archive</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1218</th>\n",
       "      <td>https://bio.tools/ncbi_genome_workbench</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1219</th>\n",
       "      <td>https://bio.tools/ncmhap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1220</th>\n",
       "      <td>https://bio.tools/ncov</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1221</th>\n",
       "      <td>https://bio.tools/neat-enrich</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1222</th>\n",
       "      <td>https://bio.tools/neat-ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1223</th>\n",
       "      <td>https://bio.tools/needlestack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1224</th>\n",
       "      <td>https://bio.tools/neptune</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1225</th>\n",
       "      <td>https://bio.tools/neptune_genome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1226</th>\n",
       "      <td>https://bio.tools/nessm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1227</th>\n",
       "      <td>https://bio.tools/netgor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1228</th>\n",
       "      <td>https://bio.tools/netsmooth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1229</th>\n",
       "      <td>https://bio.tools/nettime</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1230</th>\n",
       "      <td>https://bio.tools/neupred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1231</th>\n",
       "      <td>https://bio.tools/new_targets</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1232</th>\n",
       "      <td>https://bio.tools/nextgene</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1233</th>\n",
       "      <td>https://bio.tools/nextsv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1234</th>\n",
       "      <td>https://bio.tools/nf-core-cageseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1235</th>\n",
       "      <td>https://bio.tools/nf-core-viralrecon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1236</th>\n",
       "      <td>https://bio.tools/nglyalign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1237</th>\n",
       "      <td>https://bio.tools/ngmlr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1238</th>\n",
       "      <td>https://bio.tools/ngs-integrator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1239</th>\n",
       "      <td>https://bio.tools/ngs-primerplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1240</th>\n",
       "      <td>https://bio.tools/ngs-trex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1241</th>\n",
       "      <td>https://bio.tools/ngsRelateV2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1242</th>\n",
       "      <td>https://bio.tools/ngsReports</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1243</th>\n",
       "      <td>https://bio.tools/ngs_simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1244</th>\n",
       "      <td>https://bio.tools/ngscat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1245</th>\n",
       "      <td>https://bio.tools/ngscheckmate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1246</th>\n",
       "      <td>https://bio.tools/ngscomposer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1247</th>\n",
       "      <td>https://bio.tools/ngscopy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1248</th>\n",
       "      <td>https://bio.tools/ngseasy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1249</th>\n",
       "      <td>https://bio.tools/ngsmethdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1250</th>\n",
       "      <td>https://bio.tools/ngsml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1251</th>\n",
       "      <td>https://bio.tools/ngsoptwin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1252</th>\n",
       "      <td>https://bio.tools/ngsrelate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1253</th>\n",
       "      <td>https://bio.tools/ngsremix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1254</th>\n",
       "      <td>https://bio.tools/ngsrich</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1255</th>\n",
       "      <td>https://bio.tools/ngstools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1256</th>\n",
       "      <td>https://bio.tools/njplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1257</th>\n",
       "      <td>https://bio.tools/nldmseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1258</th>\n",
       "      <td>https://bio.tools/nobadwordscombiner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1259</th>\n",
       "      <td>https://bio.tools/noiseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1260</th>\n",
       "      <td>https://bio.tools/normr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1261</th>\n",
       "      <td>https://bio.tools/novohmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1262</th>\n",
       "      <td>https://bio.tools/novowrap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1263</th>\n",
       "      <td>https://bio.tools/npars</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1264</th>\n",
       "      <td>https://bio.tools/npbss</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1265</th>\n",
       "      <td>https://bio.tools/npscarf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1266</th>\n",
       "      <td>https://bio.tools/npseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1267</th>\n",
       "      <td>https://bio.tools/nrcfv_reader</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1268</th>\n",
       "      <td>https://bio.tools/nrps-pks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1269</th>\n",
       "      <td>https://bio.tools/ntcard</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1270</th>\n",
       "      <td>https://bio.tools/nthash2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1271</th>\n",
       "      <td>https://bio.tools/nthits</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1272</th>\n",
       "      <td>https://bio.tools/ntmg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1273</th>\n",
       "      <td>https://bio.tools/nucler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1274</th>\n",
       "      <td>https://bio.tools/nxtrim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1275</th>\n",
       "      <td>https://bio.tools/obitools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1276</th>\n",
       "      <td>https://bio.tools/occupeak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1277</th>\n",
       "      <td>https://bio.tools/ocean-barcode-atlas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1278</th>\n",
       "      <td>https://bio.tools/octopus-toolkit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1279</th>\n",
       "      <td>https://bio.tools/ohana</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1280</th>\n",
       "      <td>https://bio.tools/oligomap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1281</th>\n",
       "      <td>https://bio.tools/oligoprime</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1282</th>\n",
       "      <td>https://bio.tools/omicplotr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1283</th>\n",
       "      <td>https://bio.tools/omicsview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1284</th>\n",
       "      <td>https://bio.tools/omware</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1285</th>\n",
       "      <td>https://bio.tools/oncocnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1286</th>\n",
       "      <td>https://bio.tools/oncogenomiclandscapes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1287</th>\n",
       "      <td>https://bio.tools/oncomix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1288</th>\n",
       "      <td>https://bio.tools/onetool</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1289</th>\n",
       "      <td>https://bio.tools/ontofast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1290</th>\n",
       "      <td>https://bio.tools/opencontami</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1291</th>\n",
       "      <td>https://bio.tools/opengenomebrowser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1292</th>\n",
       "      <td>https://bio.tools/opt-sgrna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1293</th>\n",
       "      <td>https://bio.tools/optitype</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1294</th>\n",
       "      <td>https://bio.tools/optocoder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1295</th>\n",
       "      <td>https://bio.tools/orcom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1296</th>\n",
       "      <td>https://bio.tools/orfik</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1297</th>\n",
       "      <td>https://bio.tools/orfpredictor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1298</th>\n",
       "      <td>https://bio.tools/orio</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1299</th>\n",
       "      <td>https://bio.tools/orna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1300</th>\n",
       "      <td>https://bio.tools/orso</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1301</th>\n",
       "      <td>https://bio.tools/orthsearchservice</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1302</th>\n",
       "      <td>https://bio.tools/ortsuite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1303</th>\n",
       "      <td>https://bio.tools/outlierd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1304</th>\n",
       "      <td>https://bio.tools/ovnip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1305</th>\n",
       "      <td>https://bio.tools/ovpdt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1306</th>\n",
       "      <td>https://bio.tools/p-smoother</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1307</th>\n",
       "      <td>https://bio.tools/pTrimmer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1308</th>\n",
       "      <td>https://bio.tools/pacas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1309</th>\n",
       "      <td>https://bio.tools/paintshop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1310</th>\n",
       "      <td>https://bio.tools/paivs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1311</th>\n",
       "      <td>https://bio.tools/pan4draft</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1312</th>\n",
       "      <td>https://bio.tools/panaroo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1313</th>\n",
       "      <td>https://bio.tools/panelpro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1314</th>\n",
       "      <td>https://bio.tools/pangea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1315</th>\n",
       "      <td>https://bio.tools/pangia</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1316</th>\n",
       "      <td>https://bio.tools/panisa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1317</th>\n",
       "      <td>https://bio.tools/pannotator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1318</th>\n",
       "      <td>https://bio.tools/pansvr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1319</th>\n",
       "      <td>https://bio.tools/paratome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1320</th>\n",
       "      <td>https://bio.tools/pardre</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1321</th>\n",
       "      <td>https://bio.tools/parsesnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1322</th>\n",
       "      <td>https://bio.tools/pasd-qc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1323</th>\n",
       "      <td>https://bio.tools/pathdeseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1324</th>\n",
       "      <td>https://bio.tools/pathfinder-cancer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1325</th>\n",
       "      <td>https://bio.tools/pathoscope</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1326</th>\n",
       "      <td>https://bio.tools/pathostat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1327</th>\n",
       "      <td>https://bio.tools/pathway_explorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1328</th>\n",
       "      <td>https://bio.tools/pathwaysplice</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1329</th>\n",
       "      <td>https://bio.tools/patmatch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1330</th>\n",
       "      <td>https://bio.tools/patsearch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1331</th>\n",
       "      <td>https://bio.tools/pbgi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1332</th>\n",
       "      <td>https://bio.tools/pbsim2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1333</th>\n",
       "      <td>https://bio.tools/pcaexplorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1334</th>\n",
       "      <td>https://bio.tools/pcip-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1335</th>\n",
       "      <td>https://bio.tools/pcrtiler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1336</th>\n",
       "      <td>https://bio.tools/pda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1337</th>\n",
       "      <td>https://bio.tools/pdl1binder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1338</th>\n",
       "      <td>https://bio.tools/pe-analyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1339</th>\n",
       "      <td>https://bio.tools/pe-designer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1340</th>\n",
       "      <td>https://bio.tools/peace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1341</th>\n",
       "      <td>https://bio.tools/peak2bed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1342</th>\n",
       "      <td>https://bio.tools/pecanpy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1343</th>\n",
       "      <td>https://bio.tools/pecnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1344</th>\n",
       "      <td>https://bio.tools/pegfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1345</th>\n",
       "      <td>https://bio.tools/pep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1346</th>\n",
       "      <td>https://bio.tools/pepper-deepvariant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1347</th>\n",
       "      <td>https://bio.tools/pepsea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1348</th>\n",
       "      <td>https://bio.tools/perga</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1349</th>\n",
       "      <td>https://bio.tools/periodicdna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1350</th>\n",
       "      <td>https://bio.tools/pet-abc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>https://bio.tools/petkit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1352</th>\n",
       "      <td>https://bio.tools/pgcloser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1353</th>\n",
       "      <td>https://bio.tools/pgg_sv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1354</th>\n",
       "      <td>https://bio.tools/pgmicrod</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1355</th>\n",
       "      <td>https://bio.tools/pgrc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1356</th>\n",
       "      <td>https://bio.tools/pgsa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1357</th>\n",
       "      <td>https://bio.tools/pgx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1358</th>\n",
       "      <td>https://bio.tools/phageai</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1359</th>\n",
       "      <td>https://bio.tools/phageterm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1360</th>\n",
       "      <td>https://bio.tools/pharaoh-multi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1361</th>\n",
       "      <td>https://bio.tools/pharmvip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1362</th>\n",
       "      <td>https://bio.tools/pharp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1363</th>\n",
       "      <td>https://bio.tools/phasebook_haplotype</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1364</th>\n",
       "      <td>https://bio.tools/phast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1365</th>\n",
       "      <td>https://bio.tools/phaster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1366</th>\n",
       "      <td>https://bio.tools/pheniqs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1367</th>\n",
       "      <td>https://bio.tools/phenoexam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1368</th>\n",
       "      <td>https://bio.tools/phenofam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1369</th>\n",
       "      <td>https://bio.tools/phigaro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1370</th>\n",
       "      <td>https://bio.tools/phirbo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1371</th>\n",
       "      <td>https://bio.tools/phire</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1372</th>\n",
       "      <td>https://bio.tools/phred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1373</th>\n",
       "      <td>https://bio.tools/phredphrapconsed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1374</th>\n",
       "      <td>https://bio.tools/phuser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1375</th>\n",
       "      <td>https://bio.tools/phydose</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1376</th>\n",
       "      <td>https://bio.tools/phylics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1377</th>\n",
       "      <td>https://bio.tools/phylogibbs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1378</th>\n",
       "      <td>https://bio.tools/phyloligo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1379</th>\n",
       "      <td>https://bio.tools/phylomda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1380</th>\n",
       "      <td>https://bio.tools/phylonium</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1381</th>\n",
       "      <td>https://bio.tools/phyloprimer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1382</th>\n",
       "      <td>https://bio.tools/picall</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1383</th>\n",
       "      <td>https://bio.tools/picard_tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1384</th>\n",
       "      <td>https://bio.tools/picky</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1385</th>\n",
       "      <td>https://bio.tools/picopore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1386</th>\n",
       "      <td>https://bio.tools/pics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1387</th>\n",
       "      <td>https://bio.tools/pifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1388</th>\n",
       "      <td>https://bio.tools/pileline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1389</th>\n",
       "      <td>https://bio.tools/pileup2cov</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1390</th>\n",
       "      <td>https://bio.tools/pimba</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1391</th>\n",
       "      <td>https://bio.tools/pin_hic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1392</th>\n",
       "      <td>https://bio.tools/pincage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1393</th>\n",
       "      <td>https://bio.tools/pine-cone</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1394</th>\n",
       "      <td>https://bio.tools/ping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1395</th>\n",
       "      <td>https://bio.tools/pio</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1396</th>\n",
       "      <td>https://bio.tools/pipealign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1397</th>\n",
       "      <td>https://bio.tools/pipeline_pilot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1398</th>\n",
       "      <td>https://bio.tools/pipeliner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1399</th>\n",
       "      <td>https://bio.tools/pipipes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1400</th>\n",
       "      <td>https://bio.tools/pirbase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1401</th>\n",
       "      <td>https://bio.tools/pirouette</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1402</th>\n",
       "      <td>https://bio.tools/pisces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1403</th>\n",
       "      <td>https://bio.tools/pisces-mrna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1404</th>\n",
       "      <td>https://bio.tools/pixy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1405</th>\n",
       "      <td>https://bio.tools/plantdeepsea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1406</th>\n",
       "      <td>https://bio.tools/plantrep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1407</th>\n",
       "      <td>https://bio.tools/plast-ncrna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1408</th>\n",
       "      <td>https://bio.tools/plastid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1409</th>\n",
       "      <td>https://bio.tools/plek</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1410</th>\n",
       "      <td>https://bio.tools/plnseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1411</th>\n",
       "      <td>https://bio.tools/ploidyngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1412</th>\n",
       "      <td>https://bio.tools/plotcoverage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1413</th>\n",
       "      <td>https://bio.tools/plotgardener</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1414</th>\n",
       "      <td>https://bio.tools/plotgenomecoverage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1415</th>\n",
       "      <td>https://bio.tools/plotmi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1416</th>\n",
       "      <td>https://bio.tools/plpe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1417</th>\n",
       "      <td>https://bio.tools/pm4ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1418</th>\n",
       "      <td>https://bio.tools/pmdscam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1419</th>\n",
       "      <td>https://bio.tools/pngseqr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1420</th>\n",
       "      <td>https://bio.tools/polyatailor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1421</th>\n",
       "      <td>https://bio.tools/polyester</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1422</th>\n",
       "      <td>https://bio.tools/polyphred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1423</th>\n",
       "      <td>https://bio.tools/polyploid-genotyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1424</th>\n",
       "      <td>https://bio.tools/pombase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1425</th>\n",
       "      <td>https://bio.tools/pool-hmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1426</th>\n",
       "      <td>https://bio.tools/poolhap2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1427</th>\n",
       "      <td>https://bio.tools/pooling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1428</th>\n",
       "      <td>https://bio.tools/pore</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1429</th>\n",
       "      <td>https://bio.tools/poretools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1430</th>\n",
       "      <td>https://bio.tools/porthomcl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1431</th>\n",
       "      <td>https://bio.tools/powerexplorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1432</th>\n",
       "      <td>https://bio.tools/powertools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1433</th>\n",
       "      <td>https://bio.tools/ppai</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1434</th>\n",
       "      <td>https://bio.tools/ppmdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1435</th>\n",
       "      <td>https://bio.tools/pr2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1436</th>\n",
       "      <td>https://bio.tools/pr2-primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1437</th>\n",
       "      <td>https://bio.tools/prada-rnaseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1438</th>\n",
       "      <td>https://bio.tools/precogx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1439</th>\n",
       "      <td>https://bio.tools/pred-gpcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>https://bio.tools/predhpi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>https://bio.tools/prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>https://bio.tools/preseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>https://bio.tools/presto</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>https://bio.tools/price</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1445</th>\n",
       "      <td>https://bio.tools/price-orf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1446</th>\n",
       "      <td>https://bio.tools/prider</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1447</th>\n",
       "      <td>https://bio.tools/prifi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1448</th>\n",
       "      <td>https://bio.tools/primaclade</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1449</th>\n",
       "      <td>https://bio.tools/primer3plus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1450</th>\n",
       "      <td>https://bio.tools/primerbank</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1451</th>\n",
       "      <td>https://bio.tools/primerclip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1452</th>\n",
       "      <td>https://bio.tools/primerdesign-m</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1453</th>\n",
       "      <td>https://bio.tools/primerstation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1454</th>\n",
       "      <td>https://bio.tools/primerxl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1455</th>\n",
       "      <td>https://bio.tools/priset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1456</th>\n",
       "      <td>https://bio.tools/prism-primer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1457</th>\n",
       "      <td>https://bio.tools/probamr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1458</th>\n",
       "      <td>https://bio.tools/probealign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1459</th>\n",
       "      <td>https://bio.tools/probelynx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1460</th>\n",
       "      <td>https://bio.tools/probematch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1461</th>\n",
       "      <td>https://bio.tools/progmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1462</th>\n",
       "      <td>https://bio.tools/projector_2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1463</th>\n",
       "      <td>https://bio.tools/promotech</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1464</th>\n",
       "      <td>https://bio.tools/proovread</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1465</th>\n",
       "      <td>https://bio.tools/propgap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1466</th>\n",
       "      <td>https://bio.tools/proq3d</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1467</th>\n",
       "      <td>https://bio.tools/pross</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1468</th>\n",
       "      <td>https://bio.tools/psafe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1469</th>\n",
       "      <td>https://bio.tools/psea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1470</th>\n",
       "      <td>https://bio.tools/psea_ensemble</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1471</th>\n",
       "      <td>https://bio.tools/psi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1472</th>\n",
       "      <td>https://bio.tools/pts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1473</th>\n",
       "      <td>https://bio.tools/ptv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1474</th>\n",
       "      <td>https://bio.tools/pumas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1475</th>\n",
       "      <td>https://bio.tools/purbayes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1476</th>\n",
       "      <td>https://bio.tools/pyampli</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1477</th>\n",
       "      <td>https://bio.tools/pycoqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1478</th>\n",
       "      <td>https://bio.tools/pyfastx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1479</th>\n",
       "      <td>https://bio.tools/pyir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1480</th>\n",
       "      <td>https://bio.tools/pysam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1481</th>\n",
       "      <td>https://bio.tools/pysamstats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1482</th>\n",
       "      <td>https://bio.tools/pyseq2500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1483</th>\n",
       "      <td>https://bio.tools/pysradb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1484</th>\n",
       "      <td>https://bio.tools/pysster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1485</th>\n",
       "      <td>https://bio.tools/q2-feature-classifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1486</th>\n",
       "      <td>https://bio.tools/q2-fondue</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1487</th>\n",
       "      <td>https://bio.tools/qap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1488</th>\n",
       "      <td>https://bio.tools/qc-chain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1489</th>\n",
       "      <td>https://bio.tools/qc3c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1490</th>\n",
       "      <td>https://bio.tools/qcgwas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1491</th>\n",
       "      <td>https://bio.tools/qgrs_mapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1492</th>\n",
       "      <td>https://bio.tools/qrqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1493</th>\n",
       "      <td>https://bio.tools/qsea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1494</th>\n",
       "      <td>https://bio.tools/qseq2fastq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1495</th>\n",
       "      <td>https://bio.tools/quacrs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1496</th>\n",
       "      <td>https://bio.tools/quadfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1497</th>\n",
       "      <td>https://bio.tools/quake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1498</th>\n",
       "      <td>https://bio.tools/quality_assessment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1499</th>\n",
       "      <td>https://bio.tools/qualitysnpng</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1500</th>\n",
       "      <td>https://bio.tools/qualtofastq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1501</th>\n",
       "      <td>https://bio.tools/quasar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1502</th>\n",
       "      <td>https://bio.tools/quasirecomb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1503</th>\n",
       "      <td>https://bio.tools/quasiseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1504</th>\n",
       "      <td>https://bio.tools/quasitools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1505</th>\n",
       "      <td>https://bio.tools/quasr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1506</th>\n",
       "      <td>https://bio.tools/quickfigures</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1507</th>\n",
       "      <td>https://bio.tools/qvz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1508</th>\n",
       "      <td>https://bio.tools/r2r</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1509</th>\n",
       "      <td>https://bio.tools/r453plus1toolbox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1510</th>\n",
       "      <td>https://bio.tools/rabbitfx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1511</th>\n",
       "      <td>https://bio.tools/rabbitv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1512</th>\n",
       "      <td>https://bio.tools/radis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1513</th>\n",
       "      <td>https://bio.tools/radtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1514</th>\n",
       "      <td>https://bio.tools/radtyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1515</th>\n",
       "      <td>https://bio.tools/rafah</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1516</th>\n",
       "      <td>https://bio.tools/rambo-k</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1517</th>\n",
       "      <td>https://bio.tools/ramr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1518</th>\n",
       "      <td>https://bio.tools/ranbow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1519</th>\n",
       "      <td>https://bio.tools/rap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1520</th>\n",
       "      <td>https://bio.tools/rapidr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1521</th>\n",
       "      <td>https://bio.tools/raptgen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1522</th>\n",
       "      <td>https://bio.tools/raptor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1523</th>\n",
       "      <td>https://bio.tools/raptranker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1524</th>\n",
       "      <td>https://bio.tools/raremetal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1525</th>\n",
       "      <td>https://bio.tools/rarevariantvis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1526</th>\n",
       "      <td>https://bio.tools/rattle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1527</th>\n",
       "      <td>https://bio.tools/ravaq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1528</th>\n",
       "      <td>https://bio.tools/razers3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1529</th>\n",
       "      <td>https://bio.tools/rbceq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1530</th>\n",
       "      <td>https://bio.tools/rbowtie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1531</th>\n",
       "      <td>https://bio.tools/rcandy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1532</th>\n",
       "      <td>https://bio.tools/rck</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1533</th>\n",
       "      <td>https://bio.tools/rcorrector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1534</th>\n",
       "      <td>https://bio.tools/re-searcher</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1535</th>\n",
       "      <td>https://bio.tools/readaligner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1536</th>\n",
       "      <td>https://bio.tools/readbouncer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1537</th>\n",
       "      <td>https://bio.tools/readdepth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1538</th>\n",
       "      <td>https://bio.tools/readfish</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1539</th>\n",
       "      <td>https://bio.tools/readscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1540</th>\n",
       "      <td>https://bio.tools/real</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1541</th>\n",
       "      <td>https://bio.tools/reconcnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1542</th>\n",
       "      <td>https://bio.tools/recount_bioconductor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1543</th>\n",
       "      <td>https://bio.tools/recovery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1544</th>\n",
       "      <td>https://bio.tools/recruitploteasy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1545</th>\n",
       "      <td>https://bio.tools/red</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1546</th>\n",
       "      <td>https://bio.tools/redhorse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1547</th>\n",
       "      <td>https://bio.tools/rediportal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1548</th>\n",
       "      <td>https://bio.tools/redoak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1549</th>\n",
       "      <td>https://bio.tools/refhic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1550</th>\n",
       "      <td>https://bio.tools/refinehmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1551</th>\n",
       "      <td>https://bio.tools/reformat_fastq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1552</th>\n",
       "      <td>https://bio.tools/refrgim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1553</th>\n",
       "      <td>https://bio.tools/regresshaplo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1554</th>\n",
       "      <td>https://bio.tools/regsp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1555</th>\n",
       "      <td>https://bio.tools/relics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1556</th>\n",
       "      <td>https://bio.tools/remodnav</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1557</th>\n",
       "      <td>https://bio.tools/remp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1558</th>\n",
       "      <td>https://bio.tools/repahr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1559</th>\n",
       "      <td>https://bio.tools/repdenovo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1560</th>\n",
       "      <td>https://bio.tools/reps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1561</th>\n",
       "      <td>https://bio.tools/rescript</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1562</th>\n",
       "      <td>https://bio.tools/reseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1563</th>\n",
       "      <td>https://bio.tools/resistoxplorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1564</th>\n",
       "      <td>https://bio.tools/restfulse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1565</th>\n",
       "      <td>https://bio.tools/reva</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1566</th>\n",
       "      <td>https://bio.tools/reveel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1567</th>\n",
       "      <td>https://bio.tools/rgen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1568</th>\n",
       "      <td>https://bio.tools/rhat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1569</th>\n",
       "      <td>https://bio.tools/rhizobindingsites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1570</th>\n",
       "      <td>https://bio.tools/ribbon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1571</th>\n",
       "      <td>https://bio.tools/riboa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1572</th>\n",
       "      <td>https://bio.tools/ribose-map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1573</th>\n",
       "      <td>https://bio.tools/riboswitch_scanner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1574</th>\n",
       "      <td>https://bio.tools/rimeta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1575</th>\n",
       "      <td>https://bio.tools/rims-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1576</th>\n",
       "      <td>https://bio.tools/rjmcmcnucleosomes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1577</th>\n",
       "      <td>https://bio.tools/rna-express</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1578</th>\n",
       "      <td>https://bio.tools/rnabrowse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1579</th>\n",
       "      <td>https://bio.tools/rnaprobr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1580</th>\n",
       "      <td>https://bio.tools/rnaseqpowercalculator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1581</th>\n",
       "      <td>https://bio.tools/rnaspades</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1582</th>\n",
       "      <td>https://bio.tools/rnf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1583</th>\n",
       "      <td>https://bio.tools/rnnotator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1584</th>\n",
       "      <td>https://bio.tools/robocop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1585</th>\n",
       "      <td>https://bio.tools/rosettaantibody</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1586</th>\n",
       "      <td>https://bio.tools/rpmcmc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1587</th>\n",
       "      <td>https://bio.tools/rprimer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1588</th>\n",
       "      <td>https://bio.tools/rqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1589</th>\n",
       "      <td>https://bio.tools/rresolver</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1590</th>\n",
       "      <td>https://bio.tools/rseqflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1591</th>\n",
       "      <td>https://bio.tools/rsssite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1592</th>\n",
       "      <td>https://bio.tools/rsubread</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1593</th>\n",
       "      <td>https://bio.tools/rtanalyzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1594</th>\n",
       "      <td>https://bio.tools/rubioseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1595</th>\n",
       "      <td>https://bio.tools/runcer-necromancer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1596</th>\n",
       "      <td>https://bio.tools/runibic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1597</th>\n",
       "      <td>https://bio.tools/rust-bio</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1598</th>\n",
       "      <td>https://bio.tools/rvboost</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1599</th>\n",
       "      <td>https://bio.tools/rvd-variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1600</th>\n",
       "      <td>https://bio.tools/rvfam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1601</th>\n",
       "      <td>https://bio.tools/rvista</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1602</th>\n",
       "      <td>https://bio.tools/rvs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1603</th>\n",
       "      <td>https://bio.tools/rvs_software</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1604</th>\n",
       "      <td>https://bio.tools/rvsel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1605</th>\n",
       "      <td>https://bio.tools/rvtests</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1606</th>\n",
       "      <td>https://bio.tools/sRNAanno</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1607</th>\n",
       "      <td>https://bio.tools/sa-ssr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1608</th>\n",
       "      <td>https://bio.tools/saap-bs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1609</th>\n",
       "      <td>https://bio.tools/sacsann</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1610</th>\n",
       "      <td>https://bio.tools/sailfish</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1611</th>\n",
       "      <td>https://bio.tools/salai-net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1612</th>\n",
       "      <td>https://bio.tools/salt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1613</th>\n",
       "      <td>https://bio.tools/salt_snp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1614</th>\n",
       "      <td>https://bio.tools/samba</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1615</th>\n",
       "      <td>https://bio.tools/samblaster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1616</th>\n",
       "      <td>https://bio.tools/samcluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1617</th>\n",
       "      <td>https://bio.tools/samexplorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1618</th>\n",
       "      <td>https://bio.tools/samnetweb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1619</th>\n",
       "      <td>https://bio.tools/sampleseq2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1620</th>\n",
       "      <td>https://bio.tools/samql</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1621</th>\n",
       "      <td>https://bio.tools/samsvm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1622</th>\n",
       "      <td>https://bio.tools/samtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1623</th>\n",
       "      <td>https://bio.tools/samtools_filter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1624</th>\n",
       "      <td>https://bio.tools/samtools_flagstat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1625</th>\n",
       "      <td>https://bio.tools/samtools_idxstats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1626</th>\n",
       "      <td>https://bio.tools/samtools_sort</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1627</th>\n",
       "      <td>https://bio.tools/sangeranalyser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1628</th>\n",
       "      <td>https://bio.tools/sara-cnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1629</th>\n",
       "      <td>https://bio.tools/sartools_edger</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1630</th>\n",
       "      <td>https://bio.tools/sativa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1631</th>\n",
       "      <td>https://bio.tools/sav-pred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1632</th>\n",
       "      <td>https://bio.tools/savant_genome_browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1633</th>\n",
       "      <td>https://bio.tools/sbars</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1634</th>\n",
       "      <td>https://bio.tools/sbwt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1635</th>\n",
       "      <td>https://bio.tools/scATAC-pro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1636</th>\n",
       "      <td>https://bio.tools/scMNase-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1637</th>\n",
       "      <td>https://bio.tools/scaffmatch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1638</th>\n",
       "      <td>https://bio.tools/scaleqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1639</th>\n",
       "      <td>https://bio.tools/scalpel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1640</th>\n",
       "      <td>https://bio.tools/scampi2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1641</th>\n",
       "      <td>https://bio.tools/scarpa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1642</th>\n",
       "      <td>https://bio.tools/sccoda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1643</th>\n",
       "      <td>https://bio.tools/scct</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1644</th>\n",
       "      <td>https://bio.tools/scec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1645</th>\n",
       "      <td>https://bio.tools/scfind</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1646</th>\n",
       "      <td>https://bio.tools/schaplotyper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1647</th>\n",
       "      <td>https://bio.tools/schicnorm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1648</th>\n",
       "      <td>https://bio.tools/schicsrs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1649</th>\n",
       "      <td>https://bio.tools/schictools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1650</th>\n",
       "      <td>https://bio.tools/sci-phy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1651</th>\n",
       "      <td>https://bio.tools/scim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1652</th>\n",
       "      <td>https://bio.tools/scissor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1653</th>\n",
       "      <td>https://bio.tools/sclvm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1654</th>\n",
       "      <td>https://bio.tools/scnym</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1655</th>\n",
       "      <td>https://bio.tools/scone</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1656</th>\n",
       "      <td>https://bio.tools/scone_bioconductor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1657</th>\n",
       "      <td>https://bio.tools/score-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1658</th>\n",
       "      <td>https://bio.tools/scpipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1659</th>\n",
       "      <td>https://bio.tools/screadcounts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1660</th>\n",
       "      <td>https://bio.tools/screengarden</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1661</th>\n",
       "      <td>https://bio.tools/scsilicon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1662</th>\n",
       "      <td>https://bio.tools/scsim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1663</th>\n",
       "      <td>https://bio.tools/sctam-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1664</th>\n",
       "      <td>https://bio.tools/seal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1665</th>\n",
       "      <td>https://bio.tools/seal-suite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1666</th>\n",
       "      <td>https://bio.tools/seance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1667</th>\n",
       "      <td>https://bio.tools/sear</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1668</th>\n",
       "      <td>https://bio.tools/searchgtr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1669</th>\n",
       "      <td>https://bio.tools/secedo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1670</th>\n",
       "      <td>https://bio.tools/secnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1671</th>\n",
       "      <td>https://bio.tools/secureseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1672</th>\n",
       "      <td>https://bio.tools/seeker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1673</th>\n",
       "      <td>https://bio.tools/segemehl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1674</th>\n",
       "      <td>https://bio.tools/segmer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1675</th>\n",
       "      <td>https://bio.tools/segseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1676</th>\n",
       "      <td>https://bio.tools/selox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1677</th>\n",
       "      <td>https://bio.tools/semisup</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1678</th>\n",
       "      <td>https://bio.tools/sepia</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1679</th>\n",
       "      <td>https://bio.tools/sepia-phase</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1680</th>\n",
       "      <td>https://bio.tools/sepia-sim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1681</th>\n",
       "      <td>https://bio.tools/seqQscorer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1682</th>\n",
       "      <td>https://bio.tools/seqacademy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1683</th>\n",
       "      <td>https://bio.tools/seqatoms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1684</th>\n",
       "      <td>https://bio.tools/seqcat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1685</th>\n",
       "      <td>https://bio.tools/seqcna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1686</th>\n",
       "      <td>https://bio.tools/seqcons</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1687</th>\n",
       "      <td>https://bio.tools/seqcontrol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1688</th>\n",
       "      <td>https://bio.tools/seqeditor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1689</th>\n",
       "      <td>https://bio.tools/seqenhdl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1690</th>\n",
       "      <td>https://bio.tools/seqentropy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1691</th>\n",
       "      <td>https://bio.tools/seqfire</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1692</th>\n",
       "      <td>https://bio.tools/seqhound</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1693</th>\n",
       "      <td>https://bio.tools/seqpower</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1694</th>\n",
       "      <td>https://bio.tools/seqprep</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1695</th>\n",
       "      <td>https://bio.tools/seqr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1696</th>\n",
       "      <td>https://bio.tools/seqrepo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1697</th>\n",
       "      <td>https://bio.tools/seqseg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1698</th>\n",
       "      <td>https://bio.tools/seqsetvis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1699</th>\n",
       "      <td>https://bio.tools/seqsqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1700</th>\n",
       "      <td>https://bio.tools/seqtrace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1701</th>\n",
       "      <td>https://bio.tools/seqtrim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1702</th>\n",
       "      <td>https://bio.tools/seqtrimmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1703</th>\n",
       "      <td>https://bio.tools/sequeltools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1704</th>\n",
       "      <td>https://bio.tools/sequence-assembly-app</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1705</th>\n",
       "      <td>https://bio.tools/sequence_harmony</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1706</th>\n",
       "      <td>https://bio.tools/sequence_maneuverer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1707</th>\n",
       "      <td>https://bio.tools/sequencerr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1708</th>\n",
       "      <td>https://bio.tools/seqware</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1709</th>\n",
       "      <td>https://bio.tools/seqwho</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1710</th>\n",
       "      <td>https://bio.tools/serpentine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1711</th>\n",
       "      <td>https://bio.tools/sesam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1712</th>\n",
       "      <td>https://bio.tools/setuppromoter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1713</th>\n",
       "      <td>https://bio.tools/sff2fastq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1714</th>\n",
       "      <td>https://bio.tools/sgcocaller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1715</th>\n",
       "      <td>https://bio.tools/sgrp_blast_server</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1716</th>\n",
       "      <td>https://bio.tools/sgtarget</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1717</th>\n",
       "      <td>https://bio.tools/sgx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1718</th>\n",
       "      <td>https://bio.tools/shadowcaster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1719</th>\n",
       "      <td>https://bio.tools/shear</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1720</th>\n",
       "      <td>https://bio.tools/shepherd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1721</th>\n",
       "      <td>https://bio.tools/sherman</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1722</th>\n",
       "      <td>https://bio.tools/shifty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1723</th>\n",
       "      <td>https://bio.tools/shorah</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1724</th>\n",
       "      <td>https://bio.tools/shortfuse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1725</th>\n",
       "      <td>https://bio.tools/shotgun</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1726</th>\n",
       "      <td>https://bio.tools/shrec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1727</th>\n",
       "      <td>https://bio.tools/shrimp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1728</th>\n",
       "      <td>https://bio.tools/sib-blast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1729</th>\n",
       "      <td>https://bio.tools/sibeliaz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1730</th>\n",
       "      <td>https://bio.tools/sibis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1731</th>\n",
       "      <td>https://bio.tools/sictools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1732</th>\n",
       "      <td>https://bio.tools/sidr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1733</th>\n",
       "      <td>https://bio.tools/sigma</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1734</th>\n",
       "      <td>https://bio.tools/signalalign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1735</th>\n",
       "      <td>https://bio.tools/sigtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1736</th>\n",
       "      <td>https://bio.tools/sim-it</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1737</th>\n",
       "      <td>https://bio.tools/sim1000G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1738</th>\n",
       "      <td>https://bio.tools/sim3c</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1739</th>\n",
       "      <td>https://bio.tools/sim4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1740</th>\n",
       "      <td>https://bio.tools/simffpe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1741</th>\n",
       "      <td>https://bio.tools/simngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1742</th>\n",
       "      <td>https://bio.tools/simplifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1743</th>\n",
       "      <td>https://bio.tools/simsearch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1744</th>\n",
       "      <td>https://bio.tools/simuscop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1745</th>\n",
       "      <td>https://bio.tools/sina</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1746</th>\n",
       "      <td>https://bio.tools/sinoduplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1747</th>\n",
       "      <td>https://bio.tools/siq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1748</th>\n",
       "      <td>https://bio.tools/sisrs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1749</th>\n",
       "      <td>https://bio.tools/sissrs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1750</th>\n",
       "      <td>https://bio.tools/sitk-ibex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1751</th>\n",
       "      <td>https://bio.tools/skewer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1752</th>\n",
       "      <td>https://bio.tools/slag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1753</th>\n",
       "      <td>https://bio.tools/slalom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1754</th>\n",
       "      <td>https://bio.tools/slapnap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1755</th>\n",
       "      <td>https://bio.tools/slicembler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1756</th>\n",
       "      <td>https://bio.tools/slidesort-bpr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1757</th>\n",
       "      <td>https://bio.tools/slimfinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1758</th>\n",
       "      <td>https://bio.tools/smaca</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1759</th>\n",
       "      <td>https://bio.tools/smap_design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1760</th>\n",
       "      <td>https://bio.tools/smmip-tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1761</th>\n",
       "      <td>https://bio.tools/smrt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1762</th>\n",
       "      <td>https://bio.tools/smrt_R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1763</th>\n",
       "      <td>https://bio.tools/smrt_analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1764</th>\n",
       "      <td>https://bio.tools/snackntm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1765</th>\n",
       "      <td>https://bio.tools/snackvar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1766</th>\n",
       "      <td>https://bio.tools/snapatac</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1767</th>\n",
       "      <td>https://bio.tools/sniffles</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1768</th>\n",
       "      <td>https://bio.tools/snip-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1769</th>\n",
       "      <td>https://bio.tools/snooper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1770</th>\n",
       "      <td>https://bio.tools/snp-crispr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1771</th>\n",
       "      <td>https://bio.tools/snpbox</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1772</th>\n",
       "      <td>https://bio.tools/snpgenie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1773</th>\n",
       "      <td>https://bio.tools/snpnexus-covid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1774</th>\n",
       "      <td>https://bio.tools/snptools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1775</th>\n",
       "      <td>https://bio.tools/sns-align</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1776</th>\n",
       "      <td>https://bio.tools/snvmix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1777</th>\n",
       "      <td>https://bio.tools/soapdenovo2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1778</th>\n",
       "      <td>https://bio.tools/soapmetas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1779</th>\n",
       "      <td>https://bio.tools/soapnuke</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1780</th>\n",
       "      <td>https://bio.tools/soaptyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1781</th>\n",
       "      <td>https://bio.tools/soba</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1782</th>\n",
       "      <td>https://bio.tools/solexatools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1783</th>\n",
       "      <td>https://bio.tools/solodel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1784</th>\n",
       "      <td>https://bio.tools/soma</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1785</th>\n",
       "      <td>https://bio.tools/somaticsignatures</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1786</th>\n",
       "      <td>https://bio.tools/somatosim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1787</th>\n",
       "      <td>https://bio.tools/somvarius</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1788</th>\n",
       "      <td>https://bio.tools/sop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1789</th>\n",
       "      <td>https://bio.tools/sopra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1790</th>\n",
       "      <td>https://bio.tools/sorva</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1791</th>\n",
       "      <td>https://bio.tools/spacer2pam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1792</th>\n",
       "      <td>https://bio.tools/spads</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1793</th>\n",
       "      <td>https://bio.tools/spark-ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1794</th>\n",
       "      <td>https://bio.tools/sparkec</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1795</th>\n",
       "      <td>https://bio.tools/spatemhtp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1796</th>\n",
       "      <td>https://bio.tools/spdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1797</th>\n",
       "      <td>https://bio.tools/spde</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1798</th>\n",
       "      <td>https://bio.tools/speciesprimer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1799</th>\n",
       "      <td>https://bio.tools/spectracular</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1800</th>\n",
       "      <td>https://bio.tools/spiky</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1801</th>\n",
       "      <td>https://bio.tools/spindel_workbench</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1802</th>\n",
       "      <td>https://bio.tools/spiral_genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1803</th>\n",
       "      <td>https://bio.tools/splace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1804</th>\n",
       "      <td>https://bio.tools/splatter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1805</th>\n",
       "      <td>https://bio.tools/splicing_express</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1806</th>\n",
       "      <td>https://bio.tools/splicingcompass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1807</th>\n",
       "      <td>https://bio.tools/splitmem</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>https://bio.tools/spotyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>https://bio.tools/spring</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>https://bio.tools/sprint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>https://bio.tools/sprint-rna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>https://bio.tools/sprite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1813</th>\n",
       "      <td>https://bio.tools/sprite-pipeline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1814</th>\n",
       "      <td>https://bio.tools/sprites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1815</th>\n",
       "      <td>https://bio.tools/spumoni</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1816</th>\n",
       "      <td>https://bio.tools/sra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1817</th>\n",
       "      <td>https://bio.tools/sra-disk-diffusion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1818</th>\n",
       "      <td>https://bio.tools/sra-tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1819</th>\n",
       "      <td>https://bio.tools/sramp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1820</th>\n",
       "      <td>https://bio.tools/srfim</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1821</th>\n",
       "      <td>https://bio.tools/srinversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1822</th>\n",
       "      <td>https://bio.tools/srnaloop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1823</th>\n",
       "      <td>https://bio.tools/srprism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1824</th>\n",
       "      <td>https://bio.tools/srta-012</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1825</th>\n",
       "      <td>https://bio.tools/ssake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1826</th>\n",
       "      <td>https://bio.tools/ssam-lite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1827</th>\n",
       "      <td>https://bio.tools/sscpred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1828</th>\n",
       "      <td>https://bio.tools/ssmirna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1829</th>\n",
       "      <td>https://bio.tools/ssrenricher</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1830</th>\n",
       "      <td>https://bio.tools/ssrmmd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1831</th>\n",
       "      <td>https://bio.tools/staccato</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1832</th>\n",
       "      <td>https://bio.tools/staden_io_lib</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1833</th>\n",
       "      <td>https://bio.tools/stamp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1834</th>\n",
       "      <td>https://bio.tools/stamp-3d</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1835</th>\n",
       "      <td>https://bio.tools/stamp-metagenomic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1836</th>\n",
       "      <td>https://bio.tools/stampr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1837</th>\n",
       "      <td>https://bio.tools/stampy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1838</th>\n",
       "      <td>https://bio.tools/stampy_indexer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1839</th>\n",
       "      <td>https://bio.tools/standardized_velvet_assembly_report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1840</th>\n",
       "      <td>https://bio.tools/star</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1841</th>\n",
       "      <td>https://bio.tools/star-genome-browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1842</th>\n",
       "      <td>https://bio.tools/star-retinex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1843</th>\n",
       "      <td>https://bio.tools/starbeast3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1844</th>\n",
       "      <td>https://bio.tools/stargazer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1845</th>\n",
       "      <td>https://bio.tools/startlink</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1846</th>\n",
       "      <td>https://bio.tools/steak</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1847</th>\n",
       "      <td>https://bio.tools/stereogene</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1848</th>\n",
       "      <td>https://bio.tools/stitchr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1849</th>\n",
       "      <td>https://bio.tools/storm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1850</th>\n",
       "      <td>https://bio.tools/storm-riboregulator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1851</th>\n",
       "      <td>https://bio.tools/strainline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1852</th>\n",
       "      <td>https://bio.tools/straitrazor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1853</th>\n",
       "      <td>https://bio.tools/strand_ngs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1854</th>\n",
       "      <td>https://bio.tools/stride-assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1855</th>\n",
       "      <td>https://bio.tools/stringgraph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1856</th>\n",
       "      <td>https://bio.tools/stringmlst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1857</th>\n",
       "      <td>https://bio.tools/strong</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1858</th>\n",
       "      <td>https://bio.tools/strspy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1859</th>\n",
       "      <td>https://bio.tools/structurelab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1860</th>\n",
       "      <td>https://bio.tools/structurly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1861</th>\n",
       "      <td>https://bio.tools/subhmm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1862</th>\n",
       "      <td>https://bio.tools/subjunc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1863</th>\n",
       "      <td>https://bio.tools/subread</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1864</th>\n",
       "      <td>https://bio.tools/suffixerator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1865</th>\n",
       "      <td>https://bio.tools/suitemsa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1866</th>\n",
       "      <td>https://bio.tools/suldex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1867</th>\n",
       "      <td>https://bio.tools/summarizedexperiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1868</th>\n",
       "      <td>https://bio.tools/superbaits</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1869</th>\n",
       "      <td>https://bio.tools/supernova</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1870</th>\n",
       "      <td>https://bio.tools/survmicro</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1871</th>\n",
       "      <td>https://bio.tools/sv-bay</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1872</th>\n",
       "      <td>https://bio.tools/svat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1873</th>\n",
       "      <td>https://bio.tools/svim-asm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1874</th>\n",
       "      <td>https://bio.tools/svlr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1875</th>\n",
       "      <td>https://bio.tools/svreg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1876</th>\n",
       "      <td>https://bio.tools/svviz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1877</th>\n",
       "      <td>https://bio.tools/swap-assembler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1878</th>\n",
       "      <td>https://bio.tools/swcuda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1879</th>\n",
       "      <td>https://bio.tools/swdmr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880</th>\n",
       "      <td>https://bio.tools/sweblast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1881</th>\n",
       "      <td>https://bio.tools/swift-illumina</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1882</th>\n",
       "      <td>https://bio.tools/switchdna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1883</th>\n",
       "      <td>https://bio.tools/sxoligosearch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1884</th>\n",
       "      <td>https://bio.tools/syapse</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1885</th>\n",
       "      <td>https://bio.tools/syllable-pbwt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1886</th>\n",
       "      <td>https://bio.tools/symbiont-screener</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1887</th>\n",
       "      <td>https://bio.tools/synteny_portal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1888</th>\n",
       "      <td>https://bio.tools/syntview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1889</th>\n",
       "      <td>https://bio.tools/t-cnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1890</th>\n",
       "      <td>https://bio.tools/taasrad19</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1891</th>\n",
       "      <td>https://bio.tools/tabusearch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1892</th>\n",
       "      <td>https://bio.tools/tag-db</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1893</th>\n",
       "      <td>https://bio.tools/tag-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1894</th>\n",
       "      <td>https://bio.tools/tagdust2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1895</th>\n",
       "      <td>https://bio.tools/taggd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1896</th>\n",
       "      <td>https://bio.tools/tale</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1897</th>\n",
       "      <td>https://bio.tools/targetseqview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1898</th>\n",
       "      <td>https://bio.tools/tarpan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1899</th>\n",
       "      <td>https://bio.tools/tarseqqc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1900</th>\n",
       "      <td>https://bio.tools/tarsvm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1901</th>\n",
       "      <td>https://bio.tools/taxman</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>https://bio.tools/taxmapper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1903</th>\n",
       "      <td>https://bio.tools/taxonomy_summary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1904</th>\n",
       "      <td>https://bio.tools/tbdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1905</th>\n",
       "      <td>https://bio.tools/tbtools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1906</th>\n",
       "      <td>https://bio.tools/tcgabiolinksgui</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1907</th>\n",
       "      <td>https://bio.tools/tcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1908</th>\n",
       "      <td>https://bio.tools/tcs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1909</th>\n",
       "      <td>https://bio.tools/tcs-msa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1910</th>\n",
       "      <td>https://bio.tools/tcseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1911</th>\n",
       "      <td>https://bio.tools/team</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1912</th>\n",
       "      <td>https://bio.tools/tefinder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1913</th>\n",
       "      <td>https://bio.tools/telescoper</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1914</th>\n",
       "      <td>https://bio.tools/temp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1915</th>\n",
       "      <td>https://bio.tools/tex-mst</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1916</th>\n",
       "      <td>https://bio.tools/tgge-star</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1917</th>\n",
       "      <td>https://bio.tools/tgs-tb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>https://bio.tools/tiara</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1919</th>\n",
       "      <td>https://bio.tools/timiner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1920</th>\n",
       "      <td>https://bio.tools/tis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1921</th>\n",
       "      <td>https://bio.tools/tissueenrich</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1922</th>\n",
       "      <td>https://bio.tools/tm-aligner</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1923</th>\n",
       "      <td>https://bio.tools/toggle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1924</th>\n",
       "      <td>https://bio.tools/tools-taggi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1925</th>\n",
       "      <td>https://bio.tools/tophap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1926</th>\n",
       "      <td>https://bio.tools/trace-rrbs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1927</th>\n",
       "      <td>https://bio.tools/tracts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1928</th>\n",
       "      <td>https://bio.tools/transit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1929</th>\n",
       "      <td>https://bio.tools/translatorx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1930</th>\n",
       "      <td>https://bio.tools/transposable_element_finder</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1931</th>\n",
       "      <td>https://bio.tools/transposonultimate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1932</th>\n",
       "      <td>https://bio.tools/trap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1933</th>\n",
       "      <td>https://bio.tools/trap-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1934</th>\n",
       "      <td>https://bio.tools/trawlerweb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1935</th>\n",
       "      <td>https://bio.tools/treedet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1936</th>\n",
       "      <td>https://bio.tools/treq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1937</th>\n",
       "      <td>https://bio.tools/treqcg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1938</th>\n",
       "      <td>https://bio.tools/triage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1939</th>\n",
       "      <td>https://bio.tools/triagetools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1940</th>\n",
       "      <td>https://bio.tools/trimadaptor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1941</th>\n",
       "      <td>https://bio.tools/trimal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1942</th>\n",
       "      <td>https://bio.tools/trimmomatic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1943</th>\n",
       "      <td>https://bio.tools/trimsequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1944</th>\n",
       "      <td>https://bio.tools/triocaller</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1945</th>\n",
       "      <td>https://bio.tools/triodenovo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1946</th>\n",
       "      <td>https://bio.tools/triophaser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1947</th>\n",
       "      <td>https://bio.tools/triovis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1948</th>\n",
       "      <td>https://bio.tools/tronco</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1949</th>\n",
       "      <td>https://bio.tools/trowel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1950</th>\n",
       "      <td>https://bio.tools/trumicount</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1951</th>\n",
       "      <td>https://bio.tools/trumpet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1952</th>\n",
       "      <td>https://bio.tools/ts4msbh</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1953</th>\n",
       "      <td>https://bio.tools/tseta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1954</th>\n",
       "      <td>https://bio.tools/tsrchitect</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1955</th>\n",
       "      <td>https://bio.tools/tssar</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1956</th>\n",
       "      <td>https://bio.tools/tssi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1957</th>\n",
       "      <td>https://bio.tools/turf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1958</th>\n",
       "      <td>https://bio.tools/twinscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1959</th>\n",
       "      <td>https://bio.tools/uap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1960</th>\n",
       "      <td>https://bio.tools/ub-isap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1961</th>\n",
       "      <td>https://bio.tools/ubin</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1962</th>\n",
       "      <td>https://bio.tools/ugdr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1963</th>\n",
       "      <td>https://bio.tools/ultraplex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1964</th>\n",
       "      <td>https://bio.tools/ulysses</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1965</th>\n",
       "      <td>https://bio.tools/umi-gen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1966</th>\n",
       "      <td>https://bio.tools/umi-tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1967</th>\n",
       "      <td>https://bio.tools/umi4cats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1968</th>\n",
       "      <td>https://bio.tools/umic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1969</th>\n",
       "      <td>https://bio.tools/unccombo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1970</th>\n",
       "      <td>https://bio.tools/unfazed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1971</th>\n",
       "      <td>https://bio.tools/unicycler</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1972</th>\n",
       "      <td>https://bio.tools/uniprime2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1973</th>\n",
       "      <td>https://bio.tools/unwords</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1974</th>\n",
       "      <td>https://bio.tools/urmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1975</th>\n",
       "      <td>https://bio.tools/uropa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1976</th>\n",
       "      <td>https://bio.tools/urqt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1977</th>\n",
       "      <td>https://bio.tools/usat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1978</th>\n",
       "      <td>https://bio.tools/usearch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1979</th>\n",
       "      <td>https://bio.tools/v-pipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1980</th>\n",
       "      <td>https://bio.tools/v-xtractor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1981</th>\n",
       "      <td>https://bio.tools/vacsol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1982</th>\n",
       "      <td>https://bio.tools/valencia</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1983</th>\n",
       "      <td>https://bio.tools/varben</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1984</th>\n",
       "      <td>https://bio.tools/variantbam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1985</th>\n",
       "      <td>https://bio.tools/varid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1986</th>\n",
       "      <td>https://bio.tools/varscan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1987</th>\n",
       "      <td>https://bio.tools/varsifter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1988</th>\n",
       "      <td>https://bio.tools/vat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1989</th>\n",
       "      <td>https://bio.tools/vcf2caps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1990</th>\n",
       "      <td>https://bio.tools/vcf2fhir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1991</th>\n",
       "      <td>https://bio.tools/vcfpop</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1992</th>\n",
       "      <td>https://bio.tools/vcfutils</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1993</th>\n",
       "      <td>https://bio.tools/vcfview</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1994</th>\n",
       "      <td>https://bio.tools/vdjviz</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1995</th>\n",
       "      <td>https://bio.tools/vechat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1996</th>\n",
       "      <td>https://bio.tools/vecscreen_plus_taxonomy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1997</th>\n",
       "      <td>https://bio.tools/vectornti</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1998</th>\n",
       "      <td>https://bio.tools/vega_annotation_browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1999</th>\n",
       "      <td>https://bio.tools/vespa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2000</th>\n",
       "      <td>https://bio.tools/vespa-evolution</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2001</th>\n",
       "      <td>https://bio.tools/vga</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2002</th>\n",
       "      <td>https://bio.tools/vgea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2003</th>\n",
       "      <td>https://bio.tools/viadbg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2004</th>\n",
       "      <td>https://bio.tools/viewbs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2005</th>\n",
       "      <td>https://bio.tools/ving</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2006</th>\n",
       "      <td>https://bio.tools/vip-hl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2007</th>\n",
       "      <td>https://bio.tools/viquas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2008</th>\n",
       "      <td>https://bio.tools/viquf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2009</th>\n",
       "      <td>https://bio.tools/viral_bioinformatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2010</th>\n",
       "      <td>https://bio.tools/viramp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2011</th>\n",
       "      <td>https://bio.tools/virapipe</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2012</th>\n",
       "      <td>https://bio.tools/viremashiny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2013</th>\n",
       "      <td>https://bio.tools/viroscope</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2014</th>\n",
       "      <td>https://bio.tools/virstrain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2015</th>\n",
       "      <td>https://bio.tools/virtual_pharmacist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2016</th>\n",
       "      <td>https://bio.tools/virus_genotyping_tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2017</th>\n",
       "      <td>https://bio.tools/virusurf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2018</th>\n",
       "      <td>https://bio.tools/virxicon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2019</th>\n",
       "      <td>https://bio.tools/viscose</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2020</th>\n",
       "      <td>https://bio.tools/vistcr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2021</th>\n",
       "      <td>https://bio.tools/viva-variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2022</th>\n",
       "      <td>https://bio.tools/vmatch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2023</th>\n",
       "      <td>https://bio.tools/vonode</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2024</th>\n",
       "      <td>https://bio.tools/vrnetzer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2025</th>\n",
       "      <td>https://bio.tools/vt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2026</th>\n",
       "      <td>https://bio.tools/w-chipeaks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2027</th>\n",
       "      <td>https://bio.tools/w4cseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2028</th>\n",
       "      <td>https://bio.tools/walter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2029</th>\n",
       "      <td>https://bio.tools/washu_epigenome_browser</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2030</th>\n",
       "      <td>https://bio.tools/wavis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2031</th>\n",
       "      <td>https://bio.tools/webgmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2032</th>\n",
       "      <td>https://bio.tools/webmam</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2033</th>\n",
       "      <td>https://bio.tools/webprinses</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2034</th>\n",
       "      <td>https://bio.tools/wfa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2035</th>\n",
       "      <td>https://bio.tools/wgbssuite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2036</th>\n",
       "      <td>https://bio.tools/wglink</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2037</th>\n",
       "      <td>https://bio.tools/wham-variants</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2038</th>\n",
       "      <td>https://bio.tools/wmap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2039</th>\n",
       "      <td>https://bio.tools/wordcluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2040</th>\n",
       "      <td>https://bio.tools/x-cnv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2041</th>\n",
       "      <td>https://bio.tools/x-mate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2042</th>\n",
       "      <td>https://bio.tools/xcavator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2043</th>\n",
       "      <td>https://bio.tools/xparal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2044</th>\n",
       "      <td>https://bio.tools/xs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2045</th>\n",
       "      <td>https://bio.tools/xstk</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2046</th>\n",
       "      <td>https://bio.tools/xvector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2047</th>\n",
       "      <td>https://bio.tools/y-lineagetracker</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2048</th>\n",
       "      <td>https://bio.tools/y2h-scores</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2049</th>\n",
       "      <td>https://bio.tools/yass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2050</th>\n",
       "      <td>https://bio.tools/yass-spike</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2051</th>\n",
       "      <td>https://bio.tools/zelda</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2052</th>\n",
       "      <td>https://bio.tools/zinba</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2053</th>\n",
       "      <td>https://bio.tools/zinbwave</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'sequencing'seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "id": "dfa9dff8-7fa2-4284-8672-f2b771465f22",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "   }.\n",
    "   FILTER not EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "    ?y rdfs:subClassOf ?topic .\n",
    "    ?y rdfs:label ?y_label\n",
    "    \n",
    "    FILTER(?x != ?y).\n",
    "   }.\n",
    "     \n",
    "  \n",
    "    FILTER (?topic = <http://edamontology.org/topic_3168>)\n",
    "   \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "id": "28523d55-f7c0-48c9-bb93-411cef5925e4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'sequencing'bout de chaine: 0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'sequencing'bout de chaine: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "id": "4b0839e7-efe5-47cf-8a77-035d29827712",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "   }.\n",
    "   FILTER not EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3168> .\n",
    "    \n",
    "    FILTER(?x != ?y).\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3168>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "id": "0edc1614-f6d0-4898-8f7d-d623fab540c7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'sequencing'sans'sequencing' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3168</td>\n",
       "      <td>Sequencing</td>\n",
       "      <td>2161</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'sequencing'sans'sequencing' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "66535430-ca6a-4a41-8aa8-3102467c90a5",
   "metadata": {},
   "source": [
    "### biologie"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "a744b04c-b7b2-4e64-84e7-c5bcc29f307c",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3070> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3070>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "02a5371c-f445-486a-924f-948d79dda593",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'BIOLOGY': 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3070</td>\n",
       "      <td>Biology</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'BIOLOGY': {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "165c591e-7bc9-47cf-ae87-581eca5d5bd9",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3070>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3070>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "b3686e1f-d9da-490d-81c5-15046765e6ab",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'biology' seul: 0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'biology' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "1e519a8e-1f0c-41e9-b019-f17100f1ce20",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3070> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3070>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "   FILTER (?topic = <http://edamontology.org/topic_3070>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "2796ba40-edb3-466a-8dd8-bc7695ff00e2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'biology' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3070</td>\n",
       "      <td>Biology</td>\n",
       "      <td>475</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'biology' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b88a7349-64e3-4b75-98f5-185100061575",
   "metadata": {},
   "outputs": [],
   "source": [
    "### protein"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "a9327698-0923-4527-8298-978b3e563903",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0078> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0078>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "6360399a-ae82-4759-8920-26bbf458e734",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Protein' : 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "      <td>1138</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Protein' : {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "6bb36307-f010-4e0e-bd84-770ac2717d00",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0078>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0078>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "cb853628-703c-4455-b3b5-68b08813365c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'protein' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'protein' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "212ecd3c-7e5f-4205-b733-a5df25d53711",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0078> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0078>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0078>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "a8bc8fed-b5db-4b16-bbbf-de16caecaeaf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Protein' non seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>Proteins</td>\n",
       "      <td>1134</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Protein' non seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5590b1c0-9859-45c4-902a-3ffb172bd251",
   "metadata": {},
   "outputs": [],
   "source": [
    "### RNA 0099"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "086bf856-c404-49a4-a925-bfc19a80744b",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0099> .\n",
    "   }.\n",
    "     FILTER (?topic = <http://edamontology.org/topic_0099>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "1ba43db9-e5ed-4e53-9c13-aab541b1ae53",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'RNA' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0099</td>\n",
       "      <td>RNA</td>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'RNA' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "eaf72e5b-1d6d-452e-8cb4-b5238bd37376",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0099>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0099>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "2e02e316-1453-4922-9af2-41fb8ad19b9e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'RNA' seul: 0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'RNA' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "a2573e8c-2cf8-4e62-9fe6-563d5c4d7bed",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0099> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0099>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "   FILTER (?topic = <http://edamontology.org/topic_0099>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "268b67aa-bfa8-4bf4-9a59-50176b4ae168",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'RNA' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0099</td>\n",
       "      <td>RNA</td>\n",
       "      <td>835</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'RNA' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "80330af1-e26f-46f7-9a9f-038a5fe95c85",
   "metadata": {},
   "outputs": [],
   "source": [
    "### machine learning 3474"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "ebc5f9c1-06ca-4ecb-b930-9b0785cbe42b",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3474> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3474>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "3977505c-f4bf-4d4d-92a4-61d2884b56d1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'machine learning' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3474</td>\n",
       "      <td>Machine learning</td>\n",
       "      <td>3050</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'machine learning' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "21f1058a-bce2-47f2-8fb8-8ef109ef285e",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3474>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3474>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "59cb9ca8-0e77-4e78-8806-ececa90386c2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'machine learning' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3474</td>\n",
       "      <td>Machine learning</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'machine learning' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "b9053ef0-93e8-4478-ab35-92e17f96dce0",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3474> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3474>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "   FILTER (?topic = <http://edamontology.org/topic_3474>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "3ee1d289-6fea-4620-bd8b-73623834e773",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'machine learning' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3474</td>\n",
       "      <td>Machine learning</td>\n",
       "      <td>3038</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'machine learning' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "708f0f57-00a0-40e8-a315-f884d0f0edef",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Genetics 3053"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "ac7fb0bb-533a-486c-a642-ce075786657a",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3053> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3053>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "9c7ecb5b-c62a-4884-8420-2e255730d33c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Genetics' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3053</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>1107</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Genetics' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "e659318c-17e0-41f1-8542-e273b6ebdb32",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3053>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "     FILTER (?topic = <http://edamontology.org/topic_3053>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "95d2eae0-77c7-46eb-845e-9f7eaa82609c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Genetics' seul: 0\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Genetics' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "6954b3dd-c49d-4375-88fc-f431affabc15",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3053> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_3053>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "    FILTER (?topic = <http://edamontology.org/topic_3053>)\n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "ce8a2882-62e6-48d3-8231-90f17e828082",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Genetics' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_3053</td>\n",
       "      <td>Genetics</td>\n",
       "      <td>1107</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Genetics' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "11cb5e01-4942-4823-83f3-50d8e1077497",
   "metadata": {},
   "outputs": [],
   "source": [
    "### DNA 0654"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "e2c1cce2-f4d2-434b-ad1a-85473ebf9619",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0654> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0654>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "3e93b579-9b39-4f5b-8c77-c0fb465c2cab",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'DNA' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0654</td>\n",
       "      <td>DNA</td>\n",
       "      <td>1272</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'DNA' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "ae8f6b98-abee-4e77-acde-04fac2740cf7",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0654>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0654>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "33a07ab2-0387-4cc0-921a-6fe5a0ad135d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'DNA' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0654</td>\n",
       "      <td>DNA</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'DNA' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "90dbf275-7815-451e-a131-3acd42bee886",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0654> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0654>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "   FILTER (?topic = <http://edamontology.org/topic_0654>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "28e1929f-db9c-470d-90cb-58e88a196e8e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'DNA' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0654</td>\n",
       "      <td>DNA</td>\n",
       "      <td>1270</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'DNA' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "52b55fdb-903c-4ebe-9dd8-b33adf753ca7",
   "metadata": {},
   "outputs": [],
   "source": [
    "### Molecular interactions, pathways and networks 0602"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "0129880a-8975-4517-806e-088a3cea3b9c",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0602> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0602>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "5c9ec4da-fbe2-4436-961a-6b389dfb8790",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Molecular interactions, pathways and networks' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "      <td>2717</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Molecular interactions, pathways and networks' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "82415d02-c965-4e3c-95bc-650af56fe88f",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0602>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0602>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "3093ea56-3fe3-43eb-98e8-c8b9fa863d44",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Molecular interactions, pathways and networks' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Molecular interactions, pathways and networks' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "c4798d6c-d81a-437d-b42e-45e6abfbb7e0",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0602> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0602>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0602>) \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "e566146d-1358-4a84-bc56-46371e756234",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Molecular interactions, pathways and networks' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "      <td>2701</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Molecular interactions, pathways and networks' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a230ab1e-1d2f-4180-a089-d0c1accbc44f",
   "metadata": {},
   "outputs": [],
   "source": [
    "### \tGene expression 0203"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "59f8e43e-0e1d-491e-b681-b818e63f6e81",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0203> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0203>)  \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "4c19758e-920a-4f0d-91f2-0fda21845e5e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic '\tGene expression' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "      <td>2665</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic '\tGene expression' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "cda71fd0-f162-4039-89e4-a853e7306a3f",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0203>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "     FILTER (?topic = <http://edamontology.org/topic_0203>)  \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "084298c7-cc48-4193-8f37-2fe80d418fbc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic '\tGene expression' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic '\tGene expression' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "id": "8123fc4c-ed9d-4d44-a070-b410efae6466",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0203> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0203>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0203>)  \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "397154fd-5e97-46fc-9b54-3be11a7d7eab",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic '\tGene expression' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>Gene expression</td>\n",
       "      <td>2655</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic '\tGene expression' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "57fbf131-0ed4-4f75-995d-ea3d3fe4bec1",
   "metadata": {},
   "outputs": [],
   "source": [
    "###Small molecules 0154"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "83d9f52f-3b77-43cf-9755-dfa8eca26804",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0154> .\n",
    "   }.\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0154>)   \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "4c77b82d-954d-4e7f-b94d-6c8b383dbf1e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Small molecules' tous: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "      <td>2427</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Small molecules' tous: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "id": "0353d5a0-e890-416d-b7e8-9f283997d3f8",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0154>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0154>)   \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?topic)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "0b38f326-4038-47b6-8d9c-ecc7d916e938",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic '\tSmall molecules' seul: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic '\tSmall molecules' seul: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "b41da075-b878-48bf-a058-6395deea71d4",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "\n",
    "SELECT  DISTINCT\n",
    "?topic \n",
    "?label \n",
    "(COUNT(?topic) as ?count) \n",
    "WHERE {\n",
    "   ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "    \n",
    "    FILTER EXISTs\n",
    "    { \n",
    "     ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0154> .\n",
    "   }.\n",
    "   filter not exists{\n",
    "    ?x rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> ?topic .\n",
    "   ?topic rdfs:label ?label .\n",
    "   \n",
    "   ?y rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   <http://schema.org/name> ?name ; \n",
    "   <http://schema.org/applicationSubCategory> <http://edamontology.org/topic_0154>.\n",
    "    \n",
    "    FILTER(?x != ?y)\n",
    "   }\n",
    "    FILTER (?topic = <http://edamontology.org/topic_0154>)  \n",
    "} GROUP BY ?topic ?label\n",
    "ORDER BY DESC(?count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "id": "40772814-957f-4faa-9dc5-5e9e7a680cc0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "topic 'Small molecules' autre: 1\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>topic</th>\n",
       "      <th>label</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>Small molecules</td>\n",
       "      <td>2409</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"topic 'Small molecules' autre: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "19335233-d4ff-47e1-a1b2-7638f306abbe",
   "metadata": {},
   "outputs": [],
   "source": [
    "##########################################################################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4de9aafc-fec4-4edb-9b41-0422c9989afe",
   "metadata": {},
   "outputs": [],
   "source": [
    "## data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "id": "3173c2ed-ef71-482f-b2d3-e86034dfba51",
   "metadata": {},
   "outputs": [],
   "source": [
    "q=\"\"\"\n",
    "PREFIX edam:<http://edamontology.org/>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX bsc: <http://bioschemas.org/>\n",
    "PREFIX bsct: <http://bioschemas.org/types/>\n",
    "\n",
    "PREFIX sc: <http://schema.org/>\n",
    "\n",
    "SELECT DISTINCT ?data (COUNT(?data) as ?count) ?label\n",
    "WHERE {\n",
    "\n",
    "    ?biotools_id rdf:type <http://schema.org/SoftwareApplication> ;\n",
    "   bsc:input [sc:additionalType ?data]. \n",
    "   ?data rdfs:label ?label .\n",
    "\n",
    "}\n",
    "GROUP BY ?data ?label\n",
    "ORDER BY ASC(?data)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "id": "95b42392-cf17-4a62-9980-39a2640dec9c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EDAM data used to annotate bio.tools: 3535\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div style='height: 200px; overflow: auto; width: fit-content'><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>data</th>\n",
       "      <th>count</th>\n",
       "      <th>label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>http://edamontology.org/citation</td>\n",
       "      <td>18</td>\n",
       "      <td>Citation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>http://edamontology.org/created_in</td>\n",
       "      <td>18</td>\n",
       "      <td>Created in</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>http://edamontology.org/data_0005</td>\n",
       "      <td>18</td>\n",
       "      <td>Resource type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>http://edamontology.org/data_0006</td>\n",
       "      <td>18</td>\n",
       "      <td>Data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>http://edamontology.org/data_0007</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>http://edamontology.org/data_0581</td>\n",
       "      <td>18</td>\n",
       "      <td>Database</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>http://edamontology.org/data_0582</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>http://edamontology.org/data_0583</td>\n",
       "      <td>18</td>\n",
       "      <td>Directory metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>http://edamontology.org/data_0831</td>\n",
       "      <td>18</td>\n",
       "      <td>MeSH vocabulary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>http://edamontology.org/data_0832</td>\n",
       "      <td>18</td>\n",
       "      <td>HGNC vocabulary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>http://edamontology.org/data_0835</td>\n",
       "      <td>18</td>\n",
       "      <td>UMLS vocabulary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>http://edamontology.org/data_0842</td>\n",
       "      <td>18</td>\n",
       "      <td>Identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>http://edamontology.org/data_0843</td>\n",
       "      <td>18</td>\n",
       "      <td>Database entry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>http://edamontology.org/data_0844</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular mass</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>http://edamontology.org/data_0845</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular charge</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>http://edamontology.org/data_0846</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical formula</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>http://edamontology.org/data_0847</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>http://edamontology.org/data_0848</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>http://edamontology.org/data_0849</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>http://edamontology.org/data_0850</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>http://edamontology.org/data_0851</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence mask character</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>http://edamontology.org/data_0852</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence mask type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>http://edamontology.org/data_0853</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA sense specification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>http://edamontology.org/data_0854</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence length specification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>http://edamontology.org/data_0855</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>http://edamontology.org/data_0856</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature source</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>http://edamontology.org/data_0857</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence search results</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>http://edamontology.org/data_0858</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature matches</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>http://edamontology.org/data_0859</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>http://edamontology.org/data_0860</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>http://edamontology.org/data_0861</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment (words)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>http://edamontology.org/data_0862</td>\n",
       "      <td>18</td>\n",
       "      <td>Dotplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>http://edamontology.org/data_0863</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>http://edamontology.org/data_0864</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>http://edamontology.org/data_0865</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence similarity score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>http://edamontology.org/data_0866</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>http://edamontology.org/data_0867</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>http://edamontology.org/data_0868</td>\n",
       "      <td>18</td>\n",
       "      <td>Profile-profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>http://edamontology.org/data_0869</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>http://edamontology.org/data_0870</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence distance matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>http://edamontology.org/data_0871</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic character data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>http://edamontology.org/data_0872</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>http://edamontology.org/data_0874</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>http://edamontology.org/data_0875</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein topology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>http://edamontology.org/data_0876</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (secondary structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>http://edamontology.org/data_0877</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (super-secondary)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>http://edamontology.org/data_0878</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>http://edamontology.org/data_0879</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure alignment metadata (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>http://edamontology.org/data_0880</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>http://edamontology.org/data_0881</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>http://edamontology.org/data_0882</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure alignment metadata (RNA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>http://edamontology.org/data_0883</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>http://edamontology.org/data_0884</td>\n",
       "      <td>18</td>\n",
       "      <td>Tertiary structure record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>http://edamontology.org/data_0885</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure database search results</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>http://edamontology.org/data_0886</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>http://edamontology.org/data_0887</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>http://edamontology.org/data_0888</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure similarity score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>http://edamontology.org/data_0889</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>http://edamontology.org/data_0890</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural (3D) profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>http://edamontology.org/data_0891</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-3D profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>http://edamontology.org/data_0892</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence-structure scoring matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>http://edamontology.org/data_0893</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>http://edamontology.org/data_0894</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>http://edamontology.org/data_0895</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>http://edamontology.org/data_0896</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>http://edamontology.org/data_0897</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>http://edamontology.org/data_0899</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>http://edamontology.org/data_0900</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>http://edamontology.org/data_0901</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (domains)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>http://edamontology.org/data_0902</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein architecture report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>http://edamontology.org/data_0903</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>http://edamontology.org/data_0904</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (mutation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>http://edamontology.org/data_0905</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction raw data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>http://edamontology.org/data_0906</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>http://edamontology.org/data_0907</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>http://edamontology.org/data_0909</td>\n",
       "      <td>18</td>\n",
       "      <td>Vmax</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>http://edamontology.org/data_0910</td>\n",
       "      <td>18</td>\n",
       "      <td>Km</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>http://edamontology.org/data_0911</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleotide base annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>http://edamontology.org/data_0912</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>http://edamontology.org/data_0914</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>http://edamontology.org/data_0916</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>http://edamontology.org/data_0917</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>http://edamontology.org/data_0918</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>http://edamontology.org/data_0919</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosome report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>http://edamontology.org/data_0920</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype/phenotype report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>http://edamontology.org/data_0923</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR experiment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>http://edamontology.org/data_0924</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence trace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>http://edamontology.org/data_0925</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>http://edamontology.org/data_0926</td>\n",
       "      <td>18</td>\n",
       "      <td>RH scores</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>http://edamontology.org/data_0927</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic linkage report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>http://edamontology.org/data_0928</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>http://edamontology.org/data_0931</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray experiment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>http://edamontology.org/data_0932</td>\n",
       "      <td>18</td>\n",
       "      <td>Oligonucleotide probe data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>http://edamontology.org/data_0933</td>\n",
       "      <td>18</td>\n",
       "      <td>SAGE experimental data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>http://edamontology.org/data_0934</td>\n",
       "      <td>18</td>\n",
       "      <td>MPSS experimental data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>http://edamontology.org/data_0935</td>\n",
       "      <td>18</td>\n",
       "      <td>SBS experimental data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>http://edamontology.org/data_0936</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence tag profile (with gene assignment)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>http://edamontology.org/data_0937</td>\n",
       "      <td>18</td>\n",
       "      <td>Electron density map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>http://edamontology.org/data_0938</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw NMR data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>http://edamontology.org/data_0939</td>\n",
       "      <td>18</td>\n",
       "      <td>CD spectra</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>http://edamontology.org/data_0940</td>\n",
       "      <td>18</td>\n",
       "      <td>Volume map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>http://edamontology.org/data_0941</td>\n",
       "      <td>18</td>\n",
       "      <td>Electron microscopy model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>http://edamontology.org/data_0942</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>http://edamontology.org/data_0943</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>http://edamontology.org/data_0944</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide mass fingerprint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>http://edamontology.org/data_0945</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>http://edamontology.org/data_0946</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>http://edamontology.org/data_0947</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological pathway map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>http://edamontology.org/data_0948</td>\n",
       "      <td>18</td>\n",
       "      <td>Data resource definition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>http://edamontology.org/data_0949</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflow metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>http://edamontology.org/data_0950</td>\n",
       "      <td>18</td>\n",
       "      <td>Mathematical model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>http://edamontology.org/data_0951</td>\n",
       "      <td>18</td>\n",
       "      <td>Statistical estimate score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>http://edamontology.org/data_0952</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS database resource definition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>http://edamontology.org/data_0953</td>\n",
       "      <td>18</td>\n",
       "      <td>Version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>http://edamontology.org/data_0954</td>\n",
       "      <td>18</td>\n",
       "      <td>Database cross-mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>http://edamontology.org/data_0955</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>http://edamontology.org/data_0956</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>http://edamontology.org/data_0957</td>\n",
       "      <td>18</td>\n",
       "      <td>Database metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>http://edamontology.org/data_0958</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>http://edamontology.org/data_0959</td>\n",
       "      <td>18</td>\n",
       "      <td>Job metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>http://edamontology.org/data_0960</td>\n",
       "      <td>18</td>\n",
       "      <td>User metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>http://edamontology.org/data_0962</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecule report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>http://edamontology.org/data_0963</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>http://edamontology.org/data_0964</td>\n",
       "      <td>18</td>\n",
       "      <td>Scent annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>http://edamontology.org/data_0966</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology term</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>http://edamontology.org/data_0967</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>http://edamontology.org/data_0968</td>\n",
       "      <td>18</td>\n",
       "      <td>Keyword</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>http://edamontology.org/data_0970</td>\n",
       "      <td>18</td>\n",
       "      <td>Citation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>http://edamontology.org/data_0971</td>\n",
       "      <td>18</td>\n",
       "      <td>Article</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>http://edamontology.org/data_0972</td>\n",
       "      <td>18</td>\n",
       "      <td>Text mining report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>http://edamontology.org/data_0974</td>\n",
       "      <td>18</td>\n",
       "      <td>Entity identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>http://edamontology.org/data_0975</td>\n",
       "      <td>18</td>\n",
       "      <td>Data resource identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>http://edamontology.org/data_0976</td>\n",
       "      <td>18</td>\n",
       "      <td>Identifier (by type of entity)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>http://edamontology.org/data_0977</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>http://edamontology.org/data_0978</td>\n",
       "      <td>18</td>\n",
       "      <td>Discrete entity identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>http://edamontology.org/data_0979</td>\n",
       "      <td>18</td>\n",
       "      <td>Entity feature identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>http://edamontology.org/data_0980</td>\n",
       "      <td>18</td>\n",
       "      <td>Entity collection identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>http://edamontology.org/data_0981</td>\n",
       "      <td>18</td>\n",
       "      <td>Phenomenon identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>http://edamontology.org/data_0982</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>http://edamontology.org/data_0983</td>\n",
       "      <td>18</td>\n",
       "      <td>Atom ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>http://edamontology.org/data_0984</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>http://edamontology.org/data_0985</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>http://edamontology.org/data_0986</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>http://edamontology.org/data_0987</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosome name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>http://edamontology.org/data_0988</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>http://edamontology.org/data_0989</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>http://edamontology.org/data_0990</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>http://edamontology.org/data_0991</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical registry number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>http://edamontology.org/data_0992</td>\n",
       "      <td>18</td>\n",
       "      <td>Ligand identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>http://edamontology.org/data_0993</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>http://edamontology.org/data_0994</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>http://edamontology.org/data_0995</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleotide identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>http://edamontology.org/data_0996</td>\n",
       "      <td>18</td>\n",
       "      <td>Monosaccharide identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>http://edamontology.org/data_0997</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical name (ChEBI)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>http://edamontology.org/data_0998</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical name (IUPAC)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>http://edamontology.org/data_0999</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical name (INN)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>http://edamontology.org/data_1000</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical name (brand)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>http://edamontology.org/data_1001</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical name (synonymous)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>http://edamontology.org/data_1002</td>\n",
       "      <td>18</td>\n",
       "      <td>CAS number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>http://edamontology.org/data_1003</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical registry number (Beilstein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>http://edamontology.org/data_1004</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical registry number (Gmelin)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>http://edamontology.org/data_1005</td>\n",
       "      <td>18</td>\n",
       "      <td>HET group name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>http://edamontology.org/data_1006</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>http://edamontology.org/data_1007</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleotide code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>http://edamontology.org/data_1008</td>\n",
       "      <td>18</td>\n",
       "      <td>Polypeptide chain ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>http://edamontology.org/data_1009</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>http://edamontology.org/data_1010</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>http://edamontology.org/data_1011</td>\n",
       "      <td>18</td>\n",
       "      <td>EC number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>http://edamontology.org/data_1012</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>http://edamontology.org/data_1013</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction enzyme name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>http://edamontology.org/data_1014</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence position specification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>http://edamontology.org/data_1015</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>http://edamontology.org/data_1016</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence position</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>http://edamontology.org/data_1017</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence range</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>http://edamontology.org/data_1018</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid feature identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>http://edamontology.org/data_1019</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein feature identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>http://edamontology.org/data_1020</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature key</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>http://edamontology.org/data_1021</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature qualifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>http://edamontology.org/data_1022</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature label</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>http://edamontology.org/data_1023</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS Uniform Feature Object</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>http://edamontology.org/data_1024</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>http://edamontology.org/data_1025</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>http://edamontology.org/data_1026</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene symbol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>http://edamontology.org/data_1027</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (NCBI)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>http://edamontology.org/data_1028</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene identifier (NCBI RefSeq)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>http://edamontology.org/data_1029</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene identifier (NCBI UniGene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>http://edamontology.org/data_1030</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene identifier (Entrez)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>http://edamontology.org/data_1031</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (CGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>http://edamontology.org/data_1032</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (DictyBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>http://edamontology.org/data_1033</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl gene ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>http://edamontology.org/data_1034</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (SGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>http://edamontology.org/data_1035</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>http://edamontology.org/data_1036</td>\n",
       "      <td>18</td>\n",
       "      <td>TIGR identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>http://edamontology.org/data_1037</td>\n",
       "      <td>18</td>\n",
       "      <td>TAIR accession (gene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>http://edamontology.org/data_1038</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>http://edamontology.org/data_1039</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP domain identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>http://edamontology.org/data_1040</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH domain ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>http://edamontology.org/data_1041</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP concise classification string (sccs)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>http://edamontology.org/data_1042</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP sunid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>http://edamontology.org/data_1043</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH node ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>http://edamontology.org/data_1044</td>\n",
       "      <td>18</td>\n",
       "      <td>Kingdom name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>http://edamontology.org/data_1045</td>\n",
       "      <td>18</td>\n",
       "      <td>Species name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>http://edamontology.org/data_1046</td>\n",
       "      <td>18</td>\n",
       "      <td>Strain name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>http://edamontology.org/data_1047</td>\n",
       "      <td>18</td>\n",
       "      <td>URI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>http://edamontology.org/data_1048</td>\n",
       "      <td>18</td>\n",
       "      <td>Database ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>http://edamontology.org/data_1049</td>\n",
       "      <td>18</td>\n",
       "      <td>Directory name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>http://edamontology.org/data_1050</td>\n",
       "      <td>18</td>\n",
       "      <td>File name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>http://edamontology.org/data_1051</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>http://edamontology.org/data_1052</td>\n",
       "      <td>18</td>\n",
       "      <td>URL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>http://edamontology.org/data_1053</td>\n",
       "      <td>18</td>\n",
       "      <td>URN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>http://edamontology.org/data_1055</td>\n",
       "      <td>18</td>\n",
       "      <td>LSID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>http://edamontology.org/data_1056</td>\n",
       "      <td>18</td>\n",
       "      <td>Database name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>http://edamontology.org/data_1057</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>http://edamontology.org/data_1058</td>\n",
       "      <td>18</td>\n",
       "      <td>Enumerated file name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>http://edamontology.org/data_1059</td>\n",
       "      <td>18</td>\n",
       "      <td>File name extension</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>http://edamontology.org/data_1060</td>\n",
       "      <td>18</td>\n",
       "      <td>File base name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>http://edamontology.org/data_1061</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>http://edamontology.org/data_1062</td>\n",
       "      <td>18</td>\n",
       "      <td>Database entry identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>http://edamontology.org/data_1063</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>http://edamontology.org/data_1064</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>http://edamontology.org/data_1065</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>http://edamontology.org/data_1066</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>http://edamontology.org/data_1067</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic distance matrix identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>223</th>\n",
       "      <td>http://edamontology.org/data_1068</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>http://edamontology.org/data_1069</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>http://edamontology.org/data_1070</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>http://edamontology.org/data_1071</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural (3D) profile ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>http://edamontology.org/data_1072</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>http://edamontology.org/data_1073</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>http://edamontology.org/data_1074</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>230</th>\n",
       "      <td>http://edamontology.org/data_1075</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>231</th>\n",
       "      <td>http://edamontology.org/data_1076</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>232</th>\n",
       "      <td>http://edamontology.org/data_1077</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factor identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>233</th>\n",
       "      <td>http://edamontology.org/data_1078</td>\n",
       "      <td>18</td>\n",
       "      <td>Experiment annotation ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>http://edamontology.org/data_1079</td>\n",
       "      <td>18</td>\n",
       "      <td>Electron microscopy model ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>235</th>\n",
       "      <td>http://edamontology.org/data_1080</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression report ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>http://edamontology.org/data_1081</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype and phenotype annotation ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>http://edamontology.org/data_1082</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>238</th>\n",
       "      <td>http://edamontology.org/data_1083</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflow ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>239</th>\n",
       "      <td>http://edamontology.org/data_1084</td>\n",
       "      <td>18</td>\n",
       "      <td>Data resource definition ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>240</th>\n",
       "      <td>http://edamontology.org/data_1085</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological model ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>241</th>\n",
       "      <td>http://edamontology.org/data_1086</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>http://edamontology.org/data_1087</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>243</th>\n",
       "      <td>http://edamontology.org/data_1088</td>\n",
       "      <td>18</td>\n",
       "      <td>Article ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>244</th>\n",
       "      <td>http://edamontology.org/data_1089</td>\n",
       "      <td>18</td>\n",
       "      <td>FlyBase ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>245</th>\n",
       "      <td>http://edamontology.org/data_1091</td>\n",
       "      <td>18</td>\n",
       "      <td>WormBase name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>246</th>\n",
       "      <td>http://edamontology.org/data_1092</td>\n",
       "      <td>18</td>\n",
       "      <td>WormBase class</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>http://edamontology.org/data_1093</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>http://edamontology.org/data_1094</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>249</th>\n",
       "      <td>http://edamontology.org/data_1095</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS Uniform Sequence Address</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250</th>\n",
       "      <td>http://edamontology.org/data_1096</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence accession (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>251</th>\n",
       "      <td>http://edamontology.org/data_1097</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence accession (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>252</th>\n",
       "      <td>http://edamontology.org/data_1098</td>\n",
       "      <td>18</td>\n",
       "      <td>RefSeq accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>http://edamontology.org/data_1099</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt accession (extended)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>254</th>\n",
       "      <td>http://edamontology.org/data_1100</td>\n",
       "      <td>18</td>\n",
       "      <td>PIR identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>http://edamontology.org/data_1101</td>\n",
       "      <td>18</td>\n",
       "      <td>TREMBL accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>256</th>\n",
       "      <td>http://edamontology.org/data_1102</td>\n",
       "      <td>18</td>\n",
       "      <td>Gramene primary identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>257</th>\n",
       "      <td>http://edamontology.org/data_1103</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL/GenBank/DDBJ ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>http://edamontology.org/data_1104</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (UniGene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>259</th>\n",
       "      <td>http://edamontology.org/data_1105</td>\n",
       "      <td>18</td>\n",
       "      <td>dbEST accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>260</th>\n",
       "      <td>http://edamontology.org/data_1106</td>\n",
       "      <td>18</td>\n",
       "      <td>dbSNP ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>261</th>\n",
       "      <td>http://edamontology.org/data_1110</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS sequence type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>262</th>\n",
       "      <td>http://edamontology.org/data_1111</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS listfile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>263</th>\n",
       "      <td>http://edamontology.org/data_1112</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>264</th>\n",
       "      <td>http://edamontology.org/data_1113</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (COG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>265</th>\n",
       "      <td>http://edamontology.org/data_1114</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>http://edamontology.org/data_1115</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>267</th>\n",
       "      <td>http://edamontology.org/data_1116</td>\n",
       "      <td>18</td>\n",
       "      <td>ELM ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>http://edamontology.org/data_1117</td>\n",
       "      <td>18</td>\n",
       "      <td>Prosite accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>269</th>\n",
       "      <td>http://edamontology.org/data_1118</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER hidden Markov model ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>270</th>\n",
       "      <td>http://edamontology.org/data_1119</td>\n",
       "      <td>18</td>\n",
       "      <td>JASPAR profile ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>http://edamontology.org/data_1120</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>http://edamontology.org/data_1121</td>\n",
       "      <td>18</td>\n",
       "      <td>BLAST sequence alignment type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>273</th>\n",
       "      <td>http://edamontology.org/data_1122</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>http://edamontology.org/data_1123</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeBASE study accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>http://edamontology.org/data_1124</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeFam accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>http://edamontology.org/data_1125</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>http://edamontology.org/data_1126</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>http://edamontology.org/data_1127</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>279</th>\n",
       "      <td>http://edamontology.org/data_1128</td>\n",
       "      <td>18</td>\n",
       "      <td>AAindex ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>280</th>\n",
       "      <td>http://edamontology.org/data_1129</td>\n",
       "      <td>18</td>\n",
       "      <td>BIND accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>http://edamontology.org/data_1130</td>\n",
       "      <td>18</td>\n",
       "      <td>IntAct accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>http://edamontology.org/data_1131</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>http://edamontology.org/data_1132</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro entry name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>http://edamontology.org/data_1133</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>http://edamontology.org/data_1134</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro secondary accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>286</th>\n",
       "      <td>http://edamontology.org/data_1135</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene3D ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>287</th>\n",
       "      <td>http://edamontology.org/data_1136</td>\n",
       "      <td>18</td>\n",
       "      <td>PIRSF ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>http://edamontology.org/data_1137</td>\n",
       "      <td>18</td>\n",
       "      <td>PRINTS code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>http://edamontology.org/data_1138</td>\n",
       "      <td>18</td>\n",
       "      <td>Pfam accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>http://edamontology.org/data_1139</td>\n",
       "      <td>18</td>\n",
       "      <td>SMART accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>http://edamontology.org/data_1140</td>\n",
       "      <td>18</td>\n",
       "      <td>Superfamily hidden Markov model number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>http://edamontology.org/data_1141</td>\n",
       "      <td>18</td>\n",
       "      <td>TIGRFam ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>http://edamontology.org/data_1142</td>\n",
       "      <td>18</td>\n",
       "      <td>ProDom accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>http://edamontology.org/data_1143</td>\n",
       "      <td>18</td>\n",
       "      <td>TRANSFAC accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>http://edamontology.org/data_1144</td>\n",
       "      <td>18</td>\n",
       "      <td>ArrayExpress accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>http://edamontology.org/data_1145</td>\n",
       "      <td>18</td>\n",
       "      <td>PRIDE experiment accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>http://edamontology.org/data_1146</td>\n",
       "      <td>18</td>\n",
       "      <td>EMDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>http://edamontology.org/data_1147</td>\n",
       "      <td>18</td>\n",
       "      <td>GEO accession number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>http://edamontology.org/data_1148</td>\n",
       "      <td>18</td>\n",
       "      <td>GermOnline ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>http://edamontology.org/data_1149</td>\n",
       "      <td>18</td>\n",
       "      <td>EMAGE ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>http://edamontology.org/data_1150</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>http://edamontology.org/data_1151</td>\n",
       "      <td>18</td>\n",
       "      <td>HGVbase ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>http://edamontology.org/data_1152</td>\n",
       "      <td>18</td>\n",
       "      <td>HIVDB identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>http://edamontology.org/data_1153</td>\n",
       "      <td>18</td>\n",
       "      <td>OMIM ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>http://edamontology.org/data_1154</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG object identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>http://edamontology.org/data_1155</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (reactome)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>http://edamontology.org/data_1156</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (aMAZE)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>http://edamontology.org/data_1157</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (BioCyc)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>http://edamontology.org/data_1158</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (INOH)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>http://edamontology.org/data_1159</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (PATIKA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>http://edamontology.org/data_1160</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (CPDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>http://edamontology.org/data_1161</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (Panther)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>http://edamontology.org/data_1162</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>http://edamontology.org/data_1163</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM data type name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>http://edamontology.org/data_1164</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM URI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>http://edamontology.org/data_1165</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM data type primary name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>http://edamontology.org/data_1166</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM data type synonymous name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>http://edamontology.org/data_1167</td>\n",
       "      <td>18</td>\n",
       "      <td>Taverna workflow ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>http://edamontology.org/data_1170</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological model name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>http://edamontology.org/data_1171</td>\n",
       "      <td>18</td>\n",
       "      <td>BioModel ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>http://edamontology.org/data_1172</td>\n",
       "      <td>18</td>\n",
       "      <td>PubChem CID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>http://edamontology.org/data_1173</td>\n",
       "      <td>18</td>\n",
       "      <td>ChemSpider ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>http://edamontology.org/data_1174</td>\n",
       "      <td>18</td>\n",
       "      <td>ChEBI ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>324</th>\n",
       "      <td>http://edamontology.org/data_1175</td>\n",
       "      <td>18</td>\n",
       "      <td>BioPax concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>http://edamontology.org/data_1176</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>http://edamontology.org/data_1177</td>\n",
       "      <td>18</td>\n",
       "      <td>MeSH concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>http://edamontology.org/data_1178</td>\n",
       "      <td>18</td>\n",
       "      <td>HGNC concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>http://edamontology.org/data_1179</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI taxonomy ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>http://edamontology.org/data_1180</td>\n",
       "      <td>18</td>\n",
       "      <td>Plant Ontology concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>330</th>\n",
       "      <td>http://edamontology.org/data_1181</td>\n",
       "      <td>18</td>\n",
       "      <td>UMLS concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>331</th>\n",
       "      <td>http://edamontology.org/data_1182</td>\n",
       "      <td>18</td>\n",
       "      <td>FMA concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>332</th>\n",
       "      <td>http://edamontology.org/data_1183</td>\n",
       "      <td>18</td>\n",
       "      <td>EMAP concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>333</th>\n",
       "      <td>http://edamontology.org/data_1184</td>\n",
       "      <td>18</td>\n",
       "      <td>ChEBI concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>334</th>\n",
       "      <td>http://edamontology.org/data_1185</td>\n",
       "      <td>18</td>\n",
       "      <td>MGED concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>335</th>\n",
       "      <td>http://edamontology.org/data_1186</td>\n",
       "      <td>18</td>\n",
       "      <td>myGrid concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>http://edamontology.org/data_1187</td>\n",
       "      <td>18</td>\n",
       "      <td>PubMed ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>337</th>\n",
       "      <td>http://edamontology.org/data_1188</td>\n",
       "      <td>18</td>\n",
       "      <td>DOI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>338</th>\n",
       "      <td>http://edamontology.org/data_1189</td>\n",
       "      <td>18</td>\n",
       "      <td>Medline UI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>339</th>\n",
       "      <td>http://edamontology.org/data_1190</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>340</th>\n",
       "      <td>http://edamontology.org/data_1191</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name (signature)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>341</th>\n",
       "      <td>http://edamontology.org/data_1192</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name (BLAST)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>342</th>\n",
       "      <td>http://edamontology.org/data_1193</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name (FASTA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>343</th>\n",
       "      <td>http://edamontology.org/data_1194</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name (EMBOSS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>344</th>\n",
       "      <td>http://edamontology.org/data_1195</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool name (EMBASSY package)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>345</th>\n",
       "      <td>http://edamontology.org/data_1201</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor (constitutional)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>346</th>\n",
       "      <td>http://edamontology.org/data_1202</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor (electronic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>347</th>\n",
       "      <td>http://edamontology.org/data_1203</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor (geometrical)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>348</th>\n",
       "      <td>http://edamontology.org/data_1204</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor (topological)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>349</th>\n",
       "      <td>http://edamontology.org/data_1205</td>\n",
       "      <td>18</td>\n",
       "      <td>QSAR descriptor (molecular)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>350</th>\n",
       "      <td>http://edamontology.org/data_1233</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>http://edamontology.org/data_1234</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>352</th>\n",
       "      <td>http://edamontology.org/data_1235</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>353</th>\n",
       "      <td>http://edamontology.org/data_1236</td>\n",
       "      <td>18</td>\n",
       "      <td>Psiblast checkpoint file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>354</th>\n",
       "      <td>http://edamontology.org/data_1237</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER synthetic sequences set</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>355</th>\n",
       "      <td>http://edamontology.org/data_1238</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteolytic digest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>356</th>\n",
       "      <td>http://edamontology.org/data_1239</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction digest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>357</th>\n",
       "      <td>http://edamontology.org/data_1240</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>358</th>\n",
       "      <td>http://edamontology.org/data_1241</td>\n",
       "      <td>18</td>\n",
       "      <td>vectorstrip cloning vector definition file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>http://edamontology.org/data_1242</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer3 internal oligo mishybridizing library</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>360</th>\n",
       "      <td>http://edamontology.org/data_1243</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer3 mispriming library file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>http://edamontology.org/data_1244</td>\n",
       "      <td>18</td>\n",
       "      <td>primersearch primer pairs sequence record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>362</th>\n",
       "      <td>http://edamontology.org/data_1245</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>363</th>\n",
       "      <td>http://edamontology.org/data_1246</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>364</th>\n",
       "      <td>http://edamontology.org/data_1249</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence length</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>365</th>\n",
       "      <td>http://edamontology.org/data_1250</td>\n",
       "      <td>18</td>\n",
       "      <td>Word size</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>366</th>\n",
       "      <td>http://edamontology.org/data_1251</td>\n",
       "      <td>18</td>\n",
       "      <td>Window size</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>367</th>\n",
       "      <td>http://edamontology.org/data_1252</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence length range</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>368</th>\n",
       "      <td>http://edamontology.org/data_1253</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence information report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>369</th>\n",
       "      <td>http://edamontology.org/data_1254</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>370</th>\n",
       "      <td>http://edamontology.org/data_1255</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>371</th>\n",
       "      <td>http://edamontology.org/data_1256</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features (comparative)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>372</th>\n",
       "      <td>http://edamontology.org/data_1257</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence property (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>http://edamontology.org/data_1258</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence property (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>374</th>\n",
       "      <td>http://edamontology.org/data_1259</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence complexity report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>http://edamontology.org/data_1260</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence ambiguity report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>http://edamontology.org/data_1261</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>http://edamontology.org/data_1262</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide molecular weight hits</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>http://edamontology.org/data_1263</td>\n",
       "      <td>18</td>\n",
       "      <td>Base position variability plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>http://edamontology.org/data_1264</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>380</th>\n",
       "      <td>http://edamontology.org/data_1265</td>\n",
       "      <td>18</td>\n",
       "      <td>Base frequencies table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>381</th>\n",
       "      <td>http://edamontology.org/data_1266</td>\n",
       "      <td>18</td>\n",
       "      <td>Base word frequencies table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>http://edamontology.org/data_1267</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid frequencies table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>http://edamontology.org/data_1268</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid word frequencies table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>http://edamontology.org/data_1269</td>\n",
       "      <td>18</td>\n",
       "      <td>DAS sequence feature annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>http://edamontology.org/data_1270</td>\n",
       "      <td>18</td>\n",
       "      <td>Feature table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>http://edamontology.org/data_1274</td>\n",
       "      <td>18</td>\n",
       "      <td>Map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>http://edamontology.org/data_1276</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>388</th>\n",
       "      <td>http://edamontology.org/data_1277</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>389</th>\n",
       "      <td>http://edamontology.org/data_1278</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>http://edamontology.org/data_1279</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>391</th>\n",
       "      <td>http://edamontology.org/data_1280</td>\n",
       "      <td>18</td>\n",
       "      <td>Physical map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>392</th>\n",
       "      <td>http://edamontology.org/data_1281</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>393</th>\n",
       "      <td>http://edamontology.org/data_1283</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytogenetic map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>394</th>\n",
       "      <td>http://edamontology.org/data_1284</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA transduction map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>395</th>\n",
       "      <td>http://edamontology.org/data_1285</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>396</th>\n",
       "      <td>http://edamontology.org/data_1286</td>\n",
       "      <td>18</td>\n",
       "      <td>Plasmid map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>http://edamontology.org/data_1288</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>398</th>\n",
       "      <td>http://edamontology.org/data_1289</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>http://edamontology.org/data_1290</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro compact match image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>400</th>\n",
       "      <td>http://edamontology.org/data_1291</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro detailed match image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>401</th>\n",
       "      <td>http://edamontology.org/data_1292</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro architecture image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>402</th>\n",
       "      <td>http://edamontology.org/data_1293</td>\n",
       "      <td>18</td>\n",
       "      <td>SMART protein schematic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>http://edamontology.org/data_1294</td>\n",
       "      <td>18</td>\n",
       "      <td>GlobPlot domain image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>404</th>\n",
       "      <td>http://edamontology.org/data_1298</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif matches</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>405</th>\n",
       "      <td>http://edamontology.org/data_1299</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features (repeats)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>406</th>\n",
       "      <td>http://edamontology.org/data_1300</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene and transcript structure (report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>407</th>\n",
       "      <td>http://edamontology.org/data_1301</td>\n",
       "      <td>18</td>\n",
       "      <td>Mobile genetic elements</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>408</th>\n",
       "      <td>http://edamontology.org/data_1303</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (quadruplexes)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>http://edamontology.org/data_1306</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleosome exclusion sequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>410</th>\n",
       "      <td>http://edamontology.org/data_1309</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene features (exonic splicing enhancer)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>411</th>\n",
       "      <td>http://edamontology.org/data_1310</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (microRNA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>412</th>\n",
       "      <td>http://edamontology.org/data_1313</td>\n",
       "      <td>18</td>\n",
       "      <td>Coding region</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>413</th>\n",
       "      <td>http://edamontology.org/data_1314</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene features (SECIS element)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>414</th>\n",
       "      <td>http://edamontology.org/data_1315</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factor binding sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>415</th>\n",
       "      <td>http://edamontology.org/data_1321</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>http://edamontology.org/data_1322</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (signal peptides)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>417</th>\n",
       "      <td>http://edamontology.org/data_1323</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (cleavage sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>http://edamontology.org/data_1324</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (post-translation modifications)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>419</th>\n",
       "      <td>http://edamontology.org/data_1325</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (active sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>420</th>\n",
       "      <td>http://edamontology.org/data_1326</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (binding sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>421</th>\n",
       "      <td>http://edamontology.org/data_1327</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (epitopes)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>http://edamontology.org/data_1328</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (nucleic acid binding sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>423</th>\n",
       "      <td>http://edamontology.org/data_1329</td>\n",
       "      <td>18</td>\n",
       "      <td>MHC Class I epitopes report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>424</th>\n",
       "      <td>http://edamontology.org/data_1330</td>\n",
       "      <td>18</td>\n",
       "      <td>MHC Class II epitopes report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>425</th>\n",
       "      <td>http://edamontology.org/data_1331</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (PEST sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>426</th>\n",
       "      <td>http://edamontology.org/data_1338</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database hits scores list</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>427</th>\n",
       "      <td>http://edamontology.org/data_1339</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database hits alignments list</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>428</th>\n",
       "      <td>http://edamontology.org/data_1340</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database hits evaluation data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>429</th>\n",
       "      <td>http://edamontology.org/data_1344</td>\n",
       "      <td>18</td>\n",
       "      <td>MEME motif alphabet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>430</th>\n",
       "      <td>http://edamontology.org/data_1345</td>\n",
       "      <td>18</td>\n",
       "      <td>MEME background frequencies file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>431</th>\n",
       "      <td>http://edamontology.org/data_1346</td>\n",
       "      <td>18</td>\n",
       "      <td>MEME motifs directive file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>432</th>\n",
       "      <td>http://edamontology.org/data_1347</td>\n",
       "      <td>18</td>\n",
       "      <td>Dirichlet distribution</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>433</th>\n",
       "      <td>http://edamontology.org/data_1348</td>\n",
       "      <td>18</td>\n",
       "      <td>HMM emission and transition counts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>434</th>\n",
       "      <td>http://edamontology.org/data_1352</td>\n",
       "      <td>18</td>\n",
       "      <td>Regular expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>435</th>\n",
       "      <td>http://edamontology.org/data_1353</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>http://edamontology.org/data_1354</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>437</th>\n",
       "      <td>http://edamontology.org/data_1355</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>438</th>\n",
       "      <td>http://edamontology.org/data_1358</td>\n",
       "      <td>18</td>\n",
       "      <td>Prosite nucleotide pattern</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>439</th>\n",
       "      <td>http://edamontology.org/data_1359</td>\n",
       "      <td>18</td>\n",
       "      <td>Prosite protein pattern</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>440</th>\n",
       "      <td>http://edamontology.org/data_1361</td>\n",
       "      <td>18</td>\n",
       "      <td>Position frequency matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>441</th>\n",
       "      <td>http://edamontology.org/data_1362</td>\n",
       "      <td>18</td>\n",
       "      <td>Position weight matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>442</th>\n",
       "      <td>http://edamontology.org/data_1363</td>\n",
       "      <td>18</td>\n",
       "      <td>Information content matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>443</th>\n",
       "      <td>http://edamontology.org/data_1364</td>\n",
       "      <td>18</td>\n",
       "      <td>Hidden Markov model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>444</th>\n",
       "      <td>http://edamontology.org/data_1365</td>\n",
       "      <td>18</td>\n",
       "      <td>Fingerprint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>445</th>\n",
       "      <td>http://edamontology.org/data_1368</td>\n",
       "      <td>18</td>\n",
       "      <td>Domainatrix signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>446</th>\n",
       "      <td>http://edamontology.org/data_1371</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER NULL hidden Markov model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>447</th>\n",
       "      <td>http://edamontology.org/data_1372</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>448</th>\n",
       "      <td>http://edamontology.org/data_1373</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>449</th>\n",
       "      <td>http://edamontology.org/data_1374</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein region signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>450</th>\n",
       "      <td>http://edamontology.org/data_1375</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein repeat signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>451</th>\n",
       "      <td>http://edamontology.org/data_1376</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein site signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>452</th>\n",
       "      <td>http://edamontology.org/data_1377</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein conserved site signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>453</th>\n",
       "      <td>http://edamontology.org/data_1378</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein active site signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>454</th>\n",
       "      <td>http://edamontology.org/data_1379</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein binding site signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>455</th>\n",
       "      <td>http://edamontology.org/data_1380</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein post-translational modification signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>456</th>\n",
       "      <td>http://edamontology.org/data_1381</td>\n",
       "      <td>18</td>\n",
       "      <td>Pair sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>http://edamontology.org/data_1382</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment (multiple)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>http://edamontology.org/data_1383</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>http://edamontology.org/data_1384</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>http://edamontology.org/data_1385</td>\n",
       "      <td>18</td>\n",
       "      <td>Hybrid sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>461</th>\n",
       "      <td>http://edamontology.org/data_1386</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment (nucleic acid pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>462</th>\n",
       "      <td>http://edamontology.org/data_1387</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment (protein pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>463</th>\n",
       "      <td>http://edamontology.org/data_1388</td>\n",
       "      <td>18</td>\n",
       "      <td>Hybrid sequence alignment (pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>464</th>\n",
       "      <td>http://edamontology.org/data_1389</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple nucleotide sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>465</th>\n",
       "      <td>http://edamontology.org/data_1390</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple protein sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>466</th>\n",
       "      <td>http://edamontology.org/data_1394</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment score or penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>467</th>\n",
       "      <td>http://edamontology.org/data_1395</td>\n",
       "      <td>18</td>\n",
       "      <td>Score end gaps control</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>468</th>\n",
       "      <td>http://edamontology.org/data_1396</td>\n",
       "      <td>18</td>\n",
       "      <td>Aligned sequence order</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>469</th>\n",
       "      <td>http://edamontology.org/data_1397</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap opening penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>http://edamontology.org/data_1398</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap extension penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>471</th>\n",
       "      <td>http://edamontology.org/data_1399</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap separation penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>472</th>\n",
       "      <td>http://edamontology.org/data_1400</td>\n",
       "      <td>18</td>\n",
       "      <td>Terminal gap penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>473</th>\n",
       "      <td>http://edamontology.org/data_1401</td>\n",
       "      <td>18</td>\n",
       "      <td>Match reward score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>http://edamontology.org/data_1402</td>\n",
       "      <td>18</td>\n",
       "      <td>Mismatch penalty score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>http://edamontology.org/data_1403</td>\n",
       "      <td>18</td>\n",
       "      <td>Drop off score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>476</th>\n",
       "      <td>http://edamontology.org/data_1404</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap opening penalty (integer)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>477</th>\n",
       "      <td>http://edamontology.org/data_1405</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap opening penalty (float)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>478</th>\n",
       "      <td>http://edamontology.org/data_1406</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap extension penalty (integer)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>http://edamontology.org/data_1407</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap extension penalty (float)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>480</th>\n",
       "      <td>http://edamontology.org/data_1408</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap separation penalty (integer)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>481</th>\n",
       "      <td>http://edamontology.org/data_1409</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap separation penalty (float)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>482</th>\n",
       "      <td>http://edamontology.org/data_1410</td>\n",
       "      <td>18</td>\n",
       "      <td>Terminal gap opening penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>483</th>\n",
       "      <td>http://edamontology.org/data_1411</td>\n",
       "      <td>18</td>\n",
       "      <td>Terminal gap extension penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>484</th>\n",
       "      <td>http://edamontology.org/data_1412</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence identity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>485</th>\n",
       "      <td>http://edamontology.org/data_1413</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence similarity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>486</th>\n",
       "      <td>http://edamontology.org/data_1414</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment metadata (quality report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>487</th>\n",
       "      <td>http://edamontology.org/data_1415</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment report (site conservation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>488</th>\n",
       "      <td>http://edamontology.org/data_1416</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment report (site correlation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>489</th>\n",
       "      <td>http://edamontology.org/data_1417</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment (Domainatrix signature)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>490</th>\n",
       "      <td>http://edamontology.org/data_1418</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment (HMM)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>491</th>\n",
       "      <td>http://edamontology.org/data_1420</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment (fingerprint)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>492</th>\n",
       "      <td>http://edamontology.org/data_1426</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic continuous quantitative data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>493</th>\n",
       "      <td>http://edamontology.org/data_1427</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic discrete data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>494</th>\n",
       "      <td>http://edamontology.org/data_1428</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic character cliques</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>495</th>\n",
       "      <td>http://edamontology.org/data_1429</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic invariants</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>496</th>\n",
       "      <td>http://edamontology.org/data_1438</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>497</th>\n",
       "      <td>http://edamontology.org/data_1439</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA substitution model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>498</th>\n",
       "      <td>http://edamontology.org/data_1440</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (tree shape)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>499</th>\n",
       "      <td>http://edamontology.org/data_1441</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (tree evaluation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>500</th>\n",
       "      <td>http://edamontology.org/data_1442</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree distances</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>http://edamontology.org/data_1443</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (tree stratigraphic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>502</th>\n",
       "      <td>http://edamontology.org/data_1444</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic character contrasts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>http://edamontology.org/data_1446</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix (integers)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>http://edamontology.org/data_1447</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix (floats)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>http://edamontology.org/data_1448</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix (nucleotide)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>http://edamontology.org/data_1449</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison matrix (amino acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>http://edamontology.org/data_1450</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleotide comparison matrix (integers)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>508</th>\n",
       "      <td>http://edamontology.org/data_1451</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleotide comparison matrix (floats)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>509</th>\n",
       "      <td>http://edamontology.org/data_1452</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid comparison matrix (integers)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>510</th>\n",
       "      <td>http://edamontology.org/data_1453</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid comparison matrix (floats)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>511</th>\n",
       "      <td>http://edamontology.org/data_1459</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>512</th>\n",
       "      <td>http://edamontology.org/data_1460</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>513</th>\n",
       "      <td>http://edamontology.org/data_1461</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-ligand complex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>514</th>\n",
       "      <td>http://edamontology.org/data_1462</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>515</th>\n",
       "      <td>http://edamontology.org/data_1463</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecule structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>516</th>\n",
       "      <td>http://edamontology.org/data_1464</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>517</th>\n",
       "      <td>http://edamontology.org/data_1465</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>518</th>\n",
       "      <td>http://edamontology.org/data_1466</td>\n",
       "      <td>18</td>\n",
       "      <td>tRNA structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>519</th>\n",
       "      <td>http://edamontology.org/data_1467</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein chain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>520</th>\n",
       "      <td>http://edamontology.org/data_1468</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>521</th>\n",
       "      <td>http://edamontology.org/data_1469</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure (all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>522</th>\n",
       "      <td>http://edamontology.org/data_1470</td>\n",
       "      <td>18</td>\n",
       "      <td>C-alpha trace</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>523</th>\n",
       "      <td>http://edamontology.org/data_1471</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein chain (all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>524</th>\n",
       "      <td>http://edamontology.org/data_1472</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein chain (C-alpha atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>525</th>\n",
       "      <td>http://edamontology.org/data_1473</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain (all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>526</th>\n",
       "      <td>http://edamontology.org/data_1474</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain (C-alpha atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>527</th>\n",
       "      <td>http://edamontology.org/data_1479</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>528</th>\n",
       "      <td>http://edamontology.org/data_1480</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (multiple)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>529</th>\n",
       "      <td>http://edamontology.org/data_1481</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>530</th>\n",
       "      <td>http://edamontology.org/data_1482</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>531</th>\n",
       "      <td>http://edamontology.org/data_1483</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (protein pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>532</th>\n",
       "      <td>http://edamontology.org/data_1484</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple protein tertiary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>533</th>\n",
       "      <td>http://edamontology.org/data_1485</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (protein all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>http://edamontology.org/data_1486</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (protein C-alpha atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>535</th>\n",
       "      <td>http://edamontology.org/data_1487</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise protein tertiary structure alignment (all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>536</th>\n",
       "      <td>http://edamontology.org/data_1488</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise protein tertiary structure alignment (C-alpha atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>537</th>\n",
       "      <td>http://edamontology.org/data_1489</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple protein tertiary structure alignment (all atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>538</th>\n",
       "      <td>http://edamontology.org/data_1490</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple protein tertiary structure alignment (C-alpha atoms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>539</th>\n",
       "      <td>http://edamontology.org/data_1491</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (nucleic acid pair)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>http://edamontology.org/data_1492</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple nucleic acid tertiary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>541</th>\n",
       "      <td>http://edamontology.org/data_1493</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>542</th>\n",
       "      <td>http://edamontology.org/data_1494</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural transformation matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>543</th>\n",
       "      <td>http://edamontology.org/data_1495</td>\n",
       "      <td>18</td>\n",
       "      <td>DaliLite hit table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>http://edamontology.org/data_1496</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular similarity score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>545</th>\n",
       "      <td>http://edamontology.org/data_1497</td>\n",
       "      <td>18</td>\n",
       "      <td>Root-mean-square deviation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>546</th>\n",
       "      <td>http://edamontology.org/data_1498</td>\n",
       "      <td>18</td>\n",
       "      <td>Tanimoto similarity score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>547</th>\n",
       "      <td>http://edamontology.org/data_1499</td>\n",
       "      <td>18</td>\n",
       "      <td>3D-1D scoring matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>548</th>\n",
       "      <td>http://edamontology.org/data_1501</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>549</th>\n",
       "      <td>http://edamontology.org/data_1502</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index (chemical classes)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>550</th>\n",
       "      <td>http://edamontology.org/data_1503</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid pair-wise contact potentials</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>551</th>\n",
       "      <td>http://edamontology.org/data_1505</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index (molecular weight)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>http://edamontology.org/data_1506</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index (hydropathy)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>http://edamontology.org/data_1507</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index (White-Wimley data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>http://edamontology.org/data_1508</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index (van der Waals radii)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>http://edamontology.org/data_1509</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>http://edamontology.org/data_1517</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction enzyme report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>557</th>\n",
       "      <td>http://edamontology.org/data_1519</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide molecular weights</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>558</th>\n",
       "      <td>http://edamontology.org/data_1520</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide hydrophobic moment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>559</th>\n",
       "      <td>http://edamontology.org/data_1521</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein aliphatic index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>560</th>\n",
       "      <td>http://edamontology.org/data_1522</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence hydropathy plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>561</th>\n",
       "      <td>http://edamontology.org/data_1523</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein charge plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>562</th>\n",
       "      <td>http://edamontology.org/data_1524</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein solubility</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>http://edamontology.org/data_1525</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein crystallizability</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>564</th>\n",
       "      <td>http://edamontology.org/data_1526</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein globularity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>565</th>\n",
       "      <td>http://edamontology.org/data_1527</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein titration curve</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>566</th>\n",
       "      <td>http://edamontology.org/data_1528</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein isoelectric point</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>567</th>\n",
       "      <td>http://edamontology.org/data_1529</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein pKa value</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>568</th>\n",
       "      <td>http://edamontology.org/data_1530</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydrogen exchange rate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>http://edamontology.org/data_1531</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein extinction coefficient</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>http://edamontology.org/data_1532</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein optical density</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>http://edamontology.org/data_1533</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein subcellular localisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>http://edamontology.org/data_1534</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide immunogenicity data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>http://edamontology.org/data_1536</td>\n",
       "      <td>18</td>\n",
       "      <td>MHC peptide immunogenicity report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>574</th>\n",
       "      <td>http://edamontology.org/data_1537</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>575</th>\n",
       "      <td>http://edamontology.org/data_1539</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structural quality report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>576</th>\n",
       "      <td>http://edamontology.org/data_1540</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein non-covalent interactions report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>577</th>\n",
       "      <td>http://edamontology.org/data_1541</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein flexibility or motion report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>http://edamontology.org/data_1542</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein solvent accessibility</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>579</th>\n",
       "      <td>http://edamontology.org/data_1543</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein surface report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>580</th>\n",
       "      <td>http://edamontology.org/data_1544</td>\n",
       "      <td>18</td>\n",
       "      <td>Ramachandran plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>581</th>\n",
       "      <td>http://edamontology.org/data_1545</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein dipole moment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>582</th>\n",
       "      <td>http://edamontology.org/data_1546</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein distance matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>583</th>\n",
       "      <td>http://edamontology.org/data_1547</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein contact map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>584</th>\n",
       "      <td>http://edamontology.org/data_1548</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue 3D cluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>http://edamontology.org/data_1549</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydrogen bonds</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>586</th>\n",
       "      <td>http://edamontology.org/data_1550</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein non-canonical interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>http://edamontology.org/data_1553</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH node</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>http://edamontology.org/data_1554</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP node</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>http://edamontology.org/data_1555</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBASSY domain classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>http://edamontology.org/data_1556</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH class</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>591</th>\n",
       "      <td>http://edamontology.org/data_1557</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH architecture</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>592</th>\n",
       "      <td>http://edamontology.org/data_1558</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH topology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>593</th>\n",
       "      <td>http://edamontology.org/data_1559</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH homologous superfamily</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>594</th>\n",
       "      <td>http://edamontology.org/data_1560</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH structurally similar group</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>595</th>\n",
       "      <td>http://edamontology.org/data_1561</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH functional category</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>596</th>\n",
       "      <td>http://edamontology.org/data_1564</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein fold recognition report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>597</th>\n",
       "      <td>http://edamontology.org/data_1565</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interaction report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>598</th>\n",
       "      <td>http://edamontology.org/data_1566</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-ligand interaction report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>599</th>\n",
       "      <td>http://edamontology.org/data_1567</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-nucleic acid interactions report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>600</th>\n",
       "      <td>http://edamontology.org/data_1583</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>601</th>\n",
       "      <td>http://edamontology.org/data_1584</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid enthalpy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>602</th>\n",
       "      <td>http://edamontology.org/data_1585</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid entropy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>603</th>\n",
       "      <td>http://edamontology.org/data_1586</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting temperature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>http://edamontology.org/data_1587</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid stitch profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>http://edamontology.org/data_1588</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA base pair stacking energies data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>http://edamontology.org/data_1589</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA base pair twist angle data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>http://edamontology.org/data_1590</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA base trimer roll angles data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>http://edamontology.org/data_1591</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna RNA parameters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>609</th>\n",
       "      <td>http://edamontology.org/data_1592</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna RNA structure constraints</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>610</th>\n",
       "      <td>http://edamontology.org/data_1593</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna RNA concentration data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>611</th>\n",
       "      <td>http://edamontology.org/data_1594</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna RNA calculated energy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>612</th>\n",
       "      <td>http://edamontology.org/data_1595</td>\n",
       "      <td>18</td>\n",
       "      <td>Base pairing probability matrix dotplot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>613</th>\n",
       "      <td>http://edamontology.org/data_1596</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>614</th>\n",
       "      <td>http://edamontology.org/data_1597</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>615</th>\n",
       "      <td>http://edamontology.org/data_1598</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>616</th>\n",
       "      <td>http://edamontology.org/data_1599</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon adaptation index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>617</th>\n",
       "      <td>http://edamontology.org/data_1600</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage bias plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>618</th>\n",
       "      <td>http://edamontology.org/data_1601</td>\n",
       "      <td>18</td>\n",
       "      <td>Nc statistic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>619</th>\n",
       "      <td>http://edamontology.org/data_1602</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage fraction difference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>620</th>\n",
       "      <td>http://edamontology.org/data_1621</td>\n",
       "      <td>18</td>\n",
       "      <td>Pharmacogenomic test report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>621</th>\n",
       "      <td>http://edamontology.org/data_1622</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>622</th>\n",
       "      <td>http://edamontology.org/data_1634</td>\n",
       "      <td>18</td>\n",
       "      <td>Linkage disequilibrium (report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>623</th>\n",
       "      <td>http://edamontology.org/data_1636</td>\n",
       "      <td>18</td>\n",
       "      <td>Heat map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>624</th>\n",
       "      <td>http://edamontology.org/data_1642</td>\n",
       "      <td>18</td>\n",
       "      <td>Affymetrix probe sets library file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>625</th>\n",
       "      <td>http://edamontology.org/data_1643</td>\n",
       "      <td>18</td>\n",
       "      <td>Affymetrix probe sets information library file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>626</th>\n",
       "      <td>http://edamontology.org/data_1646</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular weights standard fingerprint</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>627</th>\n",
       "      <td>http://edamontology.org/data_1656</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic pathway report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>628</th>\n",
       "      <td>http://edamontology.org/data_1657</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic information processing pathway report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>629</th>\n",
       "      <td>http://edamontology.org/data_1658</td>\n",
       "      <td>18</td>\n",
       "      <td>Environmental information processing pathway report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>630</th>\n",
       "      <td>http://edamontology.org/data_1659</td>\n",
       "      <td>18</td>\n",
       "      <td>Signal transduction pathway report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>631</th>\n",
       "      <td>http://edamontology.org/data_1660</td>\n",
       "      <td>18</td>\n",
       "      <td>Cellular process pathways report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>632</th>\n",
       "      <td>http://edamontology.org/data_1661</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease pathway or network report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>633</th>\n",
       "      <td>http://edamontology.org/data_1662</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug structure relationship map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>634</th>\n",
       "      <td>http://edamontology.org/data_1663</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>635</th>\n",
       "      <td>http://edamontology.org/data_1664</td>\n",
       "      <td>18</td>\n",
       "      <td>MIRIAM datatype</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>636</th>\n",
       "      <td>http://edamontology.org/data_1667</td>\n",
       "      <td>18</td>\n",
       "      <td>E-value</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>637</th>\n",
       "      <td>http://edamontology.org/data_1668</td>\n",
       "      <td>18</td>\n",
       "      <td>Z-value</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>638</th>\n",
       "      <td>http://edamontology.org/data_1669</td>\n",
       "      <td>18</td>\n",
       "      <td>P-value</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>639</th>\n",
       "      <td>http://edamontology.org/data_1670</td>\n",
       "      <td>18</td>\n",
       "      <td>Database version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>640</th>\n",
       "      <td>http://edamontology.org/data_1671</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>641</th>\n",
       "      <td>http://edamontology.org/data_1672</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>642</th>\n",
       "      <td>http://edamontology.org/data_1673</td>\n",
       "      <td>18</td>\n",
       "      <td>Swiss-Prot to PDB mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>643</th>\n",
       "      <td>http://edamontology.org/data_1674</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database cross-references</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>644</th>\n",
       "      <td>http://edamontology.org/data_1675</td>\n",
       "      <td>18</td>\n",
       "      <td>Job status</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>645</th>\n",
       "      <td>http://edamontology.org/data_1676</td>\n",
       "      <td>18</td>\n",
       "      <td>Job ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>646</th>\n",
       "      <td>http://edamontology.org/data_1677</td>\n",
       "      <td>18</td>\n",
       "      <td>Job type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>647</th>\n",
       "      <td>http://edamontology.org/data_1678</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool log</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>648</th>\n",
       "      <td>http://edamontology.org/data_1679</td>\n",
       "      <td>18</td>\n",
       "      <td>DaliLite log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>649</th>\n",
       "      <td>http://edamontology.org/data_1680</td>\n",
       "      <td>18</td>\n",
       "      <td>STRIDE log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>650</th>\n",
       "      <td>http://edamontology.org/data_1681</td>\n",
       "      <td>18</td>\n",
       "      <td>NACCESS log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>651</th>\n",
       "      <td>http://edamontology.org/data_1682</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS wordfinder log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>652</th>\n",
       "      <td>http://edamontology.org/data_1683</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS domainatrix log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>653</th>\n",
       "      <td>http://edamontology.org/data_1684</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS sites log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>654</th>\n",
       "      <td>http://edamontology.org/data_1685</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS supermatcher error file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>655</th>\n",
       "      <td>http://edamontology.org/data_1686</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS megamerger log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>656</th>\n",
       "      <td>http://edamontology.org/data_1687</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS whichdb log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>657</th>\n",
       "      <td>http://edamontology.org/data_1688</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS vectorstrip log file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>658</th>\n",
       "      <td>http://edamontology.org/data_1689</td>\n",
       "      <td>18</td>\n",
       "      <td>Username</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>659</th>\n",
       "      <td>http://edamontology.org/data_1690</td>\n",
       "      <td>18</td>\n",
       "      <td>Password</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>http://edamontology.org/data_1691</td>\n",
       "      <td>18</td>\n",
       "      <td>Email address</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>661</th>\n",
       "      <td>http://edamontology.org/data_1692</td>\n",
       "      <td>18</td>\n",
       "      <td>Person name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>662</th>\n",
       "      <td>http://edamontology.org/data_1693</td>\n",
       "      <td>18</td>\n",
       "      <td>Number of iterations</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>http://edamontology.org/data_1694</td>\n",
       "      <td>18</td>\n",
       "      <td>Number of output entities</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>http://edamontology.org/data_1695</td>\n",
       "      <td>18</td>\n",
       "      <td>Hit sort order</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>http://edamontology.org/data_1696</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>http://edamontology.org/data_1707</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>http://edamontology.org/data_1708</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>668</th>\n",
       "      <td>http://edamontology.org/data_1709</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>669</th>\n",
       "      <td>http://edamontology.org/data_1710</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>670</th>\n",
       "      <td>http://edamontology.org/data_1711</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>671</th>\n",
       "      <td>http://edamontology.org/data_1712</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>672</th>\n",
       "      <td>http://edamontology.org/data_1713</td>\n",
       "      <td>18</td>\n",
       "      <td>Fate map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>673</th>\n",
       "      <td>http://edamontology.org/data_1714</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray spots image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>674</th>\n",
       "      <td>http://edamontology.org/data_1715</td>\n",
       "      <td>18</td>\n",
       "      <td>BioPax term</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>675</th>\n",
       "      <td>http://edamontology.org/data_1716</td>\n",
       "      <td>18</td>\n",
       "      <td>GO</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>676</th>\n",
       "      <td>http://edamontology.org/data_1717</td>\n",
       "      <td>18</td>\n",
       "      <td>MeSH</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>677</th>\n",
       "      <td>http://edamontology.org/data_1718</td>\n",
       "      <td>18</td>\n",
       "      <td>HGNC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>678</th>\n",
       "      <td>http://edamontology.org/data_1719</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI taxonomy vocabulary</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>679</th>\n",
       "      <td>http://edamontology.org/data_1720</td>\n",
       "      <td>18</td>\n",
       "      <td>Plant ontology term</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>680</th>\n",
       "      <td>http://edamontology.org/data_1721</td>\n",
       "      <td>18</td>\n",
       "      <td>UMLS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>681</th>\n",
       "      <td>http://edamontology.org/data_1722</td>\n",
       "      <td>18</td>\n",
       "      <td>FMA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>682</th>\n",
       "      <td>http://edamontology.org/data_1723</td>\n",
       "      <td>18</td>\n",
       "      <td>EMAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>683</th>\n",
       "      <td>http://edamontology.org/data_1724</td>\n",
       "      <td>18</td>\n",
       "      <td>ChEBI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>684</th>\n",
       "      <td>http://edamontology.org/data_1725</td>\n",
       "      <td>18</td>\n",
       "      <td>MGED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>685</th>\n",
       "      <td>http://edamontology.org/data_1726</td>\n",
       "      <td>18</td>\n",
       "      <td>myGrid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>686</th>\n",
       "      <td>http://edamontology.org/data_1727</td>\n",
       "      <td>18</td>\n",
       "      <td>GO (biological process)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>687</th>\n",
       "      <td>http://edamontology.org/data_1728</td>\n",
       "      <td>18</td>\n",
       "      <td>GO (molecular function)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>688</th>\n",
       "      <td>http://edamontology.org/data_1729</td>\n",
       "      <td>18</td>\n",
       "      <td>GO (cellular component)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>689</th>\n",
       "      <td>http://edamontology.org/data_1730</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology relation type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>690</th>\n",
       "      <td>http://edamontology.org/data_1731</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept definition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>691</th>\n",
       "      <td>http://edamontology.org/data_1732</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept comment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>692</th>\n",
       "      <td>http://edamontology.org/data_1733</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept reference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>693</th>\n",
       "      <td>http://edamontology.org/data_1738</td>\n",
       "      <td>18</td>\n",
       "      <td>doc2loc document information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>694</th>\n",
       "      <td>http://edamontology.org/data_1742</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB residue number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>http://edamontology.org/data_1743</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic coordinate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>http://edamontology.org/data_1744</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic x coordinate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>http://edamontology.org/data_1745</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic y coordinate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>http://edamontology.org/data_1746</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic z coordinate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>http://edamontology.org/data_1748</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB atom name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>700</th>\n",
       "      <td>http://edamontology.org/data_1755</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein atom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>701</th>\n",
       "      <td>http://edamontology.org/data_1756</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>702</th>\n",
       "      <td>http://edamontology.org/data_1757</td>\n",
       "      <td>18</td>\n",
       "      <td>Atom name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>703</th>\n",
       "      <td>http://edamontology.org/data_1758</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB residue name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>704</th>\n",
       "      <td>http://edamontology.org/data_1759</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB model number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>705</th>\n",
       "      <td>http://edamontology.org/data_1762</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH domain report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>706</th>\n",
       "      <td>http://edamontology.org/data_1764</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH representative domain sequences (ATOM)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>707</th>\n",
       "      <td>http://edamontology.org/data_1765</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH representative domain sequences (COMBS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>708</th>\n",
       "      <td>http://edamontology.org/data_1766</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH domain sequences (ATOM)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>709</th>\n",
       "      <td>http://edamontology.org/data_1767</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH domain sequences (COMBS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>710</th>\n",
       "      <td>http://edamontology.org/data_1771</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence version</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>711</th>\n",
       "      <td>http://edamontology.org/data_1772</td>\n",
       "      <td>18</td>\n",
       "      <td>Score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>712</th>\n",
       "      <td>http://edamontology.org/data_1776</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report (function)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>713</th>\n",
       "      <td>http://edamontology.org/data_1783</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (ASPGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>714</th>\n",
       "      <td>http://edamontology.org/data_1784</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (CGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>715</th>\n",
       "      <td>http://edamontology.org/data_1785</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (dictyBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>716</th>\n",
       "      <td>http://edamontology.org/data_1786</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (EcoGene primary)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>717</th>\n",
       "      <td>http://edamontology.org/data_1787</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (MaizeGDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>718</th>\n",
       "      <td>http://edamontology.org/data_1788</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (SGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>719</th>\n",
       "      <td>http://edamontology.org/data_1789</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (TGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>720</th>\n",
       "      <td>http://edamontology.org/data_1790</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (CGSC)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>http://edamontology.org/data_1791</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (HGNC)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>722</th>\n",
       "      <td>http://edamontology.org/data_1792</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (MGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>723</th>\n",
       "      <td>http://edamontology.org/data_1793</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (Bacillus subtilis)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>724</th>\n",
       "      <td>http://edamontology.org/data_1794</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (PlasmoDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>725</th>\n",
       "      <td>http://edamontology.org/data_1795</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (EcoGene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>726</th>\n",
       "      <td>http://edamontology.org/data_1796</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (FlyBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>727</th>\n",
       "      <td>http://edamontology.org/data_1797</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB Glossina morsitans)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>728</th>\n",
       "      <td>http://edamontology.org/data_1798</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB Leishmania major)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>729</th>\n",
       "      <td>http://edamontology.org/data_1799</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB Plasmodium falciparum)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>730</th>\n",
       "      <td>http://edamontology.org/data_1800</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB Schizosaccharomyces pombe)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>http://edamontology.org/data_1801</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneDB Trypanosoma brucei)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>732</th>\n",
       "      <td>http://edamontology.org/data_1802</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (Gramene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>733</th>\n",
       "      <td>http://edamontology.org/data_1803</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (Virginia microbial)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>734</th>\n",
       "      <td>http://edamontology.org/data_1804</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (SGN)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>http://edamontology.org/data_1805</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (WormBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>http://edamontology.org/data_1806</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene synonym</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>http://edamontology.org/data_1807</td>\n",
       "      <td>18</td>\n",
       "      <td>ORF name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>http://edamontology.org/data_1852</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly component</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>http://edamontology.org/data_1853</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosome annotation (aberration)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>740</th>\n",
       "      <td>http://edamontology.org/data_1855</td>\n",
       "      <td>18</td>\n",
       "      <td>Clone ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>741</th>\n",
       "      <td>http://edamontology.org/data_1856</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB insertion code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>742</th>\n",
       "      <td>http://edamontology.org/data_1857</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic occupancy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>743</th>\n",
       "      <td>http://edamontology.org/data_1858</td>\n",
       "      <td>18</td>\n",
       "      <td>Isotropic B factor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>744</th>\n",
       "      <td>http://edamontology.org/data_1859</td>\n",
       "      <td>18</td>\n",
       "      <td>Deletion map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>745</th>\n",
       "      <td>http://edamontology.org/data_1860</td>\n",
       "      <td>18</td>\n",
       "      <td>QTL map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>746</th>\n",
       "      <td>http://edamontology.org/data_1863</td>\n",
       "      <td>18</td>\n",
       "      <td>Haplotype map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>http://edamontology.org/data_1864</td>\n",
       "      <td>18</td>\n",
       "      <td>Map set data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>748</th>\n",
       "      <td>http://edamontology.org/data_1865</td>\n",
       "      <td>18</td>\n",
       "      <td>Map feature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>749</th>\n",
       "      <td>http://edamontology.org/data_1866</td>\n",
       "      <td>18</td>\n",
       "      <td>Map type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>750</th>\n",
       "      <td>http://edamontology.org/data_1867</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein fold name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>http://edamontology.org/data_1868</td>\n",
       "      <td>18</td>\n",
       "      <td>Taxon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>752</th>\n",
       "      <td>http://edamontology.org/data_1869</td>\n",
       "      <td>18</td>\n",
       "      <td>Organism identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>753</th>\n",
       "      <td>http://edamontology.org/data_1870</td>\n",
       "      <td>18</td>\n",
       "      <td>Genus name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>754</th>\n",
       "      <td>http://edamontology.org/data_1872</td>\n",
       "      <td>18</td>\n",
       "      <td>Taxonomic classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>http://edamontology.org/data_1873</td>\n",
       "      <td>18</td>\n",
       "      <td>iHOP organism ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>756</th>\n",
       "      <td>http://edamontology.org/data_1874</td>\n",
       "      <td>18</td>\n",
       "      <td>Genbank common name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>757</th>\n",
       "      <td>http://edamontology.org/data_1875</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI taxon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>758</th>\n",
       "      <td>http://edamontology.org/data_1877</td>\n",
       "      <td>18</td>\n",
       "      <td>Synonym</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>759</th>\n",
       "      <td>http://edamontology.org/data_1878</td>\n",
       "      <td>18</td>\n",
       "      <td>Misspelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>760</th>\n",
       "      <td>http://edamontology.org/data_1879</td>\n",
       "      <td>18</td>\n",
       "      <td>Acronym</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>761</th>\n",
       "      <td>http://edamontology.org/data_1880</td>\n",
       "      <td>18</td>\n",
       "      <td>Misnomer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>762</th>\n",
       "      <td>http://edamontology.org/data_1881</td>\n",
       "      <td>18</td>\n",
       "      <td>Author ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>763</th>\n",
       "      <td>http://edamontology.org/data_1882</td>\n",
       "      <td>18</td>\n",
       "      <td>DragonDB author identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>http://edamontology.org/data_1883</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotated URI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>http://edamontology.org/data_1884</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt keywords</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>http://edamontology.org/data_1885</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (GeneFarm)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>http://edamontology.org/data_1886</td>\n",
       "      <td>18</td>\n",
       "      <td>Blattner number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>http://edamontology.org/data_1887</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (MIPS Maize)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>769</th>\n",
       "      <td>http://edamontology.org/data_1888</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (MIPS Medicago)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>770</th>\n",
       "      <td>http://edamontology.org/data_1889</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (DragonDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>771</th>\n",
       "      <td>http://edamontology.org/data_1890</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (Arabidopsis)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>http://edamontology.org/data_1891</td>\n",
       "      <td>18</td>\n",
       "      <td>iHOP symbol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>773</th>\n",
       "      <td>http://edamontology.org/data_1892</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (GeneFarm)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>774</th>\n",
       "      <td>http://edamontology.org/data_1893</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>775</th>\n",
       "      <td>http://edamontology.org/data_1895</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (AGI)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>776</th>\n",
       "      <td>http://edamontology.org/data_1896</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (ASPGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>777</th>\n",
       "      <td>http://edamontology.org/data_1897</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (MGG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>778</th>\n",
       "      <td>http://edamontology.org/data_1898</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (CGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>779</th>\n",
       "      <td>http://edamontology.org/data_1899</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (CMR)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>780</th>\n",
       "      <td>http://edamontology.org/data_1900</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI locus tag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>781</th>\n",
       "      <td>http://edamontology.org/data_1901</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (SGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>782</th>\n",
       "      <td>http://edamontology.org/data_1902</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (MMP)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>783</th>\n",
       "      <td>http://edamontology.org/data_1903</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (DictyBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>784</th>\n",
       "      <td>http://edamontology.org/data_1904</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (EntrezGene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>785</th>\n",
       "      <td>http://edamontology.org/data_1905</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (MaizeGDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>786</th>\n",
       "      <td>http://edamontology.org/data_1906</td>\n",
       "      <td>18</td>\n",
       "      <td>Quantitative trait locus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>787</th>\n",
       "      <td>http://edamontology.org/data_1907</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (KOME)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>788</th>\n",
       "      <td>http://edamontology.org/data_1908</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (Tropgene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>789</th>\n",
       "      <td>http://edamontology.org/data_1916</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>790</th>\n",
       "      <td>http://edamontology.org/data_1917</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>791</th>\n",
       "      <td>http://edamontology.org/data_2007</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt keyword</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>792</th>\n",
       "      <td>http://edamontology.org/data_2009</td>\n",
       "      <td>18</td>\n",
       "      <td>Ordered locus name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>793</th>\n",
       "      <td>http://edamontology.org/data_2012</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence coordinates</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>794</th>\n",
       "      <td>http://edamontology.org/data_2016</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>795</th>\n",
       "      <td>http://edamontology.org/data_2018</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>796</th>\n",
       "      <td>http://edamontology.org/data_2019</td>\n",
       "      <td>18</td>\n",
       "      <td>Map data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>797</th>\n",
       "      <td>http://edamontology.org/data_2022</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna RNA structural data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>798</th>\n",
       "      <td>http://edamontology.org/data_2023</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence mask parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>799</th>\n",
       "      <td>http://edamontology.org/data_2024</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme kinetics data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>800</th>\n",
       "      <td>http://edamontology.org/data_2025</td>\n",
       "      <td>18</td>\n",
       "      <td>Michaelis Menten plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>801</th>\n",
       "      <td>http://edamontology.org/data_2026</td>\n",
       "      <td>18</td>\n",
       "      <td>Hanes Woolf plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>802</th>\n",
       "      <td>http://edamontology.org/data_2028</td>\n",
       "      <td>18</td>\n",
       "      <td>Experimental data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>803</th>\n",
       "      <td>http://edamontology.org/data_2041</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>804</th>\n",
       "      <td>http://edamontology.org/data_2042</td>\n",
       "      <td>18</td>\n",
       "      <td>Evidence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>805</th>\n",
       "      <td>http://edamontology.org/data_2043</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record lite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>806</th>\n",
       "      <td>http://edamontology.org/data_2044</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>807</th>\n",
       "      <td>http://edamontology.org/data_2046</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence record (lite)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>808</th>\n",
       "      <td>http://edamontology.org/data_2047</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence record (lite)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>809</th>\n",
       "      <td>http://edamontology.org/data_2048</td>\n",
       "      <td>18</td>\n",
       "      <td>Report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>810</th>\n",
       "      <td>http://edamontology.org/data_2050</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular property (general)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>811</th>\n",
       "      <td>http://edamontology.org/data_2053</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>812</th>\n",
       "      <td>http://edamontology.org/data_2070</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>813</th>\n",
       "      <td>http://edamontology.org/data_2071</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>814</th>\n",
       "      <td>http://edamontology.org/data_2079</td>\n",
       "      <td>18</td>\n",
       "      <td>Search parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>815</th>\n",
       "      <td>http://edamontology.org/data_2080</td>\n",
       "      <td>18</td>\n",
       "      <td>Database search results</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>816</th>\n",
       "      <td>http://edamontology.org/data_2081</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>817</th>\n",
       "      <td>http://edamontology.org/data_2082</td>\n",
       "      <td>18</td>\n",
       "      <td>Matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>818</th>\n",
       "      <td>http://edamontology.org/data_2083</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>819</th>\n",
       "      <td>http://edamontology.org/data_2084</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>820</th>\n",
       "      <td>http://edamontology.org/data_2085</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>821</th>\n",
       "      <td>http://edamontology.org/data_2086</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>822</th>\n",
       "      <td>http://edamontology.org/data_2087</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>823</th>\n",
       "      <td>http://edamontology.org/data_2088</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA base structural data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>824</th>\n",
       "      <td>http://edamontology.org/data_2090</td>\n",
       "      <td>18</td>\n",
       "      <td>Database entry version information</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>825</th>\n",
       "      <td>http://edamontology.org/data_2091</td>\n",
       "      <td>18</td>\n",
       "      <td>Accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>826</th>\n",
       "      <td>http://edamontology.org/data_2092</td>\n",
       "      <td>18</td>\n",
       "      <td>SNP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>http://edamontology.org/data_2093</td>\n",
       "      <td>18</td>\n",
       "      <td>Data reference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>828</th>\n",
       "      <td>http://edamontology.org/data_2098</td>\n",
       "      <td>18</td>\n",
       "      <td>Job identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>829</th>\n",
       "      <td>http://edamontology.org/data_2099</td>\n",
       "      <td>18</td>\n",
       "      <td>Name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>830</th>\n",
       "      <td>http://edamontology.org/data_2100</td>\n",
       "      <td>18</td>\n",
       "      <td>Type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>831</th>\n",
       "      <td>http://edamontology.org/data_2101</td>\n",
       "      <td>18</td>\n",
       "      <td>Account authentication</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>832</th>\n",
       "      <td>http://edamontology.org/data_2102</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG organism code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>833</th>\n",
       "      <td>http://edamontology.org/data_2103</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (KEGG GENES)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>834</th>\n",
       "      <td>http://edamontology.org/data_2104</td>\n",
       "      <td>18</td>\n",
       "      <td>BioCyc ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>835</th>\n",
       "      <td>http://edamontology.org/data_2105</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (BioCyc)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>836</th>\n",
       "      <td>http://edamontology.org/data_2106</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID (BioCyc)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>837</th>\n",
       "      <td>http://edamontology.org/data_2107</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme ID (BioCyc)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>838</th>\n",
       "      <td>http://edamontology.org/data_2108</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>839</th>\n",
       "      <td>http://edamontology.org/data_2109</td>\n",
       "      <td>18</td>\n",
       "      <td>Identifier (hybrid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>840</th>\n",
       "      <td>http://edamontology.org/data_2110</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular property identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>841</th>\n",
       "      <td>http://edamontology.org/data_2111</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>842</th>\n",
       "      <td>http://edamontology.org/data_2112</td>\n",
       "      <td>18</td>\n",
       "      <td>FlyBase primary identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>843</th>\n",
       "      <td>http://edamontology.org/data_2113</td>\n",
       "      <td>18</td>\n",
       "      <td>WormBase identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>844</th>\n",
       "      <td>http://edamontology.org/data_2114</td>\n",
       "      <td>18</td>\n",
       "      <td>WormBase wormpep ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>845</th>\n",
       "      <td>http://edamontology.org/data_2116</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (codon)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>846</th>\n",
       "      <td>http://edamontology.org/data_2117</td>\n",
       "      <td>18</td>\n",
       "      <td>Map identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>847</th>\n",
       "      <td>http://edamontology.org/data_2118</td>\n",
       "      <td>18</td>\n",
       "      <td>Person identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>848</th>\n",
       "      <td>http://edamontology.org/data_2119</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>849</th>\n",
       "      <td>http://edamontology.org/data_2126</td>\n",
       "      <td>18</td>\n",
       "      <td>Translation frame specification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>850</th>\n",
       "      <td>http://edamontology.org/data_2127</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic code identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>851</th>\n",
       "      <td>http://edamontology.org/data_2128</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic code name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>852</th>\n",
       "      <td>http://edamontology.org/data_2129</td>\n",
       "      <td>18</td>\n",
       "      <td>File format name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>853</th>\n",
       "      <td>http://edamontology.org/data_2130</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>854</th>\n",
       "      <td>http://edamontology.org/data_2131</td>\n",
       "      <td>18</td>\n",
       "      <td>Operating system name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>855</th>\n",
       "      <td>http://edamontology.org/data_2132</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>856</th>\n",
       "      <td>http://edamontology.org/data_2133</td>\n",
       "      <td>18</td>\n",
       "      <td>Logical operator</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>857</th>\n",
       "      <td>http://edamontology.org/data_2134</td>\n",
       "      <td>18</td>\n",
       "      <td>Results sort order</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>858</th>\n",
       "      <td>http://edamontology.org/data_2135</td>\n",
       "      <td>18</td>\n",
       "      <td>Toggle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>859</th>\n",
       "      <td>http://edamontology.org/data_2136</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence width</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>860</th>\n",
       "      <td>http://edamontology.org/data_2137</td>\n",
       "      <td>18</td>\n",
       "      <td>Gap penalty</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>861</th>\n",
       "      <td>http://edamontology.org/data_2139</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting temperature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>862</th>\n",
       "      <td>http://edamontology.org/data_2140</td>\n",
       "      <td>18</td>\n",
       "      <td>Concentration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>863</th>\n",
       "      <td>http://edamontology.org/data_2141</td>\n",
       "      <td>18</td>\n",
       "      <td>Window step size</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>864</th>\n",
       "      <td>http://edamontology.org/data_2142</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS graph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>865</th>\n",
       "      <td>http://edamontology.org/data_2143</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>866</th>\n",
       "      <td>http://edamontology.org/data_2145</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence offset</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>867</th>\n",
       "      <td>http://edamontology.org/data_2146</td>\n",
       "      <td>18</td>\n",
       "      <td>Threshold</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>868</th>\n",
       "      <td>http://edamontology.org/data_2147</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report (transcription factor)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>869</th>\n",
       "      <td>http://edamontology.org/data_2149</td>\n",
       "      <td>18</td>\n",
       "      <td>Database category name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>870</th>\n",
       "      <td>http://edamontology.org/data_2150</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>871</th>\n",
       "      <td>http://edamontology.org/data_2151</td>\n",
       "      <td>18</td>\n",
       "      <td>Color</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>872</th>\n",
       "      <td>http://edamontology.org/data_2152</td>\n",
       "      <td>18</td>\n",
       "      <td>Rendering parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>873</th>\n",
       "      <td>http://edamontology.org/data_2154</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>874</th>\n",
       "      <td>http://edamontology.org/data_2156</td>\n",
       "      <td>18</td>\n",
       "      <td>Date</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>875</th>\n",
       "      <td>http://edamontology.org/data_2157</td>\n",
       "      <td>18</td>\n",
       "      <td>Word composition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>876</th>\n",
       "      <td>http://edamontology.org/data_2160</td>\n",
       "      <td>18</td>\n",
       "      <td>Fickett testcode plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>877</th>\n",
       "      <td>http://edamontology.org/data_2161</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence similarity plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>878</th>\n",
       "      <td>http://edamontology.org/data_2162</td>\n",
       "      <td>18</td>\n",
       "      <td>Helical wheel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>879</th>\n",
       "      <td>http://edamontology.org/data_2163</td>\n",
       "      <td>18</td>\n",
       "      <td>Helical net</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>880</th>\n",
       "      <td>http://edamontology.org/data_2164</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence properties plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>881</th>\n",
       "      <td>http://edamontology.org/data_2165</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ionisation curve</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>882</th>\n",
       "      <td>http://edamontology.org/data_2166</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>883</th>\n",
       "      <td>http://edamontology.org/data_2167</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid density plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>884</th>\n",
       "      <td>http://edamontology.org/data_2168</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence trace image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>885</th>\n",
       "      <td>http://edamontology.org/data_2169</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (siRNA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>886</th>\n",
       "      <td>http://edamontology.org/data_2173</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set (stream)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>887</th>\n",
       "      <td>http://edamontology.org/data_2174</td>\n",
       "      <td>18</td>\n",
       "      <td>FlyBase secondary identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>888</th>\n",
       "      <td>http://edamontology.org/data_2176</td>\n",
       "      <td>18</td>\n",
       "      <td>Cardinality</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>889</th>\n",
       "      <td>http://edamontology.org/data_2177</td>\n",
       "      <td>18</td>\n",
       "      <td>Exactly 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>890</th>\n",
       "      <td>http://edamontology.org/data_2178</td>\n",
       "      <td>18</td>\n",
       "      <td>1 or more</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>891</th>\n",
       "      <td>http://edamontology.org/data_2179</td>\n",
       "      <td>18</td>\n",
       "      <td>Exactly 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>892</th>\n",
       "      <td>http://edamontology.org/data_2180</td>\n",
       "      <td>18</td>\n",
       "      <td>2 or more</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>893</th>\n",
       "      <td>http://edamontology.org/data_2190</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence checksum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>894</th>\n",
       "      <td>http://edamontology.org/data_2191</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (chemical modifications)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>895</th>\n",
       "      <td>http://edamontology.org/data_2192</td>\n",
       "      <td>18</td>\n",
       "      <td>Error</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>896</th>\n",
       "      <td>http://edamontology.org/data_2193</td>\n",
       "      <td>18</td>\n",
       "      <td>Database entry metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>897</th>\n",
       "      <td>http://edamontology.org/data_2198</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene cluster</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>898</th>\n",
       "      <td>http://edamontology.org/data_2201</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record full</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>899</th>\n",
       "      <td>http://edamontology.org/data_2208</td>\n",
       "      <td>18</td>\n",
       "      <td>Plasmid identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>900</th>\n",
       "      <td>http://edamontology.org/data_2209</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>901</th>\n",
       "      <td>http://edamontology.org/data_2212</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation annotation (basic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>902</th>\n",
       "      <td>http://edamontology.org/data_2213</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation annotation (prevalence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>903</th>\n",
       "      <td>http://edamontology.org/data_2214</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation annotation (prognostic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>904</th>\n",
       "      <td>http://edamontology.org/data_2215</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation annotation (functional)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>905</th>\n",
       "      <td>http://edamontology.org/data_2216</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>906</th>\n",
       "      <td>http://edamontology.org/data_2217</td>\n",
       "      <td>18</td>\n",
       "      <td>Tumor annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>907</th>\n",
       "      <td>http://edamontology.org/data_2218</td>\n",
       "      <td>18</td>\n",
       "      <td>Server metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>908</th>\n",
       "      <td>http://edamontology.org/data_2219</td>\n",
       "      <td>18</td>\n",
       "      <td>Database field name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>909</th>\n",
       "      <td>http://edamontology.org/data_2220</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (SYSTERS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>910</th>\n",
       "      <td>http://edamontology.org/data_2223</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>911</th>\n",
       "      <td>http://edamontology.org/data_2235</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw SCOP domain classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>912</th>\n",
       "      <td>http://edamontology.org/data_2236</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw CATH domain classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>913</th>\n",
       "      <td>http://edamontology.org/data_2240</td>\n",
       "      <td>18</td>\n",
       "      <td>Heterogen annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>914</th>\n",
       "      <td>http://edamontology.org/data_2242</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic property values</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>915</th>\n",
       "      <td>http://edamontology.org/data_2245</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set (bootstrapped)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>916</th>\n",
       "      <td>http://edamontology.org/data_2247</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic consensus tree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>917</th>\n",
       "      <td>http://edamontology.org/data_2248</td>\n",
       "      <td>18</td>\n",
       "      <td>Schema</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>918</th>\n",
       "      <td>http://edamontology.org/data_2249</td>\n",
       "      <td>18</td>\n",
       "      <td>DTD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>919</th>\n",
       "      <td>http://edamontology.org/data_2250</td>\n",
       "      <td>18</td>\n",
       "      <td>XML Schema</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>920</th>\n",
       "      <td>http://edamontology.org/data_2251</td>\n",
       "      <td>18</td>\n",
       "      <td>Relax-NG schema</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>921</th>\n",
       "      <td>http://edamontology.org/data_2252</td>\n",
       "      <td>18</td>\n",
       "      <td>XSLT stylesheet</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>922</th>\n",
       "      <td>http://edamontology.org/data_2253</td>\n",
       "      <td>18</td>\n",
       "      <td>Data resource definition name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>923</th>\n",
       "      <td>http://edamontology.org/data_2254</td>\n",
       "      <td>18</td>\n",
       "      <td>OBO file format name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>924</th>\n",
       "      <td>http://edamontology.org/data_2285</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (MIPS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>925</th>\n",
       "      <td>http://edamontology.org/data_2288</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence identifier (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>926</th>\n",
       "      <td>http://edamontology.org/data_2289</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence identifier (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>927</th>\n",
       "      <td>http://edamontology.org/data_2290</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>928</th>\n",
       "      <td>http://edamontology.org/data_2291</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>929</th>\n",
       "      <td>http://edamontology.org/data_2292</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>930</th>\n",
       "      <td>http://edamontology.org/data_2293</td>\n",
       "      <td>18</td>\n",
       "      <td>Gramene secondary identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>931</th>\n",
       "      <td>http://edamontology.org/data_2294</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence variation ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>932</th>\n",
       "      <td>http://edamontology.org/data_2295</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>933</th>\n",
       "      <td>http://edamontology.org/data_2296</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (AceView)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>934</th>\n",
       "      <td>http://edamontology.org/data_2297</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (ECK)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>935</th>\n",
       "      <td>http://edamontology.org/data_2298</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (HGNC)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>936</th>\n",
       "      <td>http://edamontology.org/data_2299</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>937</th>\n",
       "      <td>http://edamontology.org/data_2300</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (NCBI)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>938</th>\n",
       "      <td>http://edamontology.org/data_2301</td>\n",
       "      <td>18</td>\n",
       "      <td>SMILES string</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>939</th>\n",
       "      <td>http://edamontology.org/data_2302</td>\n",
       "      <td>18</td>\n",
       "      <td>STRING ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>940</th>\n",
       "      <td>http://edamontology.org/data_2307</td>\n",
       "      <td>18</td>\n",
       "      <td>Virus annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>941</th>\n",
       "      <td>http://edamontology.org/data_2308</td>\n",
       "      <td>18</td>\n",
       "      <td>Virus annotation (taxonomy)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>942</th>\n",
       "      <td>http://edamontology.org/data_2309</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID (SABIO-RK)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>943</th>\n",
       "      <td>http://edamontology.org/data_2313</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>944</th>\n",
       "      <td>http://edamontology.org/data_2314</td>\n",
       "      <td>18</td>\n",
       "      <td>GI number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>945</th>\n",
       "      <td>http://edamontology.org/data_2315</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI version</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>946</th>\n",
       "      <td>http://edamontology.org/data_2316</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>947</th>\n",
       "      <td>http://edamontology.org/data_2317</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line name (exact)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>948</th>\n",
       "      <td>http://edamontology.org/data_2318</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line name (truncated)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>949</th>\n",
       "      <td>http://edamontology.org/data_2319</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line name (no punctuation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>950</th>\n",
       "      <td>http://edamontology.org/data_2320</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell line name (assonant)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>951</th>\n",
       "      <td>http://edamontology.org/data_2321</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>952</th>\n",
       "      <td>http://edamontology.org/data_2325</td>\n",
       "      <td>18</td>\n",
       "      <td>REBASE enzyme number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>953</th>\n",
       "      <td>http://edamontology.org/data_2326</td>\n",
       "      <td>18</td>\n",
       "      <td>DrugBank ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>954</th>\n",
       "      <td>http://edamontology.org/data_2327</td>\n",
       "      <td>18</td>\n",
       "      <td>GI number (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>955</th>\n",
       "      <td>http://edamontology.org/data_2335</td>\n",
       "      <td>18</td>\n",
       "      <td>Bit score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>956</th>\n",
       "      <td>http://edamontology.org/data_2336</td>\n",
       "      <td>18</td>\n",
       "      <td>Translation phase specification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>957</th>\n",
       "      <td>http://edamontology.org/data_2337</td>\n",
       "      <td>18</td>\n",
       "      <td>Resource metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>958</th>\n",
       "      <td>http://edamontology.org/data_2338</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>959</th>\n",
       "      <td>http://edamontology.org/data_2339</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>960</th>\n",
       "      <td>http://edamontology.org/data_2340</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome build identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>961</th>\n",
       "      <td>http://edamontology.org/data_2342</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>962</th>\n",
       "      <td>http://edamontology.org/data_2343</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (KEGG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>963</th>\n",
       "      <td>http://edamontology.org/data_2344</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (NCI-Nature)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>964</th>\n",
       "      <td>http://edamontology.org/data_2345</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (ConsensusPathDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>965</th>\n",
       "      <td>http://edamontology.org/data_2346</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (UniRef)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>http://edamontology.org/data_2347</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (UniRef100)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>http://edamontology.org/data_2348</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (UniRef90)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>http://edamontology.org/data_2349</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (UniRef50)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>http://edamontology.org/data_2353</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>http://edamontology.org/data_2354</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA family report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>971</th>\n",
       "      <td>http://edamontology.org/data_2355</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA family identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>972</th>\n",
       "      <td>http://edamontology.org/data_2356</td>\n",
       "      <td>18</td>\n",
       "      <td>RFAM accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>973</th>\n",
       "      <td>http://edamontology.org/data_2357</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signature type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>974</th>\n",
       "      <td>http://edamontology.org/data_2358</td>\n",
       "      <td>18</td>\n",
       "      <td>Domain-nucleic acid interaction report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>975</th>\n",
       "      <td>http://edamontology.org/data_2359</td>\n",
       "      <td>18</td>\n",
       "      <td>Domain-domain interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>976</th>\n",
       "      <td>http://edamontology.org/data_2360</td>\n",
       "      <td>18</td>\n",
       "      <td>Domain-domain interaction (indirect)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>977</th>\n",
       "      <td>http://edamontology.org/data_2362</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence accession (hybrid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>978</th>\n",
       "      <td>http://edamontology.org/data_2363</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>979</th>\n",
       "      <td>http://edamontology.org/data_2364</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>980</th>\n",
       "      <td>http://edamontology.org/data_2365</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>981</th>\n",
       "      <td>http://edamontology.org/data_2366</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>982</th>\n",
       "      <td>http://edamontology.org/data_2367</td>\n",
       "      <td>18</td>\n",
       "      <td>ASTD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>983</th>\n",
       "      <td>http://edamontology.org/data_2368</td>\n",
       "      <td>18</td>\n",
       "      <td>ASTD ID (exon)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>984</th>\n",
       "      <td>http://edamontology.org/data_2369</td>\n",
       "      <td>18</td>\n",
       "      <td>ASTD ID (intron)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>985</th>\n",
       "      <td>http://edamontology.org/data_2370</td>\n",
       "      <td>18</td>\n",
       "      <td>ASTD ID (polya)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>http://edamontology.org/data_2371</td>\n",
       "      <td>18</td>\n",
       "      <td>ASTD ID (tss)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>987</th>\n",
       "      <td>http://edamontology.org/data_2372</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE spot report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>988</th>\n",
       "      <td>http://edamontology.org/data_2373</td>\n",
       "      <td>18</td>\n",
       "      <td>Spot ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>989</th>\n",
       "      <td>http://edamontology.org/data_2374</td>\n",
       "      <td>18</td>\n",
       "      <td>Spot serial number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>990</th>\n",
       "      <td>http://edamontology.org/data_2375</td>\n",
       "      <td>18</td>\n",
       "      <td>Spot ID (HSC-2DPAGE)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>991</th>\n",
       "      <td>http://edamontology.org/data_2378</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-motif interaction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>992</th>\n",
       "      <td>http://edamontology.org/data_2379</td>\n",
       "      <td>18</td>\n",
       "      <td>Strain identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>993</th>\n",
       "      <td>http://edamontology.org/data_2380</td>\n",
       "      <td>18</td>\n",
       "      <td>CABRI accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>994</th>\n",
       "      <td>http://edamontology.org/data_2381</td>\n",
       "      <td>18</td>\n",
       "      <td>Experiment report (genotyping)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>http://edamontology.org/data_2382</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype experiment ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>http://edamontology.org/data_2383</td>\n",
       "      <td>18</td>\n",
       "      <td>EGA accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>http://edamontology.org/data_2384</td>\n",
       "      <td>18</td>\n",
       "      <td>IPI protein ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>http://edamontology.org/data_2385</td>\n",
       "      <td>18</td>\n",
       "      <td>RefSeq accession (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>http://edamontology.org/data_2386</td>\n",
       "      <td>18</td>\n",
       "      <td>EPD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1000</th>\n",
       "      <td>http://edamontology.org/data_2387</td>\n",
       "      <td>18</td>\n",
       "      <td>TAIR accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>http://edamontology.org/data_2388</td>\n",
       "      <td>18</td>\n",
       "      <td>TAIR accession (At gene)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1002</th>\n",
       "      <td>http://edamontology.org/data_2389</td>\n",
       "      <td>18</td>\n",
       "      <td>UniSTS accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1003</th>\n",
       "      <td>http://edamontology.org/data_2390</td>\n",
       "      <td>18</td>\n",
       "      <td>UNITE accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1004</th>\n",
       "      <td>http://edamontology.org/data_2391</td>\n",
       "      <td>18</td>\n",
       "      <td>UTR accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1005</th>\n",
       "      <td>http://edamontology.org/data_2392</td>\n",
       "      <td>18</td>\n",
       "      <td>UniParc accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1006</th>\n",
       "      <td>http://edamontology.org/data_2393</td>\n",
       "      <td>18</td>\n",
       "      <td>mFLJ/mKIAA number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1007</th>\n",
       "      <td>http://edamontology.org/data_2395</td>\n",
       "      <td>18</td>\n",
       "      <td>Fungi annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1008</th>\n",
       "      <td>http://edamontology.org/data_2396</td>\n",
       "      <td>18</td>\n",
       "      <td>Fungi annotation (anamorph)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1009</th>\n",
       "      <td>http://edamontology.org/data_2398</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl protein ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1010</th>\n",
       "      <td>http://edamontology.org/data_2400</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxin annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1011</th>\n",
       "      <td>http://edamontology.org/data_2401</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report (membrane protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1012</th>\n",
       "      <td>http://edamontology.org/data_2402</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-drug interaction report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1013</th>\n",
       "      <td>http://edamontology.org/data_2522</td>\n",
       "      <td>18</td>\n",
       "      <td>Map data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1014</th>\n",
       "      <td>http://edamontology.org/data_2523</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1015</th>\n",
       "      <td>http://edamontology.org/data_2524</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1016</th>\n",
       "      <td>http://edamontology.org/data_2525</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1017</th>\n",
       "      <td>http://edamontology.org/data_2526</td>\n",
       "      <td>18</td>\n",
       "      <td>Text data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1018</th>\n",
       "      <td>http://edamontology.org/data_2527</td>\n",
       "      <td>18</td>\n",
       "      <td>Parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1019</th>\n",
       "      <td>http://edamontology.org/data_2528</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1020</th>\n",
       "      <td>http://edamontology.org/data_2529</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1021</th>\n",
       "      <td>http://edamontology.org/data_2530</td>\n",
       "      <td>18</td>\n",
       "      <td>Organism report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1022</th>\n",
       "      <td>http://edamontology.org/data_2531</td>\n",
       "      <td>18</td>\n",
       "      <td>Protocol</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1023</th>\n",
       "      <td>http://edamontology.org/data_2534</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence attribute</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1024</th>\n",
       "      <td>http://edamontology.org/data_2535</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence tag profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1025</th>\n",
       "      <td>http://edamontology.org/data_2536</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrometry data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1026</th>\n",
       "      <td>http://edamontology.org/data_2537</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure raw data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1027</th>\n",
       "      <td>http://edamontology.org/data_2538</td>\n",
       "      <td>18</td>\n",
       "      <td>Mutation identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1028</th>\n",
       "      <td>http://edamontology.org/data_2539</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1029</th>\n",
       "      <td>http://edamontology.org/data_2540</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1030</th>\n",
       "      <td>http://edamontology.org/data_2563</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid name (single letter)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1031</th>\n",
       "      <td>http://edamontology.org/data_2564</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid name (three letter)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1032</th>\n",
       "      <td>http://edamontology.org/data_2565</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid name (full name)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1033</th>\n",
       "      <td>http://edamontology.org/data_2576</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxin identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1034</th>\n",
       "      <td>http://edamontology.org/data_2578</td>\n",
       "      <td>18</td>\n",
       "      <td>ArachnoServer ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1035</th>\n",
       "      <td>http://edamontology.org/data_2579</td>\n",
       "      <td>18</td>\n",
       "      <td>Expressed gene list</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1036</th>\n",
       "      <td>http://edamontology.org/data_2580</td>\n",
       "      <td>18</td>\n",
       "      <td>BindingDB Monomer ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1037</th>\n",
       "      <td>http://edamontology.org/data_2581</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1038</th>\n",
       "      <td>http://edamontology.org/data_2582</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept ID (biological process)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1039</th>\n",
       "      <td>http://edamontology.org/data_2583</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept ID (molecular function)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1040</th>\n",
       "      <td>http://edamontology.org/data_2584</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept name (cellular component)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1041</th>\n",
       "      <td>http://edamontology.org/data_2586</td>\n",
       "      <td>18</td>\n",
       "      <td>Northern blot image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1042</th>\n",
       "      <td>http://edamontology.org/data_2587</td>\n",
       "      <td>18</td>\n",
       "      <td>Blot ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1043</th>\n",
       "      <td>http://edamontology.org/data_2588</td>\n",
       "      <td>18</td>\n",
       "      <td>BlotBase blot ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1044</th>\n",
       "      <td>http://edamontology.org/data_2589</td>\n",
       "      <td>18</td>\n",
       "      <td>Hierarchy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1045</th>\n",
       "      <td>http://edamontology.org/data_2590</td>\n",
       "      <td>18</td>\n",
       "      <td>Hierarchy identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1046</th>\n",
       "      <td>http://edamontology.org/data_2591</td>\n",
       "      <td>18</td>\n",
       "      <td>Brite hierarchy ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1047</th>\n",
       "      <td>http://edamontology.org/data_2592</td>\n",
       "      <td>18</td>\n",
       "      <td>Cancer type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1048</th>\n",
       "      <td>http://edamontology.org/data_2593</td>\n",
       "      <td>18</td>\n",
       "      <td>BRENDA organism ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1049</th>\n",
       "      <td>http://edamontology.org/data_2594</td>\n",
       "      <td>18</td>\n",
       "      <td>UniGene taxon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1050</th>\n",
       "      <td>http://edamontology.org/data_2595</td>\n",
       "      <td>18</td>\n",
       "      <td>UTRdb taxon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1051</th>\n",
       "      <td>http://edamontology.org/data_2596</td>\n",
       "      <td>18</td>\n",
       "      <td>Catalogue ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1052</th>\n",
       "      <td>http://edamontology.org/data_2597</td>\n",
       "      <td>18</td>\n",
       "      <td>CABRI catalogue name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053</th>\n",
       "      <td>http://edamontology.org/data_2598</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure alignment metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1054</th>\n",
       "      <td>http://edamontology.org/data_2599</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule interaction report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1055</th>\n",
       "      <td>http://edamontology.org/data_2600</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1056</th>\n",
       "      <td>http://edamontology.org/data_2601</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecule data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1057</th>\n",
       "      <td>http://edamontology.org/data_2602</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype and phenotype data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1058</th>\n",
       "      <td>http://edamontology.org/data_2603</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1059</th>\n",
       "      <td>http://edamontology.org/data_2605</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (KEGG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1060</th>\n",
       "      <td>http://edamontology.org/data_2606</td>\n",
       "      <td>18</td>\n",
       "      <td>RFAM name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1061</th>\n",
       "      <td>http://edamontology.org/data_2608</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID (KEGG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1062</th>\n",
       "      <td>http://edamontology.org/data_2609</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug ID (KEGG)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1063</th>\n",
       "      <td>http://edamontology.org/data_2610</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1064</th>\n",
       "      <td>http://edamontology.org/data_2611</td>\n",
       "      <td>18</td>\n",
       "      <td>ICD identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1065</th>\n",
       "      <td>http://edamontology.org/data_2612</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster ID (CluSTr)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1066</th>\n",
       "      <td>http://edamontology.org/data_2613</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG Glycan ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1067</th>\n",
       "      <td>http://edamontology.org/data_2614</td>\n",
       "      <td>18</td>\n",
       "      <td>TCDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1068</th>\n",
       "      <td>http://edamontology.org/data_2615</td>\n",
       "      <td>18</td>\n",
       "      <td>MINT ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1069</th>\n",
       "      <td>http://edamontology.org/data_2616</td>\n",
       "      <td>18</td>\n",
       "      <td>DIP ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1070</th>\n",
       "      <td>http://edamontology.org/data_2617</td>\n",
       "      <td>18</td>\n",
       "      <td>Signaling Gateway protein ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1071</th>\n",
       "      <td>http://edamontology.org/data_2618</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein modification ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1072</th>\n",
       "      <td>http://edamontology.org/data_2619</td>\n",
       "      <td>18</td>\n",
       "      <td>RESID ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1073</th>\n",
       "      <td>http://edamontology.org/data_2620</td>\n",
       "      <td>18</td>\n",
       "      <td>RGD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1074</th>\n",
       "      <td>http://edamontology.org/data_2621</td>\n",
       "      <td>18</td>\n",
       "      <td>TAIR accession (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1075</th>\n",
       "      <td>http://edamontology.org/data_2622</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (HMDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1076</th>\n",
       "      <td>http://edamontology.org/data_2625</td>\n",
       "      <td>18</td>\n",
       "      <td>LIPID MAPS ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1077</th>\n",
       "      <td>http://edamontology.org/data_2626</td>\n",
       "      <td>18</td>\n",
       "      <td>PeptideAtlas ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1078</th>\n",
       "      <td>http://edamontology.org/data_2627</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular interaction ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1079</th>\n",
       "      <td>http://edamontology.org/data_2628</td>\n",
       "      <td>18</td>\n",
       "      <td>BioGRID interaction ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1080</th>\n",
       "      <td>http://edamontology.org/data_2629</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme ID (MEROPS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1081</th>\n",
       "      <td>http://edamontology.org/data_2630</td>\n",
       "      <td>18</td>\n",
       "      <td>Mobile genetic element ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1082</th>\n",
       "      <td>http://edamontology.org/data_2631</td>\n",
       "      <td>18</td>\n",
       "      <td>ACLAME ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1083</th>\n",
       "      <td>http://edamontology.org/data_2632</td>\n",
       "      <td>18</td>\n",
       "      <td>SGD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1084</th>\n",
       "      <td>http://edamontology.org/data_2633</td>\n",
       "      <td>18</td>\n",
       "      <td>Book ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1085</th>\n",
       "      <td>http://edamontology.org/data_2634</td>\n",
       "      <td>18</td>\n",
       "      <td>ISBN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1086</th>\n",
       "      <td>http://edamontology.org/data_2635</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (3DMET)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1087</th>\n",
       "      <td>http://edamontology.org/data_2636</td>\n",
       "      <td>18</td>\n",
       "      <td>MatrixDB interaction ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1088</th>\n",
       "      <td>http://edamontology.org/data_2637</td>\n",
       "      <td>18</td>\n",
       "      <td>cPath ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1089</th>\n",
       "      <td>http://edamontology.org/data_2638</td>\n",
       "      <td>18</td>\n",
       "      <td>PubChem bioassay ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1090</th>\n",
       "      <td>http://edamontology.org/data_2639</td>\n",
       "      <td>18</td>\n",
       "      <td>PubChem ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1091</th>\n",
       "      <td>http://edamontology.org/data_2641</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID (MACie)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1092</th>\n",
       "      <td>http://edamontology.org/data_2642</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (miRBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1093</th>\n",
       "      <td>http://edamontology.org/data_2643</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (ZFIN)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1094</th>\n",
       "      <td>http://edamontology.org/data_2644</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction ID (Rhea)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1095</th>\n",
       "      <td>http://edamontology.org/data_2645</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (Unipathway)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1096</th>\n",
       "      <td>http://edamontology.org/data_2646</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (ChEMBL)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1097</th>\n",
       "      <td>http://edamontology.org/data_2647</td>\n",
       "      <td>18</td>\n",
       "      <td>LGICdb identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1098</th>\n",
       "      <td>http://edamontology.org/data_2648</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction kinetics ID (SABIO-RK)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1099</th>\n",
       "      <td>http://edamontology.org/data_2649</td>\n",
       "      <td>18</td>\n",
       "      <td>PharmGKB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1100</th>\n",
       "      <td>http://edamontology.org/data_2650</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (PharmGKB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1101</th>\n",
       "      <td>http://edamontology.org/data_2651</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease ID (PharmGKB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1102</th>\n",
       "      <td>http://edamontology.org/data_2652</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug ID (PharmGKB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1103</th>\n",
       "      <td>http://edamontology.org/data_2653</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug ID (TTD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1104</th>\n",
       "      <td>http://edamontology.org/data_2654</td>\n",
       "      <td>18</td>\n",
       "      <td>Target ID (TTD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1105</th>\n",
       "      <td>http://edamontology.org/data_2655</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell type identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1106</th>\n",
       "      <td>http://edamontology.org/data_2656</td>\n",
       "      <td>18</td>\n",
       "      <td>NeuronDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1107</th>\n",
       "      <td>http://edamontology.org/data_2657</td>\n",
       "      <td>18</td>\n",
       "      <td>NeuroMorpho ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1108</th>\n",
       "      <td>http://edamontology.org/data_2658</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound ID (ChemIDplus)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1109</th>\n",
       "      <td>http://edamontology.org/data_2659</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (SMPDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1110</th>\n",
       "      <td>http://edamontology.org/data_2660</td>\n",
       "      <td>18</td>\n",
       "      <td>BioNumbers ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1111</th>\n",
       "      <td>http://edamontology.org/data_2662</td>\n",
       "      <td>18</td>\n",
       "      <td>T3DB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1112</th>\n",
       "      <td>http://edamontology.org/data_2663</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1113</th>\n",
       "      <td>http://edamontology.org/data_2664</td>\n",
       "      <td>18</td>\n",
       "      <td>GlycomeDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1114</th>\n",
       "      <td>http://edamontology.org/data_2665</td>\n",
       "      <td>18</td>\n",
       "      <td>LipidBank ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1115</th>\n",
       "      <td>http://edamontology.org/data_2666</td>\n",
       "      <td>18</td>\n",
       "      <td>CDD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1116</th>\n",
       "      <td>http://edamontology.org/data_2667</td>\n",
       "      <td>18</td>\n",
       "      <td>MMDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1117</th>\n",
       "      <td>http://edamontology.org/data_2668</td>\n",
       "      <td>18</td>\n",
       "      <td>iRefIndex ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1118</th>\n",
       "      <td>http://edamontology.org/data_2669</td>\n",
       "      <td>18</td>\n",
       "      <td>ModelDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1119</th>\n",
       "      <td>http://edamontology.org/data_2670</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (DQCS)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1120</th>\n",
       "      <td>http://edamontology.org/data_2671</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID (Homo sapiens)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1121</th>\n",
       "      <td>http://edamontology.org/data_2672</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Bos taurus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1122</th>\n",
       "      <td>http://edamontology.org/data_2673</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Canis familiaris')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1123</th>\n",
       "      <td>http://edamontology.org/data_2674</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Cavia porcellus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1124</th>\n",
       "      <td>http://edamontology.org/data_2675</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Ciona intestinalis')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1125</th>\n",
       "      <td>http://edamontology.org/data_2676</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Ciona savignyi')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1126</th>\n",
       "      <td>http://edamontology.org/data_2677</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Danio rerio')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1127</th>\n",
       "      <td>http://edamontology.org/data_2678</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Dasypus novemcinctus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1128</th>\n",
       "      <td>http://edamontology.org/data_2679</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Echinops telfairi')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1129</th>\n",
       "      <td>http://edamontology.org/data_2680</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Erinaceus europaeus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1130</th>\n",
       "      <td>http://edamontology.org/data_2681</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Felis catus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1131</th>\n",
       "      <td>http://edamontology.org/data_2682</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Gallus gallus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1132</th>\n",
       "      <td>http://edamontology.org/data_2683</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Gasterosteus aculeatus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1133</th>\n",
       "      <td>http://edamontology.org/data_2684</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Homo sapiens')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1134</th>\n",
       "      <td>http://edamontology.org/data_2685</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Loxodonta africana')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1135</th>\n",
       "      <td>http://edamontology.org/data_2686</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Macaca mulatta')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1136</th>\n",
       "      <td>http://edamontology.org/data_2687</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Monodelphis domestica')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1137</th>\n",
       "      <td>http://edamontology.org/data_2688</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Mus musculus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1138</th>\n",
       "      <td>http://edamontology.org/data_2689</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Myotis lucifugus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1139</th>\n",
       "      <td>http://edamontology.org/data_2690</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID (\"Ornithorhynchus anatinus\")</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1140</th>\n",
       "      <td>http://edamontology.org/data_2691</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Oryctolagus cuniculus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1141</th>\n",
       "      <td>http://edamontology.org/data_2692</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Oryzias latipes')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1142</th>\n",
       "      <td>http://edamontology.org/data_2693</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Otolemur garnettii')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1143</th>\n",
       "      <td>http://edamontology.org/data_2694</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Pan troglodytes')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1144</th>\n",
       "      <td>http://edamontology.org/data_2695</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Rattus norvegicus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1145</th>\n",
       "      <td>http://edamontology.org/data_2696</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Spermophilus tridecemlineatus')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1146</th>\n",
       "      <td>http://edamontology.org/data_2697</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Takifugu rubripes')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1147</th>\n",
       "      <td>http://edamontology.org/data_2698</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Tupaia belangeri')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1148</th>\n",
       "      <td>http://edamontology.org/data_2699</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl ID ('Xenopus tropicalis')</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1149</th>\n",
       "      <td>http://edamontology.org/data_2700</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1150</th>\n",
       "      <td>http://edamontology.org/data_2701</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH node ID (family)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1151</th>\n",
       "      <td>http://edamontology.org/data_2702</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme ID (CAZy)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1152</th>\n",
       "      <td>http://edamontology.org/data_2704</td>\n",
       "      <td>18</td>\n",
       "      <td>Clone ID (IMAGE)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1153</th>\n",
       "      <td>http://edamontology.org/data_2705</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept ID (cellular component)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1154</th>\n",
       "      <td>http://edamontology.org/data_2706</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosome name (BioCyc)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1155</th>\n",
       "      <td>http://edamontology.org/data_2709</td>\n",
       "      <td>18</td>\n",
       "      <td>CleanEx entry name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1156</th>\n",
       "      <td>http://edamontology.org/data_2710</td>\n",
       "      <td>18</td>\n",
       "      <td>CleanEx dataset code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1157</th>\n",
       "      <td>http://edamontology.org/data_2711</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1158</th>\n",
       "      <td>http://edamontology.org/data_2713</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (CORUM)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1159</th>\n",
       "      <td>http://edamontology.org/data_2714</td>\n",
       "      <td>18</td>\n",
       "      <td>CDD PSSM-ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1160</th>\n",
       "      <td>http://edamontology.org/data_2715</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (CuticleDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1161</th>\n",
       "      <td>http://edamontology.org/data_2716</td>\n",
       "      <td>18</td>\n",
       "      <td>DBD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1162</th>\n",
       "      <td>http://edamontology.org/data_2717</td>\n",
       "      <td>18</td>\n",
       "      <td>Oligonucleotide probe annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1163</th>\n",
       "      <td>http://edamontology.org/data_2718</td>\n",
       "      <td>18</td>\n",
       "      <td>Oligonucleotide ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1164</th>\n",
       "      <td>http://edamontology.org/data_2719</td>\n",
       "      <td>18</td>\n",
       "      <td>dbProbe ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1165</th>\n",
       "      <td>http://edamontology.org/data_2720</td>\n",
       "      <td>18</td>\n",
       "      <td>Dinucleotide property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1166</th>\n",
       "      <td>http://edamontology.org/data_2721</td>\n",
       "      <td>18</td>\n",
       "      <td>DiProDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1167</th>\n",
       "      <td>http://edamontology.org/data_2722</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (disordered structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1168</th>\n",
       "      <td>http://edamontology.org/data_2723</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (DisProt)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1169</th>\n",
       "      <td>http://edamontology.org/data_2724</td>\n",
       "      <td>18</td>\n",
       "      <td>Embryo report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1170</th>\n",
       "      <td>http://edamontology.org/data_2725</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl transcript ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1171</th>\n",
       "      <td>http://edamontology.org/data_2726</td>\n",
       "      <td>18</td>\n",
       "      <td>Inhibitor annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1172</th>\n",
       "      <td>http://edamontology.org/data_2727</td>\n",
       "      <td>18</td>\n",
       "      <td>Promoter ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1173</th>\n",
       "      <td>http://edamontology.org/data_2728</td>\n",
       "      <td>18</td>\n",
       "      <td>EST accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1174</th>\n",
       "      <td>http://edamontology.org/data_2729</td>\n",
       "      <td>18</td>\n",
       "      <td>COGEME EST ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1175</th>\n",
       "      <td>http://edamontology.org/data_2730</td>\n",
       "      <td>18</td>\n",
       "      <td>COGEME unisequence ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1176</th>\n",
       "      <td>http://edamontology.org/data_2731</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family ID (GeneFarm)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1177</th>\n",
       "      <td>http://edamontology.org/data_2732</td>\n",
       "      <td>18</td>\n",
       "      <td>Family name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1178</th>\n",
       "      <td>http://edamontology.org/data_2733</td>\n",
       "      <td>18</td>\n",
       "      <td>Genus name (virus)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1179</th>\n",
       "      <td>http://edamontology.org/data_2734</td>\n",
       "      <td>18</td>\n",
       "      <td>Family name (virus)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1180</th>\n",
       "      <td>http://edamontology.org/data_2735</td>\n",
       "      <td>18</td>\n",
       "      <td>Database name (SwissRegulon)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1181</th>\n",
       "      <td>http://edamontology.org/data_2736</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature ID (SwissRegulon)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1182</th>\n",
       "      <td>http://edamontology.org/data_2737</td>\n",
       "      <td>18</td>\n",
       "      <td>FIG ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1183</th>\n",
       "      <td>http://edamontology.org/data_2738</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (Xenbase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1184</th>\n",
       "      <td>http://edamontology.org/data_2739</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (Genolist)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1185</th>\n",
       "      <td>http://edamontology.org/data_2740</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (Genolist)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1186</th>\n",
       "      <td>http://edamontology.org/data_2741</td>\n",
       "      <td>18</td>\n",
       "      <td>ABS ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1187</th>\n",
       "      <td>http://edamontology.org/data_2742</td>\n",
       "      <td>18</td>\n",
       "      <td>AraC-XylS ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1188</th>\n",
       "      <td>http://edamontology.org/data_2743</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene name (HUGO)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1189</th>\n",
       "      <td>http://edamontology.org/data_2744</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (PseudoCAP)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1190</th>\n",
       "      <td>http://edamontology.org/data_2745</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus ID (UTR)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1191</th>\n",
       "      <td>http://edamontology.org/data_2746</td>\n",
       "      <td>18</td>\n",
       "      <td>MonosaccharideDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1192</th>\n",
       "      <td>http://edamontology.org/data_2747</td>\n",
       "      <td>18</td>\n",
       "      <td>Database name (CMD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1193</th>\n",
       "      <td>http://edamontology.org/data_2748</td>\n",
       "      <td>18</td>\n",
       "      <td>Database name (Osteogenesis)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1194</th>\n",
       "      <td>http://edamontology.org/data_2749</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1195</th>\n",
       "      <td>http://edamontology.org/data_2751</td>\n",
       "      <td>18</td>\n",
       "      <td>GenomeReviews ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1196</th>\n",
       "      <td>http://edamontology.org/data_2752</td>\n",
       "      <td>18</td>\n",
       "      <td>GlycoMap ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1197</th>\n",
       "      <td>http://edamontology.org/data_2753</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate conformational map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1198</th>\n",
       "      <td>http://edamontology.org/data_2755</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factor name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1199</th>\n",
       "      <td>http://edamontology.org/data_2756</td>\n",
       "      <td>18</td>\n",
       "      <td>TCID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1200</th>\n",
       "      <td>http://edamontology.org/data_2757</td>\n",
       "      <td>18</td>\n",
       "      <td>Pfam domain name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>http://edamontology.org/data_2758</td>\n",
       "      <td>18</td>\n",
       "      <td>Pfam clan ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1202</th>\n",
       "      <td>http://edamontology.org/data_2759</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (VectorBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1203</th>\n",
       "      <td>http://edamontology.org/data_2761</td>\n",
       "      <td>18</td>\n",
       "      <td>UTRSite ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1204</th>\n",
       "      <td>http://edamontology.org/data_2762</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence signature report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1205</th>\n",
       "      <td>http://edamontology.org/data_2763</td>\n",
       "      <td>18</td>\n",
       "      <td>Locus annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1206</th>\n",
       "      <td>http://edamontology.org/data_2764</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein name (UniProt)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1207</th>\n",
       "      <td>http://edamontology.org/data_2765</td>\n",
       "      <td>18</td>\n",
       "      <td>Term ID list</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1208</th>\n",
       "      <td>http://edamontology.org/data_2766</td>\n",
       "      <td>18</td>\n",
       "      <td>HAMAP ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1209</th>\n",
       "      <td>http://edamontology.org/data_2767</td>\n",
       "      <td>18</td>\n",
       "      <td>Identifier with metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1210</th>\n",
       "      <td>http://edamontology.org/data_2768</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene symbol annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1211</th>\n",
       "      <td>http://edamontology.org/data_2769</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcript ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1212</th>\n",
       "      <td>http://edamontology.org/data_2770</td>\n",
       "      <td>18</td>\n",
       "      <td>HIT ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1213</th>\n",
       "      <td>http://edamontology.org/data_2771</td>\n",
       "      <td>18</td>\n",
       "      <td>HIX ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1214</th>\n",
       "      <td>http://edamontology.org/data_2772</td>\n",
       "      <td>18</td>\n",
       "      <td>HPA antibody id</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1215</th>\n",
       "      <td>http://edamontology.org/data_2773</td>\n",
       "      <td>18</td>\n",
       "      <td>IMGT/HLA ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1216</th>\n",
       "      <td>http://edamontology.org/data_2774</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (JCVI)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1217</th>\n",
       "      <td>http://edamontology.org/data_2775</td>\n",
       "      <td>18</td>\n",
       "      <td>Kinase name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1218</th>\n",
       "      <td>http://edamontology.org/data_2776</td>\n",
       "      <td>18</td>\n",
       "      <td>ConsensusPathDB entity ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1219</th>\n",
       "      <td>http://edamontology.org/data_2777</td>\n",
       "      <td>18</td>\n",
       "      <td>ConsensusPathDB entity name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1220</th>\n",
       "      <td>http://edamontology.org/data_2778</td>\n",
       "      <td>18</td>\n",
       "      <td>CCAP strain number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1221</th>\n",
       "      <td>http://edamontology.org/data_2779</td>\n",
       "      <td>18</td>\n",
       "      <td>Stock number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1222</th>\n",
       "      <td>http://edamontology.org/data_2780</td>\n",
       "      <td>18</td>\n",
       "      <td>Stock number (TAIR)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1223</th>\n",
       "      <td>http://edamontology.org/data_2781</td>\n",
       "      <td>18</td>\n",
       "      <td>REDIdb ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1224</th>\n",
       "      <td>http://edamontology.org/data_2782</td>\n",
       "      <td>18</td>\n",
       "      <td>SMART domain name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1225</th>\n",
       "      <td>http://edamontology.org/data_2783</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family ID (PANTHER)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1226</th>\n",
       "      <td>http://edamontology.org/data_2784</td>\n",
       "      <td>18</td>\n",
       "      <td>RNAVirusDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1227</th>\n",
       "      <td>http://edamontology.org/data_2785</td>\n",
       "      <td>18</td>\n",
       "      <td>Virus identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1228</th>\n",
       "      <td>http://edamontology.org/data_2786</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI Genome Project ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1229</th>\n",
       "      <td>http://edamontology.org/data_2787</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI genome accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1230</th>\n",
       "      <td>http://edamontology.org/data_2788</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1231</th>\n",
       "      <td>http://edamontology.org/data_2789</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (TopDB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1232</th>\n",
       "      <td>http://edamontology.org/data_2790</td>\n",
       "      <td>18</td>\n",
       "      <td>Gel ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1233</th>\n",
       "      <td>http://edamontology.org/data_2791</td>\n",
       "      <td>18</td>\n",
       "      <td>Reference map name (SWISS-2DPAGE)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1234</th>\n",
       "      <td>http://edamontology.org/data_2792</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (PeroxiBase)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1235</th>\n",
       "      <td>http://edamontology.org/data_2793</td>\n",
       "      <td>18</td>\n",
       "      <td>SISYPHUS ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1236</th>\n",
       "      <td>http://edamontology.org/data_2794</td>\n",
       "      <td>18</td>\n",
       "      <td>ORF ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1237</th>\n",
       "      <td>http://edamontology.org/data_2795</td>\n",
       "      <td>18</td>\n",
       "      <td>ORF identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1238</th>\n",
       "      <td>http://edamontology.org/data_2796</td>\n",
       "      <td>18</td>\n",
       "      <td>LINUCS ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1239</th>\n",
       "      <td>http://edamontology.org/data_2797</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (LGICdb)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1240</th>\n",
       "      <td>http://edamontology.org/data_2798</td>\n",
       "      <td>18</td>\n",
       "      <td>MaizeDB ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1241</th>\n",
       "      <td>http://edamontology.org/data_2799</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (MfunGD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1242</th>\n",
       "      <td>http://edamontology.org/data_2800</td>\n",
       "      <td>18</td>\n",
       "      <td>Orpha number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1243</th>\n",
       "      <td>http://edamontology.org/data_2802</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (EcID)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1244</th>\n",
       "      <td>http://edamontology.org/data_2803</td>\n",
       "      <td>18</td>\n",
       "      <td>Clone ID (RefSeq)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1245</th>\n",
       "      <td>http://edamontology.org/data_2804</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (ConoServer)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1246</th>\n",
       "      <td>http://edamontology.org/data_2805</td>\n",
       "      <td>18</td>\n",
       "      <td>GeneSNP ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1247</th>\n",
       "      <td>http://edamontology.org/data_2812</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipid identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1248</th>\n",
       "      <td>http://edamontology.org/data_2831</td>\n",
       "      <td>18</td>\n",
       "      <td>Databank</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1249</th>\n",
       "      <td>http://edamontology.org/data_2832</td>\n",
       "      <td>18</td>\n",
       "      <td>Web portal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1250</th>\n",
       "      <td>http://edamontology.org/data_2835</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene ID (VBASE2)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1251</th>\n",
       "      <td>http://edamontology.org/data_2836</td>\n",
       "      <td>18</td>\n",
       "      <td>DPVweb ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1252</th>\n",
       "      <td>http://edamontology.org/data_2837</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (BioSystems)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1253</th>\n",
       "      <td>http://edamontology.org/data_2838</td>\n",
       "      <td>18</td>\n",
       "      <td>Experimental data (proteomics)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1254</th>\n",
       "      <td>http://edamontology.org/data_2849</td>\n",
       "      <td>18</td>\n",
       "      <td>Abstract</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1255</th>\n",
       "      <td>http://edamontology.org/data_2850</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipid structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1256</th>\n",
       "      <td>http://edamontology.org/data_2851</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1257</th>\n",
       "      <td>http://edamontology.org/data_2852</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxin structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1258</th>\n",
       "      <td>http://edamontology.org/data_2854</td>\n",
       "      <td>18</td>\n",
       "      <td>Position-specific scoring matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1259</th>\n",
       "      <td>http://edamontology.org/data_2855</td>\n",
       "      <td>18</td>\n",
       "      <td>Distance matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1260</th>\n",
       "      <td>http://edamontology.org/data_2856</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural distance matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1261</th>\n",
       "      <td>http://edamontology.org/data_2857</td>\n",
       "      <td>18</td>\n",
       "      <td>Article metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1262</th>\n",
       "      <td>http://edamontology.org/data_2858</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1263</th>\n",
       "      <td>http://edamontology.org/data_2865</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage bias</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1264</th>\n",
       "      <td>http://edamontology.org/data_2866</td>\n",
       "      <td>18</td>\n",
       "      <td>Northern blot report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1265</th>\n",
       "      <td>http://edamontology.org/data_2870</td>\n",
       "      <td>18</td>\n",
       "      <td>Radiation hybrid map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1266</th>\n",
       "      <td>http://edamontology.org/data_2872</td>\n",
       "      <td>18</td>\n",
       "      <td>ID list</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1267</th>\n",
       "      <td>http://edamontology.org/data_2873</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic gene frequencies data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1268</th>\n",
       "      <td>http://edamontology.org/data_2874</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence set (polymorphic)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1269</th>\n",
       "      <td>http://edamontology.org/data_2875</td>\n",
       "      <td>18</td>\n",
       "      <td>DRCAT resource</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1270</th>\n",
       "      <td>http://edamontology.org/data_2877</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein complex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1271</th>\n",
       "      <td>http://edamontology.org/data_2878</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structural motif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1272</th>\n",
       "      <td>http://edamontology.org/data_2879</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipid report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1273</th>\n",
       "      <td>http://edamontology.org/data_2880</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1274</th>\n",
       "      <td>http://edamontology.org/data_2881</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1275</th>\n",
       "      <td>http://edamontology.org/data_2882</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1276</th>\n",
       "      <td>http://edamontology.org/data_2883</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA features report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1277</th>\n",
       "      <td>http://edamontology.org/data_2884</td>\n",
       "      <td>18</td>\n",
       "      <td>Plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1278</th>\n",
       "      <td>http://edamontology.org/data_2886</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1279</th>\n",
       "      <td>http://edamontology.org/data_2887</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1280</th>\n",
       "      <td>http://edamontology.org/data_2888</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence record (full)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1281</th>\n",
       "      <td>http://edamontology.org/data_2889</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence record (full)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1282</th>\n",
       "      <td>http://edamontology.org/data_2891</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological model accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1283</th>\n",
       "      <td>http://edamontology.org/data_2892</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell type name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1284</th>\n",
       "      <td>http://edamontology.org/data_2893</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell type accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1285</th>\n",
       "      <td>http://edamontology.org/data_2894</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1286</th>\n",
       "      <td>http://edamontology.org/data_2895</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1287</th>\n",
       "      <td>http://edamontology.org/data_2896</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxin name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1288</th>\n",
       "      <td>http://edamontology.org/data_2897</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxin accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1289</th>\n",
       "      <td>http://edamontology.org/data_2898</td>\n",
       "      <td>18</td>\n",
       "      <td>Monosaccharide accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1290</th>\n",
       "      <td>http://edamontology.org/data_2899</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1291</th>\n",
       "      <td>http://edamontology.org/data_2900</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1292</th>\n",
       "      <td>http://edamontology.org/data_2901</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecule accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1293</th>\n",
       "      <td>http://edamontology.org/data_2902</td>\n",
       "      <td>18</td>\n",
       "      <td>Data resource definition accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1294</th>\n",
       "      <td>http://edamontology.org/data_2903</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1295</th>\n",
       "      <td>http://edamontology.org/data_2904</td>\n",
       "      <td>18</td>\n",
       "      <td>Map accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1296</th>\n",
       "      <td>http://edamontology.org/data_2905</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipid accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1297</th>\n",
       "      <td>http://edamontology.org/data_2906</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1298</th>\n",
       "      <td>http://edamontology.org/data_2907</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1299</th>\n",
       "      <td>http://edamontology.org/data_2908</td>\n",
       "      <td>18</td>\n",
       "      <td>Organism accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1300</th>\n",
       "      <td>http://edamontology.org/data_2909</td>\n",
       "      <td>18</td>\n",
       "      <td>Organism name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1301</th>\n",
       "      <td>http://edamontology.org/data_2910</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1302</th>\n",
       "      <td>http://edamontology.org/data_2911</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factor accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1303</th>\n",
       "      <td>http://edamontology.org/data_2912</td>\n",
       "      <td>18</td>\n",
       "      <td>Strain accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1304</th>\n",
       "      <td>http://edamontology.org/data_2913</td>\n",
       "      <td>18</td>\n",
       "      <td>Virus identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1305</th>\n",
       "      <td>http://edamontology.org/data_2914</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1306</th>\n",
       "      <td>http://edamontology.org/data_2915</td>\n",
       "      <td>18</td>\n",
       "      <td>Gramene identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1307</th>\n",
       "      <td>http://edamontology.org/data_2916</td>\n",
       "      <td>18</td>\n",
       "      <td>DDBJ accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1308</th>\n",
       "      <td>http://edamontology.org/data_2917</td>\n",
       "      <td>18</td>\n",
       "      <td>ConsensusPathDB identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1309</th>\n",
       "      <td>http://edamontology.org/data_2925</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1310</th>\n",
       "      <td>http://edamontology.org/data_2927</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1311</th>\n",
       "      <td>http://edamontology.org/data_2954</td>\n",
       "      <td>18</td>\n",
       "      <td>Article report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1312</th>\n",
       "      <td>http://edamontology.org/data_2955</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1313</th>\n",
       "      <td>http://edamontology.org/data_2956</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1314</th>\n",
       "      <td>http://edamontology.org/data_2957</td>\n",
       "      <td>18</td>\n",
       "      <td>Hopp and Woods plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1315</th>\n",
       "      <td>http://edamontology.org/data_2958</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting curve</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1316</th>\n",
       "      <td>http://edamontology.org/data_2959</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid probability profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1317</th>\n",
       "      <td>http://edamontology.org/data_2960</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid temperature profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1318</th>\n",
       "      <td>http://edamontology.org/data_2961</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulatory network report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1319</th>\n",
       "      <td>http://edamontology.org/data_2965</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE gel report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1320</th>\n",
       "      <td>http://edamontology.org/data_2966</td>\n",
       "      <td>18</td>\n",
       "      <td>Oligonucleotide probe sets annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1321</th>\n",
       "      <td>http://edamontology.org/data_2967</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1322</th>\n",
       "      <td>http://edamontology.org/data_2968</td>\n",
       "      <td>18</td>\n",
       "      <td>Image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1323</th>\n",
       "      <td>http://edamontology.org/data_2969</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1324</th>\n",
       "      <td>http://edamontology.org/data_2970</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1325</th>\n",
       "      <td>http://edamontology.org/data_2971</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflow data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1326</th>\n",
       "      <td>http://edamontology.org/data_2972</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1327</th>\n",
       "      <td>http://edamontology.org/data_2973</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1328</th>\n",
       "      <td>http://edamontology.org/data_2974</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence (raw)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1329</th>\n",
       "      <td>http://edamontology.org/data_2975</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence (raw)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1330</th>\n",
       "      <td>http://edamontology.org/data_2976</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1331</th>\n",
       "      <td>http://edamontology.org/data_2977</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1332</th>\n",
       "      <td>http://edamontology.org/data_2978</td>\n",
       "      <td>18</td>\n",
       "      <td>Reaction data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1333</th>\n",
       "      <td>http://edamontology.org/data_2979</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1334</th>\n",
       "      <td>http://edamontology.org/data_2980</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1335</th>\n",
       "      <td>http://edamontology.org/data_2981</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1336</th>\n",
       "      <td>http://edamontology.org/data_2982</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1337</th>\n",
       "      <td>http://edamontology.org/data_2983</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1338</th>\n",
       "      <td>http://edamontology.org/data_2984</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1339</th>\n",
       "      <td>http://edamontology.org/data_2985</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid thermodynamic data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1340</th>\n",
       "      <td>http://edamontology.org/data_2986</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1341</th>\n",
       "      <td>http://edamontology.org/data_2987</td>\n",
       "      <td>18</td>\n",
       "      <td>Classification report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1342</th>\n",
       "      <td>http://edamontology.org/data_2989</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (key folding sites)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1343</th>\n",
       "      <td>http://edamontology.org/data_2991</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein geometry data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1344</th>\n",
       "      <td>http://edamontology.org/data_2992</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1345</th>\n",
       "      <td>http://edamontology.org/data_2994</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic character weights</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1346</th>\n",
       "      <td>http://edamontology.org/data_3002</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation track</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1347</th>\n",
       "      <td>http://edamontology.org/data_3021</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1348</th>\n",
       "      <td>http://edamontology.org/data_3022</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI genetic code ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1349</th>\n",
       "      <td>http://edamontology.org/data_3025</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology concept identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1350</th>\n",
       "      <td>http://edamontology.org/data_3026</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept name (biological process)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>http://edamontology.org/data_3027</td>\n",
       "      <td>18</td>\n",
       "      <td>GO concept name (molecular function)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1352</th>\n",
       "      <td>http://edamontology.org/data_3028</td>\n",
       "      <td>18</td>\n",
       "      <td>Taxonomy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1353</th>\n",
       "      <td>http://edamontology.org/data_3029</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein ID (EMBL/GenBank/DDBJ)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1354</th>\n",
       "      <td>http://edamontology.org/data_3031</td>\n",
       "      <td>18</td>\n",
       "      <td>Core data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1355</th>\n",
       "      <td>http://edamontology.org/data_3034</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1356</th>\n",
       "      <td>http://edamontology.org/data_3035</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1357</th>\n",
       "      <td>http://edamontology.org/data_3036</td>\n",
       "      <td>18</td>\n",
       "      <td>Matrix identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1358</th>\n",
       "      <td>http://edamontology.org/data_3085</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence composition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1359</th>\n",
       "      <td>http://edamontology.org/data_3086</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence composition (report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1360</th>\n",
       "      <td>http://edamontology.org/data_3101</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain classification node</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1361</th>\n",
       "      <td>http://edamontology.org/data_3102</td>\n",
       "      <td>18</td>\n",
       "      <td>CAS number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1362</th>\n",
       "      <td>http://edamontology.org/data_3103</td>\n",
       "      <td>18</td>\n",
       "      <td>ATC code</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1363</th>\n",
       "      <td>http://edamontology.org/data_3104</td>\n",
       "      <td>18</td>\n",
       "      <td>UNII</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1364</th>\n",
       "      <td>http://edamontology.org/data_3105</td>\n",
       "      <td>18</td>\n",
       "      <td>Geotemporal metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1365</th>\n",
       "      <td>http://edamontology.org/data_3106</td>\n",
       "      <td>18</td>\n",
       "      <td>System metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1366</th>\n",
       "      <td>http://edamontology.org/data_3107</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1367</th>\n",
       "      <td>http://edamontology.org/data_3108</td>\n",
       "      <td>18</td>\n",
       "      <td>Experimental measurement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1368</th>\n",
       "      <td>http://edamontology.org/data_3110</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw microarray data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1369</th>\n",
       "      <td>http://edamontology.org/data_3111</td>\n",
       "      <td>18</td>\n",
       "      <td>Processed microarray data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1370</th>\n",
       "      <td>http://edamontology.org/data_3112</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1371</th>\n",
       "      <td>http://edamontology.org/data_3113</td>\n",
       "      <td>18</td>\n",
       "      <td>Sample annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1372</th>\n",
       "      <td>http://edamontology.org/data_3115</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1373</th>\n",
       "      <td>http://edamontology.org/data_3116</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray protocol annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1374</th>\n",
       "      <td>http://edamontology.org/data_3117</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray hybridisation data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1375</th>\n",
       "      <td>http://edamontology.org/data_3119</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features (compositionally-biased regions)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1376</th>\n",
       "      <td>http://edamontology.org/data_3122</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (difference and change)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1377</th>\n",
       "      <td>http://edamontology.org/data_3128</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1378</th>\n",
       "      <td>http://edamontology.org/data_3129</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features report (repeats)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1379</th>\n",
       "      <td>http://edamontology.org/data_3130</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif matches (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1380</th>\n",
       "      <td>http://edamontology.org/data_3131</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif matches (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1381</th>\n",
       "      <td>http://edamontology.org/data_3132</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (d-loop)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1382</th>\n",
       "      <td>http://edamontology.org/data_3133</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (stem loop)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1383</th>\n",
       "      <td>http://edamontology.org/data_3134</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene transcript report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1384</th>\n",
       "      <td>http://edamontology.org/data_3137</td>\n",
       "      <td>18</td>\n",
       "      <td>Non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1385</th>\n",
       "      <td>http://edamontology.org/data_3138</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptional features (report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1386</th>\n",
       "      <td>http://edamontology.org/data_3140</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (immunoglobulin gene structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1387</th>\n",
       "      <td>http://edamontology.org/data_3141</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP class</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1388</th>\n",
       "      <td>http://edamontology.org/data_3142</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP fold</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1389</th>\n",
       "      <td>http://edamontology.org/data_3143</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP superfamily</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1390</th>\n",
       "      <td>http://edamontology.org/data_3144</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP family</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1391</th>\n",
       "      <td>http://edamontology.org/data_3145</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1392</th>\n",
       "      <td>http://edamontology.org/data_3146</td>\n",
       "      <td>18</td>\n",
       "      <td>SCOP species</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1393</th>\n",
       "      <td>http://edamontology.org/data_3147</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrometry experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1394</th>\n",
       "      <td>http://edamontology.org/data_3148</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene family report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1395</th>\n",
       "      <td>http://edamontology.org/data_3153</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1396</th>\n",
       "      <td>http://edamontology.org/data_3154</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1397</th>\n",
       "      <td>http://edamontology.org/data_3165</td>\n",
       "      <td>18</td>\n",
       "      <td>NGS experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1398</th>\n",
       "      <td>http://edamontology.org/data_3181</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1399</th>\n",
       "      <td>http://edamontology.org/data_3210</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome index</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1400</th>\n",
       "      <td>http://edamontology.org/data_3231</td>\n",
       "      <td>18</td>\n",
       "      <td>GWAS report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1401</th>\n",
       "      <td>http://edamontology.org/data_3236</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytoband position</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1402</th>\n",
       "      <td>http://edamontology.org/data_3238</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell type ontology ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1403</th>\n",
       "      <td>http://edamontology.org/data_3241</td>\n",
       "      <td>18</td>\n",
       "      <td>Kinetic model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1404</th>\n",
       "      <td>http://edamontology.org/data_3264</td>\n",
       "      <td>18</td>\n",
       "      <td>COSMIC ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1405</th>\n",
       "      <td>http://edamontology.org/data_3265</td>\n",
       "      <td>18</td>\n",
       "      <td>HGMD ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1406</th>\n",
       "      <td>http://edamontology.org/data_3266</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1407</th>\n",
       "      <td>http://edamontology.org/data_3268</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1408</th>\n",
       "      <td>http://edamontology.org/data_3269</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene homology (report)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1409</th>\n",
       "      <td>http://edamontology.org/data_3270</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl gene tree ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1410</th>\n",
       "      <td>http://edamontology.org/data_3271</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene tree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1411</th>\n",
       "      <td>http://edamontology.org/data_3272</td>\n",
       "      <td>18</td>\n",
       "      <td>Species tree</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1412</th>\n",
       "      <td>http://edamontology.org/data_3273</td>\n",
       "      <td>18</td>\n",
       "      <td>Sample ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1413</th>\n",
       "      <td>http://edamontology.org/data_3274</td>\n",
       "      <td>18</td>\n",
       "      <td>MGI accession</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1414</th>\n",
       "      <td>http://edamontology.org/data_3275</td>\n",
       "      <td>18</td>\n",
       "      <td>Phenotype name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1415</th>\n",
       "      <td>http://edamontology.org/data_3354</td>\n",
       "      <td>18</td>\n",
       "      <td>Transition matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1416</th>\n",
       "      <td>http://edamontology.org/data_3355</td>\n",
       "      <td>18</td>\n",
       "      <td>Emission matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1417</th>\n",
       "      <td>http://edamontology.org/data_3356</td>\n",
       "      <td>18</td>\n",
       "      <td>Hidden Markov model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1418</th>\n",
       "      <td>http://edamontology.org/data_3358</td>\n",
       "      <td>18</td>\n",
       "      <td>Format identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1419</th>\n",
       "      <td>http://edamontology.org/data_3424</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1420</th>\n",
       "      <td>http://edamontology.org/data_3425</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrate property</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1421</th>\n",
       "      <td>http://edamontology.org/data_3426</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteomics experiment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1422</th>\n",
       "      <td>http://edamontology.org/data_3427</td>\n",
       "      <td>18</td>\n",
       "      <td>RNAi report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1423</th>\n",
       "      <td>http://edamontology.org/data_3428</td>\n",
       "      <td>18</td>\n",
       "      <td>Simulation experiment report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1424</th>\n",
       "      <td>http://edamontology.org/data_3442</td>\n",
       "      <td>18</td>\n",
       "      <td>MRI image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1425</th>\n",
       "      <td>http://edamontology.org/data_3449</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell migration track image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1426</th>\n",
       "      <td>http://edamontology.org/data_3451</td>\n",
       "      <td>18</td>\n",
       "      <td>Rate of association</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1427</th>\n",
       "      <td>http://edamontology.org/data_3479</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene order</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1428</th>\n",
       "      <td>http://edamontology.org/data_3483</td>\n",
       "      <td>18</td>\n",
       "      <td>Spectrum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1429</th>\n",
       "      <td>http://edamontology.org/data_3488</td>\n",
       "      <td>18</td>\n",
       "      <td>NMR spectrum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1430</th>\n",
       "      <td>http://edamontology.org/data_3490</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical structure sketch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1431</th>\n",
       "      <td>http://edamontology.org/data_3492</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid signature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1432</th>\n",
       "      <td>http://edamontology.org/data_3494</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1433</th>\n",
       "      <td>http://edamontology.org/data_3495</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1434</th>\n",
       "      <td>http://edamontology.org/data_3496</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA sequence (raw)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1435</th>\n",
       "      <td>http://edamontology.org/data_3497</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA sequence (raw)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1436</th>\n",
       "      <td>http://edamontology.org/data_3498</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence variations</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1437</th>\n",
       "      <td>http://edamontology.org/data_3505</td>\n",
       "      <td>18</td>\n",
       "      <td>Bibliography</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1438</th>\n",
       "      <td>http://edamontology.org/data_3509</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1439</th>\n",
       "      <td>http://edamontology.org/data_3546</td>\n",
       "      <td>18</td>\n",
       "      <td>Image metadata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>http://edamontology.org/data_3558</td>\n",
       "      <td>18</td>\n",
       "      <td>Clinical trial report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>http://edamontology.org/data_3567</td>\n",
       "      <td>18</td>\n",
       "      <td>Reference sample report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1442</th>\n",
       "      <td>http://edamontology.org/data_3568</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene Expression Atlas Experiment ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1443</th>\n",
       "      <td>http://edamontology.org/data_3667</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1444</th>\n",
       "      <td>http://edamontology.org/data_3668</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1445</th>\n",
       "      <td>http://edamontology.org/data_3669</td>\n",
       "      <td>18</td>\n",
       "      <td>Training material</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1446</th>\n",
       "      <td>http://edamontology.org/data_3670</td>\n",
       "      <td>18</td>\n",
       "      <td>Online course</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1447</th>\n",
       "      <td>http://edamontology.org/data_3671</td>\n",
       "      <td>18</td>\n",
       "      <td>Text</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1448</th>\n",
       "      <td>http://edamontology.org/data_3707</td>\n",
       "      <td>18</td>\n",
       "      <td>Biodiversity data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1449</th>\n",
       "      <td>http://edamontology.org/data_3716</td>\n",
       "      <td>18</td>\n",
       "      <td>Biosafety report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1450</th>\n",
       "      <td>http://edamontology.org/data_3717</td>\n",
       "      <td>18</td>\n",
       "      <td>Isolation report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1451</th>\n",
       "      <td>http://edamontology.org/data_3718</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathogenicity report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1452</th>\n",
       "      <td>http://edamontology.org/data_3719</td>\n",
       "      <td>18</td>\n",
       "      <td>Biosafety classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1453</th>\n",
       "      <td>http://edamontology.org/data_3720</td>\n",
       "      <td>18</td>\n",
       "      <td>Geographic location</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1454</th>\n",
       "      <td>http://edamontology.org/data_3721</td>\n",
       "      <td>18</td>\n",
       "      <td>Isolation source</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1455</th>\n",
       "      <td>http://edamontology.org/data_3722</td>\n",
       "      <td>18</td>\n",
       "      <td>Physiology parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1456</th>\n",
       "      <td>http://edamontology.org/data_3723</td>\n",
       "      <td>18</td>\n",
       "      <td>Morphology parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1457</th>\n",
       "      <td>http://edamontology.org/data_3724</td>\n",
       "      <td>18</td>\n",
       "      <td>Cultivation parameter</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1458</th>\n",
       "      <td>http://edamontology.org/data_3732</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing metadata name</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1459</th>\n",
       "      <td>http://edamontology.org/data_3733</td>\n",
       "      <td>18</td>\n",
       "      <td>Flow cell identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1460</th>\n",
       "      <td>http://edamontology.org/data_3734</td>\n",
       "      <td>18</td>\n",
       "      <td>Lane identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1461</th>\n",
       "      <td>http://edamontology.org/data_3735</td>\n",
       "      <td>18</td>\n",
       "      <td>Run number</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1462</th>\n",
       "      <td>http://edamontology.org/data_3736</td>\n",
       "      <td>18</td>\n",
       "      <td>Ecological data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1463</th>\n",
       "      <td>http://edamontology.org/data_3737</td>\n",
       "      <td>18</td>\n",
       "      <td>Alpha diversity data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1464</th>\n",
       "      <td>http://edamontology.org/data_3738</td>\n",
       "      <td>18</td>\n",
       "      <td>Beta diversity data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1465</th>\n",
       "      <td>http://edamontology.org/data_3739</td>\n",
       "      <td>18</td>\n",
       "      <td>Gamma diversity data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1466</th>\n",
       "      <td>http://edamontology.org/data_3743</td>\n",
       "      <td>18</td>\n",
       "      <td>Ordination plot</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1467</th>\n",
       "      <td>http://edamontology.org/data_3753</td>\n",
       "      <td>18</td>\n",
       "      <td>Over-representation data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1468</th>\n",
       "      <td>http://edamontology.org/data_3754</td>\n",
       "      <td>18</td>\n",
       "      <td>GO-term enrichment data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1469</th>\n",
       "      <td>http://edamontology.org/data_3756</td>\n",
       "      <td>18</td>\n",
       "      <td>Localisation score</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1470</th>\n",
       "      <td>http://edamontology.org/data_3757</td>\n",
       "      <td>18</td>\n",
       "      <td>Unimod ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1471</th>\n",
       "      <td>http://edamontology.org/data_3759</td>\n",
       "      <td>18</td>\n",
       "      <td>ProteomeXchange ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1472</th>\n",
       "      <td>http://edamontology.org/data_3768</td>\n",
       "      <td>18</td>\n",
       "      <td>Clustered expression profiles</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1473</th>\n",
       "      <td>http://edamontology.org/data_3769</td>\n",
       "      <td>18</td>\n",
       "      <td>BRENDA ontology concept ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1474</th>\n",
       "      <td>http://edamontology.org/data_3779</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotated text</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1475</th>\n",
       "      <td>http://edamontology.org/data_3786</td>\n",
       "      <td>18</td>\n",
       "      <td>Query script</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1476</th>\n",
       "      <td>http://edamontology.org/data_3805</td>\n",
       "      <td>18</td>\n",
       "      <td>3D EM Map</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1477</th>\n",
       "      <td>http://edamontology.org/data_3806</td>\n",
       "      <td>18</td>\n",
       "      <td>3D EM Mask</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1478</th>\n",
       "      <td>http://edamontology.org/data_3807</td>\n",
       "      <td>18</td>\n",
       "      <td>EM Movie</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1479</th>\n",
       "      <td>http://edamontology.org/data_3808</td>\n",
       "      <td>18</td>\n",
       "      <td>EM Micrograph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1480</th>\n",
       "      <td>http://edamontology.org/data_3842</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular simulation data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1481</th>\n",
       "      <td>http://edamontology.org/data_3856</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA central ID</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1482</th>\n",
       "      <td>http://edamontology.org/data_3861</td>\n",
       "      <td>18</td>\n",
       "      <td>Electronic health record</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1483</th>\n",
       "      <td>http://edamontology.org/data_3869</td>\n",
       "      <td>18</td>\n",
       "      <td>Simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1484</th>\n",
       "      <td>http://edamontology.org/data_3870</td>\n",
       "      <td>18</td>\n",
       "      <td>Trajectory data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1485</th>\n",
       "      <td>http://edamontology.org/data_3871</td>\n",
       "      <td>18</td>\n",
       "      <td>Forcefield parameters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1486</th>\n",
       "      <td>http://edamontology.org/data_3872</td>\n",
       "      <td>18</td>\n",
       "      <td>Topology data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1487</th>\n",
       "      <td>http://edamontology.org/data_3905</td>\n",
       "      <td>18</td>\n",
       "      <td>Histogram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1488</th>\n",
       "      <td>http://edamontology.org/data_3914</td>\n",
       "      <td>18</td>\n",
       "      <td>Quality control report</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1489</th>\n",
       "      <td>http://edamontology.org/data_3917</td>\n",
       "      <td>18</td>\n",
       "      <td>Count matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1490</th>\n",
       "      <td>http://edamontology.org/data_3924</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1491</th>\n",
       "      <td>http://edamontology.org/data_3932</td>\n",
       "      <td>18</td>\n",
       "      <td>Q-value</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1492</th>\n",
       "      <td>http://edamontology.org/data_3949</td>\n",
       "      <td>18</td>\n",
       "      <td>Profile HMM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1493</th>\n",
       "      <td>http://edamontology.org/data_3952</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway ID (WikiPathways)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1494</th>\n",
       "      <td>http://edamontology.org/data_3953</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway overrepresentation data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1495</th>\n",
       "      <td>http://edamontology.org/data_4022</td>\n",
       "      <td>18</td>\n",
       "      <td>ORCID Identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1496</th>\n",
       "      <td>http://edamontology.org/deprecation_comment</td>\n",
       "      <td>18</td>\n",
       "      <td>deprecation_comment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1497</th>\n",
       "      <td>http://edamontology.org/documentation</td>\n",
       "      <td>18</td>\n",
       "      <td>Documentation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1498</th>\n",
       "      <td>http://edamontology.org/example</td>\n",
       "      <td>18</td>\n",
       "      <td>Example</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1499</th>\n",
       "      <td>http://edamontology.org/file_extension</td>\n",
       "      <td>18</td>\n",
       "      <td>File extension</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1500</th>\n",
       "      <td>http://edamontology.org/format_1196</td>\n",
       "      <td>18</td>\n",
       "      <td>SMILES</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1501</th>\n",
       "      <td>http://edamontology.org/format_1197</td>\n",
       "      <td>18</td>\n",
       "      <td>InChI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1502</th>\n",
       "      <td>http://edamontology.org/format_1198</td>\n",
       "      <td>18</td>\n",
       "      <td>mf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1503</th>\n",
       "      <td>http://edamontology.org/format_1199</td>\n",
       "      <td>18</td>\n",
       "      <td>InChIKey</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1504</th>\n",
       "      <td>http://edamontology.org/format_1200</td>\n",
       "      <td>18</td>\n",
       "      <td>smarts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1505</th>\n",
       "      <td>http://edamontology.org/format_1206</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous pure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1506</th>\n",
       "      <td>http://edamontology.org/format_1207</td>\n",
       "      <td>18</td>\n",
       "      <td>nucleotide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1507</th>\n",
       "      <td>http://edamontology.org/format_1208</td>\n",
       "      <td>18</td>\n",
       "      <td>protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1508</th>\n",
       "      <td>http://edamontology.org/format_1209</td>\n",
       "      <td>18</td>\n",
       "      <td>consensus</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1509</th>\n",
       "      <td>http://edamontology.org/format_1210</td>\n",
       "      <td>18</td>\n",
       "      <td>pure nucleotide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1510</th>\n",
       "      <td>http://edamontology.org/format_1211</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous pure nucleotide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1511</th>\n",
       "      <td>http://edamontology.org/format_1212</td>\n",
       "      <td>18</td>\n",
       "      <td>dna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1512</th>\n",
       "      <td>http://edamontology.org/format_1213</td>\n",
       "      <td>18</td>\n",
       "      <td>rna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1513</th>\n",
       "      <td>http://edamontology.org/format_1214</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous pure dna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1514</th>\n",
       "      <td>http://edamontology.org/format_1215</td>\n",
       "      <td>18</td>\n",
       "      <td>pure dna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1515</th>\n",
       "      <td>http://edamontology.org/format_1216</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous pure rna sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1516</th>\n",
       "      <td>http://edamontology.org/format_1217</td>\n",
       "      <td>18</td>\n",
       "      <td>pure rna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1517</th>\n",
       "      <td>http://edamontology.org/format_1218</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous pure protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1518</th>\n",
       "      <td>http://edamontology.org/format_1219</td>\n",
       "      <td>18</td>\n",
       "      <td>pure protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1519</th>\n",
       "      <td>http://edamontology.org/format_1228</td>\n",
       "      <td>18</td>\n",
       "      <td>UniGene entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1520</th>\n",
       "      <td>http://edamontology.org/format_1247</td>\n",
       "      <td>18</td>\n",
       "      <td>COG sequence cluster format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1521</th>\n",
       "      <td>http://edamontology.org/format_1248</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL feature location</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1522</th>\n",
       "      <td>http://edamontology.org/format_1295</td>\n",
       "      <td>18</td>\n",
       "      <td>quicktandem</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1523</th>\n",
       "      <td>http://edamontology.org/format_1296</td>\n",
       "      <td>18</td>\n",
       "      <td>Sanger inverted repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1524</th>\n",
       "      <td>http://edamontology.org/format_1297</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS repeat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1525</th>\n",
       "      <td>http://edamontology.org/format_1316</td>\n",
       "      <td>18</td>\n",
       "      <td>est2genome format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1526</th>\n",
       "      <td>http://edamontology.org/format_1318</td>\n",
       "      <td>18</td>\n",
       "      <td>restrict format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1527</th>\n",
       "      <td>http://edamontology.org/format_1319</td>\n",
       "      <td>18</td>\n",
       "      <td>restover format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1528</th>\n",
       "      <td>http://edamontology.org/format_1320</td>\n",
       "      <td>18</td>\n",
       "      <td>REBASE restriction sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1529</th>\n",
       "      <td>http://edamontology.org/format_1332</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA search results format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1530</th>\n",
       "      <td>http://edamontology.org/format_1333</td>\n",
       "      <td>18</td>\n",
       "      <td>BLAST results</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1531</th>\n",
       "      <td>http://edamontology.org/format_1334</td>\n",
       "      <td>18</td>\n",
       "      <td>mspcrunch</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1532</th>\n",
       "      <td>http://edamontology.org/format_1335</td>\n",
       "      <td>18</td>\n",
       "      <td>Smith-Waterman format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1533</th>\n",
       "      <td>http://edamontology.org/format_1336</td>\n",
       "      <td>18</td>\n",
       "      <td>dhf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1534</th>\n",
       "      <td>http://edamontology.org/format_1337</td>\n",
       "      <td>18</td>\n",
       "      <td>lhf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1535</th>\n",
       "      <td>http://edamontology.org/format_1341</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro hits format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1536</th>\n",
       "      <td>http://edamontology.org/format_1342</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro protein view report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1537</th>\n",
       "      <td>http://edamontology.org/format_1343</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro match table format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1538</th>\n",
       "      <td>http://edamontology.org/format_1349</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER Dirichlet prior</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1539</th>\n",
       "      <td>http://edamontology.org/format_1350</td>\n",
       "      <td>18</td>\n",
       "      <td>MEME Dirichlet prior</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1540</th>\n",
       "      <td>http://edamontology.org/format_1351</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER emission and transition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1541</th>\n",
       "      <td>http://edamontology.org/format_1356</td>\n",
       "      <td>18</td>\n",
       "      <td>prosite-pattern</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1542</th>\n",
       "      <td>http://edamontology.org/format_1357</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS sequence pattern</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1543</th>\n",
       "      <td>http://edamontology.org/format_1360</td>\n",
       "      <td>18</td>\n",
       "      <td>meme-motif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1544</th>\n",
       "      <td>http://edamontology.org/format_1366</td>\n",
       "      <td>18</td>\n",
       "      <td>prosite-profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1545</th>\n",
       "      <td>http://edamontology.org/format_1367</td>\n",
       "      <td>18</td>\n",
       "      <td>JASPAR format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1546</th>\n",
       "      <td>http://edamontology.org/format_1369</td>\n",
       "      <td>18</td>\n",
       "      <td>MEME background Markov model</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1547</th>\n",
       "      <td>http://edamontology.org/format_1370</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1548</th>\n",
       "      <td>http://edamontology.org/format_1391</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER-aln</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1549</th>\n",
       "      <td>http://edamontology.org/format_1392</td>\n",
       "      <td>18</td>\n",
       "      <td>DIALIGN format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1550</th>\n",
       "      <td>http://edamontology.org/format_1393</td>\n",
       "      <td>18</td>\n",
       "      <td>daf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1551</th>\n",
       "      <td>http://edamontology.org/format_1419</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-MEME profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1552</th>\n",
       "      <td>http://edamontology.org/format_1421</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER profile alignment (sequences versus HMMs)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1553</th>\n",
       "      <td>http://edamontology.org/format_1422</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER profile alignment (HMM versus sequences)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1554</th>\n",
       "      <td>http://edamontology.org/format_1423</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip distance matrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1555</th>\n",
       "      <td>http://edamontology.org/format_1424</td>\n",
       "      <td>18</td>\n",
       "      <td>ClustalW dendrogram</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1556</th>\n",
       "      <td>http://edamontology.org/format_1425</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip tree raw</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1557</th>\n",
       "      <td>http://edamontology.org/format_1430</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip continuous quantitative characters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1558</th>\n",
       "      <td>http://edamontology.org/format_1431</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic property values format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1559</th>\n",
       "      <td>http://edamontology.org/format_1432</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip character frequencies format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1560</th>\n",
       "      <td>http://edamontology.org/format_1433</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip discrete states format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1561</th>\n",
       "      <td>http://edamontology.org/format_1434</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip cliques format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1562</th>\n",
       "      <td>http://edamontology.org/format_1435</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip tree format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1563</th>\n",
       "      <td>http://edamontology.org/format_1436</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeBASE format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1564</th>\n",
       "      <td>http://edamontology.org/format_1437</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeFam format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1565</th>\n",
       "      <td>http://edamontology.org/format_1445</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip tree distance format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1566</th>\n",
       "      <td>http://edamontology.org/format_1454</td>\n",
       "      <td>18</td>\n",
       "      <td>dssp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1567</th>\n",
       "      <td>http://edamontology.org/format_1455</td>\n",
       "      <td>18</td>\n",
       "      <td>hssp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1568</th>\n",
       "      <td>http://edamontology.org/format_1457</td>\n",
       "      <td>18</td>\n",
       "      <td>Dot-bracket format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1569</th>\n",
       "      <td>http://edamontology.org/format_1458</td>\n",
       "      <td>18</td>\n",
       "      <td>Vienna local RNA secondary structure format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1570</th>\n",
       "      <td>http://edamontology.org/format_1475</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB database entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1571</th>\n",
       "      <td>http://edamontology.org/format_1476</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1572</th>\n",
       "      <td>http://edamontology.org/format_1477</td>\n",
       "      <td>18</td>\n",
       "      <td>mmCIF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1573</th>\n",
       "      <td>http://edamontology.org/format_1478</td>\n",
       "      <td>18</td>\n",
       "      <td>PDBML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1574</th>\n",
       "      <td>http://edamontology.org/format_1500</td>\n",
       "      <td>18</td>\n",
       "      <td>Domainatrix 3D-1D scoring matrix format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1575</th>\n",
       "      <td>http://edamontology.org/format_1504</td>\n",
       "      <td>18</td>\n",
       "      <td>aaindex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1576</th>\n",
       "      <td>http://edamontology.org/format_1511</td>\n",
       "      <td>18</td>\n",
       "      <td>IntEnz enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1577</th>\n",
       "      <td>http://edamontology.org/format_1512</td>\n",
       "      <td>18</td>\n",
       "      <td>BRENDA enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1578</th>\n",
       "      <td>http://edamontology.org/format_1513</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG REACTION enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1579</th>\n",
       "      <td>http://edamontology.org/format_1514</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG ENZYME enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1580</th>\n",
       "      <td>http://edamontology.org/format_1515</td>\n",
       "      <td>18</td>\n",
       "      <td>REBASE proto enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1581</th>\n",
       "      <td>http://edamontology.org/format_1516</td>\n",
       "      <td>18</td>\n",
       "      <td>REBASE withrefm enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1582</th>\n",
       "      <td>http://edamontology.org/format_1551</td>\n",
       "      <td>18</td>\n",
       "      <td>Pcons report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1583</th>\n",
       "      <td>http://edamontology.org/format_1552</td>\n",
       "      <td>18</td>\n",
       "      <td>ProQ report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1584</th>\n",
       "      <td>http://edamontology.org/format_1563</td>\n",
       "      <td>18</td>\n",
       "      <td>SMART domain assignment report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1585</th>\n",
       "      <td>http://edamontology.org/format_1568</td>\n",
       "      <td>18</td>\n",
       "      <td>BIND entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1586</th>\n",
       "      <td>http://edamontology.org/format_1569</td>\n",
       "      <td>18</td>\n",
       "      <td>IntAct entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1587</th>\n",
       "      <td>http://edamontology.org/format_1570</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1588</th>\n",
       "      <td>http://edamontology.org/format_1571</td>\n",
       "      <td>18</td>\n",
       "      <td>InterPro entry abstract format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1589</th>\n",
       "      <td>http://edamontology.org/format_1572</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene3D entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1590</th>\n",
       "      <td>http://edamontology.org/format_1573</td>\n",
       "      <td>18</td>\n",
       "      <td>PIRSF entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1591</th>\n",
       "      <td>http://edamontology.org/format_1574</td>\n",
       "      <td>18</td>\n",
       "      <td>PRINTS entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1592</th>\n",
       "      <td>http://edamontology.org/format_1575</td>\n",
       "      <td>18</td>\n",
       "      <td>Panther Families and HMMs entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1593</th>\n",
       "      <td>http://edamontology.org/format_1576</td>\n",
       "      <td>18</td>\n",
       "      <td>Pfam entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1594</th>\n",
       "      <td>http://edamontology.org/format_1577</td>\n",
       "      <td>18</td>\n",
       "      <td>SMART entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1595</th>\n",
       "      <td>http://edamontology.org/format_1578</td>\n",
       "      <td>18</td>\n",
       "      <td>Superfamily entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1596</th>\n",
       "      <td>http://edamontology.org/format_1579</td>\n",
       "      <td>18</td>\n",
       "      <td>TIGRFam entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1597</th>\n",
       "      <td>http://edamontology.org/format_1580</td>\n",
       "      <td>18</td>\n",
       "      <td>ProDom entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1598</th>\n",
       "      <td>http://edamontology.org/format_1581</td>\n",
       "      <td>18</td>\n",
       "      <td>FSSP entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1599</th>\n",
       "      <td>http://edamontology.org/format_1582</td>\n",
       "      <td>18</td>\n",
       "      <td>findkm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1600</th>\n",
       "      <td>http://edamontology.org/format_1603</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1601</th>\n",
       "      <td>http://edamontology.org/format_1604</td>\n",
       "      <td>18</td>\n",
       "      <td>DictyBase gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1602</th>\n",
       "      <td>http://edamontology.org/format_1605</td>\n",
       "      <td>18</td>\n",
       "      <td>CGD gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1603</th>\n",
       "      <td>http://edamontology.org/format_1606</td>\n",
       "      <td>18</td>\n",
       "      <td>DragonDB gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1604</th>\n",
       "      <td>http://edamontology.org/format_1607</td>\n",
       "      <td>18</td>\n",
       "      <td>EcoCyc gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1605</th>\n",
       "      <td>http://edamontology.org/format_1608</td>\n",
       "      <td>18</td>\n",
       "      <td>FlyBase gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1606</th>\n",
       "      <td>http://edamontology.org/format_1609</td>\n",
       "      <td>18</td>\n",
       "      <td>Gramene gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1607</th>\n",
       "      <td>http://edamontology.org/format_1610</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG GENES gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1608</th>\n",
       "      <td>http://edamontology.org/format_1611</td>\n",
       "      <td>18</td>\n",
       "      <td>MaizeGDB gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1609</th>\n",
       "      <td>http://edamontology.org/format_1612</td>\n",
       "      <td>18</td>\n",
       "      <td>MGD gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1610</th>\n",
       "      <td>http://edamontology.org/format_1613</td>\n",
       "      <td>18</td>\n",
       "      <td>RGD gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1611</th>\n",
       "      <td>http://edamontology.org/format_1614</td>\n",
       "      <td>18</td>\n",
       "      <td>SGD gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1612</th>\n",
       "      <td>http://edamontology.org/format_1615</td>\n",
       "      <td>18</td>\n",
       "      <td>GeneDB gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1613</th>\n",
       "      <td>http://edamontology.org/format_1616</td>\n",
       "      <td>18</td>\n",
       "      <td>TAIR gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1614</th>\n",
       "      <td>http://edamontology.org/format_1617</td>\n",
       "      <td>18</td>\n",
       "      <td>WormBase gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1615</th>\n",
       "      <td>http://edamontology.org/format_1618</td>\n",
       "      <td>18</td>\n",
       "      <td>ZFIN gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1616</th>\n",
       "      <td>http://edamontology.org/format_1619</td>\n",
       "      <td>18</td>\n",
       "      <td>TIGR gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1617</th>\n",
       "      <td>http://edamontology.org/format_1620</td>\n",
       "      <td>18</td>\n",
       "      <td>dbSNP polymorphism report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1618</th>\n",
       "      <td>http://edamontology.org/format_1623</td>\n",
       "      <td>18</td>\n",
       "      <td>OMIM entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1619</th>\n",
       "      <td>http://edamontology.org/format_1624</td>\n",
       "      <td>18</td>\n",
       "      <td>HGVbase entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1620</th>\n",
       "      <td>http://edamontology.org/format_1625</td>\n",
       "      <td>18</td>\n",
       "      <td>HIVDB entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1621</th>\n",
       "      <td>http://edamontology.org/format_1626</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG DISEASE entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1622</th>\n",
       "      <td>http://edamontology.org/format_1627</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer3 primer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1623</th>\n",
       "      <td>http://edamontology.org/format_1628</td>\n",
       "      <td>18</td>\n",
       "      <td>ABI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1624</th>\n",
       "      <td>http://edamontology.org/format_1629</td>\n",
       "      <td>18</td>\n",
       "      <td>mira</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1625</th>\n",
       "      <td>http://edamontology.org/format_1630</td>\n",
       "      <td>18</td>\n",
       "      <td>CAF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1626</th>\n",
       "      <td>http://edamontology.org/format_1631</td>\n",
       "      <td>18</td>\n",
       "      <td>EXP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1627</th>\n",
       "      <td>http://edamontology.org/format_1632</td>\n",
       "      <td>18</td>\n",
       "      <td>SCF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1628</th>\n",
       "      <td>http://edamontology.org/format_1633</td>\n",
       "      <td>18</td>\n",
       "      <td>PHD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1629</th>\n",
       "      <td>http://edamontology.org/format_1637</td>\n",
       "      <td>18</td>\n",
       "      <td>dat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1630</th>\n",
       "      <td>http://edamontology.org/format_1638</td>\n",
       "      <td>18</td>\n",
       "      <td>cel</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1631</th>\n",
       "      <td>http://edamontology.org/format_1639</td>\n",
       "      <td>18</td>\n",
       "      <td>affymetrix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1632</th>\n",
       "      <td>http://edamontology.org/format_1640</td>\n",
       "      <td>18</td>\n",
       "      <td>ArrayExpress entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1633</th>\n",
       "      <td>http://edamontology.org/format_1641</td>\n",
       "      <td>18</td>\n",
       "      <td>affymetrix-exp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1634</th>\n",
       "      <td>http://edamontology.org/format_1644</td>\n",
       "      <td>18</td>\n",
       "      <td>CHP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1635</th>\n",
       "      <td>http://edamontology.org/format_1645</td>\n",
       "      <td>18</td>\n",
       "      <td>EMDB entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1636</th>\n",
       "      <td>http://edamontology.org/format_1647</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG PATHWAY entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1637</th>\n",
       "      <td>http://edamontology.org/format_1648</td>\n",
       "      <td>18</td>\n",
       "      <td>MetaCyc entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1638</th>\n",
       "      <td>http://edamontology.org/format_1649</td>\n",
       "      <td>18</td>\n",
       "      <td>HumanCyc entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1639</th>\n",
       "      <td>http://edamontology.org/format_1650</td>\n",
       "      <td>18</td>\n",
       "      <td>INOH entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1640</th>\n",
       "      <td>http://edamontology.org/format_1651</td>\n",
       "      <td>18</td>\n",
       "      <td>PATIKA entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1641</th>\n",
       "      <td>http://edamontology.org/format_1652</td>\n",
       "      <td>18</td>\n",
       "      <td>Reactome entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1642</th>\n",
       "      <td>http://edamontology.org/format_1653</td>\n",
       "      <td>18</td>\n",
       "      <td>aMAZE entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1643</th>\n",
       "      <td>http://edamontology.org/format_1654</td>\n",
       "      <td>18</td>\n",
       "      <td>CPDB entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1644</th>\n",
       "      <td>http://edamontology.org/format_1655</td>\n",
       "      <td>18</td>\n",
       "      <td>Panther Pathways entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1645</th>\n",
       "      <td>http://edamontology.org/format_1665</td>\n",
       "      <td>18</td>\n",
       "      <td>Taverna workflow format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1646</th>\n",
       "      <td>http://edamontology.org/format_1666</td>\n",
       "      <td>18</td>\n",
       "      <td>BioModel mathematical model format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1647</th>\n",
       "      <td>http://edamontology.org/format_1697</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG LIGAND entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1648</th>\n",
       "      <td>http://edamontology.org/format_1698</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG COMPOUND entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1649</th>\n",
       "      <td>http://edamontology.org/format_1699</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG PLANT entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1650</th>\n",
       "      <td>http://edamontology.org/format_1700</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG GLYCAN entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1651</th>\n",
       "      <td>http://edamontology.org/format_1701</td>\n",
       "      <td>18</td>\n",
       "      <td>PubChem entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1652</th>\n",
       "      <td>http://edamontology.org/format_1702</td>\n",
       "      <td>18</td>\n",
       "      <td>ChemSpider entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1653</th>\n",
       "      <td>http://edamontology.org/format_1703</td>\n",
       "      <td>18</td>\n",
       "      <td>ChEBI entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1654</th>\n",
       "      <td>http://edamontology.org/format_1704</td>\n",
       "      <td>18</td>\n",
       "      <td>MSDchem ligand dictionary entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1655</th>\n",
       "      <td>http://edamontology.org/format_1705</td>\n",
       "      <td>18</td>\n",
       "      <td>HET group dictionary entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1656</th>\n",
       "      <td>http://edamontology.org/format_1706</td>\n",
       "      <td>18</td>\n",
       "      <td>KEGG DRUG entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1657</th>\n",
       "      <td>http://edamontology.org/format_1734</td>\n",
       "      <td>18</td>\n",
       "      <td>PubMed citation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1658</th>\n",
       "      <td>http://edamontology.org/format_1735</td>\n",
       "      <td>18</td>\n",
       "      <td>Medline Display Format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1659</th>\n",
       "      <td>http://edamontology.org/format_1736</td>\n",
       "      <td>18</td>\n",
       "      <td>CiteXplore-core</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1660</th>\n",
       "      <td>http://edamontology.org/format_1737</td>\n",
       "      <td>18</td>\n",
       "      <td>CiteXplore-all</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1661</th>\n",
       "      <td>http://edamontology.org/format_1739</td>\n",
       "      <td>18</td>\n",
       "      <td>pmc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1662</th>\n",
       "      <td>http://edamontology.org/format_1740</td>\n",
       "      <td>18</td>\n",
       "      <td>iHOP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1663</th>\n",
       "      <td>http://edamontology.org/format_1741</td>\n",
       "      <td>18</td>\n",
       "      <td>OSCAR format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1664</th>\n",
       "      <td>http://edamontology.org/format_1747</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB atom record format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1665</th>\n",
       "      <td>http://edamontology.org/format_1760</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH chain report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1666</th>\n",
       "      <td>http://edamontology.org/format_1761</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH PDB report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1667</th>\n",
       "      <td>http://edamontology.org/format_1782</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1668</th>\n",
       "      <td>http://edamontology.org/format_1808</td>\n",
       "      <td>18</td>\n",
       "      <td>GeneIlluminator gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1669</th>\n",
       "      <td>http://edamontology.org/format_1809</td>\n",
       "      <td>18</td>\n",
       "      <td>BacMap gene card format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1670</th>\n",
       "      <td>http://edamontology.org/format_1810</td>\n",
       "      <td>18</td>\n",
       "      <td>ColiCard report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1671</th>\n",
       "      <td>http://edamontology.org/format_1861</td>\n",
       "      <td>18</td>\n",
       "      <td>PlasMapper TextMap</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1672</th>\n",
       "      <td>http://edamontology.org/format_1910</td>\n",
       "      <td>18</td>\n",
       "      <td>newick</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1673</th>\n",
       "      <td>http://edamontology.org/format_1911</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeCon format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1674</th>\n",
       "      <td>http://edamontology.org/format_1912</td>\n",
       "      <td>18</td>\n",
       "      <td>Nexus format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1675</th>\n",
       "      <td>http://edamontology.org/format_1915</td>\n",
       "      <td>18</td>\n",
       "      <td>Format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1676</th>\n",
       "      <td>http://edamontology.org/format_1918</td>\n",
       "      <td>18</td>\n",
       "      <td>Atomic data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1677</th>\n",
       "      <td>http://edamontology.org/format_1919</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1678</th>\n",
       "      <td>http://edamontology.org/format_1920</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1679</th>\n",
       "      <td>http://edamontology.org/format_1921</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1680</th>\n",
       "      <td>http://edamontology.org/format_1923</td>\n",
       "      <td>18</td>\n",
       "      <td>acedb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1681</th>\n",
       "      <td>http://edamontology.org/format_1924</td>\n",
       "      <td>18</td>\n",
       "      <td>clustal sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1682</th>\n",
       "      <td>http://edamontology.org/format_1925</td>\n",
       "      <td>18</td>\n",
       "      <td>codata</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1683</th>\n",
       "      <td>http://edamontology.org/format_1926</td>\n",
       "      <td>18</td>\n",
       "      <td>dbid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1684</th>\n",
       "      <td>http://edamontology.org/format_1927</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1685</th>\n",
       "      <td>http://edamontology.org/format_1928</td>\n",
       "      <td>18</td>\n",
       "      <td>Staden experiment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1686</th>\n",
       "      <td>http://edamontology.org/format_1929</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1687</th>\n",
       "      <td>http://edamontology.org/format_1930</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1688</th>\n",
       "      <td>http://edamontology.org/format_1931</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ-illumina</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1689</th>\n",
       "      <td>http://edamontology.org/format_1932</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ-sanger</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1690</th>\n",
       "      <td>http://edamontology.org/format_1933</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ-solexa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1691</th>\n",
       "      <td>http://edamontology.org/format_1934</td>\n",
       "      <td>18</td>\n",
       "      <td>fitch program</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1692</th>\n",
       "      <td>http://edamontology.org/format_1935</td>\n",
       "      <td>18</td>\n",
       "      <td>GCG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1693</th>\n",
       "      <td>http://edamontology.org/format_1936</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1694</th>\n",
       "      <td>http://edamontology.org/format_1937</td>\n",
       "      <td>18</td>\n",
       "      <td>genpept</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1695</th>\n",
       "      <td>http://edamontology.org/format_1938</td>\n",
       "      <td>18</td>\n",
       "      <td>GFF2-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1696</th>\n",
       "      <td>http://edamontology.org/format_1939</td>\n",
       "      <td>18</td>\n",
       "      <td>GFF3-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1697</th>\n",
       "      <td>http://edamontology.org/format_1940</td>\n",
       "      <td>18</td>\n",
       "      <td>giFASTA format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1698</th>\n",
       "      <td>http://edamontology.org/format_1941</td>\n",
       "      <td>18</td>\n",
       "      <td>hennig86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1699</th>\n",
       "      <td>http://edamontology.org/format_1942</td>\n",
       "      <td>18</td>\n",
       "      <td>ig</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1700</th>\n",
       "      <td>http://edamontology.org/format_1943</td>\n",
       "      <td>18</td>\n",
       "      <td>igstrict</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1701</th>\n",
       "      <td>http://edamontology.org/format_1944</td>\n",
       "      <td>18</td>\n",
       "      <td>jackknifer</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1702</th>\n",
       "      <td>http://edamontology.org/format_1945</td>\n",
       "      <td>18</td>\n",
       "      <td>mase format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1703</th>\n",
       "      <td>http://edamontology.org/format_1946</td>\n",
       "      <td>18</td>\n",
       "      <td>mega-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1704</th>\n",
       "      <td>http://edamontology.org/format_1947</td>\n",
       "      <td>18</td>\n",
       "      <td>GCG MSF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1705</th>\n",
       "      <td>http://edamontology.org/format_1948</td>\n",
       "      <td>18</td>\n",
       "      <td>nbrf/pir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1706</th>\n",
       "      <td>http://edamontology.org/format_1949</td>\n",
       "      <td>18</td>\n",
       "      <td>nexus-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1707</th>\n",
       "      <td>http://edamontology.org/format_1950</td>\n",
       "      <td>18</td>\n",
       "      <td>pdbatom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1708</th>\n",
       "      <td>http://edamontology.org/format_1951</td>\n",
       "      <td>18</td>\n",
       "      <td>pdbatomnuc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1709</th>\n",
       "      <td>http://edamontology.org/format_1952</td>\n",
       "      <td>18</td>\n",
       "      <td>pdbseqresnuc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1710</th>\n",
       "      <td>http://edamontology.org/format_1953</td>\n",
       "      <td>18</td>\n",
       "      <td>pdbseqres</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1711</th>\n",
       "      <td>http://edamontology.org/format_1954</td>\n",
       "      <td>18</td>\n",
       "      <td>Pearson format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1712</th>\n",
       "      <td>http://edamontology.org/format_1955</td>\n",
       "      <td>18</td>\n",
       "      <td>phylip sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1713</th>\n",
       "      <td>http://edamontology.org/format_1956</td>\n",
       "      <td>18</td>\n",
       "      <td>phylipnon sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1714</th>\n",
       "      <td>http://edamontology.org/format_1957</td>\n",
       "      <td>18</td>\n",
       "      <td>raw</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1715</th>\n",
       "      <td>http://edamontology.org/format_1958</td>\n",
       "      <td>18</td>\n",
       "      <td>refseqp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1716</th>\n",
       "      <td>http://edamontology.org/format_1959</td>\n",
       "      <td>18</td>\n",
       "      <td>selex sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1717</th>\n",
       "      <td>http://edamontology.org/format_1960</td>\n",
       "      <td>18</td>\n",
       "      <td>Staden format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1718</th>\n",
       "      <td>http://edamontology.org/format_1961</td>\n",
       "      <td>18</td>\n",
       "      <td>Stockholm format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1719</th>\n",
       "      <td>http://edamontology.org/format_1962</td>\n",
       "      <td>18</td>\n",
       "      <td>strider format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1720</th>\n",
       "      <td>http://edamontology.org/format_1963</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProtKB format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1721</th>\n",
       "      <td>http://edamontology.org/format_1964</td>\n",
       "      <td>18</td>\n",
       "      <td>plain text format (unformatted)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1722</th>\n",
       "      <td>http://edamontology.org/format_1965</td>\n",
       "      <td>18</td>\n",
       "      <td>treecon sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1723</th>\n",
       "      <td>http://edamontology.org/format_1966</td>\n",
       "      <td>18</td>\n",
       "      <td>ASN.1 sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1724</th>\n",
       "      <td>http://edamontology.org/format_1967</td>\n",
       "      <td>18</td>\n",
       "      <td>DAS format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1725</th>\n",
       "      <td>http://edamontology.org/format_1968</td>\n",
       "      <td>18</td>\n",
       "      <td>dasdna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1726</th>\n",
       "      <td>http://edamontology.org/format_1969</td>\n",
       "      <td>18</td>\n",
       "      <td>debug-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1727</th>\n",
       "      <td>http://edamontology.org/format_1970</td>\n",
       "      <td>18</td>\n",
       "      <td>jackknifernon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1728</th>\n",
       "      <td>http://edamontology.org/format_1971</td>\n",
       "      <td>18</td>\n",
       "      <td>meganon sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1729</th>\n",
       "      <td>http://edamontology.org/format_1972</td>\n",
       "      <td>18</td>\n",
       "      <td>NCBI format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1730</th>\n",
       "      <td>http://edamontology.org/format_1973</td>\n",
       "      <td>18</td>\n",
       "      <td>nexusnon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1731</th>\n",
       "      <td>http://edamontology.org/format_1974</td>\n",
       "      <td>18</td>\n",
       "      <td>GFF2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1732</th>\n",
       "      <td>http://edamontology.org/format_1975</td>\n",
       "      <td>18</td>\n",
       "      <td>GFF3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1733</th>\n",
       "      <td>http://edamontology.org/format_1976</td>\n",
       "      <td>18</td>\n",
       "      <td>pir</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1734</th>\n",
       "      <td>http://edamontology.org/format_1977</td>\n",
       "      <td>18</td>\n",
       "      <td>swiss feature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1735</th>\n",
       "      <td>http://edamontology.org/format_1978</td>\n",
       "      <td>18</td>\n",
       "      <td>DASGFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1736</th>\n",
       "      <td>http://edamontology.org/format_1979</td>\n",
       "      <td>18</td>\n",
       "      <td>debug-feat</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1737</th>\n",
       "      <td>http://edamontology.org/format_1980</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL feature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1738</th>\n",
       "      <td>http://edamontology.org/format_1981</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank feature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1739</th>\n",
       "      <td>http://edamontology.org/format_1982</td>\n",
       "      <td>18</td>\n",
       "      <td>ClustalW format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1740</th>\n",
       "      <td>http://edamontology.org/format_1983</td>\n",
       "      <td>18</td>\n",
       "      <td>debug</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1741</th>\n",
       "      <td>http://edamontology.org/format_1984</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA-aln</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1742</th>\n",
       "      <td>http://edamontology.org/format_1985</td>\n",
       "      <td>18</td>\n",
       "      <td>markx0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1743</th>\n",
       "      <td>http://edamontology.org/format_1986</td>\n",
       "      <td>18</td>\n",
       "      <td>markx1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1744</th>\n",
       "      <td>http://edamontology.org/format_1987</td>\n",
       "      <td>18</td>\n",
       "      <td>markx10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1745</th>\n",
       "      <td>http://edamontology.org/format_1988</td>\n",
       "      <td>18</td>\n",
       "      <td>markx2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1746</th>\n",
       "      <td>http://edamontology.org/format_1989</td>\n",
       "      <td>18</td>\n",
       "      <td>markx3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1747</th>\n",
       "      <td>http://edamontology.org/format_1990</td>\n",
       "      <td>18</td>\n",
       "      <td>match</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1748</th>\n",
       "      <td>http://edamontology.org/format_1991</td>\n",
       "      <td>18</td>\n",
       "      <td>mega</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1749</th>\n",
       "      <td>http://edamontology.org/format_1992</td>\n",
       "      <td>18</td>\n",
       "      <td>meganon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1750</th>\n",
       "      <td>http://edamontology.org/format_1993</td>\n",
       "      <td>18</td>\n",
       "      <td>msf alignment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1751</th>\n",
       "      <td>http://edamontology.org/format_1994</td>\n",
       "      <td>18</td>\n",
       "      <td>nexus alignment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1752</th>\n",
       "      <td>http://edamontology.org/format_1995</td>\n",
       "      <td>18</td>\n",
       "      <td>nexusnon alignment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1753</th>\n",
       "      <td>http://edamontology.org/format_1996</td>\n",
       "      <td>18</td>\n",
       "      <td>pair</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1754</th>\n",
       "      <td>http://edamontology.org/format_1997</td>\n",
       "      <td>18</td>\n",
       "      <td>PHYLIP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1755</th>\n",
       "      <td>http://edamontology.org/format_1998</td>\n",
       "      <td>18</td>\n",
       "      <td>PHYLIP sequential</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1756</th>\n",
       "      <td>http://edamontology.org/format_1999</td>\n",
       "      <td>18</td>\n",
       "      <td>scores format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1757</th>\n",
       "      <td>http://edamontology.org/format_2000</td>\n",
       "      <td>18</td>\n",
       "      <td>selex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1758</th>\n",
       "      <td>http://edamontology.org/format_2001</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBOSS simple format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1759</th>\n",
       "      <td>http://edamontology.org/format_2002</td>\n",
       "      <td>18</td>\n",
       "      <td>srs format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1760</th>\n",
       "      <td>http://edamontology.org/format_2003</td>\n",
       "      <td>18</td>\n",
       "      <td>srspair</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1761</th>\n",
       "      <td>http://edamontology.org/format_2004</td>\n",
       "      <td>18</td>\n",
       "      <td>T-Coffee format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1762</th>\n",
       "      <td>http://edamontology.org/format_2005</td>\n",
       "      <td>18</td>\n",
       "      <td>TreeCon-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1763</th>\n",
       "      <td>http://edamontology.org/format_2006</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1764</th>\n",
       "      <td>http://edamontology.org/format_2013</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological pathway or network format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1765</th>\n",
       "      <td>http://edamontology.org/format_2014</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1766</th>\n",
       "      <td>http://edamontology.org/format_2015</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-profile alignment (HMM) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1767</th>\n",
       "      <td>http://edamontology.org/format_2017</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid index format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1768</th>\n",
       "      <td>http://edamontology.org/format_2020</td>\n",
       "      <td>18</td>\n",
       "      <td>Article format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1769</th>\n",
       "      <td>http://edamontology.org/format_2021</td>\n",
       "      <td>18</td>\n",
       "      <td>Text mining report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1770</th>\n",
       "      <td>http://edamontology.org/format_2027</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme kinetics report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1771</th>\n",
       "      <td>http://edamontology.org/format_2030</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1772</th>\n",
       "      <td>http://edamontology.org/format_2031</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1773</th>\n",
       "      <td>http://edamontology.org/format_2032</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflow format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1774</th>\n",
       "      <td>http://edamontology.org/format_2033</td>\n",
       "      <td>18</td>\n",
       "      <td>Tertiary structure format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1775</th>\n",
       "      <td>http://edamontology.org/format_2034</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological model format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1776</th>\n",
       "      <td>http://edamontology.org/format_2035</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical formula format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1777</th>\n",
       "      <td>http://edamontology.org/format_2036</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic character data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1778</th>\n",
       "      <td>http://edamontology.org/format_2037</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic continuous quantitative character format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1779</th>\n",
       "      <td>http://edamontology.org/format_2038</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic discrete states format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1780</th>\n",
       "      <td>http://edamontology.org/format_2039</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (cliques) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1781</th>\n",
       "      <td>http://edamontology.org/format_2040</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (invariants) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1782</th>\n",
       "      <td>http://edamontology.org/format_2045</td>\n",
       "      <td>18</td>\n",
       "      <td>Electron microscopy model format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1783</th>\n",
       "      <td>http://edamontology.org/format_2049</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree report (tree distances) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1784</th>\n",
       "      <td>http://edamontology.org/format_2051</td>\n",
       "      <td>18</td>\n",
       "      <td>Polymorphism report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1785</th>\n",
       "      <td>http://edamontology.org/format_2052</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein family report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1786</th>\n",
       "      <td>http://edamontology.org/format_2054</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1787</th>\n",
       "      <td>http://edamontology.org/format_2055</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1788</th>\n",
       "      <td>http://edamontology.org/format_2056</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray experiment data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1789</th>\n",
       "      <td>http://edamontology.org/format_2057</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence trace format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1790</th>\n",
       "      <td>http://edamontology.org/format_2058</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1791</th>\n",
       "      <td>http://edamontology.org/format_2059</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype and phenotype annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1792</th>\n",
       "      <td>http://edamontology.org/format_2060</td>\n",
       "      <td>18</td>\n",
       "      <td>Map format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1793</th>\n",
       "      <td>http://edamontology.org/format_2061</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (primers) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1794</th>\n",
       "      <td>http://edamontology.org/format_2062</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1795</th>\n",
       "      <td>http://edamontology.org/format_2063</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein report (enzyme) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1796</th>\n",
       "      <td>http://edamontology.org/format_2064</td>\n",
       "      <td>18</td>\n",
       "      <td>3D-1D scoring matrix format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1797</th>\n",
       "      <td>http://edamontology.org/format_2065</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure report (quality evaluation) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1798</th>\n",
       "      <td>http://edamontology.org/format_2066</td>\n",
       "      <td>18</td>\n",
       "      <td>Database hits (sequence) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1799</th>\n",
       "      <td>http://edamontology.org/format_2067</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence distance matrix format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1800</th>\n",
       "      <td>http://edamontology.org/format_2068</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1801</th>\n",
       "      <td>http://edamontology.org/format_2069</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1802</th>\n",
       "      <td>http://edamontology.org/format_2072</td>\n",
       "      <td>18</td>\n",
       "      <td>Hidden Markov model format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1803</th>\n",
       "      <td>http://edamontology.org/format_2074</td>\n",
       "      <td>18</td>\n",
       "      <td>Dirichlet distribution format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1804</th>\n",
       "      <td>http://edamontology.org/format_2075</td>\n",
       "      <td>18</td>\n",
       "      <td>HMM emission and transition counts format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1805</th>\n",
       "      <td>http://edamontology.org/format_2076</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1806</th>\n",
       "      <td>http://edamontology.org/format_2077</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1807</th>\n",
       "      <td>http://edamontology.org/format_2078</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence range format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1808</th>\n",
       "      <td>http://edamontology.org/format_2094</td>\n",
       "      <td>18</td>\n",
       "      <td>pure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1809</th>\n",
       "      <td>http://edamontology.org/format_2095</td>\n",
       "      <td>18</td>\n",
       "      <td>unpure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1810</th>\n",
       "      <td>http://edamontology.org/format_2096</td>\n",
       "      <td>18</td>\n",
       "      <td>unambiguous sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1811</th>\n",
       "      <td>http://edamontology.org/format_2097</td>\n",
       "      <td>18</td>\n",
       "      <td>ambiguous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1812</th>\n",
       "      <td>http://edamontology.org/format_2155</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence features (repeats) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1813</th>\n",
       "      <td>http://edamontology.org/format_2158</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid features (restriction sites) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1814</th>\n",
       "      <td>http://edamontology.org/format_2159</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene features (coding region) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1815</th>\n",
       "      <td>http://edamontology.org/format_2170</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1816</th>\n",
       "      <td>http://edamontology.org/format_2171</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster format (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1817</th>\n",
       "      <td>http://edamontology.org/format_2172</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster format (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1818</th>\n",
       "      <td>http://edamontology.org/format_2175</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene cluster format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1819</th>\n",
       "      <td>http://edamontology.org/format_2181</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL-like (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1820</th>\n",
       "      <td>http://edamontology.org/format_2182</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ-like format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1821</th>\n",
       "      <td>http://edamontology.org/format_2183</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBLXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1822</th>\n",
       "      <td>http://edamontology.org/format_2184</td>\n",
       "      <td>18</td>\n",
       "      <td>cdsxml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1823</th>\n",
       "      <td>http://edamontology.org/format_2185</td>\n",
       "      <td>18</td>\n",
       "      <td>INSDSeq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1824</th>\n",
       "      <td>http://edamontology.org/format_2186</td>\n",
       "      <td>18</td>\n",
       "      <td>geneseq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1825</th>\n",
       "      <td>http://edamontology.org/format_2187</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt-like (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1826</th>\n",
       "      <td>http://edamontology.org/format_2188</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProt format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1827</th>\n",
       "      <td>http://edamontology.org/format_2189</td>\n",
       "      <td>18</td>\n",
       "      <td>ipi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1828</th>\n",
       "      <td>http://edamontology.org/format_2194</td>\n",
       "      <td>18</td>\n",
       "      <td>medline</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1829</th>\n",
       "      <td>http://edamontology.org/format_2195</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1830</th>\n",
       "      <td>http://edamontology.org/format_2196</td>\n",
       "      <td>18</td>\n",
       "      <td>OBO format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1831</th>\n",
       "      <td>http://edamontology.org/format_2197</td>\n",
       "      <td>18</td>\n",
       "      <td>OWL format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1832</th>\n",
       "      <td>http://edamontology.org/format_2200</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA-like (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1833</th>\n",
       "      <td>http://edamontology.org/format_2202</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record full format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1834</th>\n",
       "      <td>http://edamontology.org/format_2203</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record lite format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1835</th>\n",
       "      <td>http://edamontology.org/format_2204</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1836</th>\n",
       "      <td>http://edamontology.org/format_2205</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank-like format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1837</th>\n",
       "      <td>http://edamontology.org/format_2206</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature table format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1838</th>\n",
       "      <td>http://edamontology.org/format_2210</td>\n",
       "      <td>18</td>\n",
       "      <td>Strain data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1839</th>\n",
       "      <td>http://edamontology.org/format_2211</td>\n",
       "      <td>18</td>\n",
       "      <td>CIP strain data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1840</th>\n",
       "      <td>http://edamontology.org/format_2243</td>\n",
       "      <td>18</td>\n",
       "      <td>phylip property values</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1841</th>\n",
       "      <td>http://edamontology.org/format_2303</td>\n",
       "      <td>18</td>\n",
       "      <td>STRING entry format (HTML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1842</th>\n",
       "      <td>http://edamontology.org/format_2304</td>\n",
       "      <td>18</td>\n",
       "      <td>STRING entry format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1843</th>\n",
       "      <td>http://edamontology.org/format_2305</td>\n",
       "      <td>18</td>\n",
       "      <td>GFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1844</th>\n",
       "      <td>http://edamontology.org/format_2306</td>\n",
       "      <td>18</td>\n",
       "      <td>GTF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1845</th>\n",
       "      <td>http://edamontology.org/format_2310</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA-HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1846</th>\n",
       "      <td>http://edamontology.org/format_2311</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL-HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1847</th>\n",
       "      <td>http://edamontology.org/format_2322</td>\n",
       "      <td>18</td>\n",
       "      <td>BioCyc enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1848</th>\n",
       "      <td>http://edamontology.org/format_2323</td>\n",
       "      <td>18</td>\n",
       "      <td>ENZYME enzyme report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1849</th>\n",
       "      <td>http://edamontology.org/format_2328</td>\n",
       "      <td>18</td>\n",
       "      <td>PseudoCAP gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1850</th>\n",
       "      <td>http://edamontology.org/format_2329</td>\n",
       "      <td>18</td>\n",
       "      <td>GeneCards gene report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1851</th>\n",
       "      <td>http://edamontology.org/format_2330</td>\n",
       "      <td>18</td>\n",
       "      <td>Textual format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1852</th>\n",
       "      <td>http://edamontology.org/format_2331</td>\n",
       "      <td>18</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1853</th>\n",
       "      <td>http://edamontology.org/format_2332</td>\n",
       "      <td>18</td>\n",
       "      <td>XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1854</th>\n",
       "      <td>http://edamontology.org/format_2333</td>\n",
       "      <td>18</td>\n",
       "      <td>Binary format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1855</th>\n",
       "      <td>http://edamontology.org/format_2334</td>\n",
       "      <td>18</td>\n",
       "      <td>URI format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1856</th>\n",
       "      <td>http://edamontology.org/format_2341</td>\n",
       "      <td>18</td>\n",
       "      <td>NCI-Nature pathway entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1857</th>\n",
       "      <td>http://edamontology.org/format_2350</td>\n",
       "      <td>18</td>\n",
       "      <td>Format (by type of data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1858</th>\n",
       "      <td>http://edamontology.org/format_2352</td>\n",
       "      <td>18</td>\n",
       "      <td>BioXSD (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1859</th>\n",
       "      <td>http://edamontology.org/format_2376</td>\n",
       "      <td>18</td>\n",
       "      <td>RDF format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1860</th>\n",
       "      <td>http://edamontology.org/format_2532</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank-HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1861</th>\n",
       "      <td>http://edamontology.org/format_2542</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein features (domains) format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1862</th>\n",
       "      <td>http://edamontology.org/format_2543</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL-like format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1863</th>\n",
       "      <td>http://edamontology.org/format_2545</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTQ-like format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1864</th>\n",
       "      <td>http://edamontology.org/format_2546</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTA-like</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1865</th>\n",
       "      <td>http://edamontology.org/format_2547</td>\n",
       "      <td>18</td>\n",
       "      <td>uniprotkb-like format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1866</th>\n",
       "      <td>http://edamontology.org/format_2548</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature table format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1867</th>\n",
       "      <td>http://edamontology.org/format_2549</td>\n",
       "      <td>18</td>\n",
       "      <td>OBO</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1868</th>\n",
       "      <td>http://edamontology.org/format_2550</td>\n",
       "      <td>18</td>\n",
       "      <td>OBO-XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1869</th>\n",
       "      <td>http://edamontology.org/format_2551</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1870</th>\n",
       "      <td>http://edamontology.org/format_2552</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence record format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1871</th>\n",
       "      <td>http://edamontology.org/format_2553</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature table format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1872</th>\n",
       "      <td>http://edamontology.org/format_2554</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1873</th>\n",
       "      <td>http://edamontology.org/format_2555</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1874</th>\n",
       "      <td>http://edamontology.org/format_2556</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1875</th>\n",
       "      <td>http://edamontology.org/format_2557</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree format (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1876</th>\n",
       "      <td>http://edamontology.org/format_2558</td>\n",
       "      <td>18</td>\n",
       "      <td>EMBL-like (XML)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1877</th>\n",
       "      <td>http://edamontology.org/format_2559</td>\n",
       "      <td>18</td>\n",
       "      <td>GenBank-like format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1878</th>\n",
       "      <td>http://edamontology.org/format_2560</td>\n",
       "      <td>18</td>\n",
       "      <td>STRING entry format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1879</th>\n",
       "      <td>http://edamontology.org/format_2561</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1880</th>\n",
       "      <td>http://edamontology.org/format_2562</td>\n",
       "      <td>18</td>\n",
       "      <td>Amino acid identifier format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1881</th>\n",
       "      <td>http://edamontology.org/format_2566</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1882</th>\n",
       "      <td>http://edamontology.org/format_2567</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous pure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1883</th>\n",
       "      <td>http://edamontology.org/format_2568</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous pure nucleotide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1884</th>\n",
       "      <td>http://edamontology.org/format_2569</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous pure dna</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1885</th>\n",
       "      <td>http://edamontology.org/format_2570</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous pure rna sequence</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1886</th>\n",
       "      <td>http://edamontology.org/format_2571</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw sequence format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1887</th>\n",
       "      <td>http://edamontology.org/format_2572</td>\n",
       "      <td>18</td>\n",
       "      <td>BAM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1888</th>\n",
       "      <td>http://edamontology.org/format_2573</td>\n",
       "      <td>18</td>\n",
       "      <td>SAM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1889</th>\n",
       "      <td>http://edamontology.org/format_2585</td>\n",
       "      <td>18</td>\n",
       "      <td>SBML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1890</th>\n",
       "      <td>http://edamontology.org/format_2607</td>\n",
       "      <td>18</td>\n",
       "      <td>completely unambiguous pure protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1891</th>\n",
       "      <td>http://edamontology.org/format_2848</td>\n",
       "      <td>18</td>\n",
       "      <td>Bibliographic reference format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1892</th>\n",
       "      <td>http://edamontology.org/format_2919</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence annotation track format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1893</th>\n",
       "      <td>http://edamontology.org/format_2920</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment format (pair only)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1894</th>\n",
       "      <td>http://edamontology.org/format_2921</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence variation annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1895</th>\n",
       "      <td>http://edamontology.org/format_2922</td>\n",
       "      <td>18</td>\n",
       "      <td>markx0 variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1896</th>\n",
       "      <td>http://edamontology.org/format_2923</td>\n",
       "      <td>18</td>\n",
       "      <td>mega variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1897</th>\n",
       "      <td>http://edamontology.org/format_2924</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylip format variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1898</th>\n",
       "      <td>http://edamontology.org/format_3000</td>\n",
       "      <td>18</td>\n",
       "      <td>AB1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1899</th>\n",
       "      <td>http://edamontology.org/format_3001</td>\n",
       "      <td>18</td>\n",
       "      <td>ACE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1900</th>\n",
       "      <td>http://edamontology.org/format_3003</td>\n",
       "      <td>18</td>\n",
       "      <td>BED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1901</th>\n",
       "      <td>http://edamontology.org/format_3004</td>\n",
       "      <td>18</td>\n",
       "      <td>bigBed</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>http://edamontology.org/format_3005</td>\n",
       "      <td>18</td>\n",
       "      <td>WIG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1903</th>\n",
       "      <td>http://edamontology.org/format_3006</td>\n",
       "      <td>18</td>\n",
       "      <td>bigWig</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1904</th>\n",
       "      <td>http://edamontology.org/format_3007</td>\n",
       "      <td>18</td>\n",
       "      <td>PSL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1905</th>\n",
       "      <td>http://edamontology.org/format_3008</td>\n",
       "      <td>18</td>\n",
       "      <td>MAF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1906</th>\n",
       "      <td>http://edamontology.org/format_3009</td>\n",
       "      <td>18</td>\n",
       "      <td>2bit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1907</th>\n",
       "      <td>http://edamontology.org/format_3010</td>\n",
       "      <td>18</td>\n",
       "      <td>.nib</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1908</th>\n",
       "      <td>http://edamontology.org/format_3011</td>\n",
       "      <td>18</td>\n",
       "      <td>genePred</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1909</th>\n",
       "      <td>http://edamontology.org/format_3012</td>\n",
       "      <td>18</td>\n",
       "      <td>pgSnp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1910</th>\n",
       "      <td>http://edamontology.org/format_3013</td>\n",
       "      <td>18</td>\n",
       "      <td>axt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1911</th>\n",
       "      <td>http://edamontology.org/format_3014</td>\n",
       "      <td>18</td>\n",
       "      <td>LAV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1912</th>\n",
       "      <td>http://edamontology.org/format_3015</td>\n",
       "      <td>18</td>\n",
       "      <td>Pileup</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1913</th>\n",
       "      <td>http://edamontology.org/format_3016</td>\n",
       "      <td>18</td>\n",
       "      <td>VCF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1914</th>\n",
       "      <td>http://edamontology.org/format_3017</td>\n",
       "      <td>18</td>\n",
       "      <td>SRF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1915</th>\n",
       "      <td>http://edamontology.org/format_3018</td>\n",
       "      <td>18</td>\n",
       "      <td>ZTR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1916</th>\n",
       "      <td>http://edamontology.org/format_3019</td>\n",
       "      <td>18</td>\n",
       "      <td>GVF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1917</th>\n",
       "      <td>http://edamontology.org/format_3020</td>\n",
       "      <td>18</td>\n",
       "      <td>BCF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1918</th>\n",
       "      <td>http://edamontology.org/format_3033</td>\n",
       "      <td>18</td>\n",
       "      <td>Matrix format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1919</th>\n",
       "      <td>http://edamontology.org/format_3097</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain classification format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1920</th>\n",
       "      <td>http://edamontology.org/format_3098</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw SCOP domain classification format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1921</th>\n",
       "      <td>http://edamontology.org/format_3099</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw CATH domain classification format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1922</th>\n",
       "      <td>http://edamontology.org/format_3100</td>\n",
       "      <td>18</td>\n",
       "      <td>CATH domain report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1923</th>\n",
       "      <td>http://edamontology.org/format_3155</td>\n",
       "      <td>18</td>\n",
       "      <td>SBRML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1924</th>\n",
       "      <td>http://edamontology.org/format_3156</td>\n",
       "      <td>18</td>\n",
       "      <td>BioPAX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1925</th>\n",
       "      <td>http://edamontology.org/format_3157</td>\n",
       "      <td>18</td>\n",
       "      <td>EBI Application Result XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1926</th>\n",
       "      <td>http://edamontology.org/format_3158</td>\n",
       "      <td>18</td>\n",
       "      <td>PSI MI XML (MIF)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1927</th>\n",
       "      <td>http://edamontology.org/format_3159</td>\n",
       "      <td>18</td>\n",
       "      <td>phyloXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1928</th>\n",
       "      <td>http://edamontology.org/format_3160</td>\n",
       "      <td>18</td>\n",
       "      <td>NeXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1929</th>\n",
       "      <td>http://edamontology.org/format_3161</td>\n",
       "      <td>18</td>\n",
       "      <td>MAGE-ML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1930</th>\n",
       "      <td>http://edamontology.org/format_3162</td>\n",
       "      <td>18</td>\n",
       "      <td>MAGE-TAB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1931</th>\n",
       "      <td>http://edamontology.org/format_3163</td>\n",
       "      <td>18</td>\n",
       "      <td>GCDML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1932</th>\n",
       "      <td>http://edamontology.org/format_3164</td>\n",
       "      <td>18</td>\n",
       "      <td>GTrack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1933</th>\n",
       "      <td>http://edamontology.org/format_3166</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological pathway or network report format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1934</th>\n",
       "      <td>http://edamontology.org/format_3167</td>\n",
       "      <td>18</td>\n",
       "      <td>Experiment annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1935</th>\n",
       "      <td>http://edamontology.org/format_3235</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytoband format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1936</th>\n",
       "      <td>http://edamontology.org/format_3239</td>\n",
       "      <td>18</td>\n",
       "      <td>CopasiML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1937</th>\n",
       "      <td>http://edamontology.org/format_3240</td>\n",
       "      <td>18</td>\n",
       "      <td>CellML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1938</th>\n",
       "      <td>http://edamontology.org/format_3242</td>\n",
       "      <td>18</td>\n",
       "      <td>PSI MI TAB (MITAB)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1939</th>\n",
       "      <td>http://edamontology.org/format_3243</td>\n",
       "      <td>18</td>\n",
       "      <td>PSI-PAR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1940</th>\n",
       "      <td>http://edamontology.org/format_3244</td>\n",
       "      <td>18</td>\n",
       "      <td>mzML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1941</th>\n",
       "      <td>http://edamontology.org/format_3245</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrometry data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1942</th>\n",
       "      <td>http://edamontology.org/format_3246</td>\n",
       "      <td>18</td>\n",
       "      <td>TraML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1943</th>\n",
       "      <td>http://edamontology.org/format_3247</td>\n",
       "      <td>18</td>\n",
       "      <td>mzIdentML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1944</th>\n",
       "      <td>http://edamontology.org/format_3248</td>\n",
       "      <td>18</td>\n",
       "      <td>mzQuantML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1945</th>\n",
       "      <td>http://edamontology.org/format_3249</td>\n",
       "      <td>18</td>\n",
       "      <td>GelML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1946</th>\n",
       "      <td>http://edamontology.org/format_3250</td>\n",
       "      <td>18</td>\n",
       "      <td>spML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1947</th>\n",
       "      <td>http://edamontology.org/format_3252</td>\n",
       "      <td>18</td>\n",
       "      <td>OWL Functional Syntax</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1948</th>\n",
       "      <td>http://edamontology.org/format_3253</td>\n",
       "      <td>18</td>\n",
       "      <td>Manchester OWL Syntax</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1949</th>\n",
       "      <td>http://edamontology.org/format_3254</td>\n",
       "      <td>18</td>\n",
       "      <td>KRSS2 Syntax</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1950</th>\n",
       "      <td>http://edamontology.org/format_3255</td>\n",
       "      <td>18</td>\n",
       "      <td>Turtle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1951</th>\n",
       "      <td>http://edamontology.org/format_3256</td>\n",
       "      <td>18</td>\n",
       "      <td>N-Triples</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1952</th>\n",
       "      <td>http://edamontology.org/format_3257</td>\n",
       "      <td>18</td>\n",
       "      <td>Notation3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1953</th>\n",
       "      <td>http://edamontology.org/format_3261</td>\n",
       "      <td>18</td>\n",
       "      <td>RDF/XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1954</th>\n",
       "      <td>http://edamontology.org/format_3262</td>\n",
       "      <td>18</td>\n",
       "      <td>OWL/XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1955</th>\n",
       "      <td>http://edamontology.org/format_3281</td>\n",
       "      <td>18</td>\n",
       "      <td>A2M</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1956</th>\n",
       "      <td>http://edamontology.org/format_3284</td>\n",
       "      <td>18</td>\n",
       "      <td>SFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1957</th>\n",
       "      <td>http://edamontology.org/format_3285</td>\n",
       "      <td>18</td>\n",
       "      <td>MAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1958</th>\n",
       "      <td>http://edamontology.org/format_3286</td>\n",
       "      <td>18</td>\n",
       "      <td>PED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1959</th>\n",
       "      <td>http://edamontology.org/format_3287</td>\n",
       "      <td>18</td>\n",
       "      <td>Individual genetic data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1960</th>\n",
       "      <td>http://edamontology.org/format_3288</td>\n",
       "      <td>18</td>\n",
       "      <td>PED/MAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1961</th>\n",
       "      <td>http://edamontology.org/format_3309</td>\n",
       "      <td>18</td>\n",
       "      <td>CT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1962</th>\n",
       "      <td>http://edamontology.org/format_3310</td>\n",
       "      <td>18</td>\n",
       "      <td>SS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1963</th>\n",
       "      <td>http://edamontology.org/format_3311</td>\n",
       "      <td>18</td>\n",
       "      <td>RNAML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1964</th>\n",
       "      <td>http://edamontology.org/format_3312</td>\n",
       "      <td>18</td>\n",
       "      <td>GDE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1965</th>\n",
       "      <td>http://edamontology.org/format_3313</td>\n",
       "      <td>18</td>\n",
       "      <td>BLC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1966</th>\n",
       "      <td>http://edamontology.org/format_3326</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1967</th>\n",
       "      <td>http://edamontology.org/format_3327</td>\n",
       "      <td>18</td>\n",
       "      <td>BAI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1968</th>\n",
       "      <td>http://edamontology.org/format_3328</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1969</th>\n",
       "      <td>http://edamontology.org/format_3329</td>\n",
       "      <td>18</td>\n",
       "      <td>HMMER3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1970</th>\n",
       "      <td>http://edamontology.org/format_3330</td>\n",
       "      <td>18</td>\n",
       "      <td>PO</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1971</th>\n",
       "      <td>http://edamontology.org/format_3331</td>\n",
       "      <td>18</td>\n",
       "      <td>BLAST XML results format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1972</th>\n",
       "      <td>http://edamontology.org/format_3462</td>\n",
       "      <td>18</td>\n",
       "      <td>CRAM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1973</th>\n",
       "      <td>http://edamontology.org/format_3464</td>\n",
       "      <td>18</td>\n",
       "      <td>JSON</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1974</th>\n",
       "      <td>http://edamontology.org/format_3466</td>\n",
       "      <td>18</td>\n",
       "      <td>EPS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1975</th>\n",
       "      <td>http://edamontology.org/format_3467</td>\n",
       "      <td>18</td>\n",
       "      <td>GIF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1976</th>\n",
       "      <td>http://edamontology.org/format_3468</td>\n",
       "      <td>18</td>\n",
       "      <td>xls</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1977</th>\n",
       "      <td>http://edamontology.org/format_3475</td>\n",
       "      <td>18</td>\n",
       "      <td>TSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1978</th>\n",
       "      <td>http://edamontology.org/format_3476</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1979</th>\n",
       "      <td>http://edamontology.org/format_3477</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytoscape input file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1980</th>\n",
       "      <td>http://edamontology.org/format_3484</td>\n",
       "      <td>18</td>\n",
       "      <td>ebwt</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1981</th>\n",
       "      <td>http://edamontology.org/format_3485</td>\n",
       "      <td>18</td>\n",
       "      <td>RSF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1982</th>\n",
       "      <td>http://edamontology.org/format_3486</td>\n",
       "      <td>18</td>\n",
       "      <td>GCG format variant</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1983</th>\n",
       "      <td>http://edamontology.org/format_3487</td>\n",
       "      <td>18</td>\n",
       "      <td>BSML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1984</th>\n",
       "      <td>http://edamontology.org/format_3491</td>\n",
       "      <td>18</td>\n",
       "      <td>ebwtl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1985</th>\n",
       "      <td>http://edamontology.org/format_3499</td>\n",
       "      <td>18</td>\n",
       "      <td>Ensembl variation file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1986</th>\n",
       "      <td>http://edamontology.org/format_3506</td>\n",
       "      <td>18</td>\n",
       "      <td>docx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1987</th>\n",
       "      <td>http://edamontology.org/format_3507</td>\n",
       "      <td>18</td>\n",
       "      <td>Document format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1988</th>\n",
       "      <td>http://edamontology.org/format_3508</td>\n",
       "      <td>18</td>\n",
       "      <td>PDF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1989</th>\n",
       "      <td>http://edamontology.org/format_3547</td>\n",
       "      <td>18</td>\n",
       "      <td>Image format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1990</th>\n",
       "      <td>http://edamontology.org/format_3548</td>\n",
       "      <td>18</td>\n",
       "      <td>DICOM format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1991</th>\n",
       "      <td>http://edamontology.org/format_3549</td>\n",
       "      <td>18</td>\n",
       "      <td>nii</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1992</th>\n",
       "      <td>http://edamontology.org/format_3550</td>\n",
       "      <td>18</td>\n",
       "      <td>mhd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1993</th>\n",
       "      <td>http://edamontology.org/format_3551</td>\n",
       "      <td>18</td>\n",
       "      <td>nrrd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1994</th>\n",
       "      <td>http://edamontology.org/format_3554</td>\n",
       "      <td>18</td>\n",
       "      <td>R file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1995</th>\n",
       "      <td>http://edamontology.org/format_3555</td>\n",
       "      <td>18</td>\n",
       "      <td>SPSS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1996</th>\n",
       "      <td>http://edamontology.org/format_3556</td>\n",
       "      <td>18</td>\n",
       "      <td>MHTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1997</th>\n",
       "      <td>http://edamontology.org/format_3578</td>\n",
       "      <td>18</td>\n",
       "      <td>IDAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1998</th>\n",
       "      <td>http://edamontology.org/format_3579</td>\n",
       "      <td>18</td>\n",
       "      <td>JPG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1999</th>\n",
       "      <td>http://edamontology.org/format_3580</td>\n",
       "      <td>18</td>\n",
       "      <td>rcc</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2000</th>\n",
       "      <td>http://edamontology.org/format_3581</td>\n",
       "      <td>18</td>\n",
       "      <td>arff</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2001</th>\n",
       "      <td>http://edamontology.org/format_3582</td>\n",
       "      <td>18</td>\n",
       "      <td>afg</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2002</th>\n",
       "      <td>http://edamontology.org/format_3583</td>\n",
       "      <td>18</td>\n",
       "      <td>bedgraph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2003</th>\n",
       "      <td>http://edamontology.org/format_3584</td>\n",
       "      <td>18</td>\n",
       "      <td>bedstrict</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2004</th>\n",
       "      <td>http://edamontology.org/format_3585</td>\n",
       "      <td>18</td>\n",
       "      <td>bed6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2005</th>\n",
       "      <td>http://edamontology.org/format_3586</td>\n",
       "      <td>18</td>\n",
       "      <td>bed12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2006</th>\n",
       "      <td>http://edamontology.org/format_3587</td>\n",
       "      <td>18</td>\n",
       "      <td>chrominfo</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2007</th>\n",
       "      <td>http://edamontology.org/format_3588</td>\n",
       "      <td>18</td>\n",
       "      <td>customtrack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2008</th>\n",
       "      <td>http://edamontology.org/format_3589</td>\n",
       "      <td>18</td>\n",
       "      <td>csfasta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2009</th>\n",
       "      <td>http://edamontology.org/format_3590</td>\n",
       "      <td>18</td>\n",
       "      <td>HDF5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2010</th>\n",
       "      <td>http://edamontology.org/format_3591</td>\n",
       "      <td>18</td>\n",
       "      <td>TIFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2011</th>\n",
       "      <td>http://edamontology.org/format_3592</td>\n",
       "      <td>18</td>\n",
       "      <td>BMP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2012</th>\n",
       "      <td>http://edamontology.org/format_3593</td>\n",
       "      <td>18</td>\n",
       "      <td>im</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2013</th>\n",
       "      <td>http://edamontology.org/format_3594</td>\n",
       "      <td>18</td>\n",
       "      <td>pcd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2014</th>\n",
       "      <td>http://edamontology.org/format_3595</td>\n",
       "      <td>18</td>\n",
       "      <td>pcx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2015</th>\n",
       "      <td>http://edamontology.org/format_3596</td>\n",
       "      <td>18</td>\n",
       "      <td>ppm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2016</th>\n",
       "      <td>http://edamontology.org/format_3597</td>\n",
       "      <td>18</td>\n",
       "      <td>psd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2017</th>\n",
       "      <td>http://edamontology.org/format_3598</td>\n",
       "      <td>18</td>\n",
       "      <td>xbm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2018</th>\n",
       "      <td>http://edamontology.org/format_3599</td>\n",
       "      <td>18</td>\n",
       "      <td>xpm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2019</th>\n",
       "      <td>http://edamontology.org/format_3600</td>\n",
       "      <td>18</td>\n",
       "      <td>rgb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2020</th>\n",
       "      <td>http://edamontology.org/format_3601</td>\n",
       "      <td>18</td>\n",
       "      <td>pbm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2021</th>\n",
       "      <td>http://edamontology.org/format_3602</td>\n",
       "      <td>18</td>\n",
       "      <td>pgm</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2022</th>\n",
       "      <td>http://edamontology.org/format_3603</td>\n",
       "      <td>18</td>\n",
       "      <td>PNG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2023</th>\n",
       "      <td>http://edamontology.org/format_3604</td>\n",
       "      <td>18</td>\n",
       "      <td>SVG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2024</th>\n",
       "      <td>http://edamontology.org/format_3605</td>\n",
       "      <td>18</td>\n",
       "      <td>rast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2025</th>\n",
       "      <td>http://edamontology.org/format_3606</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence quality report format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2026</th>\n",
       "      <td>http://edamontology.org/format_3607</td>\n",
       "      <td>18</td>\n",
       "      <td>qual</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2027</th>\n",
       "      <td>http://edamontology.org/format_3608</td>\n",
       "      <td>18</td>\n",
       "      <td>qualsolexa</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2028</th>\n",
       "      <td>http://edamontology.org/format_3609</td>\n",
       "      <td>18</td>\n",
       "      <td>qualillumina</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2029</th>\n",
       "      <td>http://edamontology.org/format_3610</td>\n",
       "      <td>18</td>\n",
       "      <td>qualsolid</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2030</th>\n",
       "      <td>http://edamontology.org/format_3611</td>\n",
       "      <td>18</td>\n",
       "      <td>qual454</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2031</th>\n",
       "      <td>http://edamontology.org/format_3612</td>\n",
       "      <td>18</td>\n",
       "      <td>ENCODE peak format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2032</th>\n",
       "      <td>http://edamontology.org/format_3613</td>\n",
       "      <td>18</td>\n",
       "      <td>ENCODE narrow peak format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2033</th>\n",
       "      <td>http://edamontology.org/format_3614</td>\n",
       "      <td>18</td>\n",
       "      <td>ENCODE broad peak format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2034</th>\n",
       "      <td>http://edamontology.org/format_3615</td>\n",
       "      <td>18</td>\n",
       "      <td>bgzip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2035</th>\n",
       "      <td>http://edamontology.org/format_3616</td>\n",
       "      <td>18</td>\n",
       "      <td>tabix</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2036</th>\n",
       "      <td>http://edamontology.org/format_3617</td>\n",
       "      <td>18</td>\n",
       "      <td>Graph format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2037</th>\n",
       "      <td>http://edamontology.org/format_3618</td>\n",
       "      <td>18</td>\n",
       "      <td>xgmml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2038</th>\n",
       "      <td>http://edamontology.org/format_3619</td>\n",
       "      <td>18</td>\n",
       "      <td>sif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2039</th>\n",
       "      <td>http://edamontology.org/format_3620</td>\n",
       "      <td>18</td>\n",
       "      <td>xlsx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2040</th>\n",
       "      <td>http://edamontology.org/format_3621</td>\n",
       "      <td>18</td>\n",
       "      <td>SQLite format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2041</th>\n",
       "      <td>http://edamontology.org/format_3622</td>\n",
       "      <td>18</td>\n",
       "      <td>Gemini SQLite format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2042</th>\n",
       "      <td>http://edamontology.org/format_3623</td>\n",
       "      <td>18</td>\n",
       "      <td>Index format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2043</th>\n",
       "      <td>http://edamontology.org/format_3624</td>\n",
       "      <td>18</td>\n",
       "      <td>snpeffdb</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2044</th>\n",
       "      <td>http://edamontology.org/format_3626</td>\n",
       "      <td>18</td>\n",
       "      <td>MAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2045</th>\n",
       "      <td>http://edamontology.org/format_3650</td>\n",
       "      <td>18</td>\n",
       "      <td>netCDF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2046</th>\n",
       "      <td>http://edamontology.org/format_3651</td>\n",
       "      <td>18</td>\n",
       "      <td>MGF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2047</th>\n",
       "      <td>http://edamontology.org/format_3652</td>\n",
       "      <td>18</td>\n",
       "      <td>dta</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2048</th>\n",
       "      <td>http://edamontology.org/format_3653</td>\n",
       "      <td>18</td>\n",
       "      <td>pkl</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2049</th>\n",
       "      <td>http://edamontology.org/format_3654</td>\n",
       "      <td>18</td>\n",
       "      <td>mzXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2050</th>\n",
       "      <td>http://edamontology.org/format_3655</td>\n",
       "      <td>18</td>\n",
       "      <td>pepXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2051</th>\n",
       "      <td>http://edamontology.org/format_3657</td>\n",
       "      <td>18</td>\n",
       "      <td>GPML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2052</th>\n",
       "      <td>http://edamontology.org/format_3665</td>\n",
       "      <td>18</td>\n",
       "      <td>K-mer countgraph</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2053</th>\n",
       "      <td>http://edamontology.org/format_3681</td>\n",
       "      <td>18</td>\n",
       "      <td>mzTab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2054</th>\n",
       "      <td>http://edamontology.org/format_3682</td>\n",
       "      <td>18</td>\n",
       "      <td>imzML metadata file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2055</th>\n",
       "      <td>http://edamontology.org/format_3683</td>\n",
       "      <td>18</td>\n",
       "      <td>qcML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2056</th>\n",
       "      <td>http://edamontology.org/format_3684</td>\n",
       "      <td>18</td>\n",
       "      <td>PRIDE XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2057</th>\n",
       "      <td>http://edamontology.org/format_3685</td>\n",
       "      <td>18</td>\n",
       "      <td>SED-ML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2058</th>\n",
       "      <td>http://edamontology.org/format_3686</td>\n",
       "      <td>18</td>\n",
       "      <td>COMBINE OMEX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2059</th>\n",
       "      <td>http://edamontology.org/format_3687</td>\n",
       "      <td>18</td>\n",
       "      <td>ISA-TAB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2060</th>\n",
       "      <td>http://edamontology.org/format_3688</td>\n",
       "      <td>18</td>\n",
       "      <td>SBtab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2061</th>\n",
       "      <td>http://edamontology.org/format_3689</td>\n",
       "      <td>18</td>\n",
       "      <td>BCML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2062</th>\n",
       "      <td>http://edamontology.org/format_3690</td>\n",
       "      <td>18</td>\n",
       "      <td>BDML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2063</th>\n",
       "      <td>http://edamontology.org/format_3691</td>\n",
       "      <td>18</td>\n",
       "      <td>BEL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2064</th>\n",
       "      <td>http://edamontology.org/format_3692</td>\n",
       "      <td>18</td>\n",
       "      <td>SBGN-ML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2065</th>\n",
       "      <td>http://edamontology.org/format_3693</td>\n",
       "      <td>18</td>\n",
       "      <td>AGP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2066</th>\n",
       "      <td>http://edamontology.org/format_3696</td>\n",
       "      <td>18</td>\n",
       "      <td>PS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2067</th>\n",
       "      <td>http://edamontology.org/format_3698</td>\n",
       "      <td>18</td>\n",
       "      <td>SRA format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2068</th>\n",
       "      <td>http://edamontology.org/format_3699</td>\n",
       "      <td>18</td>\n",
       "      <td>VDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2069</th>\n",
       "      <td>http://edamontology.org/format_3700</td>\n",
       "      <td>18</td>\n",
       "      <td>Tabix index file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2070</th>\n",
       "      <td>http://edamontology.org/format_3701</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequin format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2071</th>\n",
       "      <td>http://edamontology.org/format_3702</td>\n",
       "      <td>18</td>\n",
       "      <td>MSF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2072</th>\n",
       "      <td>http://edamontology.org/format_3706</td>\n",
       "      <td>18</td>\n",
       "      <td>Biodiversity data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2073</th>\n",
       "      <td>http://edamontology.org/format_3708</td>\n",
       "      <td>18</td>\n",
       "      <td>ABCD format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2074</th>\n",
       "      <td>http://edamontology.org/format_3709</td>\n",
       "      <td>18</td>\n",
       "      <td>GCT/Res format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2075</th>\n",
       "      <td>http://edamontology.org/format_3710</td>\n",
       "      <td>18</td>\n",
       "      <td>WIFF format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2076</th>\n",
       "      <td>http://edamontology.org/format_3711</td>\n",
       "      <td>18</td>\n",
       "      <td>X!Tandem XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2077</th>\n",
       "      <td>http://edamontology.org/format_3712</td>\n",
       "      <td>18</td>\n",
       "      <td>Thermo RAW</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2078</th>\n",
       "      <td>http://edamontology.org/format_3713</td>\n",
       "      <td>18</td>\n",
       "      <td>Mascot .dat file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2079</th>\n",
       "      <td>http://edamontology.org/format_3714</td>\n",
       "      <td>18</td>\n",
       "      <td>MaxQuant APL peaklist format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2080</th>\n",
       "      <td>http://edamontology.org/format_3725</td>\n",
       "      <td>18</td>\n",
       "      <td>SBOL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2081</th>\n",
       "      <td>http://edamontology.org/format_3726</td>\n",
       "      <td>18</td>\n",
       "      <td>PMML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2082</th>\n",
       "      <td>http://edamontology.org/format_3727</td>\n",
       "      <td>18</td>\n",
       "      <td>OME-TIFF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2083</th>\n",
       "      <td>http://edamontology.org/format_3728</td>\n",
       "      <td>18</td>\n",
       "      <td>LocARNA PP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2084</th>\n",
       "      <td>http://edamontology.org/format_3729</td>\n",
       "      <td>18</td>\n",
       "      <td>dbGaP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2085</th>\n",
       "      <td>http://edamontology.org/format_3746</td>\n",
       "      <td>18</td>\n",
       "      <td>BIOM format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2086</th>\n",
       "      <td>http://edamontology.org/format_3747</td>\n",
       "      <td>18</td>\n",
       "      <td>protXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2087</th>\n",
       "      <td>http://edamontology.org/format_3748</td>\n",
       "      <td>18</td>\n",
       "      <td>Linked data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2088</th>\n",
       "      <td>http://edamontology.org/format_3749</td>\n",
       "      <td>18</td>\n",
       "      <td>JSON-LD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2089</th>\n",
       "      <td>http://edamontology.org/format_3750</td>\n",
       "      <td>18</td>\n",
       "      <td>YAML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2090</th>\n",
       "      <td>http://edamontology.org/format_3751</td>\n",
       "      <td>18</td>\n",
       "      <td>DSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2091</th>\n",
       "      <td>http://edamontology.org/format_3752</td>\n",
       "      <td>18</td>\n",
       "      <td>CSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2092</th>\n",
       "      <td>http://edamontology.org/format_3758</td>\n",
       "      <td>18</td>\n",
       "      <td>SEQUEST .out file</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2093</th>\n",
       "      <td>http://edamontology.org/format_3764</td>\n",
       "      <td>18</td>\n",
       "      <td>idXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2094</th>\n",
       "      <td>http://edamontology.org/format_3765</td>\n",
       "      <td>18</td>\n",
       "      <td>KNIME datatable format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2095</th>\n",
       "      <td>http://edamontology.org/format_3770</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProtKB XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2096</th>\n",
       "      <td>http://edamontology.org/format_3771</td>\n",
       "      <td>18</td>\n",
       "      <td>UniProtKB RDF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2097</th>\n",
       "      <td>http://edamontology.org/format_3772</td>\n",
       "      <td>18</td>\n",
       "      <td>BioJSON (BioXSD)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2098</th>\n",
       "      <td>http://edamontology.org/format_3773</td>\n",
       "      <td>18</td>\n",
       "      <td>BioYAML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2099</th>\n",
       "      <td>http://edamontology.org/format_3774</td>\n",
       "      <td>18</td>\n",
       "      <td>BioJSON (Jalview)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2100</th>\n",
       "      <td>http://edamontology.org/format_3775</td>\n",
       "      <td>18</td>\n",
       "      <td>GSuite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2101</th>\n",
       "      <td>http://edamontology.org/format_3776</td>\n",
       "      <td>18</td>\n",
       "      <td>BTrack</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2102</th>\n",
       "      <td>http://edamontology.org/format_3777</td>\n",
       "      <td>18</td>\n",
       "      <td>MCPD</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2103</th>\n",
       "      <td>http://edamontology.org/format_3780</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotated text format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2104</th>\n",
       "      <td>http://edamontology.org/format_3781</td>\n",
       "      <td>18</td>\n",
       "      <td>PubAnnotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2105</th>\n",
       "      <td>http://edamontology.org/format_3782</td>\n",
       "      <td>18</td>\n",
       "      <td>BioC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2106</th>\n",
       "      <td>http://edamontology.org/format_3783</td>\n",
       "      <td>18</td>\n",
       "      <td>PubTator format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2107</th>\n",
       "      <td>http://edamontology.org/format_3784</td>\n",
       "      <td>18</td>\n",
       "      <td>Open Annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2108</th>\n",
       "      <td>http://edamontology.org/format_3785</td>\n",
       "      <td>18</td>\n",
       "      <td>BioNLP Shared Task format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2109</th>\n",
       "      <td>http://edamontology.org/format_3787</td>\n",
       "      <td>18</td>\n",
       "      <td>Query language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2110</th>\n",
       "      <td>http://edamontology.org/format_3788</td>\n",
       "      <td>18</td>\n",
       "      <td>SQL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2111</th>\n",
       "      <td>http://edamontology.org/format_3789</td>\n",
       "      <td>18</td>\n",
       "      <td>XQuery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2112</th>\n",
       "      <td>http://edamontology.org/format_3790</td>\n",
       "      <td>18</td>\n",
       "      <td>SPARQL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2113</th>\n",
       "      <td>http://edamontology.org/format_3804</td>\n",
       "      <td>18</td>\n",
       "      <td>xsd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2114</th>\n",
       "      <td>http://edamontology.org/format_3811</td>\n",
       "      <td>18</td>\n",
       "      <td>XMFA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2115</th>\n",
       "      <td>http://edamontology.org/format_3812</td>\n",
       "      <td>18</td>\n",
       "      <td>GEN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2116</th>\n",
       "      <td>http://edamontology.org/format_3813</td>\n",
       "      <td>18</td>\n",
       "      <td>SAMPLE file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2117</th>\n",
       "      <td>http://edamontology.org/format_3814</td>\n",
       "      <td>18</td>\n",
       "      <td>SDF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2118</th>\n",
       "      <td>http://edamontology.org/format_3815</td>\n",
       "      <td>18</td>\n",
       "      <td>Molfile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2119</th>\n",
       "      <td>http://edamontology.org/format_3816</td>\n",
       "      <td>18</td>\n",
       "      <td>Mol2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2120</th>\n",
       "      <td>http://edamontology.org/format_3817</td>\n",
       "      <td>18</td>\n",
       "      <td>latex</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2121</th>\n",
       "      <td>http://edamontology.org/format_3818</td>\n",
       "      <td>18</td>\n",
       "      <td>ELAND format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2122</th>\n",
       "      <td>http://edamontology.org/format_3819</td>\n",
       "      <td>18</td>\n",
       "      <td>Relaxed PHYLIP Interleaved</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2123</th>\n",
       "      <td>http://edamontology.org/format_3820</td>\n",
       "      <td>18</td>\n",
       "      <td>Relaxed PHYLIP Sequential</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2124</th>\n",
       "      <td>http://edamontology.org/format_3821</td>\n",
       "      <td>18</td>\n",
       "      <td>VisML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2125</th>\n",
       "      <td>http://edamontology.org/format_3822</td>\n",
       "      <td>18</td>\n",
       "      <td>GML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2126</th>\n",
       "      <td>http://edamontology.org/format_3823</td>\n",
       "      <td>18</td>\n",
       "      <td>FASTG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2127</th>\n",
       "      <td>http://edamontology.org/format_3824</td>\n",
       "      <td>18</td>\n",
       "      <td>NMR data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2128</th>\n",
       "      <td>http://edamontology.org/format_3825</td>\n",
       "      <td>18</td>\n",
       "      <td>nmrML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2129</th>\n",
       "      <td>http://edamontology.org/format_3826</td>\n",
       "      <td>18</td>\n",
       "      <td>proBAM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2130</th>\n",
       "      <td>http://edamontology.org/format_3827</td>\n",
       "      <td>18</td>\n",
       "      <td>proBED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2131</th>\n",
       "      <td>http://edamontology.org/format_3828</td>\n",
       "      <td>18</td>\n",
       "      <td>Raw microarray data format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2132</th>\n",
       "      <td>http://edamontology.org/format_3829</td>\n",
       "      <td>18</td>\n",
       "      <td>GPR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2133</th>\n",
       "      <td>http://edamontology.org/format_3830</td>\n",
       "      <td>18</td>\n",
       "      <td>ARB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2134</th>\n",
       "      <td>http://edamontology.org/format_3832</td>\n",
       "      <td>18</td>\n",
       "      <td>consensusXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2135</th>\n",
       "      <td>http://edamontology.org/format_3833</td>\n",
       "      <td>18</td>\n",
       "      <td>featureXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2136</th>\n",
       "      <td>http://edamontology.org/format_3834</td>\n",
       "      <td>18</td>\n",
       "      <td>mzData</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2137</th>\n",
       "      <td>http://edamontology.org/format_3835</td>\n",
       "      <td>18</td>\n",
       "      <td>TIDE TXT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2138</th>\n",
       "      <td>http://edamontology.org/format_3836</td>\n",
       "      <td>18</td>\n",
       "      <td>BLAST XML v2 results format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2139</th>\n",
       "      <td>http://edamontology.org/format_3838</td>\n",
       "      <td>18</td>\n",
       "      <td>pptx</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2140</th>\n",
       "      <td>http://edamontology.org/format_3839</td>\n",
       "      <td>18</td>\n",
       "      <td>ibd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2141</th>\n",
       "      <td>http://edamontology.org/format_3841</td>\n",
       "      <td>18</td>\n",
       "      <td>NLP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2142</th>\n",
       "      <td>http://edamontology.org/format_3843</td>\n",
       "      <td>18</td>\n",
       "      <td>BEAST</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2143</th>\n",
       "      <td>http://edamontology.org/format_3844</td>\n",
       "      <td>18</td>\n",
       "      <td>Chado-XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2144</th>\n",
       "      <td>http://edamontology.org/format_3845</td>\n",
       "      <td>18</td>\n",
       "      <td>HSAML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2145</th>\n",
       "      <td>http://edamontology.org/format_3846</td>\n",
       "      <td>18</td>\n",
       "      <td>InterProScan XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2146</th>\n",
       "      <td>http://edamontology.org/format_3847</td>\n",
       "      <td>18</td>\n",
       "      <td>KGML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2147</th>\n",
       "      <td>http://edamontology.org/format_3848</td>\n",
       "      <td>18</td>\n",
       "      <td>PubMed XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2148</th>\n",
       "      <td>http://edamontology.org/format_3849</td>\n",
       "      <td>18</td>\n",
       "      <td>MSAML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2149</th>\n",
       "      <td>http://edamontology.org/format_3850</td>\n",
       "      <td>18</td>\n",
       "      <td>OrthoXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2150</th>\n",
       "      <td>http://edamontology.org/format_3851</td>\n",
       "      <td>18</td>\n",
       "      <td>PSDML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2151</th>\n",
       "      <td>http://edamontology.org/format_3852</td>\n",
       "      <td>18</td>\n",
       "      <td>SeqXML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2152</th>\n",
       "      <td>http://edamontology.org/format_3853</td>\n",
       "      <td>18</td>\n",
       "      <td>UniParc XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2153</th>\n",
       "      <td>http://edamontology.org/format_3854</td>\n",
       "      <td>18</td>\n",
       "      <td>UniRef XML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2154</th>\n",
       "      <td>http://edamontology.org/format_3857</td>\n",
       "      <td>18</td>\n",
       "      <td>CWL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2155</th>\n",
       "      <td>http://edamontology.org/format_3858</td>\n",
       "      <td>18</td>\n",
       "      <td>Waters RAW</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2156</th>\n",
       "      <td>http://edamontology.org/format_3859</td>\n",
       "      <td>18</td>\n",
       "      <td>JCAMP-DX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2157</th>\n",
       "      <td>http://edamontology.org/format_3862</td>\n",
       "      <td>18</td>\n",
       "      <td>NLP annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2158</th>\n",
       "      <td>http://edamontology.org/format_3863</td>\n",
       "      <td>18</td>\n",
       "      <td>NLP corpus format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2159</th>\n",
       "      <td>http://edamontology.org/format_3864</td>\n",
       "      <td>18</td>\n",
       "      <td>mirGFF3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2160</th>\n",
       "      <td>http://edamontology.org/format_3865</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA annotation format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2161</th>\n",
       "      <td>http://edamontology.org/format_3866</td>\n",
       "      <td>18</td>\n",
       "      <td>Trajectory format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2162</th>\n",
       "      <td>http://edamontology.org/format_3867</td>\n",
       "      <td>18</td>\n",
       "      <td>Trajectory format (binary)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2163</th>\n",
       "      <td>http://edamontology.org/format_3868</td>\n",
       "      <td>18</td>\n",
       "      <td>Trajectory format (text)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2164</th>\n",
       "      <td>http://edamontology.org/format_3873</td>\n",
       "      <td>18</td>\n",
       "      <td>HDF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2165</th>\n",
       "      <td>http://edamontology.org/format_3874</td>\n",
       "      <td>18</td>\n",
       "      <td>PCAzip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2166</th>\n",
       "      <td>http://edamontology.org/format_3875</td>\n",
       "      <td>18</td>\n",
       "      <td>XTC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2167</th>\n",
       "      <td>http://edamontology.org/format_3876</td>\n",
       "      <td>18</td>\n",
       "      <td>TNG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2168</th>\n",
       "      <td>http://edamontology.org/format_3877</td>\n",
       "      <td>18</td>\n",
       "      <td>XYZ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2169</th>\n",
       "      <td>http://edamontology.org/format_3878</td>\n",
       "      <td>18</td>\n",
       "      <td>mdcrd</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2170</th>\n",
       "      <td>http://edamontology.org/format_3879</td>\n",
       "      <td>18</td>\n",
       "      <td>Topology format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2171</th>\n",
       "      <td>http://edamontology.org/format_3880</td>\n",
       "      <td>18</td>\n",
       "      <td>GROMACS top</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2172</th>\n",
       "      <td>http://edamontology.org/format_3881</td>\n",
       "      <td>18</td>\n",
       "      <td>AMBER top</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2173</th>\n",
       "      <td>http://edamontology.org/format_3882</td>\n",
       "      <td>18</td>\n",
       "      <td>PSF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2174</th>\n",
       "      <td>http://edamontology.org/format_3883</td>\n",
       "      <td>18</td>\n",
       "      <td>GROMACS itp</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2175</th>\n",
       "      <td>http://edamontology.org/format_3884</td>\n",
       "      <td>18</td>\n",
       "      <td>FF parameter format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2176</th>\n",
       "      <td>http://edamontology.org/format_3885</td>\n",
       "      <td>18</td>\n",
       "      <td>BinPos</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2177</th>\n",
       "      <td>http://edamontology.org/format_3886</td>\n",
       "      <td>18</td>\n",
       "      <td>RST</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2178</th>\n",
       "      <td>http://edamontology.org/format_3887</td>\n",
       "      <td>18</td>\n",
       "      <td>CHARMM rtf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2179</th>\n",
       "      <td>http://edamontology.org/format_3888</td>\n",
       "      <td>18</td>\n",
       "      <td>AMBER frcmod</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2180</th>\n",
       "      <td>http://edamontology.org/format_3889</td>\n",
       "      <td>18</td>\n",
       "      <td>AMBER off</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2181</th>\n",
       "      <td>http://edamontology.org/format_3906</td>\n",
       "      <td>18</td>\n",
       "      <td>NMReDATA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2182</th>\n",
       "      <td>http://edamontology.org/format_3909</td>\n",
       "      <td>18</td>\n",
       "      <td>BpForms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2183</th>\n",
       "      <td>http://edamontology.org/format_3910</td>\n",
       "      <td>18</td>\n",
       "      <td>trr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2184</th>\n",
       "      <td>http://edamontology.org/format_3911</td>\n",
       "      <td>18</td>\n",
       "      <td>msh</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2185</th>\n",
       "      <td>http://edamontology.org/format_3913</td>\n",
       "      <td>18</td>\n",
       "      <td>Loom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2186</th>\n",
       "      <td>http://edamontology.org/format_3915</td>\n",
       "      <td>18</td>\n",
       "      <td>Zarr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2187</th>\n",
       "      <td>http://edamontology.org/format_3916</td>\n",
       "      <td>18</td>\n",
       "      <td>MTX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2188</th>\n",
       "      <td>http://edamontology.org/format_3951</td>\n",
       "      <td>18</td>\n",
       "      <td>BcForms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2189</th>\n",
       "      <td>http://edamontology.org/format_3956</td>\n",
       "      <td>18</td>\n",
       "      <td>N-Quads</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2190</th>\n",
       "      <td>http://edamontology.org/format_3969</td>\n",
       "      <td>18</td>\n",
       "      <td>Vega</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2191</th>\n",
       "      <td>http://edamontology.org/format_3970</td>\n",
       "      <td>18</td>\n",
       "      <td>Vega-lite</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2192</th>\n",
       "      <td>http://edamontology.org/format_3971</td>\n",
       "      <td>18</td>\n",
       "      <td>NeuroML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2193</th>\n",
       "      <td>http://edamontology.org/format_3972</td>\n",
       "      <td>18</td>\n",
       "      <td>BNGL</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2194</th>\n",
       "      <td>http://edamontology.org/format_3973</td>\n",
       "      <td>18</td>\n",
       "      <td>Docker image</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2195</th>\n",
       "      <td>http://edamontology.org/format_3975</td>\n",
       "      <td>18</td>\n",
       "      <td>GFA 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2196</th>\n",
       "      <td>http://edamontology.org/format_3976</td>\n",
       "      <td>18</td>\n",
       "      <td>GFA 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2197</th>\n",
       "      <td>http://edamontology.org/format_3977</td>\n",
       "      <td>18</td>\n",
       "      <td>ObjTables</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2198</th>\n",
       "      <td>http://edamontology.org/format_3978</td>\n",
       "      <td>18</td>\n",
       "      <td>CONTIG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2199</th>\n",
       "      <td>http://edamontology.org/format_3979</td>\n",
       "      <td>18</td>\n",
       "      <td>WEGO</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2200</th>\n",
       "      <td>http://edamontology.org/format_3980</td>\n",
       "      <td>18</td>\n",
       "      <td>RPKM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2201</th>\n",
       "      <td>http://edamontology.org/format_3981</td>\n",
       "      <td>18</td>\n",
       "      <td>TAR format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2202</th>\n",
       "      <td>http://edamontology.org/format_3982</td>\n",
       "      <td>18</td>\n",
       "      <td>CHAIN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2203</th>\n",
       "      <td>http://edamontology.org/format_3983</td>\n",
       "      <td>18</td>\n",
       "      <td>NET</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2204</th>\n",
       "      <td>http://edamontology.org/format_3984</td>\n",
       "      <td>18</td>\n",
       "      <td>QMAP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2205</th>\n",
       "      <td>http://edamontology.org/format_3985</td>\n",
       "      <td>18</td>\n",
       "      <td>gxformat2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2206</th>\n",
       "      <td>http://edamontology.org/format_3986</td>\n",
       "      <td>18</td>\n",
       "      <td>WMV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2207</th>\n",
       "      <td>http://edamontology.org/format_3987</td>\n",
       "      <td>18</td>\n",
       "      <td>ZIP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2208</th>\n",
       "      <td>http://edamontology.org/format_3988</td>\n",
       "      <td>18</td>\n",
       "      <td>LSM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2209</th>\n",
       "      <td>http://edamontology.org/format_3989</td>\n",
       "      <td>18</td>\n",
       "      <td>GZIP format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2210</th>\n",
       "      <td>http://edamontology.org/format_3990</td>\n",
       "      <td>18</td>\n",
       "      <td>AVI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2211</th>\n",
       "      <td>http://edamontology.org/format_3991</td>\n",
       "      <td>18</td>\n",
       "      <td>TrackDB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2212</th>\n",
       "      <td>http://edamontology.org/format_3992</td>\n",
       "      <td>18</td>\n",
       "      <td>CIGAR format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2213</th>\n",
       "      <td>http://edamontology.org/format_3993</td>\n",
       "      <td>18</td>\n",
       "      <td>Stereolithography format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2214</th>\n",
       "      <td>http://edamontology.org/format_3994</td>\n",
       "      <td>18</td>\n",
       "      <td>U3D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2215</th>\n",
       "      <td>http://edamontology.org/format_3995</td>\n",
       "      <td>18</td>\n",
       "      <td>Texture file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2216</th>\n",
       "      <td>http://edamontology.org/format_3996</td>\n",
       "      <td>18</td>\n",
       "      <td>Python script</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2217</th>\n",
       "      <td>http://edamontology.org/format_3997</td>\n",
       "      <td>18</td>\n",
       "      <td>MPEG-4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2218</th>\n",
       "      <td>http://edamontology.org/format_3998</td>\n",
       "      <td>18</td>\n",
       "      <td>Perl script</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2219</th>\n",
       "      <td>http://edamontology.org/format_3999</td>\n",
       "      <td>18</td>\n",
       "      <td>R script</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2220</th>\n",
       "      <td>http://edamontology.org/format_4000</td>\n",
       "      <td>18</td>\n",
       "      <td>R markdown</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2221</th>\n",
       "      <td>http://edamontology.org/format_4001</td>\n",
       "      <td>18</td>\n",
       "      <td>NIFTI format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2222</th>\n",
       "      <td>http://edamontology.org/format_4002</td>\n",
       "      <td>18</td>\n",
       "      <td>pickle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2223</th>\n",
       "      <td>http://edamontology.org/format_4003</td>\n",
       "      <td>18</td>\n",
       "      <td>NumPy format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2224</th>\n",
       "      <td>http://edamontology.org/format_4004</td>\n",
       "      <td>18</td>\n",
       "      <td>SimTools repertoire file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2225</th>\n",
       "      <td>http://edamontology.org/format_4005</td>\n",
       "      <td>18</td>\n",
       "      <td>Configuration file format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2226</th>\n",
       "      <td>http://edamontology.org/format_4006</td>\n",
       "      <td>18</td>\n",
       "      <td>Zstandard format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2227</th>\n",
       "      <td>http://edamontology.org/format_4007</td>\n",
       "      <td>18</td>\n",
       "      <td>MATLAB script</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2228</th>\n",
       "      <td>http://edamontology.org/format_4015</td>\n",
       "      <td>18</td>\n",
       "      <td>PEtab</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2229</th>\n",
       "      <td>http://edamontology.org/format_4018</td>\n",
       "      <td>18</td>\n",
       "      <td>gVCF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2230</th>\n",
       "      <td>http://edamontology.org/format_4023</td>\n",
       "      <td>18</td>\n",
       "      <td>cml</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2231</th>\n",
       "      <td>http://edamontology.org/format_4024</td>\n",
       "      <td>18</td>\n",
       "      <td>cif</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2232</th>\n",
       "      <td>http://edamontology.org/format_4025</td>\n",
       "      <td>18</td>\n",
       "      <td>BioSimulators format for the specifications of biosimulation tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2233</th>\n",
       "      <td>http://edamontology.org/format_4026</td>\n",
       "      <td>18</td>\n",
       "      <td>BioSimulators standard for command-line interfaces for biosimulation tools</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2234</th>\n",
       "      <td>http://edamontology.org/format_4035</td>\n",
       "      <td>18</td>\n",
       "      <td>PQR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2235</th>\n",
       "      <td>http://edamontology.org/format_4036</td>\n",
       "      <td>18</td>\n",
       "      <td>PDBQT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2236</th>\n",
       "      <td>http://edamontology.org/has_format</td>\n",
       "      <td>18</td>\n",
       "      <td>has format</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2237</th>\n",
       "      <td>http://edamontology.org/has_function</td>\n",
       "      <td>18</td>\n",
       "      <td>has function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2238</th>\n",
       "      <td>http://edamontology.org/has_identifier</td>\n",
       "      <td>18</td>\n",
       "      <td>has identifier</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2239</th>\n",
       "      <td>http://edamontology.org/has_input</td>\n",
       "      <td>18</td>\n",
       "      <td>has input</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2240</th>\n",
       "      <td>http://edamontology.org/has_output</td>\n",
       "      <td>18</td>\n",
       "      <td>has output</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2241</th>\n",
       "      <td>http://edamontology.org/has_topic</td>\n",
       "      <td>18</td>\n",
       "      <td>has topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2242</th>\n",
       "      <td>http://edamontology.org/information_standard</td>\n",
       "      <td>18</td>\n",
       "      <td>Information standard</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2243</th>\n",
       "      <td>http://edamontology.org/is_deprecation_candidate</td>\n",
       "      <td>18</td>\n",
       "      <td>deprecation_candidate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2244</th>\n",
       "      <td>http://edamontology.org/is_format_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is format of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2245</th>\n",
       "      <td>http://edamontology.org/is_function_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is function of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2246</th>\n",
       "      <td>http://edamontology.org/is_identifier_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is identifier of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2247</th>\n",
       "      <td>http://edamontology.org/is_input_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is input of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2248</th>\n",
       "      <td>http://edamontology.org/is_output_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is output of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2249</th>\n",
       "      <td>http://edamontology.org/is_refactor_candidate</td>\n",
       "      <td>18</td>\n",
       "      <td>refactor_candidate</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2250</th>\n",
       "      <td>http://edamontology.org/is_topic_of</td>\n",
       "      <td>18</td>\n",
       "      <td>is topic of</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2251</th>\n",
       "      <td>http://edamontology.org/isdebtag</td>\n",
       "      <td>18</td>\n",
       "      <td>isdebtag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2252</th>\n",
       "      <td>http://edamontology.org/media_type</td>\n",
       "      <td>18</td>\n",
       "      <td>Media type</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2253</th>\n",
       "      <td>http://edamontology.org/notRecommendedForAnnotation</td>\n",
       "      <td>18</td>\n",
       "      <td>notRecommendedForAnnotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2254</th>\n",
       "      <td>http://edamontology.org/obsolete_since</td>\n",
       "      <td>18</td>\n",
       "      <td>Obsolete since</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2255</th>\n",
       "      <td>http://edamontology.org/oldParent</td>\n",
       "      <td>18</td>\n",
       "      <td>Old parent</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2256</th>\n",
       "      <td>http://edamontology.org/oldRelated</td>\n",
       "      <td>18</td>\n",
       "      <td>Old related</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2257</th>\n",
       "      <td>http://edamontology.org/ontology_used</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology used</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2258</th>\n",
       "      <td>http://edamontology.org/operation_0004</td>\n",
       "      <td>18</td>\n",
       "      <td>Operation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2259</th>\n",
       "      <td>http://edamontology.org/operation_0224</td>\n",
       "      <td>18</td>\n",
       "      <td>Query and retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2260</th>\n",
       "      <td>http://edamontology.org/operation_0225</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (database cross-reference)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2261</th>\n",
       "      <td>http://edamontology.org/operation_0226</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2262</th>\n",
       "      <td>http://edamontology.org/operation_0227</td>\n",
       "      <td>18</td>\n",
       "      <td>Indexing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2263</th>\n",
       "      <td>http://edamontology.org/operation_0228</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2264</th>\n",
       "      <td>http://edamontology.org/operation_0229</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation retrieval (sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2265</th>\n",
       "      <td>http://edamontology.org/operation_0230</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2266</th>\n",
       "      <td>http://edamontology.org/operation_0231</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2267</th>\n",
       "      <td>http://edamontology.org/operation_0232</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence merging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2268</th>\n",
       "      <td>http://edamontology.org/operation_0233</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence conversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2269</th>\n",
       "      <td>http://edamontology.org/operation_0234</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence complexity calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2270</th>\n",
       "      <td>http://edamontology.org/operation_0235</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence ambiguity calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2271</th>\n",
       "      <td>http://edamontology.org/operation_0236</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2272</th>\n",
       "      <td>http://edamontology.org/operation_0237</td>\n",
       "      <td>18</td>\n",
       "      <td>Repeat sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2273</th>\n",
       "      <td>http://edamontology.org/operation_0238</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif discovery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2274</th>\n",
       "      <td>http://edamontology.org/operation_0239</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2275</th>\n",
       "      <td>http://edamontology.org/operation_0240</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2276</th>\n",
       "      <td>http://edamontology.org/operation_0241</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription regulatory sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2277</th>\n",
       "      <td>http://edamontology.org/operation_0242</td>\n",
       "      <td>18</td>\n",
       "      <td>Conserved transcription regulatory sequence identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2278</th>\n",
       "      <td>http://edamontology.org/operation_0243</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein property calculation (from structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2279</th>\n",
       "      <td>http://edamontology.org/operation_0244</td>\n",
       "      <td>18</td>\n",
       "      <td>Simulation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2280</th>\n",
       "      <td>http://edamontology.org/operation_0245</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural motif discovery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2281</th>\n",
       "      <td>http://edamontology.org/operation_0246</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domain recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2282</th>\n",
       "      <td>http://edamontology.org/operation_0247</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein architecture analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2283</th>\n",
       "      <td>http://edamontology.org/operation_0248</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue interaction calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2284</th>\n",
       "      <td>http://edamontology.org/operation_0249</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein geometry calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2285</th>\n",
       "      <td>http://edamontology.org/operation_0250</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein property calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2286</th>\n",
       "      <td>http://edamontology.org/operation_0252</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide immunogenicity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2287</th>\n",
       "      <td>http://edamontology.org/operation_0253</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2288</th>\n",
       "      <td>http://edamontology.org/operation_0254</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (feature table)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2289</th>\n",
       "      <td>http://edamontology.org/operation_0255</td>\n",
       "      <td>18</td>\n",
       "      <td>Feature table query</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2290</th>\n",
       "      <td>http://edamontology.org/operation_0256</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2291</th>\n",
       "      <td>http://edamontology.org/operation_0257</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (sequence alignment)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2292</th>\n",
       "      <td>http://edamontology.org/operation_0258</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2293</th>\n",
       "      <td>http://edamontology.org/operation_0259</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2294</th>\n",
       "      <td>http://edamontology.org/operation_0260</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment conversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2295</th>\n",
       "      <td>http://edamontology.org/operation_0261</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid property processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2296</th>\n",
       "      <td>http://edamontology.org/operation_0262</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid property calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2297</th>\n",
       "      <td>http://edamontology.org/operation_0264</td>\n",
       "      <td>18</td>\n",
       "      <td>Alternative splicing prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2298</th>\n",
       "      <td>http://edamontology.org/operation_0265</td>\n",
       "      <td>18</td>\n",
       "      <td>Frameshift detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2299</th>\n",
       "      <td>http://edamontology.org/operation_0266</td>\n",
       "      <td>18</td>\n",
       "      <td>Vector sequence detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2300</th>\n",
       "      <td>http://edamontology.org/operation_0267</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2301</th>\n",
       "      <td>http://edamontology.org/operation_0268</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein super-secondary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2302</th>\n",
       "      <td>http://edamontology.org/operation_0269</td>\n",
       "      <td>18</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2303</th>\n",
       "      <td>http://edamontology.org/operation_0270</td>\n",
       "      <td>18</td>\n",
       "      <td>Transmembrane protein analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2304</th>\n",
       "      <td>http://edamontology.org/operation_0271</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2305</th>\n",
       "      <td>http://edamontology.org/operation_0272</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue contact prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2306</th>\n",
       "      <td>http://edamontology.org/operation_0273</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction raw data analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2307</th>\n",
       "      <td>http://edamontology.org/operation_0274</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interaction prediction (from protein sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2308</th>\n",
       "      <td>http://edamontology.org/operation_0275</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interaction prediction (from protein structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2309</th>\n",
       "      <td>http://edamontology.org/operation_0276</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction network analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2310</th>\n",
       "      <td>http://edamontology.org/operation_0277</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2311</th>\n",
       "      <td>http://edamontology.org/operation_0278</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2312</th>\n",
       "      <td>http://edamontology.org/operation_0279</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2313</th>\n",
       "      <td>http://edamontology.org/operation_0280</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (restriction enzyme annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2314</th>\n",
       "      <td>http://edamontology.org/operation_0281</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic marker identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2315</th>\n",
       "      <td>http://edamontology.org/operation_0282</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2316</th>\n",
       "      <td>http://edamontology.org/operation_0283</td>\n",
       "      <td>18</td>\n",
       "      <td>Linkage analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2317</th>\n",
       "      <td>http://edamontology.org/operation_0284</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2318</th>\n",
       "      <td>http://edamontology.org/operation_0285</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2319</th>\n",
       "      <td>http://edamontology.org/operation_0286</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2320</th>\n",
       "      <td>http://edamontology.org/operation_0287</td>\n",
       "      <td>18</td>\n",
       "      <td>Base position variability plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2321</th>\n",
       "      <td>http://edamontology.org/operation_0288</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence word comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2322</th>\n",
       "      <td>http://edamontology.org/operation_0289</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence distance matrix generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2323</th>\n",
       "      <td>http://edamontology.org/operation_0290</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence redundancy removal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2324</th>\n",
       "      <td>http://edamontology.org/operation_0291</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2325</th>\n",
       "      <td>http://edamontology.org/operation_0292</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2326</th>\n",
       "      <td>http://edamontology.org/operation_0293</td>\n",
       "      <td>18</td>\n",
       "      <td>Hybrid sequence alignment construction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2327</th>\n",
       "      <td>http://edamontology.org/operation_0294</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure-based sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2328</th>\n",
       "      <td>http://edamontology.org/operation_0295</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2329</th>\n",
       "      <td>http://edamontology.org/operation_0296</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2330</th>\n",
       "      <td>http://edamontology.org/operation_0297</td>\n",
       "      <td>18</td>\n",
       "      <td>3D profile generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2331</th>\n",
       "      <td>http://edamontology.org/operation_0298</td>\n",
       "      <td>18</td>\n",
       "      <td>Profile-profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2332</th>\n",
       "      <td>http://edamontology.org/operation_0299</td>\n",
       "      <td>18</td>\n",
       "      <td>3D profile-to-3D profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2333</th>\n",
       "      <td>http://edamontology.org/operation_0300</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2334</th>\n",
       "      <td>http://edamontology.org/operation_0301</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence-to-3D-profile alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2335</th>\n",
       "      <td>http://edamontology.org/operation_0302</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein threading</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2336</th>\n",
       "      <td>http://edamontology.org/operation_0303</td>\n",
       "      <td>18</td>\n",
       "      <td>Fold recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2337</th>\n",
       "      <td>http://edamontology.org/operation_0304</td>\n",
       "      <td>18</td>\n",
       "      <td>Metadata retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2338</th>\n",
       "      <td>http://edamontology.org/operation_0305</td>\n",
       "      <td>18</td>\n",
       "      <td>Literature search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2339</th>\n",
       "      <td>http://edamontology.org/operation_0306</td>\n",
       "      <td>18</td>\n",
       "      <td>Text mining</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2340</th>\n",
       "      <td>http://edamontology.org/operation_0307</td>\n",
       "      <td>18</td>\n",
       "      <td>Virtual PCR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2341</th>\n",
       "      <td>http://edamontology.org/operation_0308</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2342</th>\n",
       "      <td>http://edamontology.org/operation_0309</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray probe design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2343</th>\n",
       "      <td>http://edamontology.org/operation_0310</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2344</th>\n",
       "      <td>http://edamontology.org/operation_0311</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray data standardisation and normalisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2345</th>\n",
       "      <td>http://edamontology.org/operation_0312</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing-based expression profile data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2346</th>\n",
       "      <td>http://edamontology.org/operation_0313</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression profile clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2347</th>\n",
       "      <td>http://edamontology.org/operation_0314</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2348</th>\n",
       "      <td>http://edamontology.org/operation_0315</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression profile comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2349</th>\n",
       "      <td>http://edamontology.org/operation_0316</td>\n",
       "      <td>18</td>\n",
       "      <td>Functional profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2350</th>\n",
       "      <td>http://edamontology.org/operation_0317</td>\n",
       "      <td>18</td>\n",
       "      <td>EST and cDNA sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2351</th>\n",
       "      <td>http://edamontology.org/operation_0318</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural genomics target selection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2352</th>\n",
       "      <td>http://edamontology.org/operation_0319</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure assignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2353</th>\n",
       "      <td>http://edamontology.org/operation_0320</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure assignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2354</th>\n",
       "      <td>http://edamontology.org/operation_0321</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2355</th>\n",
       "      <td>http://edamontology.org/operation_0322</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular model refinement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2356</th>\n",
       "      <td>http://edamontology.org/operation_0323</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2357</th>\n",
       "      <td>http://edamontology.org/operation_0324</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2358</th>\n",
       "      <td>http://edamontology.org/operation_0325</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2359</th>\n",
       "      <td>http://edamontology.org/operation_0326</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2360</th>\n",
       "      <td>http://edamontology.org/operation_0327</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic footprinting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2361</th>\n",
       "      <td>http://edamontology.org/operation_0328</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2362</th>\n",
       "      <td>http://edamontology.org/operation_0329</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding pathway prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2363</th>\n",
       "      <td>http://edamontology.org/operation_0330</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein SNP mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2364</th>\n",
       "      <td>http://edamontology.org/operation_0331</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant effect prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2365</th>\n",
       "      <td>http://edamontology.org/operation_0332</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunogen design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2366</th>\n",
       "      <td>http://edamontology.org/operation_0333</td>\n",
       "      <td>18</td>\n",
       "      <td>Zinc finger prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2367</th>\n",
       "      <td>http://edamontology.org/operation_0334</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzyme kinetics calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2368</th>\n",
       "      <td>http://edamontology.org/operation_0335</td>\n",
       "      <td>18</td>\n",
       "      <td>Formatting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2369</th>\n",
       "      <td>http://edamontology.org/operation_0336</td>\n",
       "      <td>18</td>\n",
       "      <td>Format validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2370</th>\n",
       "      <td>http://edamontology.org/operation_0337</td>\n",
       "      <td>18</td>\n",
       "      <td>Visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2371</th>\n",
       "      <td>http://edamontology.org/operation_0338</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2372</th>\n",
       "      <td>http://edamontology.org/operation_0339</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2373</th>\n",
       "      <td>http://edamontology.org/operation_0340</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2374</th>\n",
       "      <td>http://edamontology.org/operation_0341</td>\n",
       "      <td>18</td>\n",
       "      <td>Motif database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2375</th>\n",
       "      <td>http://edamontology.org/operation_0342</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2376</th>\n",
       "      <td>http://edamontology.org/operation_0343</td>\n",
       "      <td>18</td>\n",
       "      <td>Transmembrane protein database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2377</th>\n",
       "      <td>http://edamontology.org/operation_0344</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence retrieval (by code)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2378</th>\n",
       "      <td>http://edamontology.org/operation_0345</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence retrieval (by keyword)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2379</th>\n",
       "      <td>http://edamontology.org/operation_0346</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence similarity search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2380</th>\n",
       "      <td>http://edamontology.org/operation_0347</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by motif or pattern)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2381</th>\n",
       "      <td>http://edamontology.org/operation_0348</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by amino acid composition)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2382</th>\n",
       "      <td>http://edamontology.org/operation_0349</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by property)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2383</th>\n",
       "      <td>http://edamontology.org/operation_0350</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by sequence using word-based methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2384</th>\n",
       "      <td>http://edamontology.org/operation_0351</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by sequence using profile-based methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2385</th>\n",
       "      <td>http://edamontology.org/operation_0352</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by sequence using local alignment-based methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2386</th>\n",
       "      <td>http://edamontology.org/operation_0353</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by sequence using global alignment-based methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2387</th>\n",
       "      <td>http://edamontology.org/operation_0354</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by sequence for primer sequences)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2388</th>\n",
       "      <td>http://edamontology.org/operation_0355</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by molecular weight)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2389</th>\n",
       "      <td>http://edamontology.org/operation_0356</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search (by isoelectric point)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2390</th>\n",
       "      <td>http://edamontology.org/operation_0357</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure retrieval (by code)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2391</th>\n",
       "      <td>http://edamontology.org/operation_0358</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure retrieval (by keyword)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2392</th>\n",
       "      <td>http://edamontology.org/operation_0359</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure database search (by sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2393</th>\n",
       "      <td>http://edamontology.org/operation_0360</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural similarity search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2394</th>\n",
       "      <td>http://edamontology.org/operation_0361</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2395</th>\n",
       "      <td>http://edamontology.org/operation_0362</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2396</th>\n",
       "      <td>http://edamontology.org/operation_0363</td>\n",
       "      <td>18</td>\n",
       "      <td>Reverse complement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2397</th>\n",
       "      <td>http://edamontology.org/operation_0364</td>\n",
       "      <td>18</td>\n",
       "      <td>Random sequence generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2398</th>\n",
       "      <td>http://edamontology.org/operation_0365</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction digest</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2399</th>\n",
       "      <td>http://edamontology.org/operation_0366</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence cleavage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2400</th>\n",
       "      <td>http://edamontology.org/operation_0367</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence mutation and randomisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2401</th>\n",
       "      <td>http://edamontology.org/operation_0368</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence masking</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2402</th>\n",
       "      <td>http://edamontology.org/operation_0369</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cutting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2403</th>\n",
       "      <td>http://edamontology.org/operation_0370</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction site creation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2404</th>\n",
       "      <td>http://edamontology.org/operation_0371</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA translation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2405</th>\n",
       "      <td>http://edamontology.org/operation_0372</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2406</th>\n",
       "      <td>http://edamontology.org/operation_0377</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition calculation (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2407</th>\n",
       "      <td>http://edamontology.org/operation_0378</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition calculation (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2408</th>\n",
       "      <td>http://edamontology.org/operation_0379</td>\n",
       "      <td>18</td>\n",
       "      <td>Repeat sequence detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2409</th>\n",
       "      <td>http://edamontology.org/operation_0380</td>\n",
       "      <td>18</td>\n",
       "      <td>Repeat sequence organisation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2410</th>\n",
       "      <td>http://edamontology.org/operation_0383</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy calculation (from structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2411</th>\n",
       "      <td>http://edamontology.org/operation_0384</td>\n",
       "      <td>18</td>\n",
       "      <td>Accessible surface calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2412</th>\n",
       "      <td>http://edamontology.org/operation_0385</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy cluster calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2413</th>\n",
       "      <td>http://edamontology.org/operation_0386</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein dipole moment calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2414</th>\n",
       "      <td>http://edamontology.org/operation_0387</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular surface calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2415</th>\n",
       "      <td>http://edamontology.org/operation_0388</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein binding site prediction (from structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2416</th>\n",
       "      <td>http://edamontology.org/operation_0389</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-nucleic acid interaction analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2417</th>\n",
       "      <td>http://edamontology.org/operation_0390</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein peeling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2418</th>\n",
       "      <td>http://edamontology.org/operation_0391</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein distance matrix calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2419</th>\n",
       "      <td>http://edamontology.org/operation_0392</td>\n",
       "      <td>18</td>\n",
       "      <td>Contact map calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2420</th>\n",
       "      <td>http://edamontology.org/operation_0393</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue cluster calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2421</th>\n",
       "      <td>http://edamontology.org/operation_0394</td>\n",
       "      <td>18</td>\n",
       "      <td>Hydrogen bond calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2422</th>\n",
       "      <td>http://edamontology.org/operation_0395</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue non-canonical interaction detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2423</th>\n",
       "      <td>http://edamontology.org/operation_0396</td>\n",
       "      <td>18</td>\n",
       "      <td>Ramachandran plot calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2424</th>\n",
       "      <td>http://edamontology.org/operation_0397</td>\n",
       "      <td>18</td>\n",
       "      <td>Ramachandran plot validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2425</th>\n",
       "      <td>http://edamontology.org/operation_0398</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein molecular weight calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2426</th>\n",
       "      <td>http://edamontology.org/operation_0399</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein extinction coefficient calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2427</th>\n",
       "      <td>http://edamontology.org/operation_0400</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein pKa calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2428</th>\n",
       "      <td>http://edamontology.org/operation_0401</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy calculation (from sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2429</th>\n",
       "      <td>http://edamontology.org/operation_0402</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein titration curve plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2430</th>\n",
       "      <td>http://edamontology.org/operation_0403</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein isoelectric point calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2431</th>\n",
       "      <td>http://edamontology.org/operation_0404</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydrogen exchange rate calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2432</th>\n",
       "      <td>http://edamontology.org/operation_0405</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydrophobic region calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2433</th>\n",
       "      <td>http://edamontology.org/operation_0406</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein aliphatic index calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2434</th>\n",
       "      <td>http://edamontology.org/operation_0407</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydrophobic moment plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2435</th>\n",
       "      <td>http://edamontology.org/operation_0408</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein globularity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2436</th>\n",
       "      <td>http://edamontology.org/operation_0409</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein solubility prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2437</th>\n",
       "      <td>http://edamontology.org/operation_0410</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein crystallizability prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2438</th>\n",
       "      <td>http://edamontology.org/operation_0411</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signal peptide detection (eukaryotes)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2439</th>\n",
       "      <td>http://edamontology.org/operation_0412</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signal peptide detection (bacteria)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2440</th>\n",
       "      <td>http://edamontology.org/operation_0413</td>\n",
       "      <td>18</td>\n",
       "      <td>MHC peptide immunogenicity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2441</th>\n",
       "      <td>http://edamontology.org/operation_0414</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein feature prediction (from sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2442</th>\n",
       "      <td>http://edamontology.org/operation_0415</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2443</th>\n",
       "      <td>http://edamontology.org/operation_0416</td>\n",
       "      <td>18</td>\n",
       "      <td>Epitope mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2444</th>\n",
       "      <td>http://edamontology.org/operation_0417</td>\n",
       "      <td>18</td>\n",
       "      <td>Post-translational modification site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2445</th>\n",
       "      <td>http://edamontology.org/operation_0418</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signal peptide detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2446</th>\n",
       "      <td>http://edamontology.org/operation_0419</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein binding site prediction (from sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2447</th>\n",
       "      <td>http://edamontology.org/operation_0420</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acids-binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2448</th>\n",
       "      <td>http://edamontology.org/operation_0421</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2449</th>\n",
       "      <td>http://edamontology.org/operation_0422</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein cleavage site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2450</th>\n",
       "      <td>http://edamontology.org/operation_0423</td>\n",
       "      <td>18</td>\n",
       "      <td>Epitope mapping (MHC Class I)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2451</th>\n",
       "      <td>http://edamontology.org/operation_0424</td>\n",
       "      <td>18</td>\n",
       "      <td>Epitope mapping (MHC Class II)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2452</th>\n",
       "      <td>http://edamontology.org/operation_0425</td>\n",
       "      <td>18</td>\n",
       "      <td>Whole gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2453</th>\n",
       "      <td>http://edamontology.org/operation_0426</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene component prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2454</th>\n",
       "      <td>http://edamontology.org/operation_0427</td>\n",
       "      <td>18</td>\n",
       "      <td>Transposon prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2455</th>\n",
       "      <td>http://edamontology.org/operation_0428</td>\n",
       "      <td>18</td>\n",
       "      <td>PolyA signal detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2456</th>\n",
       "      <td>http://edamontology.org/operation_0429</td>\n",
       "      <td>18</td>\n",
       "      <td>Quadruplex formation site detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2457</th>\n",
       "      <td>http://edamontology.org/operation_0430</td>\n",
       "      <td>18</td>\n",
       "      <td>CpG island and isochore detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2458</th>\n",
       "      <td>http://edamontology.org/operation_0431</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction site recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2459</th>\n",
       "      <td>http://edamontology.org/operation_0432</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleosome position prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2460</th>\n",
       "      <td>http://edamontology.org/operation_0433</td>\n",
       "      <td>18</td>\n",
       "      <td>Splice site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2461</th>\n",
       "      <td>http://edamontology.org/operation_0434</td>\n",
       "      <td>18</td>\n",
       "      <td>Integrated gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2462</th>\n",
       "      <td>http://edamontology.org/operation_0435</td>\n",
       "      <td>18</td>\n",
       "      <td>Operon prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2463</th>\n",
       "      <td>http://edamontology.org/operation_0436</td>\n",
       "      <td>18</td>\n",
       "      <td>Coding region prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2464</th>\n",
       "      <td>http://edamontology.org/operation_0437</td>\n",
       "      <td>18</td>\n",
       "      <td>SECIS element prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2465</th>\n",
       "      <td>http://edamontology.org/operation_0438</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptional regulatory element prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2466</th>\n",
       "      <td>http://edamontology.org/operation_0439</td>\n",
       "      <td>18</td>\n",
       "      <td>Translation initiation site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2467</th>\n",
       "      <td>http://edamontology.org/operation_0440</td>\n",
       "      <td>18</td>\n",
       "      <td>Promoter prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2468</th>\n",
       "      <td>http://edamontology.org/operation_0441</td>\n",
       "      <td>18</td>\n",
       "      <td>cis-regulatory element prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2469</th>\n",
       "      <td>http://edamontology.org/operation_0442</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptional regulatory element prediction (RNA-cis)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2470</th>\n",
       "      <td>http://edamontology.org/operation_0443</td>\n",
       "      <td>18</td>\n",
       "      <td>trans-regulatory element prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2471</th>\n",
       "      <td>http://edamontology.org/operation_0444</td>\n",
       "      <td>18</td>\n",
       "      <td>S/MAR prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2472</th>\n",
       "      <td>http://edamontology.org/operation_0445</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factor binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2473</th>\n",
       "      <td>http://edamontology.org/operation_0446</td>\n",
       "      <td>18</td>\n",
       "      <td>Exonic splicing enhancer prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2474</th>\n",
       "      <td>http://edamontology.org/operation_0447</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2475</th>\n",
       "      <td>http://edamontology.org/operation_0448</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment analysis (conservation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2476</th>\n",
       "      <td>http://edamontology.org/operation_0449</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment analysis (site correlation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2477</th>\n",
       "      <td>http://edamontology.org/operation_0450</td>\n",
       "      <td>18</td>\n",
       "      <td>Chimera detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2478</th>\n",
       "      <td>http://edamontology.org/operation_0451</td>\n",
       "      <td>18</td>\n",
       "      <td>Recombination detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2479</th>\n",
       "      <td>http://edamontology.org/operation_0452</td>\n",
       "      <td>18</td>\n",
       "      <td>Indel detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2480</th>\n",
       "      <td>http://edamontology.org/operation_0453</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleosome formation potential prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2481</th>\n",
       "      <td>http://edamontology.org/operation_0455</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid thermodynamic property calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2482</th>\n",
       "      <td>http://edamontology.org/operation_0456</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting profile plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2483</th>\n",
       "      <td>http://edamontology.org/operation_0457</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid stitch profile plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2484</th>\n",
       "      <td>http://edamontology.org/operation_0458</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid melting curve plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2485</th>\n",
       "      <td>http://edamontology.org/operation_0459</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid probability profile plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2486</th>\n",
       "      <td>http://edamontology.org/operation_0460</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid temperature profile plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2487</th>\n",
       "      <td>http://edamontology.org/operation_0461</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid curvature calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2488</th>\n",
       "      <td>http://edamontology.org/operation_0463</td>\n",
       "      <td>18</td>\n",
       "      <td>miRNA target prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2489</th>\n",
       "      <td>http://edamontology.org/operation_0464</td>\n",
       "      <td>18</td>\n",
       "      <td>tRNA gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2490</th>\n",
       "      <td>http://edamontology.org/operation_0465</td>\n",
       "      <td>18</td>\n",
       "      <td>siRNA binding specificity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2491</th>\n",
       "      <td>http://edamontology.org/operation_0467</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction (integrated)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2492</th>\n",
       "      <td>http://edamontology.org/operation_0468</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction (helices)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2493</th>\n",
       "      <td>http://edamontology.org/operation_0469</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction (turns)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2494</th>\n",
       "      <td>http://edamontology.org/operation_0470</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction (coils)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2495</th>\n",
       "      <td>http://edamontology.org/operation_0471</td>\n",
       "      <td>18</td>\n",
       "      <td>Disulfide bond prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2496</th>\n",
       "      <td>http://edamontology.org/operation_0472</td>\n",
       "      <td>18</td>\n",
       "      <td>GPCR prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2497</th>\n",
       "      <td>http://edamontology.org/operation_0473</td>\n",
       "      <td>18</td>\n",
       "      <td>GPCR analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2498</th>\n",
       "      <td>http://edamontology.org/operation_0474</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2499</th>\n",
       "      <td>http://edamontology.org/operation_0475</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2500</th>\n",
       "      <td>http://edamontology.org/operation_0476</td>\n",
       "      <td>18</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2501</th>\n",
       "      <td>http://edamontology.org/operation_0477</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2502</th>\n",
       "      <td>http://edamontology.org/operation_0478</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular docking</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2503</th>\n",
       "      <td>http://edamontology.org/operation_0479</td>\n",
       "      <td>18</td>\n",
       "      <td>Backbone modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2504</th>\n",
       "      <td>http://edamontology.org/operation_0480</td>\n",
       "      <td>18</td>\n",
       "      <td>Side chain modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2505</th>\n",
       "      <td>http://edamontology.org/operation_0481</td>\n",
       "      <td>18</td>\n",
       "      <td>Loop modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2506</th>\n",
       "      <td>http://edamontology.org/operation_0482</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-ligand docking</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2507</th>\n",
       "      <td>http://edamontology.org/operation_0483</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA inverse folding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2508</th>\n",
       "      <td>http://edamontology.org/operation_0484</td>\n",
       "      <td>18</td>\n",
       "      <td>SNP detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2509</th>\n",
       "      <td>http://edamontology.org/operation_0485</td>\n",
       "      <td>18</td>\n",
       "      <td>Radiation Hybrid Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2510</th>\n",
       "      <td>http://edamontology.org/operation_0486</td>\n",
       "      <td>18</td>\n",
       "      <td>Functional mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2511</th>\n",
       "      <td>http://edamontology.org/operation_0487</td>\n",
       "      <td>18</td>\n",
       "      <td>Haplotype mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2512</th>\n",
       "      <td>http://edamontology.org/operation_0488</td>\n",
       "      <td>18</td>\n",
       "      <td>Linkage disequilibrium calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2513</th>\n",
       "      <td>http://edamontology.org/operation_0489</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic code prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2514</th>\n",
       "      <td>http://edamontology.org/operation_0490</td>\n",
       "      <td>18</td>\n",
       "      <td>Dot plot plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2515</th>\n",
       "      <td>http://edamontology.org/operation_0491</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2516</th>\n",
       "      <td>http://edamontology.org/operation_0492</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2517</th>\n",
       "      <td>http://edamontology.org/operation_0493</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise sequence alignment generation (local)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2518</th>\n",
       "      <td>http://edamontology.org/operation_0494</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise sequence alignment generation (global)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2519</th>\n",
       "      <td>http://edamontology.org/operation_0495</td>\n",
       "      <td>18</td>\n",
       "      <td>Local alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2520</th>\n",
       "      <td>http://edamontology.org/operation_0496</td>\n",
       "      <td>18</td>\n",
       "      <td>Global alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2521</th>\n",
       "      <td>http://edamontology.org/operation_0497</td>\n",
       "      <td>18</td>\n",
       "      <td>Constrained sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2522</th>\n",
       "      <td>http://edamontology.org/operation_0498</td>\n",
       "      <td>18</td>\n",
       "      <td>Consensus-based sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2523</th>\n",
       "      <td>http://edamontology.org/operation_0499</td>\n",
       "      <td>18</td>\n",
       "      <td>Tree-based sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2524</th>\n",
       "      <td>http://edamontology.org/operation_0500</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure alignment generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2525</th>\n",
       "      <td>http://edamontology.org/operation_0501</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure alignment generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2526</th>\n",
       "      <td>http://edamontology.org/operation_0502</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2527</th>\n",
       "      <td>http://edamontology.org/operation_0503</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2528</th>\n",
       "      <td>http://edamontology.org/operation_0504</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2529</th>\n",
       "      <td>http://edamontology.org/operation_0505</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2530</th>\n",
       "      <td>http://edamontology.org/operation_0506</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment (RNA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2531</th>\n",
       "      <td>http://edamontology.org/operation_0507</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise structure alignment generation (local)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2532</th>\n",
       "      <td>http://edamontology.org/operation_0508</td>\n",
       "      <td>18</td>\n",
       "      <td>Pairwise structure alignment generation (global)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2533</th>\n",
       "      <td>http://edamontology.org/operation_0509</td>\n",
       "      <td>18</td>\n",
       "      <td>Local structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2534</th>\n",
       "      <td>http://edamontology.org/operation_0510</td>\n",
       "      <td>18</td>\n",
       "      <td>Global structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2535</th>\n",
       "      <td>http://edamontology.org/operation_0511</td>\n",
       "      <td>18</td>\n",
       "      <td>Profile-profile alignment (pairwise)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2536</th>\n",
       "      <td>http://edamontology.org/operation_0512</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment generation (multiple profile)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2537</th>\n",
       "      <td>http://edamontology.org/operation_0513</td>\n",
       "      <td>18</td>\n",
       "      <td>3D profile-to-3D profile alignment (pairwise)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2538</th>\n",
       "      <td>http://edamontology.org/operation_0514</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural profile alignment generation (multiple)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2539</th>\n",
       "      <td>http://edamontology.org/operation_0515</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (tool metadata)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2540</th>\n",
       "      <td>http://edamontology.org/operation_0516</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (database metadata)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2541</th>\n",
       "      <td>http://edamontology.org/operation_0517</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (for large scale sequencing)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2542</th>\n",
       "      <td>http://edamontology.org/operation_0518</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (for genotyping polymorphisms)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2543</th>\n",
       "      <td>http://edamontology.org/operation_0519</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (for gene transcription profiling)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2544</th>\n",
       "      <td>http://edamontology.org/operation_0520</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (for conserved primers)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2545</th>\n",
       "      <td>http://edamontology.org/operation_0521</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (based on gene structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2546</th>\n",
       "      <td>http://edamontology.org/operation_0522</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR primer design (for methylation PCRs)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2547</th>\n",
       "      <td>http://edamontology.org/operation_0523</td>\n",
       "      <td>18</td>\n",
       "      <td>Mapping assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2548</th>\n",
       "      <td>http://edamontology.org/operation_0524</td>\n",
       "      <td>18</td>\n",
       "      <td>De-novo assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2549</th>\n",
       "      <td>http://edamontology.org/operation_0525</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2550</th>\n",
       "      <td>http://edamontology.org/operation_0526</td>\n",
       "      <td>18</td>\n",
       "      <td>EST assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2551</th>\n",
       "      <td>http://edamontology.org/operation_0527</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence tag mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2552</th>\n",
       "      <td>http://edamontology.org/operation_0528</td>\n",
       "      <td>18</td>\n",
       "      <td>SAGE data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2553</th>\n",
       "      <td>http://edamontology.org/operation_0529</td>\n",
       "      <td>18</td>\n",
       "      <td>MPSS data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2554</th>\n",
       "      <td>http://edamontology.org/operation_0530</td>\n",
       "      <td>18</td>\n",
       "      <td>SBS data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2555</th>\n",
       "      <td>http://edamontology.org/operation_0531</td>\n",
       "      <td>18</td>\n",
       "      <td>Heat map generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2556</th>\n",
       "      <td>http://edamontology.org/operation_0532</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression profile analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2557</th>\n",
       "      <td>http://edamontology.org/operation_0533</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression profile pathway mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2558</th>\n",
       "      <td>http://edamontology.org/operation_0534</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure assignment (from coordinate data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2559</th>\n",
       "      <td>http://edamontology.org/operation_0535</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure assignment (from CD data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2560</th>\n",
       "      <td>http://edamontology.org/operation_0536</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure assignment (from X-ray crystallographic data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2561</th>\n",
       "      <td>http://edamontology.org/operation_0537</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure assignment (from NMR data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2562</th>\n",
       "      <td>http://edamontology.org/operation_0538</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (data centric)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2563</th>\n",
       "      <td>http://edamontology.org/operation_0539</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (method centric)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2564</th>\n",
       "      <td>http://edamontology.org/operation_0540</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (from molecular sequences)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2565</th>\n",
       "      <td>http://edamontology.org/operation_0541</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (from continuous quantitative characters)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2566</th>\n",
       "      <td>http://edamontology.org/operation_0542</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (from gene frequencies)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2567</th>\n",
       "      <td>http://edamontology.org/operation_0543</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (from polymorphism data)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2568</th>\n",
       "      <td>http://edamontology.org/operation_0544</td>\n",
       "      <td>18</td>\n",
       "      <td>Species tree construction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2569</th>\n",
       "      <td>http://edamontology.org/operation_0545</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (parsimony methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2570</th>\n",
       "      <td>http://edamontology.org/operation_0546</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (minimum distance methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2571</th>\n",
       "      <td>http://edamontology.org/operation_0547</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (maximum likelihood and Bayesian methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2572</th>\n",
       "      <td>http://edamontology.org/operation_0548</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (quartet methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2573</th>\n",
       "      <td>http://edamontology.org/operation_0549</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic inference (AI methods)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2574</th>\n",
       "      <td>http://edamontology.org/operation_0550</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA substitution modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2575</th>\n",
       "      <td>http://edamontology.org/operation_0551</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree topology analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2576</th>\n",
       "      <td>http://edamontology.org/operation_0552</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree bootstrapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2577</th>\n",
       "      <td>http://edamontology.org/operation_0553</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene tree construction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2578</th>\n",
       "      <td>http://edamontology.org/operation_0554</td>\n",
       "      <td>18</td>\n",
       "      <td>Allele frequency distribution analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2579</th>\n",
       "      <td>http://edamontology.org/operation_0555</td>\n",
       "      <td>18</td>\n",
       "      <td>Consensus tree construction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2580</th>\n",
       "      <td>http://edamontology.org/operation_0556</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic sub/super tree construction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2581</th>\n",
       "      <td>http://edamontology.org/operation_0557</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree distances calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2582</th>\n",
       "      <td>http://edamontology.org/operation_0558</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2583</th>\n",
       "      <td>http://edamontology.org/operation_0559</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunogenicity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2584</th>\n",
       "      <td>http://edamontology.org/operation_0560</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA vaccine design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2585</th>\n",
       "      <td>http://edamontology.org/operation_0561</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence formatting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2586</th>\n",
       "      <td>http://edamontology.org/operation_0562</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment formatting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2587</th>\n",
       "      <td>http://edamontology.org/operation_0563</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table formatting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2588</th>\n",
       "      <td>http://edamontology.org/operation_0564</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2589</th>\n",
       "      <td>http://edamontology.org/operation_0565</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2590</th>\n",
       "      <td>http://edamontology.org/operation_0566</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2591</th>\n",
       "      <td>http://edamontology.org/operation_0567</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2592</th>\n",
       "      <td>http://edamontology.org/operation_0568</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2593</th>\n",
       "      <td>http://edamontology.org/operation_0569</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2594</th>\n",
       "      <td>http://edamontology.org/operation_0570</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2595</th>\n",
       "      <td>http://edamontology.org/operation_0571</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2596</th>\n",
       "      <td>http://edamontology.org/operation_0572</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction network visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2597</th>\n",
       "      <td>http://edamontology.org/operation_0573</td>\n",
       "      <td>18</td>\n",
       "      <td>Map drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2598</th>\n",
       "      <td>http://edamontology.org/operation_0574</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif rendering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2599</th>\n",
       "      <td>http://edamontology.org/operation_0575</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction map drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2600</th>\n",
       "      <td>http://edamontology.org/operation_0577</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA linear map rendering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2601</th>\n",
       "      <td>http://edamontology.org/operation_0578</td>\n",
       "      <td>18</td>\n",
       "      <td>Plasmid map drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2602</th>\n",
       "      <td>http://edamontology.org/operation_0579</td>\n",
       "      <td>18</td>\n",
       "      <td>Operon drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2603</th>\n",
       "      <td>http://edamontology.org/operation_1768</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding family identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2604</th>\n",
       "      <td>http://edamontology.org/operation_1769</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding energy calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2605</th>\n",
       "      <td>http://edamontology.org/operation_1774</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2606</th>\n",
       "      <td>http://edamontology.org/operation_1777</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein function prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2607</th>\n",
       "      <td>http://edamontology.org/operation_1778</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein function comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2608</th>\n",
       "      <td>http://edamontology.org/operation_1780</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence submission</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2609</th>\n",
       "      <td>http://edamontology.org/operation_1781</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulatory network analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2610</th>\n",
       "      <td>http://edamontology.org/operation_1812</td>\n",
       "      <td>18</td>\n",
       "      <td>Parsing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2611</th>\n",
       "      <td>http://edamontology.org/operation_1813</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2612</th>\n",
       "      <td>http://edamontology.org/operation_1814</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2613</th>\n",
       "      <td>http://edamontology.org/operation_1816</td>\n",
       "      <td>18</td>\n",
       "      <td>Surface rendering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2614</th>\n",
       "      <td>http://edamontology.org/operation_1817</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein atom surface calculation (accessible)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2615</th>\n",
       "      <td>http://edamontology.org/operation_1818</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein atom surface calculation (accessible molecular)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2616</th>\n",
       "      <td>http://edamontology.org/operation_1819</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue surface calculation (accessible)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2617</th>\n",
       "      <td>http://edamontology.org/operation_1820</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue surface calculation (vacuum accessible)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2618</th>\n",
       "      <td>http://edamontology.org/operation_1821</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue surface calculation (accessible molecular)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2619</th>\n",
       "      <td>http://edamontology.org/operation_1822</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue surface calculation (vacuum molecular)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2620</th>\n",
       "      <td>http://edamontology.org/operation_1823</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein surface calculation (accessible molecular)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2621</th>\n",
       "      <td>http://edamontology.org/operation_1824</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein surface calculation (accessible)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2622</th>\n",
       "      <td>http://edamontology.org/operation_1825</td>\n",
       "      <td>18</td>\n",
       "      <td>Backbone torsion angle calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2623</th>\n",
       "      <td>http://edamontology.org/operation_1826</td>\n",
       "      <td>18</td>\n",
       "      <td>Full torsion angle calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2624</th>\n",
       "      <td>http://edamontology.org/operation_1827</td>\n",
       "      <td>18</td>\n",
       "      <td>Cysteine torsion angle calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2625</th>\n",
       "      <td>http://edamontology.org/operation_1828</td>\n",
       "      <td>18</td>\n",
       "      <td>Tau angle calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2626</th>\n",
       "      <td>http://edamontology.org/operation_1829</td>\n",
       "      <td>18</td>\n",
       "      <td>Cysteine bridge detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2627</th>\n",
       "      <td>http://edamontology.org/operation_1830</td>\n",
       "      <td>18</td>\n",
       "      <td>Free cysteine detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2628</th>\n",
       "      <td>http://edamontology.org/operation_1831</td>\n",
       "      <td>18</td>\n",
       "      <td>Metal-bound cysteine detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2629</th>\n",
       "      <td>http://edamontology.org/operation_1832</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue contact calculation (residue-nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2630</th>\n",
       "      <td>http://edamontology.org/operation_1834</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-metal contact calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2631</th>\n",
       "      <td>http://edamontology.org/operation_1835</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue contact calculation (residue-negative ion)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2632</th>\n",
       "      <td>http://edamontology.org/operation_1836</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue bump detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2633</th>\n",
       "      <td>http://edamontology.org/operation_1837</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue symmetry contact calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2634</th>\n",
       "      <td>http://edamontology.org/operation_1838</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue contact calculation (residue-ligand)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2635</th>\n",
       "      <td>http://edamontology.org/operation_1839</td>\n",
       "      <td>18</td>\n",
       "      <td>Salt bridge calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2636</th>\n",
       "      <td>http://edamontology.org/operation_1841</td>\n",
       "      <td>18</td>\n",
       "      <td>Rotamer likelihood prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2637</th>\n",
       "      <td>http://edamontology.org/operation_1842</td>\n",
       "      <td>18</td>\n",
       "      <td>Proline mutation value calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2638</th>\n",
       "      <td>http://edamontology.org/operation_1843</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue packing validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2639</th>\n",
       "      <td>http://edamontology.org/operation_1844</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein geometry validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2640</th>\n",
       "      <td>http://edamontology.org/operation_1845</td>\n",
       "      <td>18</td>\n",
       "      <td>PDB file sequence retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2641</th>\n",
       "      <td>http://edamontology.org/operation_1846</td>\n",
       "      <td>18</td>\n",
       "      <td>HET group detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2642</th>\n",
       "      <td>http://edamontology.org/operation_1847</td>\n",
       "      <td>18</td>\n",
       "      <td>DSSP secondary structure assignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2643</th>\n",
       "      <td>http://edamontology.org/operation_1848</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure formatting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2644</th>\n",
       "      <td>http://edamontology.org/operation_1850</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein cysteine and disulfide bond assignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2645</th>\n",
       "      <td>http://edamontology.org/operation_1913</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2646</th>\n",
       "      <td>http://edamontology.org/operation_1914</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure retrieval (water)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2647</th>\n",
       "      <td>http://edamontology.org/operation_2008</td>\n",
       "      <td>18</td>\n",
       "      <td>siRNA duplex prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2648</th>\n",
       "      <td>http://edamontology.org/operation_2089</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment refinement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2649</th>\n",
       "      <td>http://edamontology.org/operation_2120</td>\n",
       "      <td>18</td>\n",
       "      <td>Listfile processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2650</th>\n",
       "      <td>http://edamontology.org/operation_2121</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence file editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2651</th>\n",
       "      <td>http://edamontology.org/operation_2122</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment file processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2652</th>\n",
       "      <td>http://edamontology.org/operation_2123</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecule data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2653</th>\n",
       "      <td>http://edamontology.org/operation_2222</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (ontology annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2654</th>\n",
       "      <td>http://edamontology.org/operation_2224</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (ontology concept)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2655</th>\n",
       "      <td>http://edamontology.org/operation_2233</td>\n",
       "      <td>18</td>\n",
       "      <td>Representative sequence identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2656</th>\n",
       "      <td>http://edamontology.org/operation_2234</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure file processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2657</th>\n",
       "      <td>http://edamontology.org/operation_2237</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (sequence profile)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2658</th>\n",
       "      <td>http://edamontology.org/operation_2238</td>\n",
       "      <td>18</td>\n",
       "      <td>Statistical calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2659</th>\n",
       "      <td>http://edamontology.org/operation_2239</td>\n",
       "      <td>18</td>\n",
       "      <td>3D-1D scoring matrix generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2660</th>\n",
       "      <td>http://edamontology.org/operation_2241</td>\n",
       "      <td>18</td>\n",
       "      <td>Transmembrane protein visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2661</th>\n",
       "      <td>http://edamontology.org/operation_2246</td>\n",
       "      <td>18</td>\n",
       "      <td>Demonstration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2662</th>\n",
       "      <td>http://edamontology.org/operation_2264</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (pathway or network)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2663</th>\n",
       "      <td>http://edamontology.org/operation_2265</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (identifier)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2664</th>\n",
       "      <td>http://edamontology.org/operation_2284</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid density plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2665</th>\n",
       "      <td>http://edamontology.org/operation_2403</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2666</th>\n",
       "      <td>http://edamontology.org/operation_2404</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2667</th>\n",
       "      <td>http://edamontology.org/operation_2405</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2668</th>\n",
       "      <td>http://edamontology.org/operation_2406</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2669</th>\n",
       "      <td>http://edamontology.org/operation_2407</td>\n",
       "      <td>18</td>\n",
       "      <td>Annotation processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2670</th>\n",
       "      <td>http://edamontology.org/operation_2408</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2671</th>\n",
       "      <td>http://edamontology.org/operation_2409</td>\n",
       "      <td>18</td>\n",
       "      <td>Data handling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2672</th>\n",
       "      <td>http://edamontology.org/operation_2410</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2673</th>\n",
       "      <td>http://edamontology.org/operation_2411</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural profile processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2674</th>\n",
       "      <td>http://edamontology.org/operation_2412</td>\n",
       "      <td>18</td>\n",
       "      <td>Data index processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2675</th>\n",
       "      <td>http://edamontology.org/operation_2413</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profile processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2676</th>\n",
       "      <td>http://edamontology.org/operation_2414</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2677</th>\n",
       "      <td>http://edamontology.org/operation_2415</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2678</th>\n",
       "      <td>http://edamontology.org/operation_2416</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2679</th>\n",
       "      <td>http://edamontology.org/operation_2417</td>\n",
       "      <td>18</td>\n",
       "      <td>Physicochemical property data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2680</th>\n",
       "      <td>http://edamontology.org/operation_2419</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer and probe design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2681</th>\n",
       "      <td>http://edamontology.org/operation_2420</td>\n",
       "      <td>18</td>\n",
       "      <td>Operation (typed)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2682</th>\n",
       "      <td>http://edamontology.org/operation_2421</td>\n",
       "      <td>18</td>\n",
       "      <td>Database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2683</th>\n",
       "      <td>http://edamontology.org/operation_2422</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2684</th>\n",
       "      <td>http://edamontology.org/operation_2423</td>\n",
       "      <td>18</td>\n",
       "      <td>Prediction and recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2685</th>\n",
       "      <td>http://edamontology.org/operation_2424</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2686</th>\n",
       "      <td>http://edamontology.org/operation_2425</td>\n",
       "      <td>18</td>\n",
       "      <td>Optimisation and refinement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2687</th>\n",
       "      <td>http://edamontology.org/operation_2426</td>\n",
       "      <td>18</td>\n",
       "      <td>Modelling and simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2688</th>\n",
       "      <td>http://edamontology.org/operation_2427</td>\n",
       "      <td>18</td>\n",
       "      <td>Data handling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2689</th>\n",
       "      <td>http://edamontology.org/operation_2428</td>\n",
       "      <td>18</td>\n",
       "      <td>Validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2690</th>\n",
       "      <td>http://edamontology.org/operation_2429</td>\n",
       "      <td>18</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2691</th>\n",
       "      <td>http://edamontology.org/operation_2430</td>\n",
       "      <td>18</td>\n",
       "      <td>Design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2692</th>\n",
       "      <td>http://edamontology.org/operation_2432</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2693</th>\n",
       "      <td>http://edamontology.org/operation_2433</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage table processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2694</th>\n",
       "      <td>http://edamontology.org/operation_2434</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (codon usage table)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2695</th>\n",
       "      <td>http://edamontology.org/operation_2435</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression profile processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2696</th>\n",
       "      <td>http://edamontology.org/operation_2436</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene-set enrichment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2697</th>\n",
       "      <td>http://edamontology.org/operation_2437</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulatory network prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2698</th>\n",
       "      <td>http://edamontology.org/operation_2438</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2699</th>\n",
       "      <td>http://edamontology.org/operation_2439</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2700</th>\n",
       "      <td>http://edamontology.org/operation_2440</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure processing (RNA)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2701</th>\n",
       "      <td>http://edamontology.org/operation_2441</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2702</th>\n",
       "      <td>http://edamontology.org/operation_2442</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2703</th>\n",
       "      <td>http://edamontology.org/operation_2443</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2704</th>\n",
       "      <td>http://edamontology.org/operation_2444</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2705</th>\n",
       "      <td>http://edamontology.org/operation_2445</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction network processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2706</th>\n",
       "      <td>http://edamontology.org/operation_2446</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2707</th>\n",
       "      <td>http://edamontology.org/operation_2447</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence processing (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2708</th>\n",
       "      <td>http://edamontology.org/operation_2448</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence processing (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2709</th>\n",
       "      <td>http://edamontology.org/operation_2451</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2710</th>\n",
       "      <td>http://edamontology.org/operation_2452</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence cluster processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2711</th>\n",
       "      <td>http://edamontology.org/operation_2453</td>\n",
       "      <td>18</td>\n",
       "      <td>Feature table processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2712</th>\n",
       "      <td>http://edamontology.org/operation_2454</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2713</th>\n",
       "      <td>http://edamontology.org/operation_2456</td>\n",
       "      <td>18</td>\n",
       "      <td>GPCR classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2714</th>\n",
       "      <td>http://edamontology.org/operation_2457</td>\n",
       "      <td>18</td>\n",
       "      <td>GPCR coupling selectivity prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2715</th>\n",
       "      <td>http://edamontology.org/operation_2459</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure processing (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2716</th>\n",
       "      <td>http://edamontology.org/operation_2460</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein atom surface calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2717</th>\n",
       "      <td>http://edamontology.org/operation_2461</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue surface calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2718</th>\n",
       "      <td>http://edamontology.org/operation_2462</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein surface calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2719</th>\n",
       "      <td>http://edamontology.org/operation_2463</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2720</th>\n",
       "      <td>http://edamontology.org/operation_2464</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2721</th>\n",
       "      <td>http://edamontology.org/operation_2465</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2722</th>\n",
       "      <td>http://edamontology.org/operation_2466</td>\n",
       "      <td>18</td>\n",
       "      <td>Map annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2723</th>\n",
       "      <td>http://edamontology.org/operation_2467</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (protein annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2724</th>\n",
       "      <td>http://edamontology.org/operation_2468</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (phylogenetic tree)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2725</th>\n",
       "      <td>http://edamontology.org/operation_2469</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (protein interaction annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2726</th>\n",
       "      <td>http://edamontology.org/operation_2470</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (protein family annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2727</th>\n",
       "      <td>http://edamontology.org/operation_2471</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (RNA family annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2728</th>\n",
       "      <td>http://edamontology.org/operation_2472</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (gene annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2729</th>\n",
       "      <td>http://edamontology.org/operation_2473</td>\n",
       "      <td>18</td>\n",
       "      <td>Data retrieval (genotype and phenotype annotation)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2730</th>\n",
       "      <td>http://edamontology.org/operation_2474</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein architecture comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2731</th>\n",
       "      <td>http://edamontology.org/operation_2475</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein architecture recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2732</th>\n",
       "      <td>http://edamontology.org/operation_2476</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2733</th>\n",
       "      <td>http://edamontology.org/operation_2478</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2734</th>\n",
       "      <td>http://edamontology.org/operation_2479</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2735</th>\n",
       "      <td>http://edamontology.org/operation_2480</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2736</th>\n",
       "      <td>http://edamontology.org/operation_2481</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2737</th>\n",
       "      <td>http://edamontology.org/operation_2482</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2738</th>\n",
       "      <td>http://edamontology.org/operation_2483</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2739</th>\n",
       "      <td>http://edamontology.org/operation_2485</td>\n",
       "      <td>18</td>\n",
       "      <td>Helical wheel drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2740</th>\n",
       "      <td>http://edamontology.org/operation_2486</td>\n",
       "      <td>18</td>\n",
       "      <td>Topology diagram drawing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2741</th>\n",
       "      <td>http://edamontology.org/operation_2487</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2742</th>\n",
       "      <td>http://edamontology.org/operation_2488</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2743</th>\n",
       "      <td>http://edamontology.org/operation_2489</td>\n",
       "      <td>18</td>\n",
       "      <td>Subcellular localisation prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2744</th>\n",
       "      <td>http://edamontology.org/operation_2490</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue contact calculation (residue-residue)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2745</th>\n",
       "      <td>http://edamontology.org/operation_2491</td>\n",
       "      <td>18</td>\n",
       "      <td>Hydrogen bond calculation (inter-residue)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2746</th>\n",
       "      <td>http://edamontology.org/operation_2492</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2747</th>\n",
       "      <td>http://edamontology.org/operation_2493</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2748</th>\n",
       "      <td>http://edamontology.org/operation_2495</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2749</th>\n",
       "      <td>http://edamontology.org/operation_2496</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulatory network processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2750</th>\n",
       "      <td>http://edamontology.org/operation_2497</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2751</th>\n",
       "      <td>http://edamontology.org/operation_2498</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing-based expression profile data analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2752</th>\n",
       "      <td>http://edamontology.org/operation_2499</td>\n",
       "      <td>18</td>\n",
       "      <td>Splicing analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2753</th>\n",
       "      <td>http://edamontology.org/operation_2500</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray raw data analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2754</th>\n",
       "      <td>http://edamontology.org/operation_2501</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2755</th>\n",
       "      <td>http://edamontology.org/operation_2502</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2756</th>\n",
       "      <td>http://edamontology.org/operation_2503</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2757</th>\n",
       "      <td>http://edamontology.org/operation_2504</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2758</th>\n",
       "      <td>http://edamontology.org/operation_2505</td>\n",
       "      <td>18</td>\n",
       "      <td>Text processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2759</th>\n",
       "      <td>http://edamontology.org/operation_2506</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence alignment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2760</th>\n",
       "      <td>http://edamontology.org/operation_2507</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence alignment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2761</th>\n",
       "      <td>http://edamontology.org/operation_2508</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2762</th>\n",
       "      <td>http://edamontology.org/operation_2509</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2763</th>\n",
       "      <td>http://edamontology.org/operation_2510</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA back-translation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2764</th>\n",
       "      <td>http://edamontology.org/operation_2511</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence editing (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2765</th>\n",
       "      <td>http://edamontology.org/operation_2512</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence editing (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2766</th>\n",
       "      <td>http://edamontology.org/operation_2513</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence generation (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2767</th>\n",
       "      <td>http://edamontology.org/operation_2514</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence generation (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2768</th>\n",
       "      <td>http://edamontology.org/operation_2515</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2769</th>\n",
       "      <td>http://edamontology.org/operation_2516</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2770</th>\n",
       "      <td>http://edamontology.org/operation_2518</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2771</th>\n",
       "      <td>http://edamontology.org/operation_2519</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure processing (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2772</th>\n",
       "      <td>http://edamontology.org/operation_2520</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2773</th>\n",
       "      <td>http://edamontology.org/operation_2521</td>\n",
       "      <td>18</td>\n",
       "      <td>Map data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2774</th>\n",
       "      <td>http://edamontology.org/operation_2574</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2775</th>\n",
       "      <td>http://edamontology.org/operation_2575</td>\n",
       "      <td>18</td>\n",
       "      <td>Binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2776</th>\n",
       "      <td>http://edamontology.org/operation_2844</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2777</th>\n",
       "      <td>http://edamontology.org/operation_2871</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence tagged site (STS) mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2778</th>\n",
       "      <td>http://edamontology.org/operation_2928</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2779</th>\n",
       "      <td>http://edamontology.org/operation_2929</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein fragment weight comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2780</th>\n",
       "      <td>http://edamontology.org/operation_2930</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein property comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2781</th>\n",
       "      <td>http://edamontology.org/operation_2931</td>\n",
       "      <td>18</td>\n",
       "      <td>Secondary structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2782</th>\n",
       "      <td>http://edamontology.org/operation_2932</td>\n",
       "      <td>18</td>\n",
       "      <td>Hopp and Woods plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2783</th>\n",
       "      <td>http://edamontology.org/operation_2934</td>\n",
       "      <td>18</td>\n",
       "      <td>Cluster textual view generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2784</th>\n",
       "      <td>http://edamontology.org/operation_2935</td>\n",
       "      <td>18</td>\n",
       "      <td>Clustering profile plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2785</th>\n",
       "      <td>http://edamontology.org/operation_2936</td>\n",
       "      <td>18</td>\n",
       "      <td>Dendrograph plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2786</th>\n",
       "      <td>http://edamontology.org/operation_2937</td>\n",
       "      <td>18</td>\n",
       "      <td>Proximity map plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2787</th>\n",
       "      <td>http://edamontology.org/operation_2938</td>\n",
       "      <td>18</td>\n",
       "      <td>Dendrogram visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2788</th>\n",
       "      <td>http://edamontology.org/operation_2939</td>\n",
       "      <td>18</td>\n",
       "      <td>Principal component visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2789</th>\n",
       "      <td>http://edamontology.org/operation_2940</td>\n",
       "      <td>18</td>\n",
       "      <td>Scatter plot plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2790</th>\n",
       "      <td>http://edamontology.org/operation_2941</td>\n",
       "      <td>18</td>\n",
       "      <td>Whole microarray graph plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2791</th>\n",
       "      <td>http://edamontology.org/operation_2942</td>\n",
       "      <td>18</td>\n",
       "      <td>Treemap visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2792</th>\n",
       "      <td>http://edamontology.org/operation_2943</td>\n",
       "      <td>18</td>\n",
       "      <td>Box-Whisker plot plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2793</th>\n",
       "      <td>http://edamontology.org/operation_2944</td>\n",
       "      <td>18</td>\n",
       "      <td>Physical mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2794</th>\n",
       "      <td>http://edamontology.org/operation_2945</td>\n",
       "      <td>18</td>\n",
       "      <td>Analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2795</th>\n",
       "      <td>http://edamontology.org/operation_2946</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2796</th>\n",
       "      <td>http://edamontology.org/operation_2947</td>\n",
       "      <td>18</td>\n",
       "      <td>Article analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2797</th>\n",
       "      <td>http://edamontology.org/operation_2948</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular interaction analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2798</th>\n",
       "      <td>http://edamontology.org/operation_2949</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interaction analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2799</th>\n",
       "      <td>http://edamontology.org/operation_2950</td>\n",
       "      <td>18</td>\n",
       "      <td>Residue distance calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2800</th>\n",
       "      <td>http://edamontology.org/operation_2951</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2801</th>\n",
       "      <td>http://edamontology.org/operation_2952</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2802</th>\n",
       "      <td>http://edamontology.org/operation_2962</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage bias calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2803</th>\n",
       "      <td>http://edamontology.org/operation_2963</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage bias plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2804</th>\n",
       "      <td>http://edamontology.org/operation_2964</td>\n",
       "      <td>18</td>\n",
       "      <td>Codon usage fraction calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2805</th>\n",
       "      <td>http://edamontology.org/operation_2990</td>\n",
       "      <td>18</td>\n",
       "      <td>Classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2806</th>\n",
       "      <td>http://edamontology.org/operation_2993</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular interaction data processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2807</th>\n",
       "      <td>http://edamontology.org/operation_2995</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2808</th>\n",
       "      <td>http://edamontology.org/operation_2996</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2809</th>\n",
       "      <td>http://edamontology.org/operation_2997</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2810</th>\n",
       "      <td>http://edamontology.org/operation_2998</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2811</th>\n",
       "      <td>http://edamontology.org/operation_3023</td>\n",
       "      <td>18</td>\n",
       "      <td>Prediction and recognition (protein)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2812</th>\n",
       "      <td>http://edamontology.org/operation_3024</td>\n",
       "      <td>18</td>\n",
       "      <td>Prediction and recognition (nucleic acid)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2813</th>\n",
       "      <td>http://edamontology.org/operation_3080</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2814</th>\n",
       "      <td>http://edamontology.org/operation_3081</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2815</th>\n",
       "      <td>http://edamontology.org/operation_3083</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2816</th>\n",
       "      <td>http://edamontology.org/operation_3084</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein function prediction (from sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2817</th>\n",
       "      <td>http://edamontology.org/operation_3087</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2818</th>\n",
       "      <td>http://edamontology.org/operation_3088</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein property calculation (from sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2819</th>\n",
       "      <td>http://edamontology.org/operation_3090</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein feature prediction (from structure)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2820</th>\n",
       "      <td>http://edamontology.org/operation_3092</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2821</th>\n",
       "      <td>http://edamontology.org/operation_3093</td>\n",
       "      <td>18</td>\n",
       "      <td>Database search (by sequence)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2822</th>\n",
       "      <td>http://edamontology.org/operation_3094</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction network prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2823</th>\n",
       "      <td>http://edamontology.org/operation_3095</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2824</th>\n",
       "      <td>http://edamontology.org/operation_3096</td>\n",
       "      <td>18</td>\n",
       "      <td>Editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2825</th>\n",
       "      <td>http://edamontology.org/operation_3180</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly validation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2826</th>\n",
       "      <td>http://edamontology.org/operation_3182</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2827</th>\n",
       "      <td>http://edamontology.org/operation_3183</td>\n",
       "      <td>18</td>\n",
       "      <td>Localised reassembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2828</th>\n",
       "      <td>http://edamontology.org/operation_3184</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2829</th>\n",
       "      <td>http://edamontology.org/operation_3185</td>\n",
       "      <td>18</td>\n",
       "      <td>Base-calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2830</th>\n",
       "      <td>http://edamontology.org/operation_3186</td>\n",
       "      <td>18</td>\n",
       "      <td>Bisulfite mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2831</th>\n",
       "      <td>http://edamontology.org/operation_3187</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence contamination filtering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2832</th>\n",
       "      <td>http://edamontology.org/operation_3189</td>\n",
       "      <td>18</td>\n",
       "      <td>Trim ends</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2833</th>\n",
       "      <td>http://edamontology.org/operation_3190</td>\n",
       "      <td>18</td>\n",
       "      <td>Trim vector</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2834</th>\n",
       "      <td>http://edamontology.org/operation_3191</td>\n",
       "      <td>18</td>\n",
       "      <td>Trim to reference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2835</th>\n",
       "      <td>http://edamontology.org/operation_3192</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence trimming</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2836</th>\n",
       "      <td>http://edamontology.org/operation_3194</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome feature comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2837</th>\n",
       "      <td>http://edamontology.org/operation_3195</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing error detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2838</th>\n",
       "      <td>http://edamontology.org/operation_3196</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotyping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2839</th>\n",
       "      <td>http://edamontology.org/operation_3197</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic variation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2840</th>\n",
       "      <td>http://edamontology.org/operation_3198</td>\n",
       "      <td>18</td>\n",
       "      <td>Read mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2841</th>\n",
       "      <td>http://edamontology.org/operation_3199</td>\n",
       "      <td>18</td>\n",
       "      <td>Split read mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2842</th>\n",
       "      <td>http://edamontology.org/operation_3200</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA barcoding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2843</th>\n",
       "      <td>http://edamontology.org/operation_3201</td>\n",
       "      <td>18</td>\n",
       "      <td>SNP calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2844</th>\n",
       "      <td>http://edamontology.org/operation_3202</td>\n",
       "      <td>18</td>\n",
       "      <td>Polymorphism detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2845</th>\n",
       "      <td>http://edamontology.org/operation_3203</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromatogram visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2846</th>\n",
       "      <td>http://edamontology.org/operation_3204</td>\n",
       "      <td>18</td>\n",
       "      <td>Methylation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2847</th>\n",
       "      <td>http://edamontology.org/operation_3205</td>\n",
       "      <td>18</td>\n",
       "      <td>Methylation calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2848</th>\n",
       "      <td>http://edamontology.org/operation_3206</td>\n",
       "      <td>18</td>\n",
       "      <td>Whole genome methylation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2849</th>\n",
       "      <td>http://edamontology.org/operation_3207</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene methylation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2850</th>\n",
       "      <td>http://edamontology.org/operation_3208</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2851</th>\n",
       "      <td>http://edamontology.org/operation_3209</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2852</th>\n",
       "      <td>http://edamontology.org/operation_3211</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome indexing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2853</th>\n",
       "      <td>http://edamontology.org/operation_3212</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome indexing (Burrows-Wheeler)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2854</th>\n",
       "      <td>http://edamontology.org/operation_3213</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome indexing (suffix arrays)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2855</th>\n",
       "      <td>http://edamontology.org/operation_3214</td>\n",
       "      <td>18</td>\n",
       "      <td>Spectral analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2856</th>\n",
       "      <td>http://edamontology.org/operation_3215</td>\n",
       "      <td>18</td>\n",
       "      <td>Peak detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2857</th>\n",
       "      <td>http://edamontology.org/operation_3216</td>\n",
       "      <td>18</td>\n",
       "      <td>Scaffolding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2858</th>\n",
       "      <td>http://edamontology.org/operation_3217</td>\n",
       "      <td>18</td>\n",
       "      <td>Scaffold gap completion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2859</th>\n",
       "      <td>http://edamontology.org/operation_3218</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing quality control</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2860</th>\n",
       "      <td>http://edamontology.org/operation_3219</td>\n",
       "      <td>18</td>\n",
       "      <td>Read pre-processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2861</th>\n",
       "      <td>http://edamontology.org/operation_3221</td>\n",
       "      <td>18</td>\n",
       "      <td>Species frequency estimation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2862</th>\n",
       "      <td>http://edamontology.org/operation_3222</td>\n",
       "      <td>18</td>\n",
       "      <td>Peak calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2863</th>\n",
       "      <td>http://edamontology.org/operation_3223</td>\n",
       "      <td>18</td>\n",
       "      <td>Differential gene expression profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2864</th>\n",
       "      <td>http://edamontology.org/operation_3224</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene set testing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2865</th>\n",
       "      <td>http://edamontology.org/operation_3225</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2866</th>\n",
       "      <td>http://edamontology.org/operation_3226</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant prioritisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2867</th>\n",
       "      <td>http://edamontology.org/operation_3227</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2868</th>\n",
       "      <td>http://edamontology.org/operation_3228</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural variation detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2869</th>\n",
       "      <td>http://edamontology.org/operation_3229</td>\n",
       "      <td>18</td>\n",
       "      <td>Exome assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2870</th>\n",
       "      <td>http://edamontology.org/operation_3230</td>\n",
       "      <td>18</td>\n",
       "      <td>Read depth analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2871</th>\n",
       "      <td>http://edamontology.org/operation_3232</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression QTL analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2872</th>\n",
       "      <td>http://edamontology.org/operation_3233</td>\n",
       "      <td>18</td>\n",
       "      <td>Copy number estimation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2873</th>\n",
       "      <td>http://edamontology.org/operation_3237</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer removal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2874</th>\n",
       "      <td>http://edamontology.org/operation_3258</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptome assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2875</th>\n",
       "      <td>http://edamontology.org/operation_3259</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptome assembly (de novo)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2876</th>\n",
       "      <td>http://edamontology.org/operation_3260</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptome assembly (mapping)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2877</th>\n",
       "      <td>http://edamontology.org/operation_3267</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence coordinate conversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2878</th>\n",
       "      <td>http://edamontology.org/operation_3278</td>\n",
       "      <td>18</td>\n",
       "      <td>Document similarity calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2879</th>\n",
       "      <td>http://edamontology.org/operation_3279</td>\n",
       "      <td>18</td>\n",
       "      <td>Document clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2880</th>\n",
       "      <td>http://edamontology.org/operation_3280</td>\n",
       "      <td>18</td>\n",
       "      <td>Named-entity and concept recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2881</th>\n",
       "      <td>http://edamontology.org/operation_3282</td>\n",
       "      <td>18</td>\n",
       "      <td>ID mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2882</th>\n",
       "      <td>http://edamontology.org/operation_3283</td>\n",
       "      <td>18</td>\n",
       "      <td>Anonymisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2883</th>\n",
       "      <td>http://edamontology.org/operation_3289</td>\n",
       "      <td>18</td>\n",
       "      <td>ID retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2884</th>\n",
       "      <td>http://edamontology.org/operation_3348</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence checksum generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2885</th>\n",
       "      <td>http://edamontology.org/operation_3349</td>\n",
       "      <td>18</td>\n",
       "      <td>Bibliography generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2886</th>\n",
       "      <td>http://edamontology.org/operation_3350</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein quaternary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2887</th>\n",
       "      <td>http://edamontology.org/operation_3351</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular surface analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2888</th>\n",
       "      <td>http://edamontology.org/operation_3352</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2889</th>\n",
       "      <td>http://edamontology.org/operation_3353</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2890</th>\n",
       "      <td>http://edamontology.org/operation_3357</td>\n",
       "      <td>18</td>\n",
       "      <td>Format detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2891</th>\n",
       "      <td>http://edamontology.org/operation_3359</td>\n",
       "      <td>18</td>\n",
       "      <td>Splitting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2892</th>\n",
       "      <td>http://edamontology.org/operation_3429</td>\n",
       "      <td>18</td>\n",
       "      <td>Generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2893</th>\n",
       "      <td>http://edamontology.org/operation_3430</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2894</th>\n",
       "      <td>http://edamontology.org/operation_3431</td>\n",
       "      <td>18</td>\n",
       "      <td>Deposition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2895</th>\n",
       "      <td>http://edamontology.org/operation_3432</td>\n",
       "      <td>18</td>\n",
       "      <td>Clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2896</th>\n",
       "      <td>http://edamontology.org/operation_3433</td>\n",
       "      <td>18</td>\n",
       "      <td>Assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2897</th>\n",
       "      <td>http://edamontology.org/operation_3434</td>\n",
       "      <td>18</td>\n",
       "      <td>Conversion</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2898</th>\n",
       "      <td>http://edamontology.org/operation_3435</td>\n",
       "      <td>18</td>\n",
       "      <td>Standardisation and normalisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2899</th>\n",
       "      <td>http://edamontology.org/operation_3436</td>\n",
       "      <td>18</td>\n",
       "      <td>Aggregation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2900</th>\n",
       "      <td>http://edamontology.org/operation_3437</td>\n",
       "      <td>18</td>\n",
       "      <td>Article comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2901</th>\n",
       "      <td>http://edamontology.org/operation_3438</td>\n",
       "      <td>18</td>\n",
       "      <td>Calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2902</th>\n",
       "      <td>http://edamontology.org/operation_3439</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway or network prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2903</th>\n",
       "      <td>http://edamontology.org/operation_3440</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2904</th>\n",
       "      <td>http://edamontology.org/operation_3441</td>\n",
       "      <td>18</td>\n",
       "      <td>Plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2905</th>\n",
       "      <td>http://edamontology.org/operation_3443</td>\n",
       "      <td>18</td>\n",
       "      <td>Image analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2906</th>\n",
       "      <td>http://edamontology.org/operation_3445</td>\n",
       "      <td>18</td>\n",
       "      <td>Diffraction data analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2907</th>\n",
       "      <td>http://edamontology.org/operation_3446</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell migration analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2908</th>\n",
       "      <td>http://edamontology.org/operation_3447</td>\n",
       "      <td>18</td>\n",
       "      <td>Diffraction data reduction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2909</th>\n",
       "      <td>http://edamontology.org/operation_3450</td>\n",
       "      <td>18</td>\n",
       "      <td>Neurite measurement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2910</th>\n",
       "      <td>http://edamontology.org/operation_3453</td>\n",
       "      <td>18</td>\n",
       "      <td>Diffraction data integration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2911</th>\n",
       "      <td>http://edamontology.org/operation_3454</td>\n",
       "      <td>18</td>\n",
       "      <td>Phasing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2912</th>\n",
       "      <td>http://edamontology.org/operation_3455</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular replacement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2913</th>\n",
       "      <td>http://edamontology.org/operation_3456</td>\n",
       "      <td>18</td>\n",
       "      <td>Rigid body refinement</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2914</th>\n",
       "      <td>http://edamontology.org/operation_3457</td>\n",
       "      <td>18</td>\n",
       "      <td>Single particle analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2915</th>\n",
       "      <td>http://edamontology.org/operation_3458</td>\n",
       "      <td>18</td>\n",
       "      <td>Single particle alignment and classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2916</th>\n",
       "      <td>http://edamontology.org/operation_3459</td>\n",
       "      <td>18</td>\n",
       "      <td>Functional clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2917</th>\n",
       "      <td>http://edamontology.org/operation_3460</td>\n",
       "      <td>18</td>\n",
       "      <td>Taxonomic classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2918</th>\n",
       "      <td>http://edamontology.org/operation_3461</td>\n",
       "      <td>18</td>\n",
       "      <td>Virulence prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2919</th>\n",
       "      <td>http://edamontology.org/operation_3463</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression correlation analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2920</th>\n",
       "      <td>http://edamontology.org/operation_3465</td>\n",
       "      <td>18</td>\n",
       "      <td>Correlation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2921</th>\n",
       "      <td>http://edamontology.org/operation_3469</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA structure covariance model generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2922</th>\n",
       "      <td>http://edamontology.org/operation_3470</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA secondary structure prediction (shape-based)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2923</th>\n",
       "      <td>http://edamontology.org/operation_3471</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding prediction (alignment-based)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2924</th>\n",
       "      <td>http://edamontology.org/operation_3472</td>\n",
       "      <td>18</td>\n",
       "      <td>k-mer counting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2925</th>\n",
       "      <td>http://edamontology.org/operation_3478</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic reconstruction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2926</th>\n",
       "      <td>http://edamontology.org/operation_3480</td>\n",
       "      <td>18</td>\n",
       "      <td>Probabilistic data generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2927</th>\n",
       "      <td>http://edamontology.org/operation_3481</td>\n",
       "      <td>18</td>\n",
       "      <td>Probabilistic sequence generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2928</th>\n",
       "      <td>http://edamontology.org/operation_3482</td>\n",
       "      <td>18</td>\n",
       "      <td>Antimicrobial resistance prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2929</th>\n",
       "      <td>http://edamontology.org/operation_3501</td>\n",
       "      <td>18</td>\n",
       "      <td>Enrichment analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2930</th>\n",
       "      <td>http://edamontology.org/operation_3502</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical similarity enrichment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2931</th>\n",
       "      <td>http://edamontology.org/operation_3503</td>\n",
       "      <td>18</td>\n",
       "      <td>Incident curve plotting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2932</th>\n",
       "      <td>http://edamontology.org/operation_3504</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant pattern analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2933</th>\n",
       "      <td>http://edamontology.org/operation_3545</td>\n",
       "      <td>18</td>\n",
       "      <td>Mathematical modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2934</th>\n",
       "      <td>http://edamontology.org/operation_3552</td>\n",
       "      <td>18</td>\n",
       "      <td>Microscope image visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2935</th>\n",
       "      <td>http://edamontology.org/operation_3553</td>\n",
       "      <td>18</td>\n",
       "      <td>Image annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2936</th>\n",
       "      <td>http://edamontology.org/operation_3557</td>\n",
       "      <td>18</td>\n",
       "      <td>Imputation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2937</th>\n",
       "      <td>http://edamontology.org/operation_3559</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2938</th>\n",
       "      <td>http://edamontology.org/operation_3560</td>\n",
       "      <td>18</td>\n",
       "      <td>Maximum occurrence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2939</th>\n",
       "      <td>http://edamontology.org/operation_3561</td>\n",
       "      <td>18</td>\n",
       "      <td>Database comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2940</th>\n",
       "      <td>http://edamontology.org/operation_3562</td>\n",
       "      <td>18</td>\n",
       "      <td>Network simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2941</th>\n",
       "      <td>http://edamontology.org/operation_3563</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-seq read count analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2942</th>\n",
       "      <td>http://edamontology.org/operation_3564</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical redundancy removal</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2943</th>\n",
       "      <td>http://edamontology.org/operation_3565</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-seq time series data analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2944</th>\n",
       "      <td>http://edamontology.org/operation_3566</td>\n",
       "      <td>18</td>\n",
       "      <td>Simulated gene expression data generation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2945</th>\n",
       "      <td>http://edamontology.org/operation_3625</td>\n",
       "      <td>18</td>\n",
       "      <td>Relation extraction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2946</th>\n",
       "      <td>http://edamontology.org/operation_3627</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectra calibration</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2947</th>\n",
       "      <td>http://edamontology.org/operation_3628</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromatographic alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2948</th>\n",
       "      <td>http://edamontology.org/operation_3629</td>\n",
       "      <td>18</td>\n",
       "      <td>Deisotoping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2949</th>\n",
       "      <td>http://edamontology.org/operation_3630</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein quantification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2950</th>\n",
       "      <td>http://edamontology.org/operation_3631</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2951</th>\n",
       "      <td>http://edamontology.org/operation_3632</td>\n",
       "      <td>18</td>\n",
       "      <td>Isotopic distributions calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2952</th>\n",
       "      <td>http://edamontology.org/operation_3633</td>\n",
       "      <td>18</td>\n",
       "      <td>Retention time prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2953</th>\n",
       "      <td>http://edamontology.org/operation_3634</td>\n",
       "      <td>18</td>\n",
       "      <td>Label-free quantification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2954</th>\n",
       "      <td>http://edamontology.org/operation_3635</td>\n",
       "      <td>18</td>\n",
       "      <td>Labeled quantification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2955</th>\n",
       "      <td>http://edamontology.org/operation_3636</td>\n",
       "      <td>18</td>\n",
       "      <td>MRM/SRM</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2956</th>\n",
       "      <td>http://edamontology.org/operation_3637</td>\n",
       "      <td>18</td>\n",
       "      <td>Spectral counting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2957</th>\n",
       "      <td>http://edamontology.org/operation_3638</td>\n",
       "      <td>18</td>\n",
       "      <td>SILAC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2958</th>\n",
       "      <td>http://edamontology.org/operation_3639</td>\n",
       "      <td>18</td>\n",
       "      <td>iTRAQ</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2959</th>\n",
       "      <td>http://edamontology.org/operation_3640</td>\n",
       "      <td>18</td>\n",
       "      <td>18O labeling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2960</th>\n",
       "      <td>http://edamontology.org/operation_3641</td>\n",
       "      <td>18</td>\n",
       "      <td>TMT-tag</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2961</th>\n",
       "      <td>http://edamontology.org/operation_3642</td>\n",
       "      <td>18</td>\n",
       "      <td>Stable isotope dimethyl labelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2962</th>\n",
       "      <td>http://edamontology.org/operation_3643</td>\n",
       "      <td>18</td>\n",
       "      <td>Tag-based peptide identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2963</th>\n",
       "      <td>http://edamontology.org/operation_3644</td>\n",
       "      <td>18</td>\n",
       "      <td>de Novo sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2964</th>\n",
       "      <td>http://edamontology.org/operation_3645</td>\n",
       "      <td>18</td>\n",
       "      <td>PTM identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2965</th>\n",
       "      <td>http://edamontology.org/operation_3646</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptide database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2966</th>\n",
       "      <td>http://edamontology.org/operation_3647</td>\n",
       "      <td>18</td>\n",
       "      <td>Blind peptide database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2967</th>\n",
       "      <td>http://edamontology.org/operation_3648</td>\n",
       "      <td>18</td>\n",
       "      <td>Validation of peptide-spectrum matches</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2968</th>\n",
       "      <td>http://edamontology.org/operation_3649</td>\n",
       "      <td>18</td>\n",
       "      <td>Target-Decoy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2969</th>\n",
       "      <td>http://edamontology.org/operation_3658</td>\n",
       "      <td>18</td>\n",
       "      <td>Statistical inference</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2970</th>\n",
       "      <td>http://edamontology.org/operation_3659</td>\n",
       "      <td>18</td>\n",
       "      <td>Regression analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2971</th>\n",
       "      <td>http://edamontology.org/operation_3660</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic network modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2972</th>\n",
       "      <td>http://edamontology.org/operation_3661</td>\n",
       "      <td>18</td>\n",
       "      <td>SNP annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2973</th>\n",
       "      <td>http://edamontology.org/operation_3662</td>\n",
       "      <td>18</td>\n",
       "      <td>Ab-initio gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2974</th>\n",
       "      <td>http://edamontology.org/operation_3663</td>\n",
       "      <td>18</td>\n",
       "      <td>Homology-based gene prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2975</th>\n",
       "      <td>http://edamontology.org/operation_3664</td>\n",
       "      <td>18</td>\n",
       "      <td>Statistical modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2976</th>\n",
       "      <td>http://edamontology.org/operation_3666</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular surface comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2977</th>\n",
       "      <td>http://edamontology.org/operation_3672</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene functional annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2978</th>\n",
       "      <td>http://edamontology.org/operation_3675</td>\n",
       "      <td>18</td>\n",
       "      <td>Variant filtering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2979</th>\n",
       "      <td>http://edamontology.org/operation_3677</td>\n",
       "      <td>18</td>\n",
       "      <td>Differential binding analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2980</th>\n",
       "      <td>http://edamontology.org/operation_3680</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-Seq analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2981</th>\n",
       "      <td>http://edamontology.org/operation_3694</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrum visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2982</th>\n",
       "      <td>http://edamontology.org/operation_3695</td>\n",
       "      <td>18</td>\n",
       "      <td>Filtering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2983</th>\n",
       "      <td>http://edamontology.org/operation_3703</td>\n",
       "      <td>18</td>\n",
       "      <td>Reference identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2984</th>\n",
       "      <td>http://edamontology.org/operation_3704</td>\n",
       "      <td>18</td>\n",
       "      <td>Ion counting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2985</th>\n",
       "      <td>http://edamontology.org/operation_3705</td>\n",
       "      <td>18</td>\n",
       "      <td>Isotope-coded protein label</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2986</th>\n",
       "      <td>http://edamontology.org/operation_3715</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic labeling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2987</th>\n",
       "      <td>http://edamontology.org/operation_3730</td>\n",
       "      <td>18</td>\n",
       "      <td>Cross-assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2988</th>\n",
       "      <td>http://edamontology.org/operation_3731</td>\n",
       "      <td>18</td>\n",
       "      <td>Sample comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2989</th>\n",
       "      <td>http://edamontology.org/operation_3741</td>\n",
       "      <td>18</td>\n",
       "      <td>Differential protein expression profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2990</th>\n",
       "      <td>http://edamontology.org/operation_3742</td>\n",
       "      <td>18</td>\n",
       "      <td>Differential gene expression analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2991</th>\n",
       "      <td>http://edamontology.org/operation_3744</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple sample visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2992</th>\n",
       "      <td>http://edamontology.org/operation_3745</td>\n",
       "      <td>18</td>\n",
       "      <td>Ancestral reconstruction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2993</th>\n",
       "      <td>http://edamontology.org/operation_3755</td>\n",
       "      <td>18</td>\n",
       "      <td>PTM localisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2994</th>\n",
       "      <td>http://edamontology.org/operation_3760</td>\n",
       "      <td>18</td>\n",
       "      <td>Service management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2995</th>\n",
       "      <td>http://edamontology.org/operation_3761</td>\n",
       "      <td>18</td>\n",
       "      <td>Service discovery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2996</th>\n",
       "      <td>http://edamontology.org/operation_3762</td>\n",
       "      <td>18</td>\n",
       "      <td>Service composition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2997</th>\n",
       "      <td>http://edamontology.org/operation_3763</td>\n",
       "      <td>18</td>\n",
       "      <td>Service invocation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2998</th>\n",
       "      <td>http://edamontology.org/operation_3766</td>\n",
       "      <td>18</td>\n",
       "      <td>Weighted correlation network analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2999</th>\n",
       "      <td>http://edamontology.org/operation_3767</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3000</th>\n",
       "      <td>http://edamontology.org/operation_3778</td>\n",
       "      <td>18</td>\n",
       "      <td>Text annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3001</th>\n",
       "      <td>http://edamontology.org/operation_3791</td>\n",
       "      <td>18</td>\n",
       "      <td>Collapsing methods</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3002</th>\n",
       "      <td>http://edamontology.org/operation_3792</td>\n",
       "      <td>18</td>\n",
       "      <td>miRNA expression analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3003</th>\n",
       "      <td>http://edamontology.org/operation_3793</td>\n",
       "      <td>18</td>\n",
       "      <td>Read summarisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3004</th>\n",
       "      <td>http://edamontology.org/operation_3795</td>\n",
       "      <td>18</td>\n",
       "      <td>In vitro selection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3005</th>\n",
       "      <td>http://edamontology.org/operation_3797</td>\n",
       "      <td>18</td>\n",
       "      <td>Rarefaction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3006</th>\n",
       "      <td>http://edamontology.org/operation_3798</td>\n",
       "      <td>18</td>\n",
       "      <td>Read binning</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3007</th>\n",
       "      <td>http://edamontology.org/operation_3799</td>\n",
       "      <td>18</td>\n",
       "      <td>Quantification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3008</th>\n",
       "      <td>http://edamontology.org/operation_3800</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-Seq quantification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3009</th>\n",
       "      <td>http://edamontology.org/operation_3801</td>\n",
       "      <td>18</td>\n",
       "      <td>Spectral library search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3010</th>\n",
       "      <td>http://edamontology.org/operation_3802</td>\n",
       "      <td>18</td>\n",
       "      <td>Sorting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3011</th>\n",
       "      <td>http://edamontology.org/operation_3803</td>\n",
       "      <td>18</td>\n",
       "      <td>Natural product identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3012</th>\n",
       "      <td>http://edamontology.org/operation_3809</td>\n",
       "      <td>18</td>\n",
       "      <td>DMR identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3013</th>\n",
       "      <td>http://edamontology.org/operation_3840</td>\n",
       "      <td>18</td>\n",
       "      <td>Multilocus sequence typing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3014</th>\n",
       "      <td>http://edamontology.org/operation_3860</td>\n",
       "      <td>18</td>\n",
       "      <td>Spectrum calculation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3015</th>\n",
       "      <td>http://edamontology.org/operation_3890</td>\n",
       "      <td>18</td>\n",
       "      <td>Trajectory visualization</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3016</th>\n",
       "      <td>http://edamontology.org/operation_3891</td>\n",
       "      <td>18</td>\n",
       "      <td>Essential dynamics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3017</th>\n",
       "      <td>http://edamontology.org/operation_3893</td>\n",
       "      <td>18</td>\n",
       "      <td>Forcefield parameterisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3018</th>\n",
       "      <td>http://edamontology.org/operation_3894</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3019</th>\n",
       "      <td>http://edamontology.org/operation_3896</td>\n",
       "      <td>18</td>\n",
       "      <td>Active site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3020</th>\n",
       "      <td>http://edamontology.org/operation_3897</td>\n",
       "      <td>18</td>\n",
       "      <td>Ligand-binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3021</th>\n",
       "      <td>http://edamontology.org/operation_3898</td>\n",
       "      <td>18</td>\n",
       "      <td>Metal-binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3022</th>\n",
       "      <td>http://edamontology.org/operation_3899</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein docking</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3023</th>\n",
       "      <td>http://edamontology.org/operation_3900</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA-binding protein prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3024</th>\n",
       "      <td>http://edamontology.org/operation_3901</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-binding protein prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3025</th>\n",
       "      <td>http://edamontology.org/operation_3902</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3026</th>\n",
       "      <td>http://edamontology.org/operation_3903</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA binding site prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3027</th>\n",
       "      <td>http://edamontology.org/operation_3904</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein disorder prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3028</th>\n",
       "      <td>http://edamontology.org/operation_3907</td>\n",
       "      <td>18</td>\n",
       "      <td>Information extraction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3029</th>\n",
       "      <td>http://edamontology.org/operation_3908</td>\n",
       "      <td>18</td>\n",
       "      <td>Information retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3030</th>\n",
       "      <td>http://edamontology.org/operation_3918</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3031</th>\n",
       "      <td>http://edamontology.org/operation_3919</td>\n",
       "      <td>18</td>\n",
       "      <td>Methylation calling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3032</th>\n",
       "      <td>http://edamontology.org/operation_3920</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA testing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3033</th>\n",
       "      <td>http://edamontology.org/operation_3921</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence read processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3034</th>\n",
       "      <td>http://edamontology.org/operation_3925</td>\n",
       "      <td>18</td>\n",
       "      <td>Network visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3035</th>\n",
       "      <td>http://edamontology.org/operation_3926</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3036</th>\n",
       "      <td>http://edamontology.org/operation_3927</td>\n",
       "      <td>18</td>\n",
       "      <td>Network analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3037</th>\n",
       "      <td>http://edamontology.org/operation_3928</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathway analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3038</th>\n",
       "      <td>http://edamontology.org/operation_3929</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic pathway prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3039</th>\n",
       "      <td>http://edamontology.org/operation_3933</td>\n",
       "      <td>18</td>\n",
       "      <td>Demultiplexing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3040</th>\n",
       "      <td>http://edamontology.org/operation_3935</td>\n",
       "      <td>18</td>\n",
       "      <td>Dimensionality reduction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3041</th>\n",
       "      <td>http://edamontology.org/operation_3936</td>\n",
       "      <td>18</td>\n",
       "      <td>Feature selection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3042</th>\n",
       "      <td>http://edamontology.org/operation_3937</td>\n",
       "      <td>18</td>\n",
       "      <td>Feature extraction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3043</th>\n",
       "      <td>http://edamontology.org/operation_3938</td>\n",
       "      <td>18</td>\n",
       "      <td>Virtual screening</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3044</th>\n",
       "      <td>http://edamontology.org/operation_3942</td>\n",
       "      <td>18</td>\n",
       "      <td>Tree dating</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3045</th>\n",
       "      <td>http://edamontology.org/operation_3946</td>\n",
       "      <td>18</td>\n",
       "      <td>Ecological modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3046</th>\n",
       "      <td>http://edamontology.org/operation_3947</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetic tree reconciliation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3047</th>\n",
       "      <td>http://edamontology.org/operation_3950</td>\n",
       "      <td>18</td>\n",
       "      <td>Selection detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3048</th>\n",
       "      <td>http://edamontology.org/operation_3960</td>\n",
       "      <td>18</td>\n",
       "      <td>Principal component analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3049</th>\n",
       "      <td>http://edamontology.org/operation_3961</td>\n",
       "      <td>18</td>\n",
       "      <td>Copy number variation detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3050</th>\n",
       "      <td>http://edamontology.org/operation_3962</td>\n",
       "      <td>18</td>\n",
       "      <td>Deletion detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3051</th>\n",
       "      <td>http://edamontology.org/operation_3963</td>\n",
       "      <td>18</td>\n",
       "      <td>Duplication detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3052</th>\n",
       "      <td>http://edamontology.org/operation_3964</td>\n",
       "      <td>18</td>\n",
       "      <td>Complex CNV detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3053</th>\n",
       "      <td>http://edamontology.org/operation_3965</td>\n",
       "      <td>18</td>\n",
       "      <td>Amplification detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3054</th>\n",
       "      <td>http://edamontology.org/operation_3968</td>\n",
       "      <td>18</td>\n",
       "      <td>Adhesin prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3055</th>\n",
       "      <td>http://edamontology.org/operation_4008</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3056</th>\n",
       "      <td>http://edamontology.org/operation_4009</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecule design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3057</th>\n",
       "      <td>http://edamontology.org/operation_4031</td>\n",
       "      <td>18</td>\n",
       "      <td>Power test</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3058</th>\n",
       "      <td>http://edamontology.org/operation_4032</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA modification prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3059</th>\n",
       "      <td>http://edamontology.org/operation_4033</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease transmission analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3060</th>\n",
       "      <td>http://edamontology.org/operation_4034</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiple testing correction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3061</th>\n",
       "      <td>http://edamontology.org/organisation</td>\n",
       "      <td>18</td>\n",
       "      <td>Organisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3062</th>\n",
       "      <td>http://edamontology.org/refactor_comment</td>\n",
       "      <td>18</td>\n",
       "      <td>refactor_comment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3063</th>\n",
       "      <td>http://edamontology.org/regex</td>\n",
       "      <td>18</td>\n",
       "      <td>Regular expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3064</th>\n",
       "      <td>http://edamontology.org/related_term</td>\n",
       "      <td>18</td>\n",
       "      <td>Related term</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3065</th>\n",
       "      <td>http://edamontology.org/repository</td>\n",
       "      <td>18</td>\n",
       "      <td>Repository</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3066</th>\n",
       "      <td>http://edamontology.org/thematic_editor</td>\n",
       "      <td>18</td>\n",
       "      <td>thematic_editor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3067</th>\n",
       "      <td>http://edamontology.org/topic_0003</td>\n",
       "      <td>18</td>\n",
       "      <td>Topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3068</th>\n",
       "      <td>http://edamontology.org/topic_0077</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3069</th>\n",
       "      <td>http://edamontology.org/topic_0078</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3070</th>\n",
       "      <td>http://edamontology.org/topic_0079</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3071</th>\n",
       "      <td>http://edamontology.org/topic_0080</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3072</th>\n",
       "      <td>http://edamontology.org/topic_0081</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3073</th>\n",
       "      <td>http://edamontology.org/topic_0082</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3074</th>\n",
       "      <td>http://edamontology.org/topic_0083</td>\n",
       "      <td>18</td>\n",
       "      <td>Alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3075</th>\n",
       "      <td>http://edamontology.org/topic_0084</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogeny</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3076</th>\n",
       "      <td>http://edamontology.org/topic_0085</td>\n",
       "      <td>18</td>\n",
       "      <td>Functional genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3077</th>\n",
       "      <td>http://edamontology.org/topic_0089</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontology and terminology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3078</th>\n",
       "      <td>http://edamontology.org/topic_0090</td>\n",
       "      <td>18</td>\n",
       "      <td>Information retrieval</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3079</th>\n",
       "      <td>http://edamontology.org/topic_0091</td>\n",
       "      <td>18</td>\n",
       "      <td>Bioinformatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3080</th>\n",
       "      <td>http://edamontology.org/topic_0092</td>\n",
       "      <td>18</td>\n",
       "      <td>Data visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3081</th>\n",
       "      <td>http://edamontology.org/topic_0094</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid thermodynamics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3082</th>\n",
       "      <td>http://edamontology.org/topic_0097</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3083</th>\n",
       "      <td>http://edamontology.org/topic_0099</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3084</th>\n",
       "      <td>http://edamontology.org/topic_0100</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid restriction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3085</th>\n",
       "      <td>http://edamontology.org/topic_0102</td>\n",
       "      <td>18</td>\n",
       "      <td>Mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3086</th>\n",
       "      <td>http://edamontology.org/topic_0107</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic codes and codon usage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3087</th>\n",
       "      <td>http://edamontology.org/topic_0108</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3088</th>\n",
       "      <td>http://edamontology.org/topic_0109</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene finding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3089</th>\n",
       "      <td>http://edamontology.org/topic_0110</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3090</th>\n",
       "      <td>http://edamontology.org/topic_0111</td>\n",
       "      <td>18</td>\n",
       "      <td>Promoters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3091</th>\n",
       "      <td>http://edamontology.org/topic_0112</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid folding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3092</th>\n",
       "      <td>http://edamontology.org/topic_0114</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3093</th>\n",
       "      <td>http://edamontology.org/topic_0121</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3094</th>\n",
       "      <td>http://edamontology.org/topic_0122</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3095</th>\n",
       "      <td>http://edamontology.org/topic_0123</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein properties</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3096</th>\n",
       "      <td>http://edamontology.org/topic_0128</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3097</th>\n",
       "      <td>http://edamontology.org/topic_0130</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folding, stability and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3098</th>\n",
       "      <td>http://edamontology.org/topic_0133</td>\n",
       "      <td>18</td>\n",
       "      <td>Two-dimensional gel electrophoresis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3099</th>\n",
       "      <td>http://edamontology.org/topic_0134</td>\n",
       "      <td>18</td>\n",
       "      <td>Mass spectrometry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3100</th>\n",
       "      <td>http://edamontology.org/topic_0135</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein microarrays</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3101</th>\n",
       "      <td>http://edamontology.org/topic_0137</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein hydropathy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3102</th>\n",
       "      <td>http://edamontology.org/topic_0140</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein targeting and localisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3103</th>\n",
       "      <td>http://edamontology.org/topic_0141</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein cleavage sites and proteolysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3104</th>\n",
       "      <td>http://edamontology.org/topic_0143</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3105</th>\n",
       "      <td>http://edamontology.org/topic_0144</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein residue interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3106</th>\n",
       "      <td>http://edamontology.org/topic_0147</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3107</th>\n",
       "      <td>http://edamontology.org/topic_0148</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-ligand interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3108</th>\n",
       "      <td>http://edamontology.org/topic_0149</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-nucleic acid interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3109</th>\n",
       "      <td>http://edamontology.org/topic_0150</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3110</th>\n",
       "      <td>http://edamontology.org/topic_0151</td>\n",
       "      <td>18</td>\n",
       "      <td>G protein-coupled receptors (GPCR)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3111</th>\n",
       "      <td>http://edamontology.org/topic_0152</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbohydrates</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3112</th>\n",
       "      <td>http://edamontology.org/topic_0153</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3113</th>\n",
       "      <td>http://edamontology.org/topic_0154</td>\n",
       "      <td>18</td>\n",
       "      <td>Small molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3114</th>\n",
       "      <td>http://edamontology.org/topic_0156</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence editing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3115</th>\n",
       "      <td>http://edamontology.org/topic_0157</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence composition, complexity and repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3116</th>\n",
       "      <td>http://edamontology.org/topic_0158</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3117</th>\n",
       "      <td>http://edamontology.org/topic_0159</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3118</th>\n",
       "      <td>http://edamontology.org/topic_0160</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3119</th>\n",
       "      <td>http://edamontology.org/topic_0163</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3120</th>\n",
       "      <td>http://edamontology.org/topic_0164</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence clustering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3121</th>\n",
       "      <td>http://edamontology.org/topic_0166</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structural motifs and surfaces</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3122</th>\n",
       "      <td>http://edamontology.org/topic_0167</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural (3D) profiles</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3123</th>\n",
       "      <td>http://edamontology.org/topic_0172</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3124</th>\n",
       "      <td>http://edamontology.org/topic_0173</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3125</th>\n",
       "      <td>http://edamontology.org/topic_0174</td>\n",
       "      <td>18</td>\n",
       "      <td>Ab initio structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3126</th>\n",
       "      <td>http://edamontology.org/topic_0175</td>\n",
       "      <td>18</td>\n",
       "      <td>Homology modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3127</th>\n",
       "      <td>http://edamontology.org/topic_0176</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular dynamics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3128</th>\n",
       "      <td>http://edamontology.org/topic_0177</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular docking</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3129</th>\n",
       "      <td>http://edamontology.org/topic_0178</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3130</th>\n",
       "      <td>http://edamontology.org/topic_0179</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein tertiary structure prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3131</th>\n",
       "      <td>http://edamontology.org/topic_0180</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein fold recognition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3132</th>\n",
       "      <td>http://edamontology.org/topic_0182</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3133</th>\n",
       "      <td>http://edamontology.org/topic_0183</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3134</th>\n",
       "      <td>http://edamontology.org/topic_0184</td>\n",
       "      <td>18</td>\n",
       "      <td>Threading</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3135</th>\n",
       "      <td>http://edamontology.org/topic_0188</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence profiles and HMMs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3136</th>\n",
       "      <td>http://edamontology.org/topic_0191</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogeny reconstruction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3137</th>\n",
       "      <td>http://edamontology.org/topic_0194</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3138</th>\n",
       "      <td>http://edamontology.org/topic_0195</td>\n",
       "      <td>18</td>\n",
       "      <td>Virtual PCR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3139</th>\n",
       "      <td>http://edamontology.org/topic_0196</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence assembly</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3140</th>\n",
       "      <td>http://edamontology.org/topic_0199</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3141</th>\n",
       "      <td>http://edamontology.org/topic_0200</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarrays</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3142</th>\n",
       "      <td>http://edamontology.org/topic_0202</td>\n",
       "      <td>18</td>\n",
       "      <td>Pharmacology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3143</th>\n",
       "      <td>http://edamontology.org/topic_0203</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3144</th>\n",
       "      <td>http://edamontology.org/topic_0204</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3145</th>\n",
       "      <td>http://edamontology.org/topic_0208</td>\n",
       "      <td>18</td>\n",
       "      <td>Pharmacogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3146</th>\n",
       "      <td>http://edamontology.org/topic_0209</td>\n",
       "      <td>18</td>\n",
       "      <td>Medicinal chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3147</th>\n",
       "      <td>http://edamontology.org/topic_0210</td>\n",
       "      <td>18</td>\n",
       "      <td>Fish</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3148</th>\n",
       "      <td>http://edamontology.org/topic_0211</td>\n",
       "      <td>18</td>\n",
       "      <td>Flies</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3149</th>\n",
       "      <td>http://edamontology.org/topic_0213</td>\n",
       "      <td>18</td>\n",
       "      <td>Mice or rats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3150</th>\n",
       "      <td>http://edamontology.org/topic_0215</td>\n",
       "      <td>18</td>\n",
       "      <td>Worms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3151</th>\n",
       "      <td>http://edamontology.org/topic_0217</td>\n",
       "      <td>18</td>\n",
       "      <td>Literature analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3152</th>\n",
       "      <td>http://edamontology.org/topic_0218</td>\n",
       "      <td>18</td>\n",
       "      <td>Natural language processing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3153</th>\n",
       "      <td>http://edamontology.org/topic_0219</td>\n",
       "      <td>18</td>\n",
       "      <td>Data submission, annotation, and curation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3154</th>\n",
       "      <td>http://edamontology.org/topic_0220</td>\n",
       "      <td>18</td>\n",
       "      <td>Document, record and content management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3155</th>\n",
       "      <td>http://edamontology.org/topic_0221</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3156</th>\n",
       "      <td>http://edamontology.org/topic_0222</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome annotation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3157</th>\n",
       "      <td>http://edamontology.org/topic_0593</td>\n",
       "      <td>18</td>\n",
       "      <td>NMR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3158</th>\n",
       "      <td>http://edamontology.org/topic_0594</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3159</th>\n",
       "      <td>http://edamontology.org/topic_0595</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3160</th>\n",
       "      <td>http://edamontology.org/topic_0598</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence motif or profile</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3161</th>\n",
       "      <td>http://edamontology.org/topic_0601</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein modifications</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3162</th>\n",
       "      <td>http://edamontology.org/topic_0602</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular interactions, pathways and networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3163</th>\n",
       "      <td>http://edamontology.org/topic_0605</td>\n",
       "      <td>18</td>\n",
       "      <td>Informatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3164</th>\n",
       "      <td>http://edamontology.org/topic_0606</td>\n",
       "      <td>18</td>\n",
       "      <td>Literature data resources</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3165</th>\n",
       "      <td>http://edamontology.org/topic_0607</td>\n",
       "      <td>18</td>\n",
       "      <td>Laboratory information management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3166</th>\n",
       "      <td>http://edamontology.org/topic_0608</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell and tissue culture</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3167</th>\n",
       "      <td>http://edamontology.org/topic_0610</td>\n",
       "      <td>18</td>\n",
       "      <td>Ecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3168</th>\n",
       "      <td>http://edamontology.org/topic_0611</td>\n",
       "      <td>18</td>\n",
       "      <td>Electron microscopy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3169</th>\n",
       "      <td>http://edamontology.org/topic_0612</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell cycle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3170</th>\n",
       "      <td>http://edamontology.org/topic_0613</td>\n",
       "      <td>18</td>\n",
       "      <td>Peptides and amino acids</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3171</th>\n",
       "      <td>http://edamontology.org/topic_0616</td>\n",
       "      <td>18</td>\n",
       "      <td>Organelles</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3172</th>\n",
       "      <td>http://edamontology.org/topic_0617</td>\n",
       "      <td>18</td>\n",
       "      <td>Ribosomes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3173</th>\n",
       "      <td>http://edamontology.org/topic_0618</td>\n",
       "      <td>18</td>\n",
       "      <td>Scents</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3174</th>\n",
       "      <td>http://edamontology.org/topic_0620</td>\n",
       "      <td>18</td>\n",
       "      <td>Drugs and target structures</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3175</th>\n",
       "      <td>http://edamontology.org/topic_0621</td>\n",
       "      <td>18</td>\n",
       "      <td>Model organisms</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3176</th>\n",
       "      <td>http://edamontology.org/topic_0622</td>\n",
       "      <td>18</td>\n",
       "      <td>Genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3177</th>\n",
       "      <td>http://edamontology.org/topic_0623</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene and protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3178</th>\n",
       "      <td>http://edamontology.org/topic_0624</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosomes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3179</th>\n",
       "      <td>http://edamontology.org/topic_0625</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotype and phenotype</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3180</th>\n",
       "      <td>http://edamontology.org/topic_0629</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene expression and microarray</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3181</th>\n",
       "      <td>http://edamontology.org/topic_0632</td>\n",
       "      <td>18</td>\n",
       "      <td>Probes and primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3182</th>\n",
       "      <td>http://edamontology.org/topic_0634</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3183</th>\n",
       "      <td>http://edamontology.org/topic_0635</td>\n",
       "      <td>18</td>\n",
       "      <td>Specific protein resources</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3184</th>\n",
       "      <td>http://edamontology.org/topic_0637</td>\n",
       "      <td>18</td>\n",
       "      <td>Taxonomy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3185</th>\n",
       "      <td>http://edamontology.org/topic_0639</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3186</th>\n",
       "      <td>http://edamontology.org/topic_0640</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3187</th>\n",
       "      <td>http://edamontology.org/topic_0641</td>\n",
       "      <td>18</td>\n",
       "      <td>Repeat sequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3188</th>\n",
       "      <td>http://edamontology.org/topic_0642</td>\n",
       "      <td>18</td>\n",
       "      <td>Low complexity sequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3189</th>\n",
       "      <td>http://edamontology.org/topic_0644</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteome</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3190</th>\n",
       "      <td>http://edamontology.org/topic_0654</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3191</th>\n",
       "      <td>http://edamontology.org/topic_0655</td>\n",
       "      <td>18</td>\n",
       "      <td>Coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3192</th>\n",
       "      <td>http://edamontology.org/topic_0659</td>\n",
       "      <td>18</td>\n",
       "      <td>Functional, regulatory and non-coding RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3193</th>\n",
       "      <td>http://edamontology.org/topic_0660</td>\n",
       "      <td>18</td>\n",
       "      <td>rRNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3194</th>\n",
       "      <td>http://edamontology.org/topic_0663</td>\n",
       "      <td>18</td>\n",
       "      <td>tRNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3195</th>\n",
       "      <td>http://edamontology.org/topic_0694</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3196</th>\n",
       "      <td>http://edamontology.org/topic_0697</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3197</th>\n",
       "      <td>http://edamontology.org/topic_0698</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein tertiary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3198</th>\n",
       "      <td>http://edamontology.org/topic_0722</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3199</th>\n",
       "      <td>http://edamontology.org/topic_0724</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein families</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3200</th>\n",
       "      <td>http://edamontology.org/topic_0736</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein folds and structural domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3201</th>\n",
       "      <td>http://edamontology.org/topic_0740</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3202</th>\n",
       "      <td>http://edamontology.org/topic_0741</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3203</th>\n",
       "      <td>http://edamontology.org/topic_0747</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sites and features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3204</th>\n",
       "      <td>http://edamontology.org/topic_0748</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sites and features</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3205</th>\n",
       "      <td>http://edamontology.org/topic_0749</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcription factors and regulatory sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3206</th>\n",
       "      <td>http://edamontology.org/topic_0751</td>\n",
       "      <td>18</td>\n",
       "      <td>Phosphorylation sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3207</th>\n",
       "      <td>http://edamontology.org/topic_0753</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3208</th>\n",
       "      <td>http://edamontology.org/topic_0754</td>\n",
       "      <td>18</td>\n",
       "      <td>Signaling pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3209</th>\n",
       "      <td>http://edamontology.org/topic_0767</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein and peptide identification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3210</th>\n",
       "      <td>http://edamontology.org/topic_0769</td>\n",
       "      <td>18</td>\n",
       "      <td>Workflows</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3211</th>\n",
       "      <td>http://edamontology.org/topic_0770</td>\n",
       "      <td>18</td>\n",
       "      <td>Data types and objects</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3212</th>\n",
       "      <td>http://edamontology.org/topic_0771</td>\n",
       "      <td>18</td>\n",
       "      <td>Theoretical biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3213</th>\n",
       "      <td>http://edamontology.org/topic_0779</td>\n",
       "      <td>18</td>\n",
       "      <td>Mitochondria</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3214</th>\n",
       "      <td>http://edamontology.org/topic_0780</td>\n",
       "      <td>18</td>\n",
       "      <td>Plant biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3215</th>\n",
       "      <td>http://edamontology.org/topic_0781</td>\n",
       "      <td>18</td>\n",
       "      <td>Virology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3216</th>\n",
       "      <td>http://edamontology.org/topic_0782</td>\n",
       "      <td>18</td>\n",
       "      <td>Fungi</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3217</th>\n",
       "      <td>http://edamontology.org/topic_0783</td>\n",
       "      <td>18</td>\n",
       "      <td>Pathogens</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3218</th>\n",
       "      <td>http://edamontology.org/topic_0786</td>\n",
       "      <td>18</td>\n",
       "      <td>Arabidopsis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3219</th>\n",
       "      <td>http://edamontology.org/topic_0787</td>\n",
       "      <td>18</td>\n",
       "      <td>Rice</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3220</th>\n",
       "      <td>http://edamontology.org/topic_0796</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic mapping and linkage</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3221</th>\n",
       "      <td>http://edamontology.org/topic_0797</td>\n",
       "      <td>18</td>\n",
       "      <td>Comparative genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3222</th>\n",
       "      <td>http://edamontology.org/topic_0798</td>\n",
       "      <td>18</td>\n",
       "      <td>Mobile genetic elements</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3223</th>\n",
       "      <td>http://edamontology.org/topic_0803</td>\n",
       "      <td>18</td>\n",
       "      <td>Human disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3224</th>\n",
       "      <td>http://edamontology.org/topic_0804</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3225</th>\n",
       "      <td>http://edamontology.org/topic_0820</td>\n",
       "      <td>18</td>\n",
       "      <td>Membrane and lipoproteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3226</th>\n",
       "      <td>http://edamontology.org/topic_0821</td>\n",
       "      <td>18</td>\n",
       "      <td>Enzymes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3227</th>\n",
       "      <td>http://edamontology.org/topic_0922</td>\n",
       "      <td>18</td>\n",
       "      <td>Primers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3228</th>\n",
       "      <td>http://edamontology.org/topic_1302</td>\n",
       "      <td>18</td>\n",
       "      <td>PolyA signal or sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3229</th>\n",
       "      <td>http://edamontology.org/topic_1304</td>\n",
       "      <td>18</td>\n",
       "      <td>CpG island and isochores</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3230</th>\n",
       "      <td>http://edamontology.org/topic_1305</td>\n",
       "      <td>18</td>\n",
       "      <td>Restriction sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3231</th>\n",
       "      <td>http://edamontology.org/topic_1307</td>\n",
       "      <td>18</td>\n",
       "      <td>Splice sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3232</th>\n",
       "      <td>http://edamontology.org/topic_1308</td>\n",
       "      <td>18</td>\n",
       "      <td>Matrix/scaffold attachment sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3233</th>\n",
       "      <td>http://edamontology.org/topic_1311</td>\n",
       "      <td>18</td>\n",
       "      <td>Operon</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3234</th>\n",
       "      <td>http://edamontology.org/topic_1312</td>\n",
       "      <td>18</td>\n",
       "      <td>Promoters</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3235</th>\n",
       "      <td>http://edamontology.org/topic_1317</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3236</th>\n",
       "      <td>http://edamontology.org/topic_1456</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein membrane regions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3237</th>\n",
       "      <td>http://edamontology.org/topic_1770</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3238</th>\n",
       "      <td>http://edamontology.org/topic_1775</td>\n",
       "      <td>18</td>\n",
       "      <td>Function analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3239</th>\n",
       "      <td>http://edamontology.org/topic_1811</td>\n",
       "      <td>18</td>\n",
       "      <td>Prokaryotes and Archaea</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3240</th>\n",
       "      <td>http://edamontology.org/topic_2225</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein databases</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3241</th>\n",
       "      <td>http://edamontology.org/topic_2226</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure determination</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3242</th>\n",
       "      <td>http://edamontology.org/topic_2229</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3243</th>\n",
       "      <td>http://edamontology.org/topic_2230</td>\n",
       "      <td>18</td>\n",
       "      <td>Classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3244</th>\n",
       "      <td>http://edamontology.org/topic_2232</td>\n",
       "      <td>18</td>\n",
       "      <td>Lipoproteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3245</th>\n",
       "      <td>http://edamontology.org/topic_2257</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogeny visualisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3246</th>\n",
       "      <td>http://edamontology.org/topic_2258</td>\n",
       "      <td>18</td>\n",
       "      <td>Cheminformatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3247</th>\n",
       "      <td>http://edamontology.org/topic_2259</td>\n",
       "      <td>18</td>\n",
       "      <td>Systems biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3248</th>\n",
       "      <td>http://edamontology.org/topic_2269</td>\n",
       "      <td>18</td>\n",
       "      <td>Statistics and probability</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3249</th>\n",
       "      <td>http://edamontology.org/topic_2271</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure database search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3250</th>\n",
       "      <td>http://edamontology.org/topic_2275</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3251</th>\n",
       "      <td>http://edamontology.org/topic_2276</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein function prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3252</th>\n",
       "      <td>http://edamontology.org/topic_2277</td>\n",
       "      <td>18</td>\n",
       "      <td>SNP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3253</th>\n",
       "      <td>http://edamontology.org/topic_2278</td>\n",
       "      <td>18</td>\n",
       "      <td>Transmembrane protein prediction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3254</th>\n",
       "      <td>http://edamontology.org/topic_2280</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure comparison</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3255</th>\n",
       "      <td>http://edamontology.org/topic_2397</td>\n",
       "      <td>18</td>\n",
       "      <td>Exons</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3256</th>\n",
       "      <td>http://edamontology.org/topic_2399</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene transcription</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3257</th>\n",
       "      <td>http://edamontology.org/topic_2533</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA mutation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3258</th>\n",
       "      <td>http://edamontology.org/topic_2640</td>\n",
       "      <td>18</td>\n",
       "      <td>Oncology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3259</th>\n",
       "      <td>http://edamontology.org/topic_2661</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxins and targets</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3260</th>\n",
       "      <td>http://edamontology.org/topic_2754</td>\n",
       "      <td>18</td>\n",
       "      <td>Introns</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3261</th>\n",
       "      <td>http://edamontology.org/topic_2807</td>\n",
       "      <td>18</td>\n",
       "      <td>Tool topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3262</th>\n",
       "      <td>http://edamontology.org/topic_2809</td>\n",
       "      <td>18</td>\n",
       "      <td>Study topic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3263</th>\n",
       "      <td>http://edamontology.org/topic_2811</td>\n",
       "      <td>18</td>\n",
       "      <td>Nomenclature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3264</th>\n",
       "      <td>http://edamontology.org/topic_2813</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease genes and proteins</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3265</th>\n",
       "      <td>http://edamontology.org/topic_2814</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure analysis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3266</th>\n",
       "      <td>http://edamontology.org/topic_2815</td>\n",
       "      <td>18</td>\n",
       "      <td>Human biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3267</th>\n",
       "      <td>http://edamontology.org/topic_2816</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene resources</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3268</th>\n",
       "      <td>http://edamontology.org/topic_2817</td>\n",
       "      <td>18</td>\n",
       "      <td>Yeast</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3269</th>\n",
       "      <td>http://edamontology.org/topic_2818</td>\n",
       "      <td>18</td>\n",
       "      <td>Eukaryotes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3270</th>\n",
       "      <td>http://edamontology.org/topic_2819</td>\n",
       "      <td>18</td>\n",
       "      <td>Invertebrates</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3271</th>\n",
       "      <td>http://edamontology.org/topic_2820</td>\n",
       "      <td>18</td>\n",
       "      <td>Vertebrates</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3272</th>\n",
       "      <td>http://edamontology.org/topic_2821</td>\n",
       "      <td>18</td>\n",
       "      <td>Unicellular eukaryotes</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3273</th>\n",
       "      <td>http://edamontology.org/topic_2826</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein structure alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3274</th>\n",
       "      <td>http://edamontology.org/topic_2828</td>\n",
       "      <td>18</td>\n",
       "      <td>X-ray diffraction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3275</th>\n",
       "      <td>http://edamontology.org/topic_2829</td>\n",
       "      <td>18</td>\n",
       "      <td>Ontologies, nomenclature and classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3276</th>\n",
       "      <td>http://edamontology.org/topic_2830</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunoproteins and antigens</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3277</th>\n",
       "      <td>http://edamontology.org/topic_2839</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecules</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3278</th>\n",
       "      <td>http://edamontology.org/topic_2840</td>\n",
       "      <td>18</td>\n",
       "      <td>Toxicology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3279</th>\n",
       "      <td>http://edamontology.org/topic_2842</td>\n",
       "      <td>18</td>\n",
       "      <td>High-throughput sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3280</th>\n",
       "      <td>http://edamontology.org/topic_2846</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene regulatory networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3281</th>\n",
       "      <td>http://edamontology.org/topic_2847</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease (specific)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3282</th>\n",
       "      <td>http://edamontology.org/topic_2867</td>\n",
       "      <td>18</td>\n",
       "      <td>VNTR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3283</th>\n",
       "      <td>http://edamontology.org/topic_2868</td>\n",
       "      <td>18</td>\n",
       "      <td>Microsatellites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3284</th>\n",
       "      <td>http://edamontology.org/topic_2869</td>\n",
       "      <td>18</td>\n",
       "      <td>RFLP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3285</th>\n",
       "      <td>http://edamontology.org/topic_2885</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA polymorphism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3286</th>\n",
       "      <td>http://edamontology.org/topic_2953</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3287</th>\n",
       "      <td>http://edamontology.org/topic_3032</td>\n",
       "      <td>18</td>\n",
       "      <td>Primer or probe design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3288</th>\n",
       "      <td>http://edamontology.org/topic_3038</td>\n",
       "      <td>18</td>\n",
       "      <td>Structure databases</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3289</th>\n",
       "      <td>http://edamontology.org/topic_3039</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3290</th>\n",
       "      <td>http://edamontology.org/topic_3041</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence databases</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3291</th>\n",
       "      <td>http://edamontology.org/topic_3042</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3292</th>\n",
       "      <td>http://edamontology.org/topic_3043</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3293</th>\n",
       "      <td>http://edamontology.org/topic_3044</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction networks</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3294</th>\n",
       "      <td>http://edamontology.org/topic_3047</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3295</th>\n",
       "      <td>http://edamontology.org/topic_3048</td>\n",
       "      <td>18</td>\n",
       "      <td>Mammals</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3296</th>\n",
       "      <td>http://edamontology.org/topic_3050</td>\n",
       "      <td>18</td>\n",
       "      <td>Biodiversity</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3297</th>\n",
       "      <td>http://edamontology.org/topic_3052</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence clusters and classification</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3298</th>\n",
       "      <td>http://edamontology.org/topic_3053</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3299</th>\n",
       "      <td>http://edamontology.org/topic_3055</td>\n",
       "      <td>18</td>\n",
       "      <td>Quantitative genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3300</th>\n",
       "      <td>http://edamontology.org/topic_3056</td>\n",
       "      <td>18</td>\n",
       "      <td>Population genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3301</th>\n",
       "      <td>http://edamontology.org/topic_3060</td>\n",
       "      <td>18</td>\n",
       "      <td>Regulatory RNA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3302</th>\n",
       "      <td>http://edamontology.org/topic_3061</td>\n",
       "      <td>18</td>\n",
       "      <td>Documentation and help</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3303</th>\n",
       "      <td>http://edamontology.org/topic_3062</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic organisation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3304</th>\n",
       "      <td>http://edamontology.org/topic_3063</td>\n",
       "      <td>18</td>\n",
       "      <td>Medical informatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3305</th>\n",
       "      <td>http://edamontology.org/topic_3064</td>\n",
       "      <td>18</td>\n",
       "      <td>Developmental biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3306</th>\n",
       "      <td>http://edamontology.org/topic_3065</td>\n",
       "      <td>18</td>\n",
       "      <td>Embryology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3307</th>\n",
       "      <td>http://edamontology.org/topic_3067</td>\n",
       "      <td>18</td>\n",
       "      <td>Anatomy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3308</th>\n",
       "      <td>http://edamontology.org/topic_3068</td>\n",
       "      <td>18</td>\n",
       "      <td>Literature and language</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3309</th>\n",
       "      <td>http://edamontology.org/topic_3070</td>\n",
       "      <td>18</td>\n",
       "      <td>Biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3310</th>\n",
       "      <td>http://edamontology.org/topic_3071</td>\n",
       "      <td>18</td>\n",
       "      <td>Data management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3311</th>\n",
       "      <td>http://edamontology.org/topic_3072</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3312</th>\n",
       "      <td>http://edamontology.org/topic_3073</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3313</th>\n",
       "      <td>http://edamontology.org/topic_3074</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein feature detection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3314</th>\n",
       "      <td>http://edamontology.org/topic_3075</td>\n",
       "      <td>18</td>\n",
       "      <td>Biological system modelling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3315</th>\n",
       "      <td>http://edamontology.org/topic_3077</td>\n",
       "      <td>18</td>\n",
       "      <td>Data acquisition</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3316</th>\n",
       "      <td>http://edamontology.org/topic_3078</td>\n",
       "      <td>18</td>\n",
       "      <td>Genes and proteins resources</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3317</th>\n",
       "      <td>http://edamontology.org/topic_3118</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein topological domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3318</th>\n",
       "      <td>http://edamontology.org/topic_3120</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein variants</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3319</th>\n",
       "      <td>http://edamontology.org/topic_3123</td>\n",
       "      <td>18</td>\n",
       "      <td>Expression signals</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3320</th>\n",
       "      <td>http://edamontology.org/topic_3125</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA binding sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3321</th>\n",
       "      <td>http://edamontology.org/topic_3126</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3322</th>\n",
       "      <td>http://edamontology.org/topic_3127</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA replication and recombination</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3323</th>\n",
       "      <td>http://edamontology.org/topic_3135</td>\n",
       "      <td>18</td>\n",
       "      <td>Signal or transit peptide</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3324</th>\n",
       "      <td>http://edamontology.org/topic_3139</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence tagged sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3325</th>\n",
       "      <td>http://edamontology.org/topic_3168</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3326</th>\n",
       "      <td>http://edamontology.org/topic_3169</td>\n",
       "      <td>18</td>\n",
       "      <td>ChIP-seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3327</th>\n",
       "      <td>http://edamontology.org/topic_3170</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-Seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3328</th>\n",
       "      <td>http://edamontology.org/topic_3171</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA methylation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3329</th>\n",
       "      <td>http://edamontology.org/topic_3172</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3330</th>\n",
       "      <td>http://edamontology.org/topic_3173</td>\n",
       "      <td>18</td>\n",
       "      <td>Epigenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3331</th>\n",
       "      <td>http://edamontology.org/topic_3174</td>\n",
       "      <td>18</td>\n",
       "      <td>Metagenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3332</th>\n",
       "      <td>http://edamontology.org/topic_3175</td>\n",
       "      <td>18</td>\n",
       "      <td>Structural variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3333</th>\n",
       "      <td>http://edamontology.org/topic_3176</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA packaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3334</th>\n",
       "      <td>http://edamontology.org/topic_3177</td>\n",
       "      <td>18</td>\n",
       "      <td>DNA-Seq</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3335</th>\n",
       "      <td>http://edamontology.org/topic_3178</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA-Seq alignment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3336</th>\n",
       "      <td>http://edamontology.org/topic_3179</td>\n",
       "      <td>18</td>\n",
       "      <td>ChIP-on-chip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3337</th>\n",
       "      <td>http://edamontology.org/topic_3263</td>\n",
       "      <td>18</td>\n",
       "      <td>Data security</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3338</th>\n",
       "      <td>http://edamontology.org/topic_3277</td>\n",
       "      <td>18</td>\n",
       "      <td>Sample collections</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3339</th>\n",
       "      <td>http://edamontology.org/topic_3292</td>\n",
       "      <td>18</td>\n",
       "      <td>Biochemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3340</th>\n",
       "      <td>http://edamontology.org/topic_3293</td>\n",
       "      <td>18</td>\n",
       "      <td>Phylogenetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3341</th>\n",
       "      <td>http://edamontology.org/topic_3295</td>\n",
       "      <td>18</td>\n",
       "      <td>Epigenetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3342</th>\n",
       "      <td>http://edamontology.org/topic_3297</td>\n",
       "      <td>18</td>\n",
       "      <td>Biotechnology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3343</th>\n",
       "      <td>http://edamontology.org/topic_3298</td>\n",
       "      <td>18</td>\n",
       "      <td>Phenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3344</th>\n",
       "      <td>http://edamontology.org/topic_3299</td>\n",
       "      <td>18</td>\n",
       "      <td>Evolutionary biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3345</th>\n",
       "      <td>http://edamontology.org/topic_3300</td>\n",
       "      <td>18</td>\n",
       "      <td>Physiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3346</th>\n",
       "      <td>http://edamontology.org/topic_3301</td>\n",
       "      <td>18</td>\n",
       "      <td>Microbiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3347</th>\n",
       "      <td>http://edamontology.org/topic_3302</td>\n",
       "      <td>18</td>\n",
       "      <td>Parasitology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3348</th>\n",
       "      <td>http://edamontology.org/topic_3303</td>\n",
       "      <td>18</td>\n",
       "      <td>Medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3349</th>\n",
       "      <td>http://edamontology.org/topic_3304</td>\n",
       "      <td>18</td>\n",
       "      <td>Neurobiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3350</th>\n",
       "      <td>http://edamontology.org/topic_3305</td>\n",
       "      <td>18</td>\n",
       "      <td>Public health and epidemiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3351</th>\n",
       "      <td>http://edamontology.org/topic_3306</td>\n",
       "      <td>18</td>\n",
       "      <td>Biophysics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3352</th>\n",
       "      <td>http://edamontology.org/topic_3307</td>\n",
       "      <td>18</td>\n",
       "      <td>Computational biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3353</th>\n",
       "      <td>http://edamontology.org/topic_3308</td>\n",
       "      <td>18</td>\n",
       "      <td>Transcriptomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3354</th>\n",
       "      <td>http://edamontology.org/topic_3314</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3355</th>\n",
       "      <td>http://edamontology.org/topic_3315</td>\n",
       "      <td>18</td>\n",
       "      <td>Mathematics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3356</th>\n",
       "      <td>http://edamontology.org/topic_3316</td>\n",
       "      <td>18</td>\n",
       "      <td>Computer science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3357</th>\n",
       "      <td>http://edamontology.org/topic_3318</td>\n",
       "      <td>18</td>\n",
       "      <td>Physics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3358</th>\n",
       "      <td>http://edamontology.org/topic_3320</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA splicing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3359</th>\n",
       "      <td>http://edamontology.org/topic_3321</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3360</th>\n",
       "      <td>http://edamontology.org/topic_3322</td>\n",
       "      <td>18</td>\n",
       "      <td>Respiratory medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3361</th>\n",
       "      <td>http://edamontology.org/topic_3323</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3362</th>\n",
       "      <td>http://edamontology.org/topic_3324</td>\n",
       "      <td>18</td>\n",
       "      <td>Infectious disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3363</th>\n",
       "      <td>http://edamontology.org/topic_3325</td>\n",
       "      <td>18</td>\n",
       "      <td>Rare diseases</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3364</th>\n",
       "      <td>http://edamontology.org/topic_3332</td>\n",
       "      <td>18</td>\n",
       "      <td>Computational chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3365</th>\n",
       "      <td>http://edamontology.org/topic_3334</td>\n",
       "      <td>18</td>\n",
       "      <td>Neurology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3366</th>\n",
       "      <td>http://edamontology.org/topic_3335</td>\n",
       "      <td>18</td>\n",
       "      <td>Cardiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3367</th>\n",
       "      <td>http://edamontology.org/topic_3336</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug discovery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3368</th>\n",
       "      <td>http://edamontology.org/topic_3337</td>\n",
       "      <td>18</td>\n",
       "      <td>Biobank</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3369</th>\n",
       "      <td>http://edamontology.org/topic_3338</td>\n",
       "      <td>18</td>\n",
       "      <td>Mouse clinic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3370</th>\n",
       "      <td>http://edamontology.org/topic_3339</td>\n",
       "      <td>18</td>\n",
       "      <td>Microbial collection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3371</th>\n",
       "      <td>http://edamontology.org/topic_3340</td>\n",
       "      <td>18</td>\n",
       "      <td>Cell culture collection</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3372</th>\n",
       "      <td>http://edamontology.org/topic_3341</td>\n",
       "      <td>18</td>\n",
       "      <td>Clone library</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3373</th>\n",
       "      <td>http://edamontology.org/topic_3342</td>\n",
       "      <td>18</td>\n",
       "      <td>Translational medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3374</th>\n",
       "      <td>http://edamontology.org/topic_3343</td>\n",
       "      <td>18</td>\n",
       "      <td>Compound libraries and screening</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3375</th>\n",
       "      <td>http://edamontology.org/topic_3344</td>\n",
       "      <td>18</td>\n",
       "      <td>Biomedical science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3376</th>\n",
       "      <td>http://edamontology.org/topic_3345</td>\n",
       "      <td>18</td>\n",
       "      <td>Data identity and mapping</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3377</th>\n",
       "      <td>http://edamontology.org/topic_3346</td>\n",
       "      <td>18</td>\n",
       "      <td>Sequence search</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3378</th>\n",
       "      <td>http://edamontology.org/topic_3360</td>\n",
       "      <td>18</td>\n",
       "      <td>Biomarkers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3379</th>\n",
       "      <td>http://edamontology.org/topic_3361</td>\n",
       "      <td>18</td>\n",
       "      <td>Laboratory techniques</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3380</th>\n",
       "      <td>http://edamontology.org/topic_3365</td>\n",
       "      <td>18</td>\n",
       "      <td>Data architecture, analysis and design</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3381</th>\n",
       "      <td>http://edamontology.org/topic_3366</td>\n",
       "      <td>18</td>\n",
       "      <td>Data integration and warehousing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3382</th>\n",
       "      <td>http://edamontology.org/topic_3368</td>\n",
       "      <td>18</td>\n",
       "      <td>Biomaterials</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3383</th>\n",
       "      <td>http://edamontology.org/topic_3369</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemical biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3384</th>\n",
       "      <td>http://edamontology.org/topic_3370</td>\n",
       "      <td>18</td>\n",
       "      <td>Analytical chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3385</th>\n",
       "      <td>http://edamontology.org/topic_3371</td>\n",
       "      <td>18</td>\n",
       "      <td>Synthetic chemistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3386</th>\n",
       "      <td>http://edamontology.org/topic_3372</td>\n",
       "      <td>18</td>\n",
       "      <td>Software engineering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3387</th>\n",
       "      <td>http://edamontology.org/topic_3373</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug development</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3388</th>\n",
       "      <td>http://edamontology.org/topic_3374</td>\n",
       "      <td>18</td>\n",
       "      <td>Biotherapeutics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3389</th>\n",
       "      <td>http://edamontology.org/topic_3375</td>\n",
       "      <td>18</td>\n",
       "      <td>Drug metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3390</th>\n",
       "      <td>http://edamontology.org/topic_3376</td>\n",
       "      <td>18</td>\n",
       "      <td>Medicines research and development</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3391</th>\n",
       "      <td>http://edamontology.org/topic_3377</td>\n",
       "      <td>18</td>\n",
       "      <td>Safety sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3392</th>\n",
       "      <td>http://edamontology.org/topic_3378</td>\n",
       "      <td>18</td>\n",
       "      <td>Pharmacovigilance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3393</th>\n",
       "      <td>http://edamontology.org/topic_3379</td>\n",
       "      <td>18</td>\n",
       "      <td>Preclinical and clinical studies</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3394</th>\n",
       "      <td>http://edamontology.org/topic_3382</td>\n",
       "      <td>18</td>\n",
       "      <td>Imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3395</th>\n",
       "      <td>http://edamontology.org/topic_3383</td>\n",
       "      <td>18</td>\n",
       "      <td>Bioimaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3396</th>\n",
       "      <td>http://edamontology.org/topic_3384</td>\n",
       "      <td>18</td>\n",
       "      <td>Medical imaging</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3397</th>\n",
       "      <td>http://edamontology.org/topic_3385</td>\n",
       "      <td>18</td>\n",
       "      <td>Light microscopy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3398</th>\n",
       "      <td>http://edamontology.org/topic_3386</td>\n",
       "      <td>18</td>\n",
       "      <td>Laboratory animal science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3399</th>\n",
       "      <td>http://edamontology.org/topic_3387</td>\n",
       "      <td>18</td>\n",
       "      <td>Marine biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3400</th>\n",
       "      <td>http://edamontology.org/topic_3388</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3401</th>\n",
       "      <td>http://edamontology.org/topic_3390</td>\n",
       "      <td>18</td>\n",
       "      <td>Nutritional science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3402</th>\n",
       "      <td>http://edamontology.org/topic_3391</td>\n",
       "      <td>18</td>\n",
       "      <td>Omics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3403</th>\n",
       "      <td>http://edamontology.org/topic_3393</td>\n",
       "      <td>18</td>\n",
       "      <td>Quality affairs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3404</th>\n",
       "      <td>http://edamontology.org/topic_3394</td>\n",
       "      <td>18</td>\n",
       "      <td>Regulatory affairs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3405</th>\n",
       "      <td>http://edamontology.org/topic_3395</td>\n",
       "      <td>18</td>\n",
       "      <td>Regenerative medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3406</th>\n",
       "      <td>http://edamontology.org/topic_3396</td>\n",
       "      <td>18</td>\n",
       "      <td>Systems medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3407</th>\n",
       "      <td>http://edamontology.org/topic_3397</td>\n",
       "      <td>18</td>\n",
       "      <td>Veterinary medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3408</th>\n",
       "      <td>http://edamontology.org/topic_3398</td>\n",
       "      <td>18</td>\n",
       "      <td>Bioengineering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3409</th>\n",
       "      <td>http://edamontology.org/topic_3399</td>\n",
       "      <td>18</td>\n",
       "      <td>Geriatric medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3410</th>\n",
       "      <td>http://edamontology.org/topic_3400</td>\n",
       "      <td>18</td>\n",
       "      <td>Allergy, clinical immunology and immunotherapeutics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3411</th>\n",
       "      <td>http://edamontology.org/topic_3401</td>\n",
       "      <td>18</td>\n",
       "      <td>Pain medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3412</th>\n",
       "      <td>http://edamontology.org/topic_3402</td>\n",
       "      <td>18</td>\n",
       "      <td>Anaesthesiology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3413</th>\n",
       "      <td>http://edamontology.org/topic_3403</td>\n",
       "      <td>18</td>\n",
       "      <td>Critical care medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3414</th>\n",
       "      <td>http://edamontology.org/topic_3404</td>\n",
       "      <td>18</td>\n",
       "      <td>Dermatology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3415</th>\n",
       "      <td>http://edamontology.org/topic_3405</td>\n",
       "      <td>18</td>\n",
       "      <td>Dentistry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3416</th>\n",
       "      <td>http://edamontology.org/topic_3406</td>\n",
       "      <td>18</td>\n",
       "      <td>Ear, nose and throat medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3417</th>\n",
       "      <td>http://edamontology.org/topic_3407</td>\n",
       "      <td>18</td>\n",
       "      <td>Endocrinology and metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3418</th>\n",
       "      <td>http://edamontology.org/topic_3408</td>\n",
       "      <td>18</td>\n",
       "      <td>Haematology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3419</th>\n",
       "      <td>http://edamontology.org/topic_3409</td>\n",
       "      <td>18</td>\n",
       "      <td>Gastroenterology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3420</th>\n",
       "      <td>http://edamontology.org/topic_3410</td>\n",
       "      <td>18</td>\n",
       "      <td>Gender medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3421</th>\n",
       "      <td>http://edamontology.org/topic_3411</td>\n",
       "      <td>18</td>\n",
       "      <td>Gynaecology and obstetrics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3422</th>\n",
       "      <td>http://edamontology.org/topic_3412</td>\n",
       "      <td>18</td>\n",
       "      <td>Hepatic and biliary medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3423</th>\n",
       "      <td>http://edamontology.org/topic_3413</td>\n",
       "      <td>18</td>\n",
       "      <td>Infectious tropical disease</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3424</th>\n",
       "      <td>http://edamontology.org/topic_3414</td>\n",
       "      <td>18</td>\n",
       "      <td>Trauma medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3425</th>\n",
       "      <td>http://edamontology.org/topic_3415</td>\n",
       "      <td>18</td>\n",
       "      <td>Medical toxicology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3426</th>\n",
       "      <td>http://edamontology.org/topic_3416</td>\n",
       "      <td>18</td>\n",
       "      <td>Musculoskeletal medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3427</th>\n",
       "      <td>http://edamontology.org/topic_3417</td>\n",
       "      <td>18</td>\n",
       "      <td>Ophthalmology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3428</th>\n",
       "      <td>http://edamontology.org/topic_3418</td>\n",
       "      <td>18</td>\n",
       "      <td>Paediatrics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3429</th>\n",
       "      <td>http://edamontology.org/topic_3419</td>\n",
       "      <td>18</td>\n",
       "      <td>Psychiatry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3430</th>\n",
       "      <td>http://edamontology.org/topic_3420</td>\n",
       "      <td>18</td>\n",
       "      <td>Reproductive health</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3431</th>\n",
       "      <td>http://edamontology.org/topic_3421</td>\n",
       "      <td>18</td>\n",
       "      <td>Surgery</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3432</th>\n",
       "      <td>http://edamontology.org/topic_3422</td>\n",
       "      <td>18</td>\n",
       "      <td>Urology and nephrology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3433</th>\n",
       "      <td>http://edamontology.org/topic_3423</td>\n",
       "      <td>18</td>\n",
       "      <td>Complementary medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3434</th>\n",
       "      <td>http://edamontology.org/topic_3444</td>\n",
       "      <td>18</td>\n",
       "      <td>MRI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3435</th>\n",
       "      <td>http://edamontology.org/topic_3448</td>\n",
       "      <td>18</td>\n",
       "      <td>Neutron diffraction</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3436</th>\n",
       "      <td>http://edamontology.org/topic_3452</td>\n",
       "      <td>18</td>\n",
       "      <td>Tomography</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3437</th>\n",
       "      <td>http://edamontology.org/topic_3473</td>\n",
       "      <td>18</td>\n",
       "      <td>Data mining</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3438</th>\n",
       "      <td>http://edamontology.org/topic_3474</td>\n",
       "      <td>18</td>\n",
       "      <td>Machine learning</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3439</th>\n",
       "      <td>http://edamontology.org/topic_3489</td>\n",
       "      <td>18</td>\n",
       "      <td>Database management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3440</th>\n",
       "      <td>http://edamontology.org/topic_3500</td>\n",
       "      <td>18</td>\n",
       "      <td>Zoology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3441</th>\n",
       "      <td>http://edamontology.org/topic_3510</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3442</th>\n",
       "      <td>http://edamontology.org/topic_3511</td>\n",
       "      <td>18</td>\n",
       "      <td>Nucleic acid sites, features and motifs</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3443</th>\n",
       "      <td>http://edamontology.org/topic_3512</td>\n",
       "      <td>18</td>\n",
       "      <td>Gene transcripts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3444</th>\n",
       "      <td>http://edamontology.org/topic_3514</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-ligand interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3445</th>\n",
       "      <td>http://edamontology.org/topic_3515</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-drug interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3446</th>\n",
       "      <td>http://edamontology.org/topic_3516</td>\n",
       "      <td>18</td>\n",
       "      <td>Genotyping experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3447</th>\n",
       "      <td>http://edamontology.org/topic_3517</td>\n",
       "      <td>18</td>\n",
       "      <td>GWAS study</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3448</th>\n",
       "      <td>http://edamontology.org/topic_3518</td>\n",
       "      <td>18</td>\n",
       "      <td>Microarray experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3449</th>\n",
       "      <td>http://edamontology.org/topic_3519</td>\n",
       "      <td>18</td>\n",
       "      <td>PCR experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3450</th>\n",
       "      <td>http://edamontology.org/topic_3520</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteomics experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3451</th>\n",
       "      <td>http://edamontology.org/topic_3521</td>\n",
       "      <td>18</td>\n",
       "      <td>2D PAGE experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3452</th>\n",
       "      <td>http://edamontology.org/topic_3522</td>\n",
       "      <td>18</td>\n",
       "      <td>Northern blot experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3453</th>\n",
       "      <td>http://edamontology.org/topic_3523</td>\n",
       "      <td>18</td>\n",
       "      <td>RNAi experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3454</th>\n",
       "      <td>http://edamontology.org/topic_3524</td>\n",
       "      <td>18</td>\n",
       "      <td>Simulation experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3455</th>\n",
       "      <td>http://edamontology.org/topic_3525</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-nucleic acid interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3456</th>\n",
       "      <td>http://edamontology.org/topic_3526</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-protein interactions</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3457</th>\n",
       "      <td>http://edamontology.org/topic_3527</td>\n",
       "      <td>18</td>\n",
       "      <td>Cellular process pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3458</th>\n",
       "      <td>http://edamontology.org/topic_3528</td>\n",
       "      <td>18</td>\n",
       "      <td>Disease pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3459</th>\n",
       "      <td>http://edamontology.org/topic_3529</td>\n",
       "      <td>18</td>\n",
       "      <td>Environmental information processing pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3460</th>\n",
       "      <td>http://edamontology.org/topic_3530</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic information processing pathways</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3461</th>\n",
       "      <td>http://edamontology.org/topic_3531</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein super-secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3462</th>\n",
       "      <td>http://edamontology.org/topic_3533</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein active sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3463</th>\n",
       "      <td>http://edamontology.org/topic_3534</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein binding sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3464</th>\n",
       "      <td>http://edamontology.org/topic_3535</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein-nucleic acid binding sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3465</th>\n",
       "      <td>http://edamontology.org/topic_3536</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein cleavage sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3466</th>\n",
       "      <td>http://edamontology.org/topic_3537</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein chemical modifications</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3467</th>\n",
       "      <td>http://edamontology.org/topic_3538</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein disordered structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3468</th>\n",
       "      <td>http://edamontology.org/topic_3539</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein domains</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3469</th>\n",
       "      <td>http://edamontology.org/topic_3540</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein key folding sites</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3470</th>\n",
       "      <td>http://edamontology.org/topic_3541</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein post-translational modifications</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3471</th>\n",
       "      <td>http://edamontology.org/topic_3542</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein secondary structure</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3472</th>\n",
       "      <td>http://edamontology.org/topic_3543</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein sequence repeats</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3473</th>\n",
       "      <td>http://edamontology.org/topic_3544</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein signal peptides</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3474</th>\n",
       "      <td>http://edamontology.org/topic_3569</td>\n",
       "      <td>18</td>\n",
       "      <td>Applied mathematics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3475</th>\n",
       "      <td>http://edamontology.org/topic_3570</td>\n",
       "      <td>18</td>\n",
       "      <td>Pure mathematics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3476</th>\n",
       "      <td>http://edamontology.org/topic_3571</td>\n",
       "      <td>18</td>\n",
       "      <td>Data governance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3477</th>\n",
       "      <td>http://edamontology.org/topic_3572</td>\n",
       "      <td>18</td>\n",
       "      <td>Data quality management</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3478</th>\n",
       "      <td>http://edamontology.org/topic_3573</td>\n",
       "      <td>18</td>\n",
       "      <td>Freshwater biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3479</th>\n",
       "      <td>http://edamontology.org/topic_3574</td>\n",
       "      <td>18</td>\n",
       "      <td>Human genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3480</th>\n",
       "      <td>http://edamontology.org/topic_3575</td>\n",
       "      <td>18</td>\n",
       "      <td>Tropical medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3481</th>\n",
       "      <td>http://edamontology.org/topic_3576</td>\n",
       "      <td>18</td>\n",
       "      <td>Medical biotechnology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3482</th>\n",
       "      <td>http://edamontology.org/topic_3577</td>\n",
       "      <td>18</td>\n",
       "      <td>Personalised medicine</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3483</th>\n",
       "      <td>http://edamontology.org/topic_3656</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunoprecipitation experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3484</th>\n",
       "      <td>http://edamontology.org/topic_3673</td>\n",
       "      <td>18</td>\n",
       "      <td>Whole genome sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3485</th>\n",
       "      <td>http://edamontology.org/topic_3674</td>\n",
       "      <td>18</td>\n",
       "      <td>Methylated DNA immunoprecipitation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3486</th>\n",
       "      <td>http://edamontology.org/topic_3676</td>\n",
       "      <td>18</td>\n",
       "      <td>Exome sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3487</th>\n",
       "      <td>http://edamontology.org/topic_3678</td>\n",
       "      <td>18</td>\n",
       "      <td>Experimental design and studies</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3488</th>\n",
       "      <td>http://edamontology.org/topic_3679</td>\n",
       "      <td>18</td>\n",
       "      <td>Animal study</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3489</th>\n",
       "      <td>http://edamontology.org/topic_3697</td>\n",
       "      <td>18</td>\n",
       "      <td>Microbial ecology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3490</th>\n",
       "      <td>http://edamontology.org/topic_3794</td>\n",
       "      <td>18</td>\n",
       "      <td>RNA immunoprecipitation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3491</th>\n",
       "      <td>http://edamontology.org/topic_3796</td>\n",
       "      <td>18</td>\n",
       "      <td>Population genomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3492</th>\n",
       "      <td>http://edamontology.org/topic_3810</td>\n",
       "      <td>18</td>\n",
       "      <td>Agricultural science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3493</th>\n",
       "      <td>http://edamontology.org/topic_3837</td>\n",
       "      <td>18</td>\n",
       "      <td>Metagenomic sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3494</th>\n",
       "      <td>http://edamontology.org/topic_3855</td>\n",
       "      <td>18</td>\n",
       "      <td>Environmental sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3495</th>\n",
       "      <td>http://edamontology.org/topic_3892</td>\n",
       "      <td>18</td>\n",
       "      <td>Biomolecular simulation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3496</th>\n",
       "      <td>http://edamontology.org/topic_3895</td>\n",
       "      <td>18</td>\n",
       "      <td>Synthetic biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3497</th>\n",
       "      <td>http://edamontology.org/topic_3912</td>\n",
       "      <td>18</td>\n",
       "      <td>Genetic engineering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3498</th>\n",
       "      <td>http://edamontology.org/topic_3922</td>\n",
       "      <td>18</td>\n",
       "      <td>Proteogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3499</th>\n",
       "      <td>http://edamontology.org/topic_3923</td>\n",
       "      <td>18</td>\n",
       "      <td>Genome resequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3500</th>\n",
       "      <td>http://edamontology.org/topic_3930</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunogenetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3501</th>\n",
       "      <td>http://edamontology.org/topic_3931</td>\n",
       "      <td>18</td>\n",
       "      <td>Chemometrics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3502</th>\n",
       "      <td>http://edamontology.org/topic_3934</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytometry</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3503</th>\n",
       "      <td>http://edamontology.org/topic_3939</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabolic engineering</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3504</th>\n",
       "      <td>http://edamontology.org/topic_3940</td>\n",
       "      <td>18</td>\n",
       "      <td>Chromosome conformation capture</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3505</th>\n",
       "      <td>http://edamontology.org/topic_3941</td>\n",
       "      <td>18</td>\n",
       "      <td>Metatranscriptomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3506</th>\n",
       "      <td>http://edamontology.org/topic_3943</td>\n",
       "      <td>18</td>\n",
       "      <td>Paleogenomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3507</th>\n",
       "      <td>http://edamontology.org/topic_3944</td>\n",
       "      <td>18</td>\n",
       "      <td>Cladistics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3508</th>\n",
       "      <td>http://edamontology.org/topic_3945</td>\n",
       "      <td>18</td>\n",
       "      <td>Molecular evolution</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3509</th>\n",
       "      <td>http://edamontology.org/topic_3948</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunoinformatics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3510</th>\n",
       "      <td>http://edamontology.org/topic_3954</td>\n",
       "      <td>18</td>\n",
       "      <td>Echography</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3511</th>\n",
       "      <td>http://edamontology.org/topic_3955</td>\n",
       "      <td>18</td>\n",
       "      <td>Fluxomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3512</th>\n",
       "      <td>http://edamontology.org/topic_3957</td>\n",
       "      <td>18</td>\n",
       "      <td>Protein interaction experiment</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3513</th>\n",
       "      <td>http://edamontology.org/topic_3958</td>\n",
       "      <td>18</td>\n",
       "      <td>Copy number variation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3514</th>\n",
       "      <td>http://edamontology.org/topic_3959</td>\n",
       "      <td>18</td>\n",
       "      <td>Cytogenetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3515</th>\n",
       "      <td>http://edamontology.org/topic_3966</td>\n",
       "      <td>18</td>\n",
       "      <td>Vaccinology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3516</th>\n",
       "      <td>http://edamontology.org/topic_3967</td>\n",
       "      <td>18</td>\n",
       "      <td>Immunomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3517</th>\n",
       "      <td>http://edamontology.org/topic_3974</td>\n",
       "      <td>18</td>\n",
       "      <td>Epistasis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3518</th>\n",
       "      <td>http://edamontology.org/topic_4010</td>\n",
       "      <td>18</td>\n",
       "      <td>Open science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3519</th>\n",
       "      <td>http://edamontology.org/topic_4011</td>\n",
       "      <td>18</td>\n",
       "      <td>Data rescue</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3520</th>\n",
       "      <td>http://edamontology.org/topic_4012</td>\n",
       "      <td>18</td>\n",
       "      <td>FAIR data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3521</th>\n",
       "      <td>http://edamontology.org/topic_4013</td>\n",
       "      <td>18</td>\n",
       "      <td>Antimicrobial Resistance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3522</th>\n",
       "      <td>http://edamontology.org/topic_4014</td>\n",
       "      <td>18</td>\n",
       "      <td>Electroencephalography</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3523</th>\n",
       "      <td>http://edamontology.org/topic_4016</td>\n",
       "      <td>18</td>\n",
       "      <td>Electrocardiography</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3524</th>\n",
       "      <td>http://edamontology.org/topic_4017</td>\n",
       "      <td>18</td>\n",
       "      <td>Cryogenic electron microscopy</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3525</th>\n",
       "      <td>http://edamontology.org/topic_4019</td>\n",
       "      <td>18</td>\n",
       "      <td>Biosciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3526</th>\n",
       "      <td>http://edamontology.org/topic_4020</td>\n",
       "      <td>18</td>\n",
       "      <td>Carbon cycle</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3527</th>\n",
       "      <td>http://edamontology.org/topic_4021</td>\n",
       "      <td>18</td>\n",
       "      <td>Multiomics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3528</th>\n",
       "      <td>http://edamontology.org/topic_4027</td>\n",
       "      <td>18</td>\n",
       "      <td>Ribosome Profiling</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3529</th>\n",
       "      <td>http://edamontology.org/topic_4028</td>\n",
       "      <td>18</td>\n",
       "      <td>Single-Cell Sequencing</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3530</th>\n",
       "      <td>http://edamontology.org/topic_4029</td>\n",
       "      <td>18</td>\n",
       "      <td>Acoustics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3531</th>\n",
       "      <td>http://edamontology.org/topic_4030</td>\n",
       "      <td>18</td>\n",
       "      <td>Microfluidics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3532</th>\n",
       "      <td>http://edamontology.org/topic_4037</td>\n",
       "      <td>18</td>\n",
       "      <td>Genomic imprinting</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3533</th>\n",
       "      <td>http://edamontology.org/topic_4038</td>\n",
       "      <td>18</td>\n",
       "      <td>Metabarcoding</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3534</th>\n",
       "      <td>http://www.geneontology.org/formats/oboInOwl#ObsoleteClass</td>\n",
       "      <td>18</td>\n",
       "      <td>Obsolete concept (EDAM)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table></div>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(f\"EDAM data used to annotate bio.tools: {len(remoteQuery(query=q, endpoint=ep_biotools))}\")\n",
    "results=remoteQuery(query=q, endpoint=ep_biotools)\n",
    "\n",
    "display(HTML(\"<div style='height: 200px; overflow: auto; width: fit-content'>\" +\n",
    "             results.to_html() +\n",
    "             \"</div>\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "id": "7d9f6de0-d920-4bec-8a91-e0d26007cd02",
   "metadata": {},
   "outputs": [],
   "source": [
    "q= \"\"\"\n",
    "PREFIX biotools: <https://bio.tools/ontology/>\n",
    "PREFIX bsc: <http://bioschemas.org/>\n",
    "PREFIX bsct: <http://bioschemas.org/types/>\n",
    "PREFIX dct: <http://purl.org/dc/terms/>\n",
    "PREFIX edam: <http://edamontology.org/>\n",
    "PREFIX owl: <http://www.w3.org/2002/07/owl#>\n",
    "PREFIX sc: <http://schema.org/>\n",
    "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
    "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\n",
    "PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>\n",
    "\n",
    "SELECT DISTINCT\n",
    "    ?format (COUNT(?format) as ?count) ?label\n",
    "   \n",
    "WHERE\n",
    "{\n",
    "    ?x bsc:input [sc:encodingFormat ?format] .\n",
    "    ?format rdfs:label ?label .   # Neeeded because the old data with URIs as literals for Data and Formats\n",
    "    ?y bsc:input [sc:encodingFormat edam:format_1212] .\n",
    "    FILTER(?x != ?y)\n",
    "}\n",
    "GROUP BY ?format ?label\n",
    "ORDER BY ASC(?format)\n",
    "#LIMIT 8\n",
    "\"\"\""
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
